<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url(https://themes.googleusercontent.com/fonts/css?kit=fpjTOVmNbO4Lz34iLyptLUXza5VhXqVC6o75Eld_V98);.lst-kix_list_98-3>li:before{content:"\0025cf   "}.lst-kix_list_98-7>li:before{content:"\0025cf   "}.lst-kix_list_60-0>li:before{content:"\0025cf   "}.lst-kix_list_130-4>li:before{content:"\0025cf   "}ul.lst-kix_list_93-8{list-style-type:none}.lst-kix_list_42-1>li:before{content:"\0025cb   "}ul.lst-kix_list_93-7{list-style-type:none}ul.lst-kix_list_93-6{list-style-type:none}ul.lst-kix_list_105-2{list-style-type:none}ul.lst-kix_list_105-3{list-style-type:none}ul.lst-kix_list_105-0{list-style-type:none}ul.lst-kix_list_105-1{list-style-type:none}ul.lst-kix_list_93-1{list-style-type:none}ul.lst-kix_list_105-6{list-style-type:none}ul.lst-kix_list_93-0{list-style-type:none}ul.lst-kix_list_105-7{list-style-type:none}ul.lst-kix_list_105-4{list-style-type:none}ul.lst-kix_list_105-5{list-style-type:none}ul.lst-kix_list_93-5{list-style-type:none}ul.lst-kix_list_93-4{list-style-type:none}.lst-kix_list_112-5>li:before{content:"\0025cf   "}ul.lst-kix_list_93-3{list-style-type:none}ul.lst-kix_list_105-8{list-style-type:none}ul.lst-kix_list_93-2{list-style-type:none}.lst-kix_list_130-0>li:before{content:"\0025cf   "}.lst-kix_list_42-5>li:before{content:"\0025a0   "}.lst-kix_list_149-2>li:before{content:"" counter(lst-ctn-kix_list_149-2,decimal) ". "}ul.lst-kix_list_80-8{list-style-type:none}.lst-kix_list_79-6>li:before{content:"" counter(lst-ctn-kix_list_79-6,decimal) ". "}ul.lst-kix_list_118-0{list-style-type:none}ul.lst-kix_list_27-0{list-style-type:none}ul.lst-kix_list_118-1{list-style-type:none}ul.lst-kix_list_27-1{list-style-type:none}ul.lst-kix_list_27-2{list-style-type:none}ul.lst-kix_list_80-2{list-style-type:none}ul.lst-kix_list_118-4{list-style-type:none}ul.lst-kix_list_80-3{list-style-type:none}ul.lst-kix_list_118-5{list-style-type:none}ul.lst-kix_list_80-0{list-style-type:none}ul.lst-kix_list_118-2{list-style-type:none}ul.lst-kix_list_80-1{list-style-type:none}ul.lst-kix_list_118-3{list-style-type:none}.lst-kix_list_79-2>li:before{content:"" counter(lst-ctn-kix_list_79-2,decimal) ". "}ul.lst-kix_list_80-6{list-style-type:none}ul.lst-kix_list_118-8{list-style-type:none}ul.lst-kix_list_80-7{list-style-type:none}ul.lst-kix_list_80-4{list-style-type:none}.lst-kix_list_24-2>li:before{content:"-  "}ul.lst-kix_list_118-6{list-style-type:none}ul.lst-kix_list_80-5{list-style-type:none}.lst-kix_list_149-6>li:before{content:"" counter(lst-ctn-kix_list_149-6,decimal) ". "}ul.lst-kix_list_118-7{list-style-type:none}ul.lst-kix_list_27-3{list-style-type:none}ul.lst-kix_list_27-4{list-style-type:none}ul.lst-kix_list_27-5{list-style-type:none}ul.lst-kix_list_27-6{list-style-type:none}ul.lst-kix_list_27-7{list-style-type:none}ul.lst-kix_list_27-8{list-style-type:none}.lst-kix_list_24-6>li:before{content:"-  "}ol.lst-kix_list_138-4.start{counter-reset:lst-ctn-kix_list_138-4 0}.lst-kix_list_23-0>li:before{content:"\0025cf   "}.lst-kix_list_23-8>li:before{content:"\0025a0   "}.lst-kix_list_23-4>li:before{content:"\0025cb   "}.lst-kix_list_43-3>li:before{content:"\0025cf   "}.lst-kix_list_80-3>li:before{content:"\0025cf   "}.lst-kix_list_80-7>li:before{content:"\0025cf   "}.lst-kix_list_22-2>li:before{content:"\0025cf   "}.lst-kix_list_22-6>li:before{content:"\0025cf   "}.lst-kix_list_43-7>li:before{content:"\0025cf   "}.lst-kix_e77bf81k0na-0>li:before{content:"\0025cf   "}.lst-kix_4sgfft60r4dw-4>li:before{content:"\0025cb   "}.lst-kix_e77bf81k0na-4>li:before{content:"\0025cb   "}.lst-kix_4sgfft60r4dw-8>li:before{content:"\0025a0   "}.lst-kix_list_99-5>li:before{content:"" counter(lst-ctn-kix_list_99-5,decimal) ". "}.lst-kix_list_99-1>li:before{content:"" counter(lst-ctn-kix_list_99-1,decimal) ". "}.lst-kix_list_62-8>li:before{content:"\0025cf   "}.lst-kix_e77bf81k0na-8>li:before{content:"\0025a0   "}.lst-kix_list_41-7>li:before{content:"\0025cb   "}.lst-kix_list_78-0>li:before{content:"\0025cf   "}.lst-kix_list_78-8>li:before{content:"\0025a0   "}.lst-kix_4sgfft60r4dw-0>li:before{content:"\0025cf   "}.lst-kix_list_78-4>li:before{content:"\0025cb   "}.lst-kix_list_62-0>li:before{content:"\0025cf   "}.lst-kix_list_40-5>li:before{content:"\0025cf   "}.lst-kix_list_61-6>li:before{content:"\0025cf   "}.lst-kix_list_96-3>li:before{content:"\0025cf   "}.lst-kix_list_62-4>li:before{content:"\0025cf   "}.lst-kix_list_41-3>li:before{content:"\0025cf   "}.lst-kix_list_60-8>li:before{content:"\0025cf   "}.lst-kix_list_97-1>li:before{content:"o  "}.lst-kix_list_60-4>li:before{content:"\0025cf   "}.lst-kix_list_81-5>li:before{content:"\0025a0   "}.lst-kix_list_96-7>li:before{content:"\0025cf   "}.lst-kix_list_97-5>li:before{content:"\0025cf   "}.lst-kix_list_40-1>li:before{content:"o  "}.lst-kix_list_61-2>li:before{content:"\0025a0   "}.lst-kix_list_77-2>li:before{content:"\0025cf   "}ul.lst-kix_list_136-1{list-style-type:none}ul.lst-kix_list_136-0{list-style-type:none}ol.lst-kix_list_144-1.start{counter-reset:lst-ctn-kix_list_144-1 0}ul.lst-kix_list_136-5{list-style-type:none}ul.lst-kix_list_136-4{list-style-type:none}ul.lst-kix_list_136-3{list-style-type:none}ul.lst-kix_list_136-2{list-style-type:none}.lst-kix_list_77-6>li:before{content:"\0025cf   "}ul.lst-kix_list_136-8{list-style-type:none}ul.lst-kix_list_136-7{list-style-type:none}ul.lst-kix_list_136-6{list-style-type:none}.lst-kix_list_81-1>li:before{content:"\0025cb   "}.lst-kix_list_21-8>li:before{content:"" counter(lst-ctn-kix_list_21-8,decimal) ". "}.lst-kix_list_21-0>li:before{content:"" counter(lst-ctn-kix_list_21-0,decimal) ". "}.lst-kix_list_63-2>li:before{content:"\0025cf   "}.lst-kix_list_95-8>li{counter-increment:lst-ctn-kix_list_95-8}.lst-kix_list_63-6>li:before{content:"\0025cf   "}.lst-kix_list_21-4>li:before{content:"" counter(lst-ctn-kix_list_21-4,decimal) ". "}.lst-kix_list_95-1>li:before{content:"" counter(lst-ctn-kix_list_95-1,decimal) ". "}.lst-kix_list_58-1>li:before{content:"\0025cf   "}.lst-kix_list_115-7>li:before{content:"\0025cf   "}.lst-kix_list_25-0>li:before{content:"\0025cf   "}.lst-kix_list_129-3>li:before{content:"\0025cf   "}.lst-kix_list_58-5>li:before{content:"\0025cf   "}ul.lst-kix_list_58-0{list-style-type:none}ul.lst-kix_list_58-2{list-style-type:none}ul.lst-kix_list_58-1{list-style-type:none}ul.lst-kix_list_58-4{list-style-type:none}.lst-kix_list_39-0>li:before{content:"\0025cf   "}ul.lst-kix_list_58-3{list-style-type:none}.lst-kix_list_129-7>li:before{content:"\0025cf   "}.lst-kix_list_44-1>li:before{content:"\0025cf   "}ol.lst-kix_list_20-6.start{counter-reset:lst-ctn-kix_list_20-6 0}ul.lst-kix_list_58-6{list-style-type:none}ul.lst-kix_list_58-5{list-style-type:none}ul.lst-kix_list_58-8{list-style-type:none}ul.lst-kix_list_58-7{list-style-type:none}.lst-kix_list_44-5>li:before{content:"\0025cf   "}.lst-kix_list_115-3>li:before{content:"\0025cf   "}.lst-kix_list_113-3>li:before{content:"\0025cf   "}.lst-kix_list_113-7>li:before{content:"\0025cf   "}.lst-kix_list_134-4>li:before{content:"\0025cf   "}.lst-kix_list_134-8>li:before{content:"\0025cf   "}.lst-kix_list_99-3>li{counter-increment:lst-ctn-kix_list_99-3}.lst-kix_list_20-2>li{counter-increment:lst-ctn-kix_list_20-2}.lst-kix_list_27-4>li:before{content:"\0025cf   "}.lst-kix_list_148-0>li:before{content:"\0025cf   "}.lst-kix_list_39-4>li:before{content:"\0025cb   "}.lst-kix_list_39-8>li:before{content:"\0025a0   "}.lst-kix_list_27-0>li:before{content:"\0025cf   "}.lst-kix_list_127-7>li:before{content:"\0025cf   "}.lst-kix_list_148-8>li:before{content:"\0025a0   "}.lst-kix_list_127-3>li:before{content:"\0025cf   "}.lst-kix_list_148-4>li:before{content:"\0025cb   "}ol.lst-kix_list_20-1.start{counter-reset:lst-ctn-kix_list_20-1 0}.lst-kix_list_25-4>li:before{content:"\0025cb   "}.lst-kix_list_134-0>li:before{content:"\0025cf   "}.lst-kix_list_46-5>li:before{content:"\0025cf   "}.lst-kix_list_25-8>li:before{content:"\0025a0   "}.lst-kix_list_95-1>li{counter-increment:lst-ctn-kix_list_95-1}.lst-kix_list_111-3>li:before{content:"\0025cf   "}.lst-kix_list_132-4>li:before{content:"\0025cb   "}.lst-kix_list_111-7>li:before{content:"\0025cf   "}.lst-kix_list_132-8>li:before{content:"\0025a0   "}ul.lst-kix_list_123-2{list-style-type:none}ul.lst-kix_list_123-3{list-style-type:none}.lst-kix_list_146-0>li:before{content:"\002794   "}ul.lst-kix_list_123-0{list-style-type:none}ul.lst-kix_list_123-1{list-style-type:none}.lst-kix_list_28-3>li{counter-increment:lst-ctn-kix_list_28-3}.lst-kix_list_130-8>li:before{content:"\0025cf   "}ul.lst-kix_list_123-6{list-style-type:none}ul.lst-kix_list_123-7{list-style-type:none}ul.lst-kix_list_123-4{list-style-type:none}ul.lst-kix_list_123-5{list-style-type:none}ul.lst-kix_list_14-4{list-style-type:none}ul.lst-kix_list_14-3{list-style-type:none}ul.lst-kix_list_14-2{list-style-type:none}.lst-kix_list_146-4>li:before{content:"\0025c6   "}ul.lst-kix_list_123-8{list-style-type:none}ul.lst-kix_list_14-1{list-style-type:none}ul.lst-kix_list_14-0{list-style-type:none}.lst-kix_list_146-8>li:before{content:"\0025cf   "}ul.lst-kix_list_14-8{list-style-type:none}ul.lst-kix_list_14-7{list-style-type:none}ul.lst-kix_list_14-6{list-style-type:none}ul.lst-kix_list_14-5{list-style-type:none}.lst-kix_list_27-8>li:before{content:"\0025cf   "}.lst-kix_list_132-0>li:before{content:"\0025cf   "}ul.lst-kix_list_89-2{list-style-type:none}ul.lst-kix_list_89-1{list-style-type:none}.lst-kix_list_19-1>li:before{content:"\0025cb   "}ul.lst-kix_list_89-0{list-style-type:none}.lst-kix_list_126-5>li:before{content:"\0025cf   "}ul.lst-kix_list_89-6{list-style-type:none}.lst-kix_list_79-4>li{counter-increment:lst-ctn-kix_list_79-4}ul.lst-kix_list_89-5{list-style-type:none}ul.lst-kix_list_89-4{list-style-type:none}.lst-kix_list_93-5>li:before{content:"\0025cf   "}ul.lst-kix_list_89-3{list-style-type:none}.lst-kix_list_56-1>li:before{content:"\0025cf   "}ul.lst-kix_list_89-8{list-style-type:none}ul.lst-kix_list_89-7{list-style-type:none}ul.lst-kix_list_32-1{list-style-type:none}ul.lst-kix_list_32-2{list-style-type:none}ul.lst-kix_list_32-3{list-style-type:none}ul.lst-kix_list_32-4{list-style-type:none}ul.lst-kix_list_32-5{list-style-type:none}.lst-kix_list_84-3>li:before{content:"\0025cf   "}ul.lst-kix_list_32-6{list-style-type:none}ul.lst-kix_list_32-7{list-style-type:none}ul.lst-kix_list_32-8{list-style-type:none}.lst-kix_list_138-2>li{counter-increment:lst-ctn-kix_list_138-2}.lst-kix_list_47-3>li:before{content:"\0025cf   "}ol.lst-kix_list_99-0{list-style-type:none}ol.lst-kix_list_99-1{list-style-type:none}ul.lst-kix_list_32-0{list-style-type:none}ol.lst-kix_list_99-2{list-style-type:none}.lst-kix_list_37-0>li:before{content:"\0025cf   "}ol.lst-kix_list_99-3{list-style-type:none}.lst-kix_list_144-4>li:before{content:"" counter(lst-ctn-kix_list_144-4,lower-latin) ". "}ol.lst-kix_list_99-4{list-style-type:none}ol.lst-kix_list_99-5{list-style-type:none}ol.lst-kix_list_95-3.start{counter-reset:lst-ctn-kix_list_95-3 0}ol.lst-kix_list_99-6{list-style-type:none}ol.lst-kix_list_99-7{list-style-type:none}ol.lst-kix_list_99-8{list-style-type:none}.lst-kix_list_117-7>li:before{content:"\0025cf   "}.lst-kix_list_37-8>li:before{content:"\0025a0   "}ul.lst-kix_list_110-4{list-style-type:none}ul.lst-kix_list_110-5{list-style-type:none}ul.lst-kix_list_110-2{list-style-type:none}ul.lst-kix_list_110-3{list-style-type:none}.lst-kix_tm0xq8syd7t0-5>li:before{content:"\0025a0   "}ul.lst-kix_list_110-8{list-style-type:none}ul.lst-kix_list_110-6{list-style-type:none}ul.lst-kix_list_110-7{list-style-type:none}.lst-kix_list_66-4>li:before{content:"-  "}.lst-kix_list_107-4>li:before{content:"\0025cf   "}.lst-kix_list_136-8>li:before{content:"\0025cf   "}.lst-kix_list_116-5>li:before{content:"\0025cf   "}.lst-kix_list_46-1>li:before{content:"\0025cf   "}.lst-kix_2x6t2se3panh-3>li:before{content:"\0025cf   "}ul.lst-kix_list_110-0{list-style-type:none}ul.lst-kix_list_110-1{list-style-type:none}.lst-kix_list_21-6>li{counter-increment:lst-ctn-kix_list_21-6}.lst-kix_list_10-3>li{counter-increment:lst-ctn-kix_list_10-3}.lst-kix_list_65-2>li:before{content:"\0025cf   "}.lst-kix_list_135-6>li:before{content:"\0025cf   "}.lst-kix_list_92-3>li:before{content:"\0025cf   "}.lst-kix_list_136-0>li:before{content:"\0025cf   "}.lst-kix_list_108-6>li:before{content:"\0025cf   "}.lst-kix_list_38-2>li:before{content:"\0025a0   "}ul.lst-kix_list_45-0{list-style-type:none}ul.lst-kix_list_45-2{list-style-type:none}.lst-kix_list_64-4>li:before{content:"\0025cf   "}ul.lst-kix_list_45-1{list-style-type:none}ul.lst-kix_list_45-4{list-style-type:none}ul.lst-kix_list_45-3{list-style-type:none}ul.lst-kix_list_45-6{list-style-type:none}ul.lst-kix_list_45-5{list-style-type:none}ol.lst-kix_list_55-8{list-style-type:none}ul.lst-kix_tm0xq8syd7t0-5{list-style-type:none}ul.lst-kix_tm0xq8syd7t0-4{list-style-type:none}ol.lst-kix_list_55-6{list-style-type:none}ul.lst-kix_tm0xq8syd7t0-7{list-style-type:none}ol.lst-kix_list_55-7{list-style-type:none}ul.lst-kix_tm0xq8syd7t0-6{list-style-type:none}ol.lst-kix_list_55-4{list-style-type:none}ol.lst-kix_list_55-5{list-style-type:none}ul.lst-kix_tm0xq8syd7t0-8{list-style-type:none}ol.lst-kix_list_55-2{list-style-type:none}ol.lst-kix_list_55-3{list-style-type:none}ol.lst-kix_list_55-0{list-style-type:none}ol.lst-kix_list_55-1{list-style-type:none}ul.lst-kix_list_3-7{list-style-type:none}ul.lst-kix_list_3-8{list-style-type:none}.lst-kix_list_21-3>li{counter-increment:lst-ctn-kix_list_21-3}.lst-kix_list_73-2>li:before{content:"\0025a0   "}ul.lst-kix_list_45-8{list-style-type:none}ul.lst-kix_list_3-1{list-style-type:none}ul.lst-kix_list_45-7{list-style-type:none}ul.lst-kix_list_3-2{list-style-type:none}ul.lst-kix_list_3-0{list-style-type:none}.lst-kix_list_85-5>li:before{content:"\0025cf   "}.lst-kix_list_118-5>li:before{content:"\0025cf   "}ul.lst-kix_list_3-5{list-style-type:none}ul.lst-kix_list_3-6{list-style-type:none}.lst-kix_rxdmkrnvx4u4-0>li:before{content:"\0025cf   "}ul.lst-kix_list_3-3{list-style-type:none}ul.lst-kix_list_3-4{list-style-type:none}ul.lst-kix_list_141-7{list-style-type:none}ul.lst-kix_list_141-6{list-style-type:none}ul.lst-kix_list_141-5{list-style-type:none}ul.lst-kix_list_141-4{list-style-type:none}.lst-kix_36wys1csajrz-8>li:before{content:"\0025a0   "}ul.lst-kix_list_141-8{list-style-type:none}.lst-kix_list_94-3>li:before{content:"\0025cf   "}.lst-kix_list_55-7>li{counter-increment:lst-ctn-kix_list_55-7}.lst-kix_list_36-2>li:before{content:"\0025cf   "}.lst-kix_list_57-3>li:before{content:"\0025cf   "}ul.lst-kix_list_141-3{list-style-type:none}ul.lst-kix_list_141-2{list-style-type:none}.lst-kix_rxdmkrnvx4u4-8>li:before{content:"\0025a0   "}ul.lst-kix_list_141-1{list-style-type:none}ul.lst-kix_list_141-0{list-style-type:none}.lst-kix_list_74-4>li:before{content:"\0025cb   "}.lst-kix_list_95-5>li:before{content:"" counter(lst-ctn-kix_list_95-5,decimal) ". "}.lst-kix_list_83-5>li:before{content:"\0025cf   "}.lst-kix_list_20-2>li:before{content:"" counter(lst-ctn-kix_list_20-2,decimal) ". "}.lst-kix_list_101-7>li{counter-increment:lst-ctn-kix_list_101-7}.lst-kix_list_75-2>li:before{content:"\0025cf   "}.lst-kix_list_54-1>li:before{content:"\0025cf   "}.lst-kix_list_82-3>li:before{content:"\0025cf   "}ol.lst-kix_list_95-8.start{counter-reset:lst-ctn-kix_list_95-8 0}.lst-kix_list_76-4>li:before{content:"\0025cf   "}.lst-kix_list_55-3>li:before{content:"" counter(lst-ctn-kix_list_55-3,decimal) ". "}.lst-kix_list_119-7>li:before{content:"\0025cf   "}.lst-kix_list_105-4>li:before{content:"\0025cf   "}.lst-kix_list_147-6>li:before{content:"\0025cf   "}.lst-kix_list_100-5>li:before{content:"" counter(lst-ctn-kix_list_100-5,decimal) ". "}.lst-kix_list_133-6>li:before{content:"\0025cf   "}.lst-kix_list_26-2>li:before{content:"\0025a0   "}.lst-kix_list_68-4>li:before{content:"\0025cf   "}.lst-kix_list_90-3>li:before{content:"\0025cf   "}ol.lst-kix_list_16-0.start{counter-reset:lst-ctn-kix_list_16-0 1}.lst-kix_list_87-5>li:before{content:"\0025cf   "}.lst-kix_list_128-5>li:before{content:"\0025cf   "}.lst-kix_36wys1csajrz-0>li:before{content:"\0025cf   "}.lst-kix_list_17-1>li:before{content:"-  "}.lst-kix_list_45-3>li:before{content:"\0025cf   "}.lst-kix_1yoisyi8wrza-4>li:before{content:"\0025cb   "}.lst-kix_list_137-6>li:before{content:"\0025cf   "}ul.lst-kix_list_50-6{list-style-type:none}ul.lst-kix_list_50-5{list-style-type:none}ul.lst-kix_list_50-8{list-style-type:none}ul.lst-kix_list_50-7{list-style-type:none}ul.lst-kix_list_50-0{list-style-type:none}.lst-kix_list_141-6>li:before{content:"\0025cf   "}.lst-kix_list_114-5>li:before{content:"\0025cf   "}ul.lst-kix_list_50-2{list-style-type:none}ul.lst-kix_list_50-1{list-style-type:none}ul.lst-kix_list_50-4{list-style-type:none}.lst-kix_list_71-2>li:before{content:"\0025cf   "}ul.lst-kix_list_50-3{list-style-type:none}.lst-kix_list_59-3>li:before{content:"\0025cf   "}ul.lst-kix_tm0xq8syd7t0-1{list-style-type:none}ul.lst-kix_tm0xq8syd7t0-0{list-style-type:none}ul.lst-kix_tm0xq8syd7t0-3{list-style-type:none}ol.lst-kix_list_16-5.start{counter-reset:lst-ctn-kix_list_16-5 0}ul.lst-kix_tm0xq8syd7t0-2{list-style-type:none}ol.lst-kix_list_100-1.start{counter-reset:lst-ctn-kix_list_100-1 0}.lst-kix_list_109-4>li:before{content:"\0025cf   "}.lst-kix_list_150-7>li:before{content:"\0025cf   "}.lst-kix_list_2-6>li:before{content:"\0025cf   "}ul.lst-kix_list_119-0{list-style-type:none}ol.lst-kix_list_100-6.start{counter-reset:lst-ctn-kix_list_100-6 0}.lst-kix_list_7-5>li:before{content:"\0025a0   "}ul.lst-kix_list_119-3{list-style-type:none}ul.lst-kix_list_119-4{list-style-type:none}ul.lst-kix_list_119-1{list-style-type:none}ul.lst-kix_list_119-2{list-style-type:none}ul.lst-kix_list_119-7{list-style-type:none}ul.lst-kix_list_119-8{list-style-type:none}ul.lst-kix_list_119-5{list-style-type:none}ul.lst-kix_list_119-6{list-style-type:none}.lst-kix_list_122-5>li:before{content:"\0025cf   "}.lst-kix_list_143-6>li:before{content:"\0025cf   "}.lst-kix_list_10-6>li{counter-increment:lst-ctn-kix_list_10-6}ul.lst-kix_list_76-3{list-style-type:none}ul.lst-kix_list_76-4{list-style-type:none}.lst-kix_list_100-6>li{counter-increment:lst-ctn-kix_list_100-6}ul.lst-kix_list_76-1{list-style-type:none}ul.lst-kix_list_76-2{list-style-type:none}ul.lst-kix_list_76-7{list-style-type:none}ul.lst-kix_list_76-8{list-style-type:none}ul.lst-kix_list_76-5{list-style-type:none}ul.lst-kix_list_76-6{list-style-type:none}.lst-kix_list_31-3>li:before{content:"\0025cf   "}ul.lst-kix_list_76-0{list-style-type:none}.lst-kix_list_10-2>li:before{content:"" counter(lst-ctn-kix_list_10-2,lower-roman) ". "}.lst-kix_list_4-6>li:before{content:"\0025cf   "}ul.lst-kix_list_63-5{list-style-type:none}ul.lst-kix_list_63-6{list-style-type:none}ul.lst-kix_list_63-3{list-style-type:none}ul.lst-kix_list_63-4{list-style-type:none}ul.lst-kix_list_63-7{list-style-type:none}ul.lst-kix_list_63-8{list-style-type:none}ul.lst-kix_list_63-1{list-style-type:none}ul.lst-kix_list_63-2{list-style-type:none}.lst-kix_list_9-5>li:before{content:"\0025cf   "}ul.lst-kix_list_63-0{list-style-type:none}.lst-kix_list_145-6>li:before{content:"\0025cf   "}.lst-kix_list_124-5>li:before{content:"\0025a0   "}.lst-kix_list_103-4>li:before{content:"\0025cf   "}.lst-kix_list_12-2>li:before{content:"\0025a0   "}.lst-kix_list_144-6>li{counter-increment:lst-ctn-kix_list_144-6}.lst-kix_list_49-3>li:before{content:"\0025cf   "}.lst-kix_list_110-5>li:before{content:"\0025cf   "}.lst-kix_list_131-6>li:before{content:"\0025cf   "}.lst-kix_list_28-2>li:before{content:"" counter(lst-ctn-kix_list_28-2,decimal) ". "}ol.lst-kix_list_79-0.start{counter-reset:lst-ctn-kix_list_79-0 0}.lst-kix_list_14-7>li:before{content:"\0025cb   "}.lst-kix_list_51-7>li:before{content:"\0025cf   "}ol.lst-kix_list_28-3.start{counter-reset:lst-ctn-kix_list_28-3 0}.lst-kix_list_102-1>li:before{content:"\0025cf   "}.lst-kix_list_89-5>li:before{content:"\0025a0   "}.lst-kix_list_140-3>li:before{content:"\0025cf   "}.lst-kix_list_89-1>li:before{content:"\0025cb   "}.lst-kix_list_5-1>li:before{content:"\0025cf   "}.lst-kix_list_5-8>li:before{content:"\0025cf   "}.lst-kix_list_70-8>li:before{content:"\0025cf   "}.lst-kix_list_5-4>li:before{content:"\0025cf   "}.lst-kix_list_5-5>li:before{content:"\0025cf   "}ol.lst-kix_list_95-4.start{counter-reset:lst-ctn-kix_list_95-4 0}.lst-kix_list_50-1>li:before{content:"o  "}ul.lst-kix_list_92-8{list-style-type:none}ul.lst-kix_list_92-7{list-style-type:none}.lst-kix_list_6-3>li:before{content:"\0025cf   "}ul.lst-kix_list_106-1{list-style-type:none}ul.lst-kix_list_106-2{list-style-type:none}ul.lst-kix_list_39-0{list-style-type:none}ul.lst-kix_list_39-1{list-style-type:none}ul.lst-kix_list_106-0{list-style-type:none}ul.lst-kix_list_92-2{list-style-type:none}ul.lst-kix_list_106-5{list-style-type:none}ul.lst-kix_list_92-1{list-style-type:none}ul.lst-kix_list_106-6{list-style-type:none}ul.lst-kix_list_92-0{list-style-type:none}ul.lst-kix_list_106-3{list-style-type:none}ul.lst-kix_list_106-4{list-style-type:none}ul.lst-kix_list_92-6{list-style-type:none}ul.lst-kix_list_92-5{list-style-type:none}ul.lst-kix_list_92-4{list-style-type:none}ul.lst-kix_list_106-7{list-style-type:none}ul.lst-kix_list_92-3{list-style-type:none}ul.lst-kix_list_106-8{list-style-type:none}.lst-kix_list_50-8>li:before{content:"\0025cf   "}ul.lst-kix_list_39-2{list-style-type:none}.lst-kix_list_121-2>li:before{content:"\0025cf   "}ul.lst-kix_list_39-3{list-style-type:none}.lst-kix_list_50-5>li:before{content:"\0025cf   "}ul.lst-kix_list_39-4{list-style-type:none}ul.lst-kix_list_39-5{list-style-type:none}.lst-kix_list_6-7>li:before{content:"\0025cf   "}ul.lst-kix_list_39-6{list-style-type:none}ul.lst-kix_list_39-7{list-style-type:none}ul.lst-kix_list_39-8{list-style-type:none}.lst-kix_list_6-6>li:before{content:"\0025cf   "}ol.lst-kix_list_10-6.start{counter-reset:lst-ctn-kix_list_10-6 0}.lst-kix_list_139-2>li:before{content:"\0025cf   "}.lst-kix_list_31-0>li:before{content:"\0025cf   "}.lst-kix_list_122-4>li:before{content:"\0025cf   "}.lst-kix_list_95-4>li{counter-increment:lst-ctn-kix_list_95-4}.lst-kix_list_141-2>li:before{content:"\0025cf   "}.lst-kix_list_31-4>li:before{content:"\0025cf   "}.lst-kix_list_122-8>li:before{content:"\0025cf   "}.lst-kix_list_15-1>li:before{content:"\0025cf   "}.lst-kix_list_88-8>li:before{content:"\0025cf   "}.lst-kix_list_140-0>li:before{content:"\0025cf   "}ol.lst-kix_list_79-5.start{counter-reset:lst-ctn-kix_list_79-5 0}.lst-kix_list_103-3>li:before{content:"\0025cf   "}ol.lst-kix_list_101-6{list-style-type:none}ol.lst-kix_list_101-7{list-style-type:none}ol.lst-kix_list_101-8{list-style-type:none}ol.lst-kix_list_101-2{list-style-type:none}.lst-kix_list_103-7>li:before{content:"\0025cf   "}ol.lst-kix_list_101-3{list-style-type:none}.lst-kix_list_99-7>li{counter-increment:lst-ctn-kix_list_99-7}ol.lst-kix_list_101-4{list-style-type:none}ol.lst-kix_list_101-5{list-style-type:none}.lst-kix_list_12-3>li:before{content:"\0025cf   "}ol.lst-kix_list_101-0{list-style-type:none}ol.lst-kix_list_101-1{list-style-type:none}.lst-kix_list_32-6>li:before{content:"\0025cf   "}.lst-kix_list_16-3>li{counter-increment:lst-ctn-kix_list_16-3}.lst-kix_list_28-0>li{counter-increment:lst-ctn-kix_list_28-0}.lst-kix_list_13-5>li:before{content:"\0025a0   "}ol.lst-kix_list_28-8.start{counter-reset:lst-ctn-kix_list_28-8 0}.lst-kix_list_69-7>li:before{content:"\0025cf   "}.lst-kix_list_33-8>li:before{content:"\0025cf   "}.lst-kix_list_104-5>li:before{content:"\0025cf   "}.lst-kix_list_34-2>li:before{content:"\0025cf   "}.lst-kix_list_72-4>li:before{content:"\0025cb   "}.lst-kix_list_100-6>li:before{content:"" counter(lst-ctn-kix_list_100-6,decimal) ". "}.lst-kix_list_30-5>li:before{content:"\0025cf   "}.lst-kix_list_35-0>li:before{content:"\0025cf   "}.lst-kix_list_35-4>li:before{content:"\0025cf   "}.lst-kix_list_30-1>li:before{content:"o  "}.lst-kix_list_142-8>li:before{content:"\0025cf   "}.lst-kix_list_3-0>li:before{content:"\0025cf   "}.lst-kix_list_72-3>li:before{content:"\0025cf   "}.lst-kix_list_30-2>li:before{content:"\0025cf   "}.lst-kix_list_86-0>li:before{content:"\0025cf   "}.lst-kix_list_72-7>li:before{content:"\0025cb   "}.lst-kix_list_138-0>li:before{content:"" counter(lst-ctn-kix_list_138-0,decimal) ". "}ul.lst-kix_list_124-1{list-style-type:none}ul.lst-kix_list_124-2{list-style-type:none}ul.lst-kix_list_124-0{list-style-type:none}ul.lst-kix_list_124-5{list-style-type:none}ul.lst-kix_list_124-6{list-style-type:none}ul.lst-kix_list_124-3{list-style-type:none}ul.lst-kix_list_124-4{list-style-type:none}ul.lst-kix_list_13-5{list-style-type:none}ul.lst-kix_1yoisyi8wrza-4{list-style-type:none}ul.lst-kix_list_13-4{list-style-type:none}ul.lst-kix_1yoisyi8wrza-5{list-style-type:none}ul.lst-kix_list_13-3{list-style-type:none}ul.lst-kix_1yoisyi8wrza-6{list-style-type:none}ul.lst-kix_list_124-7{list-style-type:none}ul.lst-kix_list_13-2{list-style-type:none}ul.lst-kix_1yoisyi8wrza-7{list-style-type:none}ul.lst-kix_list_124-8{list-style-type:none}ul.lst-kix_list_13-1{list-style-type:none}ul.lst-kix_1yoisyi8wrza-0{list-style-type:none}ul.lst-kix_list_13-0{list-style-type:none}ul.lst-kix_1yoisyi8wrza-1{list-style-type:none}ul.lst-kix_1yoisyi8wrza-2{list-style-type:none}ul.lst-kix_1yoisyi8wrza-3{list-style-type:none}.lst-kix_list_53-2>li:before{content:"\0025cf   "}.lst-kix_list_53-3>li:before{content:"\0025cf   "}ul.lst-kix_list_13-8{list-style-type:none}.lst-kix_list_11-1>li:before{content:"\0025cf   "}ul.lst-kix_list_13-7{list-style-type:none}ul.lst-kix_list_13-6{list-style-type:none}.lst-kix_list_123-3>li:before{content:"\0025cf   "}.lst-kix_list_123-7>li:before{content:"\0025cf   "}.lst-kix_list_11-0>li:before{content:"\0025cf   "}ul.lst-kix_list_137-0{list-style-type:none}.lst-kix_list_79-1>li{counter-increment:lst-ctn-kix_list_79-1}ul.lst-kix_list_137-4{list-style-type:none}ul.lst-kix_list_137-3{list-style-type:none}ul.lst-kix_list_137-2{list-style-type:none}ul.lst-kix_list_137-1{list-style-type:none}ul.lst-kix_list_137-8{list-style-type:none}ul.lst-kix_list_137-7{list-style-type:none}ul.lst-kix_list_137-6{list-style-type:none}ul.lst-kix_list_137-5{list-style-type:none}.lst-kix_list_91-1>li:before{content:"o  "}ul.lst-kix_list_4-8{list-style-type:none}.lst-kix_list_67-3>li:before{content:"\0025cf   "}ul.lst-kix_list_4-6{list-style-type:none}ul.lst-kix_list_4-7{list-style-type:none}.lst-kix_list_91-4>li:before{content:"\0025cf   "}.lst-kix_list_91-8>li:before{content:"\0025cf   "}.lst-kix_1yoisyi8wrza-3>li:before{content:"\0025cf   "}.lst-kix_list_67-2>li:before{content:"\0025a0   "}.lst-kix_list_137-3>li:before{content:"\0025cf   "}.lst-kix_list_137-7>li:before{content:"\0025cf   "}ul.lst-kix_list_4-0{list-style-type:none}ul.lst-kix_list_4-1{list-style-type:none}.lst-kix_list_16-3>li:before{content:"" counter(lst-ctn-kix_list_16-3,decimal) ". "}ul.lst-kix_list_4-4{list-style-type:none}.lst-kix_list_91-5>li:before{content:"\0025cf   "}ul.lst-kix_list_4-5{list-style-type:none}ul.lst-kix_list_4-2{list-style-type:none}ul.lst-kix_list_4-3{list-style-type:none}.lst-kix_list_86-7>li:before{content:"\0025cf   "}.lst-kix_list_86-4>li:before{content:"\0025cf   "}.lst-kix_list_141-5>li:before{content:"\0025cf   "}.lst-kix_list_86-3>li:before{content:"\0025cf   "}.lst-kix_list_17-8>li:before{content:"-  "}.lst-kix_1yoisyi8wrza-7>li:before{content:"\0025cb   "}.lst-kix_list_100-2>li:before{content:"" counter(lst-ctn-kix_list_100-2,decimal) ". "}.lst-kix_list_72-0>li:before{content:"\0025cf   "}.lst-kix_list_7-0>li:before{content:"\0025cf   "}.lst-kix_list_142-4>li:before{content:"\0025cf   "}ul.lst-kix_list_111-3{list-style-type:none}ul.lst-kix_list_111-4{list-style-type:none}ul.lst-kix_list_111-1{list-style-type:none}ul.lst-kix_list_111-2{list-style-type:none}ul.lst-kix_list_26-0{list-style-type:none}ul.lst-kix_list_111-7{list-style-type:none}ul.lst-kix_list_26-1{list-style-type:none}ul.lst-kix_list_111-8{list-style-type:none}ul.lst-kix_list_26-2{list-style-type:none}ul.lst-kix_list_111-5{list-style-type:none}ul.lst-kix_list_26-3{list-style-type:none}ul.lst-kix_list_111-6{list-style-type:none}.lst-kix_list_118-2>li:before{content:"\0025cf   "}.lst-kix_list_48-5>li:before{content:"\0025cf   "}.lst-kix_list_13-8>li:before{content:"\0025a0   "}.lst-kix_list_18-7>li:before{content:"\0025c6   "}ul.lst-kix_list_26-4{list-style-type:none}ul.lst-kix_list_26-5{list-style-type:none}ul.lst-kix_list_26-6{list-style-type:none}ul.lst-kix_list_26-7{list-style-type:none}ul.lst-kix_list_26-8{list-style-type:none}ul.lst-kix_list_111-0{list-style-type:none}.lst-kix_list_118-6>li:before{content:"\0025cf   "}.lst-kix_list_48-1>li:before{content:"\0025cf   "}.lst-kix_list_15-4>li:before{content:"\0025cf   "}.lst-kix_list_149-2>li{counter-increment:lst-ctn-kix_list_149-2}.lst-kix_list_120-0>li:before{content:"\0025cf   "}.lst-kix_list_15-0>li:before{content:"\0025cf   "}ol.lst-kix_list_79-1.start{counter-reset:lst-ctn-kix_list_79-1 0}.lst-kix_list_67-6>li:before{content:"\0025cf   "}.lst-kix_36wys1csajrz-1>li:before{content:"\0025cb   "}.lst-kix_list_29-4>li:before{content:"\0025cf   "}.lst-kix_list_53-6>li:before{content:"\0025cf   "}.lst-kix_list_11-4>li:before{content:"\0025cf   "}ul.lst-kix_1yoisyi8wrza-8{list-style-type:none}.lst-kix_list_29-0>li:before{content:"\0025cf   "}.lst-kix_list_104-2>li:before{content:"\0025cf   "}.lst-kix_list_1-4>li:before{content:"-  "}.lst-kix_list_34-5>li:before{content:"\0025cf   "}.lst-kix_list_104-6>li:before{content:"\0025cf   "}.lst-kix_list_1-8>li:before{content:"-  "}.lst-kix_list_34-1>li:before{content:"\0025cf   "}.lst-kix_list_126-4>li:before{content:"\0025cf   "}.lst-kix_list_19-0>li:before{content:"\0025cf   "}.lst-kix_list_56-0>li:before{content:"\0025cf   "}.lst-kix_list_135-2>li:before{content:"\0025cf   "}.lst-kix_list_117-0>li:before{content:"\0025cf   "}.lst-kix_list_117-3>li:before{content:"\0025cf   "}.lst-kix_list_144-0>li:before{content:"" counter(lst-ctn-kix_list_144-0,decimal) ". "}.lst-kix_list_84-8>li:before{content:"\0025a0   "}.lst-kix_list_144-3>li:before{content:"" counter(lst-ctn-kix_list_144-3,decimal) ". "}.lst-kix_list_107-0>li:before{content:"\0025cf   "}ol.lst-kix_list_138-3.start{counter-reset:lst-ctn-kix_list_138-3 0}.lst-kix_list_107-3>li:before{content:"\0025cf   "}ul.lst-kix_list_88-3{list-style-type:none}ul.lst-kix_list_88-2{list-style-type:none}ul.lst-kix_list_88-1{list-style-type:none}.lst-kix_2x6t2se3panh-7>li:before{content:"\0025cb   "}ul.lst-kix_list_88-0{list-style-type:none}ul.lst-kix_list_88-7{list-style-type:none}ul.lst-kix_list_88-6{list-style-type:none}ul.lst-kix_list_88-5{list-style-type:none}ul.lst-kix_list_88-4{list-style-type:none}.lst-kix_list_116-1>li:before{content:"\0025cf   "}.lst-kix_tm0xq8syd7t0-1>li:before{content:"\0025cb   "}.lst-kix_list_38-1>li:before{content:"\0025cb   "}.lst-kix_tm0xq8syd7t0-8>li:before{content:"\0025a0   "}ul.lst-kix_list_88-8{list-style-type:none}.lst-kix_list_108-2>li:before{content:"\0025a0   "}ul.lst-kix_list_31-2{list-style-type:none}ul.lst-kix_list_31-3{list-style-type:none}ul.lst-kix_list_31-4{list-style-type:none}.lst-kix_list_65-7>li:before{content:"\0025cf   "}ul.lst-kix_list_31-5{list-style-type:none}ul.lst-kix_list_31-6{list-style-type:none}ul.lst-kix_list_31-7{list-style-type:none}ul.lst-kix_list_31-8{list-style-type:none}ul.lst-kix_list_31-0{list-style-type:none}ul.lst-kix_list_31-1{list-style-type:none}.lst-kix_list_66-0>li:before{content:"-  "}.lst-kix_list_136-1>li:before{content:"\0025cf   "}.lst-kix_list_93-0>li:before{content:"\0025cf   "}.lst-kix_list_92-7>li:before{content:"\0025cf   "}.lst-kix_list_136-4>li:before{content:"\0025cf   "}.lst-kix_list_2-7>li:before{content:"\0025cf   "}.lst-kix_list_27-5>li:before{content:"\0025cf   "}.lst-kix_list_64-8>li:before{content:"\0025cf   "}.lst-kix_list_85-1>li:before{content:"\0025cf   "}.lst-kix_list_109-8>li:before{content:"\0025cf   "}.lst-kix_list_39-7>li:before{content:"\0025cb   "}.lst-kix_list_55-7>li:before{content:"" counter(lst-ctn-kix_list_55-7,lower-latin) ". "}.lst-kix_list_106-1>li:before{content:"\0025cf   "}.lst-kix_list_36-6>li:before{content:"\0025cf   "}ul.lst-kix_list_57-1{list-style-type:none}ul.lst-kix_list_57-0{list-style-type:none}ul.lst-kix_list_57-3{list-style-type:none}ul.lst-kix_list_57-2{list-style-type:none}.lst-kix_list_73-6>li:before{content:"\0025cf   "}ul.lst-kix_list_57-5{list-style-type:none}ul.lst-kix_list_57-4{list-style-type:none}.lst-kix_36wys1csajrz-4>li:before{content:"\0025cb   "}ol.lst-kix_list_28-7.start{counter-reset:lst-ctn-kix_list_28-7 0}.lst-kix_list_149-5>li{counter-increment:lst-ctn-kix_list_149-5}ul.lst-kix_list_57-7{list-style-type:none}ul.lst-kix_list_57-6{list-style-type:none}.lst-kix_list_46-6>li:before{content:"\0025cf   "}ul.lst-kix_list_57-8{list-style-type:none}ul.lst-kix_list_44-1{list-style-type:none}ul.lst-kix_list_44-0{list-style-type:none}ul.lst-kix_list_44-3{list-style-type:none}ul.lst-kix_list_44-2{list-style-type:none}ul.lst-kix_list_44-5{list-style-type:none}ul.lst-kix_list_44-4{list-style-type:none}ul.lst-kix_list_44-7{list-style-type:none}ul.lst-kix_list_44-6{list-style-type:none}ol.lst-kix_list_10-7{list-style-type:none}.lst-kix_list_82-7>li:before{content:"\0025cf   "}ol.lst-kix_list_10-8{list-style-type:none}ol.lst-kix_list_10-3{list-style-type:none}.lst-kix_list_9-4>li:before{content:"\0025cf   "}ol.lst-kix_list_10-4{list-style-type:none}ol.lst-kix_list_10-5{list-style-type:none}ol.lst-kix_list_10-6{list-style-type:none}ol.lst-kix_list_10-0{list-style-type:none}ol.lst-kix_list_10-1{list-style-type:none}ol.lst-kix_list_10-2{list-style-type:none}.lst-kix_list_145-5>li:before{content:"\0025cf   "}ul.lst-kix_list_44-8{list-style-type:none}.lst-kix_list_74-8>li:before{content:"\0025a0   "}.lst-kix_list_1-1>li:before{content:"-  "}.lst-kix_list_49-4>li:before{content:"\0025cf   "}ul.lst-kix_list_142-6{list-style-type:none}.lst-kix_list_76-0>li:before{content:"\0025cf   "}ul.lst-kix_list_142-5{list-style-type:none}ul.lst-kix_list_142-4{list-style-type:none}ul.lst-kix_list_142-3{list-style-type:none}.lst-kix_list_125-2>li:before{content:"\0025cf   "}ol.lst-kix_list_10-5.start{counter-reset:lst-ctn-kix_list_10-5 0}ul.lst-kix_list_142-8{list-style-type:none}ul.lst-kix_list_142-7{list-style-type:none}ol.lst-kix_list_79-6.start{counter-reset:lst-ctn-kix_list_79-6 0}.lst-kix_list_54-5>li:before{content:"\0025cf   "}ol.lst-kix_list_138-8.start{counter-reset:lst-ctn-kix_list_138-8 0}ul.lst-kix_list_142-2{list-style-type:none}ul.lst-kix_list_142-1{list-style-type:none}ul.lst-kix_list_142-0{list-style-type:none}ul.lst-kix_list_62-6{list-style-type:none}ul.lst-kix_list_62-7{list-style-type:none}ul.lst-kix_list_62-4{list-style-type:none}ul.lst-kix_list_62-5{list-style-type:none}ul.lst-kix_list_62-8{list-style-type:none}.lst-kix_list_35-7>li:before{content:"\0025cf   "}ul.lst-kix_list_62-2{list-style-type:none}ul.lst-kix_list_62-3{list-style-type:none}ul.lst-kix_list_62-0{list-style-type:none}ul.lst-kix_list_62-1{list-style-type:none}.lst-kix_list_26-6>li:before{content:"\0025cf   "}.lst-kix_list_8-2>li:before{content:"\0025a0   "}.lst-kix_list_105-8>li:before{content:"\0025cf   "}.lst-kix_list_68-8>li:before{content:"\0025cf   "}.lst-kix_list_26-3>li:before{content:"\0025cf   "}.lst-kix_list_68-5>li:before{content:"\0025cf   "}.lst-kix_list_21-7>li:before{content:"" counter(lst-ctn-kix_list_21-7,decimal) ". "}.lst-kix_list_90-2>li:before{content:"\0025a0   "}.lst-kix_list_149-8>li{counter-increment:lst-ctn-kix_list_149-8}.lst-kix_list_58-8>li:before{content:"\0025cf   "}.lst-kix_list_16-6>li:before{content:"" counter(lst-ctn-kix_list_16-6,decimal) ". "}ol.lst-kix_list_10-1.start{counter-reset:lst-ctn-kix_list_10-1 0}.lst-kix_list_115-8>li:before{content:"\0025cf   "}.lst-kix_list_45-4>li:before{content:"\0025cf   "}.lst-kix_1yoisyi8wrza-0>li:before{content:"\0025cf   "}.lst-kix_list_87-6>li:before{content:"\0025cf   "}.lst-kix_list_45-7>li:before{content:"\0025cf   "}.lst-kix_list_71-1>li:before{content:"\0025cf   "}.lst-kix_list_59-7>li:before{content:"\0025cb   "}.lst-kix_list_17-5>li:before{content:"-  "}.lst-kix_list_150-3>li:before{content:"\0025cf   "}.lst-kix_9jhhktla6bpr-6>li:before{content:"\0025cf   "}.lst-kix_list_122-1>li:before{content:"\0025cf   "}.lst-kix_list_52-0>li:before{content:"\0025cf   "}.lst-kix_list_94-2>li:before{content:"\0025cf   "}.lst-kix_list_16-6>li{counter-increment:lst-ctn-kix_list_16-6}.lst-kix_list_64-5>li:before{content:"\0025cf   "}ol.lst-kix_list_10-0.start{counter-reset:lst-ctn-kix_list_10-0 0}.lst-kix_list_31-7>li:before{content:"\0025cf   "}ul.lst-kix_9jhhktla6bpr-8{list-style-type:none}ul.lst-kix_list_107-0{list-style-type:none}ul.lst-kix_9jhhktla6bpr-5{list-style-type:none}ul.lst-kix_list_107-1{list-style-type:none}ul.lst-kix_9jhhktla6bpr-4{list-style-type:none}ul.lst-kix_9jhhktla6bpr-7{list-style-type:none}ul.lst-kix_9jhhktla6bpr-6{list-style-type:none}ul.lst-kix_list_107-4{list-style-type:none}ul.lst-kix_list_107-5{list-style-type:none}ul.lst-kix_list_107-2{list-style-type:none}ul.lst-kix_list_107-3{list-style-type:none}ul.lst-kix_list_107-8{list-style-type:none}ul.lst-kix_list_107-6{list-style-type:none}.lst-kix_list_101-8>li:before{content:"" counter(lst-ctn-kix_list_101-8,decimal) ". "}ul.lst-kix_list_107-7{list-style-type:none}.lst-kix_list_4-2>li:before{content:"\0025cf   "}.lst-kix_list_9-1>li:before{content:"\0025cf   "}.lst-kix_list_103-0>li:before{content:"\0025cf   "}.lst-kix_list_40-8>li:before{content:"\0025cf   "}.lst-kix_list_145-2>li:before{content:"\0025cf   "}.lst-kix_list_83-6>li:before{content:"\0025cf   "}ul.lst-kix_9jhhktla6bpr-1{list-style-type:none}.lst-kix_list_75-1>li:before{content:"o  "}ul.lst-kix_9jhhktla6bpr-0{list-style-type:none}ul.lst-kix_9jhhktla6bpr-3{list-style-type:none}ul.lst-kix_9jhhktla6bpr-2{list-style-type:none}.lst-kix_list_112-1>li:before{content:"\0025cf   "}ul.lst-kix_list_75-4{list-style-type:none}ul.lst-kix_list_75-5{list-style-type:none}ul.lst-kix_list_75-2{list-style-type:none}ul.lst-kix_list_75-3{list-style-type:none}ul.lst-kix_list_75-8{list-style-type:none}.lst-kix_list_119-8>li:before{content:"\0025cf   "}ul.lst-kix_list_75-6{list-style-type:none}ul.lst-kix_list_75-7{list-style-type:none}.lst-kix_list_49-7>li:before{content:"\0025cf   "}.lst-kix_list_131-2>li:before{content:"\0025a0   "}ul.lst-kix_list_75-0{list-style-type:none}ul.lst-kix_list_75-1{list-style-type:none}ol.lst-kix_list_28-2.start{counter-reset:lst-ctn-kix_list_28-2 0}.lst-kix_list_138-1>li{counter-increment:lst-ctn-kix_list_138-1}.lst-kix_list_12-6>li:before{content:"\0025cf   "}.lst-kix_list_126-1>li:before{content:"\0025cf   "}.lst-kix_list_54-8>li:before{content:"\0025cf   "}.lst-kix_list_98-6>li:before{content:"\0025cf   "}.lst-kix_list_98-0>li:before{content:"\0025cf   "}.lst-kix_list_98-8>li:before{content:"\0025cf   "}ol.lst-kix_list_100-7{list-style-type:none}ol.lst-kix_list_100-8{list-style-type:none}ol.lst-kix_list_100-3{list-style-type:none}ol.lst-kix_list_101-8.start{counter-reset:lst-ctn-kix_list_101-8 0}ul.lst-kix_list_9-3{list-style-type:none}ol.lst-kix_list_100-4{list-style-type:none}ul.lst-kix_list_9-4{list-style-type:none}.lst-kix_list_130-3>li:before{content:"\0025cf   "}ol.lst-kix_list_100-5{list-style-type:none}ul.lst-kix_list_9-1{list-style-type:none}ol.lst-kix_list_100-6{list-style-type:none}ul.lst-kix_list_9-2{list-style-type:none}ul.lst-kix_list_9-7{list-style-type:none}ol.lst-kix_list_100-0{list-style-type:none}ul.lst-kix_list_9-8{list-style-type:none}.lst-kix_list_60-3>li:before{content:"\0025cf   "}ol.lst-kix_list_100-1{list-style-type:none}ul.lst-kix_list_9-5{list-style-type:none}ol.lst-kix_list_100-2{list-style-type:none}ul.lst-kix_list_9-6{list-style-type:none}.lst-kix_list_130-5>li:before{content:"\0025cf   "}ul.lst-kix_list_9-0{list-style-type:none}.lst-kix_list_112-6>li:before{content:"\0025cf   "}.lst-kix_list_149-3>li:before{content:"" counter(lst-ctn-kix_list_149-3,decimal) ". "}.lst-kix_list_42-6>li:before{content:"\0025cf   "}.lst-kix_list_42-4>li:before{content:"\0025cb   "}.lst-kix_list_24-7>li:before{content:"-  "}.lst-kix_list_149-5>li:before{content:"" counter(lst-ctn-kix_list_149-5,decimal) ". "}.lst-kix_list_79-1>li:before{content:"" counter(lst-ctn-kix_list_79-1,decimal) ". "}ol.lst-kix_list_20-7.start{counter-reset:lst-ctn-kix_list_20-7 0}.lst-kix_list_79-3>li:before{content:"" counter(lst-ctn-kix_list_79-3,decimal) ". "}.lst-kix_list_24-5>li:before{content:"-  "}ol.lst-kix_list_16-4.start{counter-reset:lst-ctn-kix_list_16-4 0}ul.lst-kix_list_129-1{list-style-type:none}ul.lst-kix_list_129-0{list-style-type:none}.lst-kix_list_23-7>li:before{content:"\0025cb   "}ul.lst-kix_list_129-5{list-style-type:none}ul.lst-kix_list_129-4{list-style-type:none}ul.lst-kix_list_129-3{list-style-type:none}ul.lst-kix_list_129-2{list-style-type:none}.lst-kix_list_95-0>li{counter-increment:lst-ctn-kix_list_95-0}ul.lst-kix_list_129-8{list-style-type:none}.lst-kix_list_23-1>li:before{content:"\0025cb   "}ul.lst-kix_list_129-7{list-style-type:none}ul.lst-kix_list_129-6{list-style-type:none}.lst-kix_list_24-1>li:before{content:"-  "}ul.lst-kix_list_60-8{list-style-type:none}ul.lst-kix_list_60-6{list-style-type:none}.lst-kix_list_113-4>li:before{content:"\0025cf   "}.lst-kix_list_113-6>li:before{content:"\0025cf   "}ul.lst-kix_list_60-7{list-style-type:none}.lst-kix_list_150-8>li:before{content:"\0025cf   "}ul.lst-kix_list_60-0{list-style-type:none}ul.lst-kix_list_60-1{list-style-type:none}ul.lst-kix_list_60-4{list-style-type:none}ul.lst-kix_list_60-5{list-style-type:none}ul.lst-kix_list_60-2{list-style-type:none}.lst-kix_list_150-2>li:before{content:"\0025cf   "}ul.lst-kix_list_60-3{list-style-type:none}.lst-kix_list_150-0>li:before{content:"\0025cf   "}.lst-kix_list_148-3>li:before{content:"\0025cf   "}ul.lst-kix_list_103-0{list-style-type:none}ul.lst-kix_list_103-1{list-style-type:none}ul.lst-kix_list_103-4{list-style-type:none}ul.lst-kix_list_103-5{list-style-type:none}ul.lst-kix_list_103-2{list-style-type:none}ul.lst-kix_list_103-3{list-style-type:none}ul.lst-kix_list_103-8{list-style-type:none}ul.lst-kix_list_103-6{list-style-type:none}ul.lst-kix_list_103-7{list-style-type:none}.lst-kix_list_148-5>li:before{content:"\0025a0   "}.lst-kix_list_25-5>li:before{content:"\0025a0   "}.lst-kix_list_113-0>li:before{content:"\0025cf   "}.lst-kix_list_111-0>li:before{content:"\0025cf   "}ul.lst-kix_list_116-2{list-style-type:none}ul.lst-kix_list_116-3{list-style-type:none}ul.lst-kix_list_116-0{list-style-type:none}ul.lst-kix_list_116-1{list-style-type:none}ul.lst-kix_list_116-6{list-style-type:none}ul.lst-kix_list_116-7{list-style-type:none}ul.lst-kix_list_116-4{list-style-type:none}.lst-kix_list_132-3>li:before{content:"\0025cf   "}ul.lst-kix_list_116-5{list-style-type:none}ul.lst-kix_list_116-8{list-style-type:none}.lst-kix_list_111-8>li:before{content:"\0025cf   "}.lst-kix_list_111-6>li:before{content:"\0025cf   "}.lst-kix_list_110-0>li:before{content:"\0025cf   "}ol.lst-kix_list_55-5.start{counter-reset:lst-ctn-kix_list_55-5 0}.lst-kix_list_112-0>li:before{content:"\0025cf   "}.lst-kix_list_110-2>li:before{content:"\0025aa   "}.lst-kix_list_131-3>li:before{content:"\0025cf   "}.lst-kix_list_131-5>li:before{content:"\0025a0   "}.lst-kix_list_110-8>li:before{content:"\0025cf   "}.lst-kix_list_110-6>li:before{content:"\0025cf   "}ol.lst-kix_list_21-3.start{counter-reset:lst-ctn-kix_list_21-3 0}.lst-kix_list_81-0>li:before{content:"\0025cf   "}.lst-kix_list_16-0>li{counter-increment:lst-ctn-kix_list_16-0}.lst-kix_list_26-5>li:before{content:"\0025a0   "}.lst-kix_list_10-0>li{counter-increment:lst-ctn-kix_list_10-0}.lst-kix_list_77-3>li:before{content:"\0025cf   "}ul.lst-kix_list_147-1{list-style-type:none}ul.lst-kix_list_147-0{list-style-type:none}ul.lst-kix_list_147-5{list-style-type:none}ul.lst-kix_list_147-4{list-style-type:none}ul.lst-kix_list_147-3{list-style-type:none}.lst-kix_list_133-3>li:before{content:"\0025cf   "}.lst-kix_list_21-1>li:before{content:"" counter(lst-ctn-kix_list_21-1,decimal) ". "}ul.lst-kix_list_147-2{list-style-type:none}ul.lst-kix_list_147-8{list-style-type:none}ul.lst-kix_list_147-7{list-style-type:none}ul.lst-kix_list_147-6{list-style-type:none}.lst-kix_list_128-2>li:before{content:"\0025cf   "}.lst-kix_list_63-7>li:before{content:"\0025cf   "}.lst-kix_list_133-7>li:before{content:"\0025cf   "}ol.lst-kix_list_138-0.start{counter-reset:lst-ctn-kix_list_138-0 2}ul.lst-kix_list_69-0{list-style-type:none}ol.lst-kix_list_99-3.start{counter-reset:lst-ctn-kix_list_99-3 0}.lst-kix_list_58-6>li:before{content:"\0025cf   "}ul.lst-kix_list_69-3{list-style-type:none}ul.lst-kix_list_69-4{list-style-type:none}ul.lst-kix_list_69-1{list-style-type:none}ul.lst-kix_list_69-2{list-style-type:none}ol.lst-kix_list_79-2.start{counter-reset:lst-ctn-kix_list_79-2 0}.lst-kix_list_128-6>li:before{content:"\0025cf   "}.lst-kix_list_45-2>li:before{content:"\0025cf   "}ul.lst-kix_list_69-7{list-style-type:none}ul.lst-kix_list_69-8{list-style-type:none}ul.lst-kix_list_69-5{list-style-type:none}ul.lst-kix_list_69-6{list-style-type:none}ul.lst-kix_list_12-6{list-style-type:none}.lst-kix_list_44-2>li:before{content:"\0025cf   "}ul.lst-kix_list_12-5{list-style-type:none}ul.lst-kix_list_12-4{list-style-type:none}ul.lst-kix_list_12-3{list-style-type:none}.lst-kix_list_39-1>li:before{content:"\0025cb   "}ul.lst-kix_list_12-2{list-style-type:none}ul.lst-kix_list_12-1{list-style-type:none}.lst-kix_list_129-6>li:before{content:"\0025cf   "}ul.lst-kix_list_12-0{list-style-type:none}.lst-kix_list_115-2>li:before{content:"\0025cf   "}ul.lst-kix_list_12-8{list-style-type:none}ul.lst-kix_list_12-7{list-style-type:none}.lst-kix_list_59-6>li:before{content:"\0025cf   "}.lst-kix_list_16-5>li{counter-increment:lst-ctn-kix_list_16-5}ul.lst-kix_list_134-3{list-style-type:none}ul.lst-kix_list_134-2{list-style-type:none}ul.lst-kix_list_134-1{list-style-type:none}ul.lst-kix_list_134-0{list-style-type:none}ul.lst-kix_list_134-7{list-style-type:none}ul.lst-kix_list_134-6{list-style-type:none}ul.lst-kix_list_134-5{list-style-type:none}ul.lst-kix_list_134-4{list-style-type:none}.lst-kix_list_64-7>li:before{content:"\0025cf   "}.lst-kix_list_22-1>li:before{content:"\0025cf   "}ul.lst-kix_list_134-8{list-style-type:none}ol.lst-kix_list_144-7{list-style-type:none}ol.lst-kix_list_144-8{list-style-type:none}ol.lst-kix_list_144-5{list-style-type:none}ol.lst-kix_list_144-6{list-style-type:none}ol.lst-kix_list_144-3{list-style-type:none}.lst-kix_list_97-8>li:before{content:"\0025cf   "}ol.lst-kix_list_144-4{list-style-type:none}ol.lst-kix_list_144-1{list-style-type:none}ol.lst-kix_list_144-2{list-style-type:none}.lst-kix_e77bf81k0na-3>li:before{content:"\0025cf   "}ol.lst-kix_list_144-0{list-style-type:none}.lst-kix_4sgfft60r4dw-7>li:before{content:"\0025cb   "}.lst-kix_list_80-8>li:before{content:"\0025cf   "}ul.lst-kix_list_91-8{list-style-type:none}.lst-kix_list_78-3>li:before{content:"\0025cf   "}ul.lst-kix_list_38-0{list-style-type:none}ul.lst-kix_list_38-1{list-style-type:none}.lst-kix_list_57-2>li:before{content:"\0025cf   "}.lst-kix_list_57-6>li:before{content:"\0025cf   "}ul.lst-kix_list_38-2{list-style-type:none}.lst-kix_rxdmkrnvx4u4-5>li:before{content:"\0025a0   "}ul.lst-kix_list_91-3{list-style-type:none}ul.lst-kix_list_91-2{list-style-type:none}.lst-kix_list_99-0>li:before{content:"" counter(lst-ctn-kix_list_99-0,decimal) ". "}ul.lst-kix_list_91-1{list-style-type:none}ul.lst-kix_list_91-0{list-style-type:none}ul.lst-kix_list_91-7{list-style-type:none}ul.lst-kix_list_91-6{list-style-type:none}ul.lst-kix_list_91-5{list-style-type:none}.lst-kix_list_94-4>li:before{content:"\0025cf   "}ul.lst-kix_list_91-4{list-style-type:none}.lst-kix_list_62-7>li:before{content:"\0025cf   "}.lst-kix_4sgfft60r4dw-3>li:before{content:"\0025cf   "}ul.lst-kix_list_38-3{list-style-type:none}ul.lst-kix_list_38-4{list-style-type:none}.lst-kix_list_80-0>li:before{content:"\0025cf   "}ul.lst-kix_list_38-5{list-style-type:none}ul.lst-kix_list_38-6{list-style-type:none}ul.lst-kix_list_38-7{list-style-type:none}ul.lst-kix_list_38-8{list-style-type:none}.lst-kix_list_95-8>li:before{content:"" counter(lst-ctn-kix_list_95-8,decimal) ". "}ul.lst-kix_list_25-0{list-style-type:none}ul.lst-kix_list_25-1{list-style-type:none}ul.lst-kix_list_25-2{list-style-type:none}ul.lst-kix_list_25-3{list-style-type:none}ul.lst-kix_list_25-4{list-style-type:none}.lst-kix_list_95-4>li:before{content:"" counter(lst-ctn-kix_list_95-4,decimal) ". "}.lst-kix_list_40-6>li:before{content:"\0025cf   "}.lst-kix_list_144-7>li{counter-increment:lst-ctn-kix_list_144-7}.lst-kix_list_41-6>li:before{content:"\0025cf   "}.lst-kix_list_100-7>li{counter-increment:lst-ctn-kix_list_100-7}ul.lst-kix_list_25-5{list-style-type:none}ul.lst-kix_list_25-6{list-style-type:none}ul.lst-kix_list_25-7{list-style-type:none}ul.lst-kix_list_25-8{list-style-type:none}.lst-kix_list_83-4>li:before{content:"\0025cf   "}.lst-kix_list_62-3>li:before{content:"\0025cf   "}.lst-kix_list_97-0>li:before{content:"\0025cf   "}.lst-kix_list_82-0>li:before{content:"\0025cf   "}ol.lst-kix_list_144-0.start{counter-reset:lst-ctn-kix_list_144-0 0}.lst-kix_list_61-3>li:before{content:"\0025cf   "}.lst-kix_list_82-4>li:before{content:"\0025cf   "}.lst-kix_list_75-7>li:before{content:"\0025cf   "}.lst-kix_list_96-8>li:before{content:"\0025cf   "}.lst-kix_list_93-4>li:before{content:"\0025cf   "}ul.lst-kix_list_43-2{list-style-type:none}ul.lst-kix_list_43-1{list-style-type:none}ul.lst-kix_list_43-4{list-style-type:none}ul.lst-kix_list_43-3{list-style-type:none}.lst-kix_list_56-2>li:before{content:"\0025cf   "}ul.lst-kix_list_43-6{list-style-type:none}.lst-kix_list_126-2>li:before{content:"\0025cf   "}ul.lst-kix_list_43-5{list-style-type:none}ul.lst-kix_list_1-0{list-style-type:none}ul.lst-kix_list_43-8{list-style-type:none}ul.lst-kix_list_43-7{list-style-type:none}.lst-kix_list_19-4>li:before{content:"\0025cb   "}ul.lst-kix_list_43-0{list-style-type:none}.lst-kix_list_135-1>li:before{content:"\0025cf   "}.lst-kix_list_47-6>li:before{content:"\0025cf   "}.lst-kix_list_126-8>li:before{content:"\0025cf   "}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_1-4{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}ul.lst-kix_list_1-2{list-style-type:none}ul.lst-kix_list_1-7{list-style-type:none}ul.lst-kix_list_1-8{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}.lst-kix_list_47-8>li:before{content:"\0025cf   "}ul.lst-kix_list_1-6{list-style-type:none}.lst-kix_list_84-4>li:before{content:"\0025cb   "}.lst-kix_list_144-7>li:before{content:"" counter(lst-ctn-kix_list_144-7,lower-latin) ". "}.lst-kix_list_19-6>li:before{content:"\0025cf   "}.lst-kix_list_74-1>li:before{content:"\0025cb   "}.lst-kix_list_47-0>li:before{content:"\0025cf   "}.lst-kix_2x6t2se3panh-6>li:before{content:"\0025cf   "}.lst-kix_list_100-2>li{counter-increment:lst-ctn-kix_list_100-2}.lst-kix_list_136-7>li:before{content:"\0025cf   "}.lst-kix_list_20-5>li{counter-increment:lst-ctn-kix_list_20-5}.lst-kix_2x6t2se3panh-0>li:before{content:"\0025cf   "}.lst-kix_list_46-2>li:before{content:"\0025cf   "}.lst-kix_list_108-3>li:before{content:"\0025cf   "}.lst-kix_list_108-5>li:before{content:"\0025a0   "}.lst-kix_list_116-2>li:before{content:"\0025cf   "}.lst-kix_list_37-3>li:before{content:"\0025cf   "}ol.lst-kix_list_100-0.start{counter-reset:lst-ctn-kix_list_100-0 2}ul.lst-kix_list_121-4{list-style-type:none}ul.lst-kix_list_121-5{list-style-type:none}.lst-kix_list_65-5>li:before{content:"\0025cf   "}ul.lst-kix_list_121-2{list-style-type:none}ul.lst-kix_list_121-3{list-style-type:none}ul.lst-kix_list_121-8{list-style-type:none}.lst-kix_list_92-0>li:before{content:"\0025cf   "}.lst-kix_list_18-0>li:before{content:"\002794   "}ul.lst-kix_list_121-6{list-style-type:none}ul.lst-kix_list_121-7{list-style-type:none}.lst-kix_list_136-5>li:before{content:"\0025cf   "}.lst-kix_list_92-6>li:before{content:"\0025cf   "}ul.lst-kix_list_121-0{list-style-type:none}ul.lst-kix_list_121-1{list-style-type:none}.lst-kix_list_134-7>li:before{content:"\0025cf   "}.lst-kix_list_27-3>li:before{content:"\0025cf   "}.lst-kix_list_143-1>li:before{content:"\0025cf   "}.lst-kix_list_143-3>li:before{content:"\0025cf   "}.lst-kix_list_20-1>li{counter-increment:lst-ctn-kix_list_20-1}.lst-kix_list_95-5>li{counter-increment:lst-ctn-kix_list_95-5}.lst-kix_list_106-5>li:before{content:"\0025cf   "}.lst-kix_list_67-5>li:before{content:"\0025a0   "}.lst-kix_list_9-0>li:before{content:"\0025cf   "}.lst-kix_list_145-1>li:before{content:"\0025cf   "}ol.lst-kix_list_79-8{list-style-type:none}ol.lst-kix_list_79-7{list-style-type:none}ol.lst-kix_list_79-6{list-style-type:none}ol.lst-kix_list_79-5{list-style-type:none}ol.lst-kix_list_79-4{list-style-type:none}ol.lst-kix_list_79-3{list-style-type:none}ol.lst-kix_list_79-2{list-style-type:none}ol.lst-kix_list_79-1{list-style-type:none}.lst-kix_list_11-3>li:before{content:"\0025cf   "}ol.lst-kix_list_79-0{list-style-type:none}.lst-kix_list_29-1>li:before{content:"\0025cf   "}.lst-kix_list_29-3>li:before{content:"\0025cf   "}.lst-kix_list_133-1>li:before{content:"\0025cf   "}ul.lst-kix_list_56-0{list-style-type:none}.lst-kix_list_28-6>li{counter-increment:lst-ctn-kix_list_28-6}ul.lst-kix_list_56-2{list-style-type:none}ul.lst-kix_list_56-1{list-style-type:none}ul.lst-kix_list_56-4{list-style-type:none}ul.lst-kix_list_56-3{list-style-type:none}ul.lst-kix_list_56-6{list-style-type:none}ul.lst-kix_list_56-5{list-style-type:none}.lst-kix_list_146-5>li:before{content:"\0025cf   "}.lst-kix_list_125-4>li:before{content:"\0025cf   "}.lst-kix_list_49-6>li:before{content:"\0025cf   "}.lst-kix_list_2-1>li:before{content:"\0025cf   "}ul.lst-kix_list_56-8{list-style-type:none}.lst-kix_list_49-0>li:before{content:"\0025cf   "}ul.lst-kix_list_56-7{list-style-type:none}.lst-kix_list_2-3>li:before{content:"\0025cf   "}.lst-kix_list_146-7>li:before{content:"\0025c6   "}.lst-kix_list_100-8>li:before{content:"" counter(lst-ctn-kix_list_100-8,decimal) ". "}.lst-kix_list_77-5>li:before{content:"\0025cf   "}.lst-kix_list_72-1>li:before{content:"\0025cb   "}ol.lst-kix_list_100-2.start{counter-reset:lst-ctn-kix_list_100-2 0}.lst-kix_list_105-7>li:before{content:"\0025cf   "}ul.lst-kix_list_74-5{list-style-type:none}ul.lst-kix_list_74-6{list-style-type:none}ul.lst-kix_list_74-3{list-style-type:none}ul.lst-kix_list_74-4{list-style-type:none}ul.lst-kix_list_74-7{list-style-type:none}.lst-kix_list_8-6>li:before{content:"\0025cf   "}ul.lst-kix_list_74-8{list-style-type:none}ul.lst-kix_list_74-1{list-style-type:none}ol.lst-kix_list_16-6.start{counter-reset:lst-ctn-kix_list_16-6 0}ul.lst-kix_list_74-2{list-style-type:none}ul.lst-kix_list_74-0{list-style-type:none}.lst-kix_list_95-2>li:before{content:"" counter(lst-ctn-kix_list_95-2,decimal) ". "}.lst-kix_list_90-6>li:before{content:"\0025cf   "}.lst-kix_list_128-0>li:before{content:"\0025cf   "}.lst-kix_list_123-6>li:before{content:"\0025cf   "}.lst-kix_list_138-5>li:before{content:"" counter(lst-ctn-kix_list_138-5,decimal) ". "}.lst-kix_list_21-3>li:before{content:"" counter(lst-ctn-kix_list_21-3,decimal) ". "}ul.lst-kix_list_61-7{list-style-type:none}ul.lst-kix_list_61-8{list-style-type:none}ul.lst-kix_list_61-5{list-style-type:none}ul.lst-kix_list_61-6{list-style-type:none}.lst-kix_list_128-8>li:before{content:"\0025cf   "}.lst-kix_list_45-8>li:before{content:"\0025cf   "}.lst-kix_list_79-7>li{counter-increment:lst-ctn-kix_list_79-7}ul.lst-kix_list_61-0{list-style-type:none}ul.lst-kix_list_61-3{list-style-type:none}ul.lst-kix_list_61-4{list-style-type:none}ul.lst-kix_list_61-1{list-style-type:none}ul.lst-kix_list_61-2{list-style-type:none}.lst-kix_list_16-2>li:before{content:"" counter(lst-ctn-kix_list_16-2,decimal) ". "}.lst-kix_list_59-0>li:before{content:"\0025cf   "}.lst-kix_1yoisyi8wrza-1>li:before{content:"\0025cb   "}.lst-kix_list_141-3>li:before{content:"\0025cf   "}.lst-kix_list_44-4>li:before{content:"\0025cf   "}.lst-kix_list_39-3>li:before{content:"\0025cf   "}.lst-kix_list_87-2>li:before{content:"\0025cf   "}.lst-kix_list_115-4>li:before{content:"\0025cf   "}.lst-kix_list_100-0>li:before{content:"" counter(lst-ctn-kix_list_100-0,decimal) ". "}.lst-kix_list_17-6>li:before{content:"-  "}ul.lst-kix_list_87-4{list-style-type:none}ul.lst-kix_list_87-3{list-style-type:none}ol.lst-kix_list_16-2.start{counter-reset:lst-ctn-kix_list_16-2 0}.lst-kix_list_85-2>li:before{content:"\0025cf   "}ul.lst-kix_list_87-2{list-style-type:none}.lst-kix_list_80-6>li:before{content:"\0025cf   "}ul.lst-kix_list_87-1{list-style-type:none}ul.lst-kix_list_87-8{list-style-type:none}ul.lst-kix_list_87-7{list-style-type:none}ul.lst-kix_list_87-6{list-style-type:none}.lst-kix_list_118-0>li:before{content:"\0025cf   "}ul.lst-kix_list_87-5{list-style-type:none}.lst-kix_list_138-6>li{counter-increment:lst-ctn-kix_list_138-6}.lst-kix_list_28-2>li{counter-increment:lst-ctn-kix_list_28-2}.lst-kix_list_55-4>li{counter-increment:lst-ctn-kix_list_55-4}.lst-kix_list_22-7>li:before{content:"\0025cf   "}ul.lst-kix_list_87-0{list-style-type:none}ol.lst-kix_list_95-2.start{counter-reset:lst-ctn-kix_list_95-2 0}.lst-kix_list_64-1>li:before{content:"\0025cf   "}.lst-kix_list_34-7>li:before{content:"\0025cf   "}.lst-kix_list_118-8>li:before{content:"\0025cf   "}.lst-kix_list_55-8>li:before{content:"" counter(lst-ctn-kix_list_55-8,lower-roman) ". "}.lst-kix_list_99-4>li{counter-increment:lst-ctn-kix_list_99-4}ul.lst-kix_list_30-3{list-style-type:none}ul.lst-kix_list_30-4{list-style-type:none}ul.lst-kix_list_30-5{list-style-type:none}ul.lst-kix_list_30-6{list-style-type:none}ul.lst-kix_list_30-7{list-style-type:none}ul.lst-kix_list_30-8{list-style-type:none}.lst-kix_list_52-4>li:before{content:"\0025cf   "}.lst-kix_rxdmkrnvx4u4-3>li:before{content:"\0025cf   "}ul.lst-kix_list_30-0{list-style-type:none}ul.lst-kix_list_30-1{list-style-type:none}.lst-kix_list_57-0>li:before{content:"\0025cf   "}.lst-kix_list_57-8>li:before{content:"\0025cf   "}ul.lst-kix_list_30-2{list-style-type:none}.lst-kix_list_116-8>li:before{content:"\0025cf   "}.lst-kix_4sgfft60r4dw-1>li:before{content:"\0025cb   "}ol.lst-kix_list_95-0.start{counter-reset:lst-ctn-kix_list_95-0 3}.lst-kix_36wys1csajrz-3>li:before{content:"\0025cf   "}.lst-kix_list_41-0>li:before{content:"\0025cf   "}.lst-kix_list_62-1>li:before{content:"\0025cf   "}.lst-kix_list_82-6>li:before{content:"\0025cf   "}.lst-kix_list_75-5>li:before{content:"\0025cf   "}.lst-kix_list_54-4>li:before{content:"\0025cf   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_list_97-2>li:before{content:"\0025cf   "}.lst-kix_list_121-6>li:before{content:"\0025cf   "}ul.lst-kix_list_117-1{list-style-type:none}ul.lst-kix_list_117-2{list-style-type:none}ul.lst-kix_list_117-0{list-style-type:none}ul.lst-kix_list_117-5{list-style-type:none}ul.lst-kix_list_117-6{list-style-type:none}ul.lst-kix_list_117-3{list-style-type:none}ul.lst-kix_list_117-4{list-style-type:none}.lst-kix_list_121-0>li:before{content:"\0025cf   "}.lst-kix_list_51-3>li:before{content:"\0025cf   "}ul.lst-kix_list_117-7{list-style-type:none}ul.lst-kix_list_117-8{list-style-type:none}.lst-kix_list_89-6>li:before{content:"\0025cf   "}.lst-kix_list_51-6>li:before{content:"\0025cf   "}.lst-kix_list_14-2>li:before{content:"\0025a0   "}ol.lst-kix_list_55-7.start{counter-reset:lst-ctn-kix_list_55-7 0}.lst-kix_list_89-0>li:before{content:"\0025cf   "}.lst-kix_list_102-0>li:before{content:"\0025cf   "}.lst-kix_list_79-2>li{counter-increment:lst-ctn-kix_list_79-2}.lst-kix_list_102-3>li:before{content:"\0025cf   "}.lst-kix_list_32-1>li:before{content:"\0025cf   "}.lst-kix_list_28-7>li{counter-increment:lst-ctn-kix_list_28-7}.lst-kix_list_102-6>li:before{content:"\0025cf   "}.lst-kix_list_89-3>li:before{content:"\0025cf   "}.lst-kix_list_140-2>li:before{content:"\0025cf   "}.lst-kix_list_70-0>li:before{content:"\0025cf   "}.lst-kix_list_5-2>li:before{content:"\0025cf   "}.lst-kix_list_140-5>li:before{content:"\0025cf   "}.lst-kix_list_70-6>li:before{content:"\0025cf   "}.lst-kix_list_88-3>li:before{content:"\0025cf   "}.lst-kix_list_70-3>li:before{content:"\0025cf   "}.lst-kix_list_140-8>li:before{content:"\0025cf   "}.lst-kix_list_88-0>li:before{content:"\0025cf   "}.lst-kix_list_50-0>li:before{content:"\0025cf   "}.lst-kix_list_50-3>li:before{content:"\0025cf   "}.lst-kix_list_120-5>li:before{content:"\0025cf   "}.lst-kix_list_6-2>li:before{content:"\0025cf   "}.lst-kix_list_120-8>li:before{content:"\0025cf   "}.lst-kix_list_6-8>li:before{content:"\0025cf   "}.lst-kix_list_51-0>li:before{content:"\0025cf   "}.lst-kix_list_50-6>li:before{content:"\0025cf   "}.lst-kix_list_6-5>li:before{content:"\0025cf   "}.lst-kix_list_121-3>li:before{content:"\0025cf   "}.lst-kix_list_69-3>li:before{content:"\0025cf   "}.lst-kix_list_139-0>li:before{content:"\0025cf   "}.lst-kix_9jhhktla6bpr-1>li:before{content:"\0025cb   "}ol.lst-kix_list_144-8.start{counter-reset:lst-ctn-kix_list_144-8 0}.lst-kix_list_122-0>li:before{content:"\0025cf   "}.lst-kix_9jhhktla6bpr-7>li:before{content:"\0025cb   "}.lst-kix_list_52-1>li:before{content:"\0025cf   "}.lst-kix_list_13-7>li:before{content:"\0025cb   "}.lst-kix_list_7-8>li:before{content:"\0025a0   "}.lst-kix_list_144-3>li{counter-increment:lst-ctn-kix_list_144-3}.lst-kix_list_101-3>li:before{content:"" counter(lst-ctn-kix_list_101-3,decimal) ". "}.lst-kix_list_139-6>li:before{content:"\0025cf   "}ul.lst-kix_list_72-7{list-style-type:none}.lst-kix_list_15-5>li:before{content:"\0025cf   "}ul.lst-kix_list_72-8{list-style-type:none}ul.lst-kix_list_72-5{list-style-type:none}.lst-kix_list_31-6>li:before{content:"\0025cf   "}ul.lst-kix_list_72-6{list-style-type:none}.lst-kix_list_100-3>li{counter-increment:lst-ctn-kix_list_100-3}.lst-kix_list_122-6>li:before{content:"\0025cf   "}ul.lst-kix_list_72-0{list-style-type:none}.lst-kix_list_4-1>li:before{content:"\0025cf   "}.lst-kix_list_141-0>li:before{content:"\0025cf   "}ul.lst-kix_list_72-3{list-style-type:none}ul.lst-kix_list_72-4{list-style-type:none}ul.lst-kix_list_72-1{list-style-type:none}ul.lst-kix_list_72-2{list-style-type:none}ul.lst-kix_list_19-7{list-style-type:none}ul.lst-kix_list_19-6{list-style-type:none}ul.lst-kix_list_19-5{list-style-type:none}ul.lst-kix_list_19-4{list-style-type:none}ul.lst-kix_list_19-3{list-style-type:none}ul.lst-kix_list_19-2{list-style-type:none}ul.lst-kix_list_19-1{list-style-type:none}.lst-kix_list_88-6>li:before{content:"\0025cf   "}.lst-kix_list_10-5>li{counter-increment:lst-ctn-kix_list_10-5}ul.lst-kix_list_19-0{list-style-type:none}ul.lst-kix_list_19-8{list-style-type:none}.lst-kix_list_20-0>li{counter-increment:lst-ctn-kix_list_20-0}.lst-kix_list_32-4>li:before{content:"\0025cf   "}.lst-kix_list_103-1>li:before{content:"\0025cf   "}.lst-kix_list_33-4>li:before{content:"\0025cf   "}.lst-kix_list_12-1>li:before{content:"\0025cb   "}.lst-kix_list_53-7>li:before{content:"\0025cf   "}.lst-kix_list_101-1>li{counter-increment:lst-ctn-kix_list_101-1}ul.lst-kix_list_104-0{list-style-type:none}.lst-kix_list_104-1>li:before{content:"\0025cf   "}ul.lst-kix_list_104-3{list-style-type:none}.lst-kix_list_21-4>li{counter-increment:lst-ctn-kix_list_21-4}ul.lst-kix_list_104-4{list-style-type:none}ul.lst-kix_list_104-1{list-style-type:none}ul.lst-kix_list_104-2{list-style-type:none}ul.lst-kix_list_104-7{list-style-type:none}ul.lst-kix_list_104-8{list-style-type:none}ul.lst-kix_list_104-5{list-style-type:none}ul.lst-kix_list_104-6{list-style-type:none}.lst-kix_list_12-7>li:before{content:"\0025cb   "}.lst-kix_list_149-4>li{counter-increment:lst-ctn-kix_list_149-4}ol.lst-kix_list_21-1.start{counter-reset:lst-ctn-kix_list_21-1 0}ol.lst-kix_list_99-1.start{counter-reset:lst-ctn-kix_list_99-1 0}.lst-kix_list_104-7>li:before{content:"\0025cf   "}.lst-kix_list_13-1>li:before{content:"\0025cb   "}ul.lst-kix_list_24-0{list-style-type:none}ul.lst-kix_list_24-1{list-style-type:none}ul.lst-kix_list_24-2{list-style-type:none}ul.lst-kix_list_24-3{list-style-type:none}.lst-kix_list_35-5>li:before{content:"\0025cf   "}ul.lst-kix_list_24-4{list-style-type:none}ul.lst-kix_list_24-5{list-style-type:none}.lst-kix_list_142-0>li:before{content:"\0025cf   "}ol.lst-kix_list_20-5.start{counter-reset:lst-ctn-kix_list_20-5 0}.lst-kix_list_68-6>li:before{content:"\0025cf   "}ul.lst-kix_list_24-6{list-style-type:none}ul.lst-kix_list_24-7{list-style-type:none}ul.lst-kix_list_24-8{list-style-type:none}ol.lst-kix_list_101-0.start{counter-reset:lst-ctn-kix_list_101-0 0}.lst-kix_list_3-4>li:before{content:"\0025cb   "}.lst-kix_list_8-0>li:before{content:"\0025cf   "}.lst-kix_list_30-6>li:before{content:"\0025cf   "}.lst-kix_list_90-0>li:before{content:"\0025cf   "}.lst-kix_list_8-3>li:before{content:"\0025cf   "}.lst-kix_list_68-3>li:before{content:"\0025cf   "}.lst-kix_list_3-7>li:before{content:"\0025cb   "}.lst-kix_list_138-8>li:before{content:"" counter(lst-ctn-kix_list_138-8,decimal) ". "}.lst-kix_list_35-8>li:before{content:"\0025cf   "}.lst-kix_list_16-8>li:before{content:"" counter(lst-ctn-kix_list_16-8,decimal) ". "}.lst-kix_list_87-8>li:before{content:"\0025cf   "}.lst-kix_list_91-0>li:before{content:"\0025cf   "}.lst-kix_list_17-0>li:before{content:"-  "}.lst-kix_list_4-7>li:before{content:"\0025cf   "}ol.lst-kix_list_149-1.start{counter-reset:lst-ctn-kix_list_149-1 0}ol.lst-kix_list_144-2.start{counter-reset:lst-ctn-kix_list_144-2 0}ul.lst-kix_list_148-0{list-style-type:none}.lst-kix_list_144-2>li{counter-increment:lst-ctn-kix_list_144-2}.lst-kix_list_71-3>li:before{content:"\0025cf   "}.lst-kix_list_86-8>li:before{content:"\0025cf   "}ul.lst-kix_list_148-4{list-style-type:none}ul.lst-kix_list_148-3{list-style-type:none}ul.lst-kix_list_148-2{list-style-type:none}.lst-kix_list_71-0>li:before{content:"\0025cf   "}ul.lst-kix_list_148-1{list-style-type:none}ul.lst-kix_list_148-8{list-style-type:none}ul.lst-kix_list_148-7{list-style-type:none}ul.lst-kix_list_148-6{list-style-type:none}ul.lst-kix_list_148-5{list-style-type:none}.lst-kix_list_101-0>li{counter-increment:lst-ctn-kix_list_101-0}.lst-kix_list_17-3>li:before{content:"-  "}.lst-kix_list_139-3>li:before{content:"\0025cf   "}.lst-kix_9jhhktla6bpr-4>li:before{content:"\0025cb   "}.lst-kix_list_69-6>li:before{content:"\0025cf   "}.lst-kix_list_10-0>li:before{content:"" counter(lst-ctn-kix_list_10-0,decimal) ". "}ol.lst-kix_list_21-7.start{counter-reset:lst-ctn-kix_list_21-7 0}ol.lst-kix_list_99-5.start{counter-reset:lst-ctn-kix_list_99-5 0}ul.lst-kix_list_90-0{list-style-type:none}.lst-kix_list_122-3>li:before{content:"\0025cf   "}.lst-kix_list_18-3>li:before{content:"\0025cb   "}.lst-kix_list_106-2>li:before{content:"\0025cf   "}.lst-kix_list_73-7>li:before{content:"\0025cb   "}.lst-kix_list_36-5>li:before{content:"\0025cf   "}ul.lst-kix_list_122-3{list-style-type:none}ul.lst-kix_list_122-4{list-style-type:none}ul.lst-kix_list_122-1{list-style-type:none}ul.lst-kix_list_122-2{list-style-type:none}ul.lst-kix_list_122-7{list-style-type:none}ul.lst-kix_list_122-8{list-style-type:none}ul.lst-kix_list_122-5{list-style-type:none}.lst-kix_list_101-6>li:before{content:"" counter(lst-ctn-kix_list_101-6,decimal) ". "}ul.lst-kix_list_122-6{list-style-type:none}.lst-kix_list_15-8>li:before{content:"\0025cf   "}.lst-kix_list_4-4>li:before{content:"\0025cf   "}.lst-kix_list_9-3>li:before{content:"\0025cf   "}ul.lst-kix_list_122-0{list-style-type:none}ul.lst-kix_list_135-2{list-style-type:none}ul.lst-kix_list_135-1{list-style-type:none}ul.lst-kix_list_135-0{list-style-type:none}ul.lst-kix_list_135-6{list-style-type:none}ul.lst-kix_list_135-5{list-style-type:none}.lst-kix_list_74-7>li:before{content:"\0025cb   "}ul.lst-kix_list_135-4{list-style-type:none}ul.lst-kix_list_135-3{list-style-type:none}ul.lst-kix_list_135-8{list-style-type:none}ul.lst-kix_list_135-7{list-style-type:none}.lst-kix_list_12-4>li:before{content:"\0025cb   "}.lst-kix_list_124-7>li:before{content:"\0025cb   "}.lst-kix_list_33-1>li:before{content:"\0025cf   "}.lst-kix_list_1-0>li:before{content:"-  "}.lst-kix_list_119-6>li:before{content:"\0025cf   "}ul.lst-kix_list_37-0{list-style-type:none}ul.lst-kix_list_37-1{list-style-type:none}ul.lst-kix_list_37-2{list-style-type:none}ul.lst-kix_list_37-3{list-style-type:none}ul.lst-kix_list_90-4{list-style-type:none}ol.lst-kix_list_99-7.start{counter-reset:lst-ctn-kix_list_99-7 0}ul.lst-kix_list_90-3{list-style-type:none}ul.lst-kix_list_90-2{list-style-type:none}ul.lst-kix_list_90-1{list-style-type:none}.lst-kix_list_13-4>li:before{content:"\0025cb   "}ul.lst-kix_list_90-8{list-style-type:none}ul.lst-kix_list_90-7{list-style-type:none}ul.lst-kix_list_90-6{list-style-type:none}ul.lst-kix_list_90-5{list-style-type:none}ul.lst-kix_list_37-4{list-style-type:none}ul.lst-kix_list_37-5{list-style-type:none}ul.lst-kix_list_37-6{list-style-type:none}ul.lst-kix_list_37-7{list-style-type:none}ul.lst-kix_list_37-8{list-style-type:none}.lst-kix_list_125-7>li:before{content:"\0025cf   "}.lst-kix_list_55-2>li:before{content:"" counter(lst-ctn-kix_list_55-2,lower-roman) ". "}.lst-kix_list_93-1>li:before{content:"\0025cf   "}.lst-kix_list_56-5>li:before{content:"\0025cf   "}.lst-kix_list_117-2>li:before{content:"\0025cf   "}ol.lst-kix_list_149-5.start{counter-reset:lst-ctn-kix_list_149-5 0}.lst-kix_list_144-1>li:before{content:"" counter(lst-ctn-kix_list_144-1,lower-latin) ". "}.lst-kix_list_117-8>li:before{content:"\0025cf   "}ol.lst-kix_list_144-4.start{counter-reset:lst-ctn-kix_list_144-4 0}.lst-kix_list_56-8>li:before{content:"\0025cf   "}.lst-kix_list_84-7>li:before{content:"\0025cb   "}.lst-kix_list_117-5>li:before{content:"\0025cf   "}ul.lst-kix_list_42-3{list-style-type:none}ol.lst-kix_list_55-1.start{counter-reset:lst-ctn-kix_list_55-1 0}ul.lst-kix_list_42-2{list-style-type:none}ul.lst-kix_list_42-5{list-style-type:none}.lst-kix_list_66-8>li:before{content:"-  "}ul.lst-kix_list_42-4{list-style-type:none}.lst-kix_list_107-8>li:before{content:"\0025cf   "}ul.lst-kix_list_42-7{list-style-type:none}ul.lst-kix_list_42-6{list-style-type:none}ul.lst-kix_list_42-8{list-style-type:none}.lst-kix_tm0xq8syd7t0-6>li:before{content:"\0025cf   "}ol.lst-kix_list_21-5.start{counter-reset:lst-ctn-kix_list_21-5 0}.lst-kix_list_38-0>li:before{content:"\0025cf   "}ul.lst-kix_list_42-1{list-style-type:none}ul.lst-kix_list_42-0{list-style-type:none}.lst-kix_list_107-5>li:before{content:"\0025cf   "}.lst-kix_tm0xq8syd7t0-0>li:before{content:"\0025cf   "}.lst-kix_list_66-5>li:before{content:"-  "}.lst-kix_tm0xq8syd7t0-3>li:before{content:"\0025cf   "}.lst-kix_list_55-3>li{counter-increment:lst-ctn-kix_list_55-3}.lst-kix_list_101-6>li{counter-increment:lst-ctn-kix_list_101-6}.lst-kix_list_144-8>li{counter-increment:lst-ctn-kix_list_144-8}.lst-kix_list_135-7>li:before{content:"\0025cf   "}.lst-kix_list_66-2>li:before{content:"-  "}.lst-kix_list_38-3>li:before{content:"\0025cf   "}.lst-kix_list_38-6>li:before{content:"\0025cf   "}ul.lst-kix_list_68-0{list-style-type:none}ul.lst-kix_list_68-1{list-style-type:none}.lst-kix_list_48-0>li:before{content:"\0025cf   "}ul.lst-kix_list_68-4{list-style-type:none}ul.lst-kix_list_68-5{list-style-type:none}ul.lst-kix_list_68-2{list-style-type:none}ul.lst-kix_list_68-3{list-style-type:none}.lst-kix_list_48-6>li:before{content:"\0025cf   "}.lst-kix_list_76-8>li:before{content:"\0025cf   "}.lst-kix_list_18-6>li:before{content:"\0025cb   "}ol.lst-kix_list_55-3.start{counter-reset:lst-ctn-kix_list_55-3 0}ul.lst-kix_list_68-8{list-style-type:none}ul.lst-kix_list_68-6{list-style-type:none}.lst-kix_list_127-2>li:before{content:"\0025cf   "}ul.lst-kix_list_68-7{list-style-type:none}.lst-kix_list_94-7>li:before{content:"\0025cf   "}.lst-kix_list_127-8>li:before{content:"\0025cf   "}.lst-kix_list_79-8>li{counter-increment:lst-ctn-kix_list_79-8}.lst-kix_list_10-3>li:before{content:"" counter(lst-ctn-kix_list_10-3,decimal) ". "}ul.lst-kix_list_11-7{list-style-type:none}ul.lst-kix_list_11-6{list-style-type:none}ul.lst-kix_list_11-5{list-style-type:none}.lst-kix_list_36-8>li:before{content:"\0025cf   "}ul.lst-kix_list_11-4{list-style-type:none}ul.lst-kix_list_11-3{list-style-type:none}ul.lst-kix_list_11-2{list-style-type:none}.lst-kix_list_137-8>li:before{content:"\0025cf   "}.lst-kix_list_73-4>li:before{content:"\0025cb   "}ul.lst-kix_list_11-1{list-style-type:none}ul.lst-kix_list_11-0{list-style-type:none}.lst-kix_list_28-1>li{counter-increment:lst-ctn-kix_list_28-1}ul.lst-kix_list_11-8{list-style-type:none}.lst-kix_list_134-1>li:before{content:"\0025cf   "}.lst-kix_list_46-8>li:before{content:"\0025cf   "}.lst-kix_list_138-5>li{counter-increment:lst-ctn-kix_list_138-5}ol.lst-kix_list_101-4.start{counter-reset:lst-ctn-kix_list_101-4 0}.lst-kix_list_20-0>li:before{content:"" counter(lst-ctn-kix_list_20-0,decimal) ". "}.lst-kix_list_9-6>li:before{content:"\0025cf   "}.lst-kix_list_83-1>li:before{content:"\0025cf   "}.lst-kix_list_20-6>li:before{content:"" counter(lst-ctn-kix_list_20-6,decimal) ". "}.lst-kix_list_145-7>li:before{content:"\0025cf   "}ul.lst-kix_list_2-8{list-style-type:none}.lst-kix_list_124-4>li:before{content:"\0025cb   "}ul.lst-kix_list_2-2{list-style-type:none}ul.lst-kix_list_2-3{list-style-type:none}ul.lst-kix_list_2-0{list-style-type:none}ul.lst-kix_list_2-1{list-style-type:none}.lst-kix_list_20-6>li{counter-increment:lst-ctn-kix_list_20-6}ul.lst-kix_list_2-6{list-style-type:none}ul.lst-kix_list_2-7{list-style-type:none}ul.lst-kix_list_2-4{list-style-type:none}ul.lst-kix_list_2-5{list-style-type:none}.lst-kix_list_1-3>li:before{content:"-  "}ol.lst-kix_list_149-3.start{counter-reset:lst-ctn-kix_list_149-3 0}.lst-kix_list_28-3>li:before{content:"" counter(lst-ctn-kix_list_28-3,decimal) ". "}ol.lst-kix_list_21-8{list-style-type:none}ol.lst-kix_list_21-7{list-style-type:none}ol.lst-kix_list_21-4{list-style-type:none}ol.lst-kix_list_21-3{list-style-type:none}ol.lst-kix_list_21-6{list-style-type:none}.lst-kix_list_119-3>li:before{content:"\0025cf   "}ol.lst-kix_list_21-5{list-style-type:none}ol.lst-kix_list_21-0{list-style-type:none}ol.lst-kix_list_101-2.start{counter-reset:lst-ctn-kix_list_101-2 0}ol.lst-kix_list_21-2{list-style-type:none}ol.lst-kix_list_21-1{list-style-type:none}.lst-kix_list_76-2>li:before{content:"\0025cf   "}ol.lst-kix_list_144-6.start{counter-reset:lst-ctn-kix_list_144-6 0}.lst-kix_list_142-6>li:before{content:"\0025cf   "}.lst-kix_list_35-2>li:before{content:"\0025cf   "}.lst-kix_list_147-5>li:before{content:"\0025cf   "}.lst-kix_list_30-3>li:before{content:"\0025cf   "}.lst-kix_list_147-2>li:before{content:"\0025cf   "}.lst-kix_list_3-1>li:before{content:"\0025cb   "}.lst-kix_list_114-1>li:before{content:"\0025cf   "}.lst-kix_list_119-0>li:before{content:"\0025cf   "}.lst-kix_list_114-4>li:before{content:"\0025cf   "}.lst-kix_list_81-3>li:before{content:"\0025cf   "}.lst-kix_list_63-1>li:before{content:"o  "}.lst-kix_list_58-0>li:before{content:"\0025cf   "}.lst-kix_list_53-1>li:before{content:"\0025cf   "}.lst-kix_list_68-0>li:before{content:"\0025cf   "}.lst-kix_list_63-4>li:before{content:"\0025cf   "}.lst-kix_list_91-3>li:before{content:"\0025cf   "}.lst-kix_list_137-2>li:before{content:"\0025cf   "}.lst-kix_list_129-0>li:before{content:"\0025cf   "}.lst-kix_list_25-2>li:before{content:"\0025a0   "}.lst-kix_list_100-8>li{counter-increment:lst-ctn-kix_list_100-8}ol.lst-kix_list_101-6.start{counter-reset:lst-ctn-kix_list_101-6 0}.lst-kix_list_91-6>li:before{content:"\0025cf   "}.lst-kix_list_137-5>li:before{content:"\0025cf   "}.lst-kix_list_58-3>li:before{content:"\0025cf   "}ul.lst-kix_list_73-6{list-style-type:none}ul.lst-kix_list_140-8{list-style-type:none}ul.lst-kix_list_73-7{list-style-type:none}ul.lst-kix_list_140-7{list-style-type:none}ul.lst-kix_list_73-4{list-style-type:none}.lst-kix_list_86-5>li:before{content:"\0025cf   "}ul.lst-kix_list_140-6{list-style-type:none}ul.lst-kix_list_73-5{list-style-type:none}ul.lst-kix_list_140-5{list-style-type:none}.lst-kix_list_109-0>li:before{content:"\0025cf   "}ul.lst-kix_list_73-8{list-style-type:none}.lst-kix_list_86-2>li:before{content:"\0025cf   "}ul.lst-kix_list_73-2{list-style-type:none}ul.lst-kix_list_73-3{list-style-type:none}ul.lst-kix_list_73-0{list-style-type:none}ul.lst-kix_list_73-1{list-style-type:none}ol.lst-kix_list_149-7.start{counter-reset:lst-ctn-kix_list_149-7 0}.lst-kix_list_30-0>li:before{content:"\0025cf   "}ul.lst-kix_list_140-0{list-style-type:none}.lst-kix_list_142-3>li:before{content:"\0025cf   "}ul.lst-kix_list_140-4{list-style-type:none}ul.lst-kix_list_140-3{list-style-type:none}ul.lst-kix_list_140-2{list-style-type:none}.lst-kix_list_71-6>li:before{content:"\0025cf   "}ul.lst-kix_list_140-1{list-style-type:none}.lst-kix_list_109-3>li:before{content:"\0025cf   "}.lst-kix_list_43-4>li:before{content:"\0025cf   "}.lst-kix_list_48-3>li:before{content:"\0025cf   "}.lst-kix_list_101-0>li:before{content:"" counter(lst-ctn-kix_list_101-0,decimal) ". "}.lst-kix_list_73-1>li:before{content:"\0025cb   "}.lst-kix_list_99-6>li:before{content:"" counter(lst-ctn-kix_list_99-6,decimal) ". "}.lst-kix_list_127-5>li:before{content:"\0025cf   "}.lst-kix_list_15-2>li:before{content:"\0025cf   "}.lst-kix_list_10-6>li:before{content:"" counter(lst-ctn-kix_list_10-6,decimal) ". "}ul.lst-kix_list_86-5{list-style-type:none}ul.lst-kix_list_86-4{list-style-type:none}ul.lst-kix_list_86-3{list-style-type:none}ul.lst-kix_list_86-2{list-style-type:none}ul.lst-kix_list_86-8{list-style-type:none}ul.lst-kix_list_86-7{list-style-type:none}ul.lst-kix_list_86-6{list-style-type:none}.lst-kix_list_124-1>li:before{content:"\0025cb   "}ul.lst-kix_list_86-1{list-style-type:none}.lst-kix_list_53-4>li:before{content:"\0025cf   "}ul.lst-kix_list_86-0{list-style-type:none}.lst-kix_list_132-6>li:before{content:"\0025cf   "}.lst-kix_list_20-3>li:before{content:"" counter(lst-ctn-kix_list_20-3,decimal) ". "}.lst-kix_list_96-2>li:before{content:"\0025cf   "}.lst-kix_list_104-4>li:before{content:"\0025cf   "}.lst-kix_list_81-6>li:before{content:"\0025cf   "}.lst-kix_list_28-6>li:before{content:"" counter(lst-ctn-kix_list_28-6,decimal) ". "}.lst-kix_list_1-6>li:before{content:"-  "}.lst-kix_list_76-5>li:before{content:"\0025cf   "}.lst-kix_list_33-7>li:before{content:"\0025cf   "}.lst-kix_list_40-0>li:before{content:"\0025cf   "}.lst-kix_list_98-1>li:before{content:"o  "}.lst-kix_list_21-8>li{counter-increment:lst-ctn-kix_list_21-8}ul.lst-kix_e77bf81k0na-8{list-style-type:none}ul.lst-kix_e77bf81k0na-7{list-style-type:none}ul.lst-kix_e77bf81k0na-2{list-style-type:none}.lst-kix_list_138-0>li{counter-increment:lst-ctn-kix_list_138-0}.lst-kix_list_130-6>li:before{content:"\0025cf   "}ul.lst-kix_e77bf81k0na-1{list-style-type:none}.lst-kix_list_98-5>li:before{content:"\0025cf   "}ul.lst-kix_e77bf81k0na-0{list-style-type:none}ul.lst-kix_e77bf81k0na-6{list-style-type:none}.lst-kix_list_60-2>li:before{content:"\0025cf   "}ul.lst-kix_e77bf81k0na-5{list-style-type:none}ul.lst-kix_e77bf81k0na-4{list-style-type:none}ul.lst-kix_e77bf81k0na-3{list-style-type:none}.lst-kix_list_112-3>li:before{content:"\0025cf   "}.lst-kix_list_42-7>li:before{content:"\0025cb   "}.lst-kix_list_79-0>li:before{content:"" counter(lst-ctn-kix_list_79-0,decimal) ". "}.lst-kix_list_130-2>li:before{content:"\0025cf   "}.lst-kix_list_149-0>li:before{content:"" counter(lst-ctn-kix_list_149-0,decimal) ". "}.lst-kix_list_149-4>li:before{content:"" counter(lst-ctn-kix_list_149-4,decimal) ". "}.lst-kix_list_112-7>li:before{content:"\0025cf   "}.lst-kix_list_42-3>li:before{content:"\0025cf   "}.lst-kix_list_79-8>li:before{content:"" counter(lst-ctn-kix_list_79-8,decimal) ". "}ol.lst-kix_list_16-5{list-style-type:none}ol.lst-kix_list_16-6{list-style-type:none}.lst-kix_list_149-0>li{counter-increment:lst-ctn-kix_list_149-0}.lst-kix_list_24-8>li:before{content:"-  "}ol.lst-kix_list_16-7{list-style-type:none}ol.lst-kix_list_16-8{list-style-type:none}.lst-kix_list_149-8>li:before{content:"" counter(lst-ctn-kix_list_149-8,decimal) ". "}ol.lst-kix_list_16-1{list-style-type:none}ol.lst-kix_list_16-2{list-style-type:none}ol.lst-kix_list_16-3{list-style-type:none}ol.lst-kix_list_16-4{list-style-type:none}ol.lst-kix_list_16-0{list-style-type:none}.lst-kix_list_24-4>li:before{content:"-  "}.lst-kix_list_79-4>li:before{content:"" counter(lst-ctn-kix_list_79-4,decimal) ". "}.lst-kix_list_23-6>li:before{content:"\0025cf   "}.lst-kix_list_23-2>li:before{content:"\0025a0   "}.lst-kix_list_24-0>li:before{content:"-  "}.lst-kix_list_80-5>li:before{content:"\0025cf   "}.lst-kix_e77bf81k0na-2>li:before{content:"\0025a0   "}.lst-kix_list_43-1>li:before{content:"\0025cf   "}.lst-kix_list_22-0>li:before{content:"\0025cf   "}.lst-kix_list_22-8>li:before{content:"\0025cf   "}.lst-kix_e77bf81k0na-6>li:before{content:"\0025cf   "}.lst-kix_4sgfft60r4dw-6>li:before{content:"\0025cf   "}.lst-kix_list_43-5>li:before{content:"\0025cf   "}.lst-kix_list_22-4>li:before{content:"\0025cf   "}.lst-kix_list_99-3>li:before{content:"" counter(lst-ctn-kix_list_99-3,decimal) ". "}.lst-kix_list_99-7>li:before{content:"" counter(lst-ctn-kix_list_99-7,decimal) ". "}.lst-kix_list_78-2>li:before{content:"\0025a0   "}.lst-kix_list_78-6>li:before{content:"\0025cf   "}.lst-kix_4sgfft60r4dw-2>li:before{content:"\0025a0   "}.lst-kix_list_80-1>li:before{content:"o  "}.lst-kix_list_41-1>li:before{content:"\0025cb   "}.lst-kix_list_61-8>li:before{content:"\0025a0   "}.lst-kix_list_40-7>li:before{content:"\0025cf   "}.lst-kix_list_41-5>li:before{content:"\0025a0   "}.lst-kix_list_96-5>li:before{content:"\0025cf   "}.lst-kix_list_62-6>li:before{content:"\0025cf   "}.lst-kix_list_96-1>li:before{content:"\0025cf   "}.lst-kix_list_62-2>li:before{content:"\0025cf   "}ul.lst-kix_list_71-8{list-style-type:none}ul.lst-kix_list_71-6{list-style-type:none}.lst-kix_list_81-7>li:before{content:"\0025cb   "}ul.lst-kix_list_71-7{list-style-type:none}ul.lst-kix_list_127-3{list-style-type:none}ul.lst-kix_list_127-2{list-style-type:none}ul.lst-kix_list_127-1{list-style-type:none}ul.lst-kix_list_127-0{list-style-type:none}ul.lst-kix_list_71-0{list-style-type:none}ul.lst-kix_list_18-0{list-style-type:none}ul.lst-kix_list_127-7{list-style-type:none}ul.lst-kix_list_71-1{list-style-type:none}ul.lst-kix_list_127-6{list-style-type:none}ul.lst-kix_list_127-5{list-style-type:none}ul.lst-kix_list_127-4{list-style-type:none}ul.lst-kix_list_71-4{list-style-type:none}.lst-kix_list_60-6>li:before{content:"\0025cf   "}ul.lst-kix_list_71-5{list-style-type:none}ul.lst-kix_list_71-2{list-style-type:none}ul.lst-kix_list_71-3{list-style-type:none}.lst-kix_list_97-7>li:before{content:"\0025cf   "}ul.lst-kix_list_127-8{list-style-type:none}ul.lst-kix_list_18-8{list-style-type:none}.lst-kix_list_40-3>li:before{content:"\0025cf   "}ul.lst-kix_list_18-7{list-style-type:none}ul.lst-kix_list_18-6{list-style-type:none}ul.lst-kix_list_18-5{list-style-type:none}ul.lst-kix_list_18-4{list-style-type:none}ul.lst-kix_list_18-3{list-style-type:none}ul.lst-kix_list_18-2{list-style-type:none}.lst-kix_list_61-0>li:before{content:"\0025cf   "}.lst-kix_list_61-4>li:before{content:"\0025cb   "}ul.lst-kix_list_18-1{list-style-type:none}.lst-kix_list_97-3>li:before{content:"\0025cf   "}.lst-kix_list_147-4>li:before{content:"\0025cf   "}.lst-kix_list_147-8>li:before{content:"\0025cf   "}.lst-kix_list_114-3>li:before{content:"\0025cf   "}.lst-kix_list_26-8>li:before{content:"\0025a0   "}ul.lst-kix_list_36-0{list-style-type:none}ul.lst-kix_list_36-1{list-style-type:none}ul.lst-kix_list_36-2{list-style-type:none}ul.lst-kix_list_36-3{list-style-type:none}ul.lst-kix_list_36-4{list-style-type:none}.lst-kix_list_133-4>li:before{content:"\0025cf   "}.lst-kix_list_26-4>li:before{content:"\0025cb   "}ol.lst-kix_list_55-4.start{counter-reset:lst-ctn-kix_list_55-4 0}.lst-kix_list_128-3>li:before{content:"\0025cf   "}ul.lst-kix_list_36-5{list-style-type:none}ul.lst-kix_list_36-6{list-style-type:none}ul.lst-kix_list_36-7{list-style-type:none}ul.lst-kix_list_36-8{list-style-type:none}.lst-kix_list_133-8>li:before{content:"\0025cf   "}.lst-kix_list_26-0>li:before{content:"\0025cf   "}ul.lst-kix_list_114-0{list-style-type:none}ul.lst-kix_list_23-0{list-style-type:none}ul.lst-kix_list_114-1{list-style-type:none}ul.lst-kix_list_23-1{list-style-type:none}ul.lst-kix_list_23-2{list-style-type:none}ul.lst-kix_list_23-3{list-style-type:none}ul.lst-kix_list_114-4{list-style-type:none}ul.lst-kix_list_23-4{list-style-type:none}.lst-kix_list_128-7>li:before{content:"\0025cf   "}ul.lst-kix_list_114-5{list-style-type:none}ul.lst-kix_list_23-5{list-style-type:none}ul.lst-kix_list_114-2{list-style-type:none}ul.lst-kix_list_23-6{list-style-type:none}ul.lst-kix_list_114-3{list-style-type:none}ul.lst-kix_list_114-8{list-style-type:none}ul.lst-kix_list_114-6{list-style-type:none}ul.lst-kix_list_114-7{list-style-type:none}.lst-kix_list_59-1>li:before{content:"\0025cb   "}ul.lst-kix_list_23-7{list-style-type:none}.lst-kix_list_45-5>li:before{content:"\0025cf   "}ul.lst-kix_list_23-8{list-style-type:none}ol.lst-kix_list_10-8.start{counter-reset:lst-ctn-kix_list_10-8 0}.lst-kix_list_114-7>li:before{content:"\0025cf   "}ol.lst-kix_list_99-4.start{counter-reset:lst-ctn-kix_list_99-4 0}ol.lst-kix_list_21-4.start{counter-reset:lst-ctn-kix_list_21-4 0}.lst-kix_list_45-1>li:before{content:"\0025cf   "}ol.lst-kix_list_79-3.start{counter-reset:lst-ctn-kix_list_79-3 0}.lst-kix_list_109-6>li:before{content:"\0025cf   "}.lst-kix_list_95-6>li{counter-increment:lst-ctn-kix_list_95-6}.lst-kix_list_109-2>li:before{content:"\0025cf   "}.lst-kix_list_59-5>li:before{content:"\0025a0   "}ul.lst-kix_list_49-0{list-style-type:none}ul.lst-kix_list_49-2{list-style-type:none}ul.lst-kix_list_49-1{list-style-type:none}.lst-kix_list_150-1>li:before{content:"\0025cf   "}.lst-kix_list_101-5>li{counter-increment:lst-ctn-kix_list_101-5}.lst-kix_list_138-7>li{counter-increment:lst-ctn-kix_list_138-7}ul.lst-kix_list_7-5{list-style-type:none}ul.lst-kix_list_7-6{list-style-type:none}ul.lst-kix_list_7-3{list-style-type:none}ul.lst-kix_list_7-4{list-style-type:none}ul.lst-kix_list_7-7{list-style-type:none}ul.lst-kix_list_7-8{list-style-type:none}ul.lst-kix_list_49-4{list-style-type:none}ul.lst-kix_list_49-3{list-style-type:none}ul.lst-kix_list_49-6{list-style-type:none}ul.lst-kix_list_49-5{list-style-type:none}.lst-kix_list_55-5>li{counter-increment:lst-ctn-kix_list_55-5}ul.lst-kix_list_49-8{list-style-type:none}ul.lst-kix_list_7-1{list-style-type:none}ul.lst-kix_list_49-7{list-style-type:none}ul.lst-kix_list_7-2{list-style-type:none}ul.lst-kix_list_7-0{list-style-type:none}ul.lst-kix_list_145-3{list-style-type:none}.lst-kix_list_20-4>li{counter-increment:lst-ctn-kix_list_20-4}ul.lst-kix_list_145-2{list-style-type:none}.lst-kix_list_99-5>li{counter-increment:lst-ctn-kix_list_99-5}ul.lst-kix_list_145-1{list-style-type:none}ul.lst-kix_list_145-0{list-style-type:none}ul.lst-kix_list_145-7{list-style-type:none}ul.lst-kix_list_145-6{list-style-type:none}ul.lst-kix_list_145-5{list-style-type:none}ul.lst-kix_list_145-4{list-style-type:none}ul.lst-kix_list_145-8{list-style-type:none}.lst-kix_list_150-5>li:before{content:"\0025cf   "}.lst-kix_list_145-0>li:before{content:"\0025cf   "}.lst-kix_list_145-4>li:before{content:"\0025cf   "}.lst-kix_list_145-8>li:before{content:"\0025cf   "}.lst-kix_list_28-8>li:before{content:"" counter(lst-ctn-kix_list_28-8,decimal) ". "}.lst-kix_list_131-0>li:before{content:"\0025cf   "}.lst-kix_list_133-0>li:before{content:"\0025cf   "}.lst-kix_list_28-4>li:before{content:"" counter(lst-ctn-kix_list_28-4,decimal) ". "}.lst-kix_list_110-3>li:before{content:"\0025cf   "}.lst-kix_list_149-7>li{counter-increment:lst-ctn-kix_list_149-7}.lst-kix_list_79-6>li{counter-increment:lst-ctn-kix_list_79-6}.lst-kix_list_131-4>li:before{content:"\0025cb   "}.lst-kix_list_28-0>li:before{content:"" counter(lst-ctn-kix_list_28-0,decimal) ". "}.lst-kix_list_147-0>li:before{content:"\0025cf   "}.lst-kix_list_131-8>li:before{content:"\0025a0   "}.lst-kix_list_110-7>li:before{content:"\0025cf   "}ol.lst-kix_list_20-3.start{counter-reset:lst-ctn-kix_list_20-3 0}ul.lst-kix_list_132-5{list-style-type:none}ul.lst-kix_list_132-4{list-style-type:none}ul.lst-kix_list_132-3{list-style-type:none}.lst-kix_list_126-3>li:before{content:"\0025cf   "}ul.lst-kix_list_132-2{list-style-type:none}ul.lst-kix_list_132-8{list-style-type:none}ul.lst-kix_list_132-7{list-style-type:none}ul.lst-kix_list_132-6{list-style-type:none}.lst-kix_list_19-3>li:before{content:"\0025cf   "}.lst-kix_list_56-7>li:before{content:"\0025cf   "}.lst-kix_list_47-5>li:before{content:"\0025cf   "}.lst-kix_list_65-0>li:before{content:"\0025cf   "}.lst-kix_list_135-4>li:before{content:"\0025cf   "}ul.lst-kix_list_132-1{list-style-type:none}ul.lst-kix_list_132-0{list-style-type:none}.lst-kix_list_93-7>li:before{content:"\0025cf   "}.lst-kix_list_84-1>li:before{content:"\0025cb   "}ol.lst-kix_list_16-3.start{counter-reset:lst-ctn-kix_list_16-3 0}.lst-kix_list_144-2>li:before{content:"" counter(lst-ctn-kix_list_144-2,lower-roman) ". "}.lst-kix_list_117-1>li:before{content:"\0025cf   "}.lst-kix_list_107-2>li:before{content:"\0025cf   "}ul.lst-kix_2x6t2se3panh-1{list-style-type:none}ul.lst-kix_2x6t2se3panh-2{list-style-type:none}ul.lst-kix_2x6t2se3panh-0{list-style-type:none}.lst-kix_2x6t2se3panh-5>li:before{content:"\0025a0   "}.lst-kix_tm0xq8syd7t0-7>li:before{content:"\0025cb   "}.lst-kix_list_37-6>li:before{content:"\0025cf   "}.lst-kix_list_108-0>li:before{content:"\0025cf   "}.lst-kix_list_28-5>li{counter-increment:lst-ctn-kix_list_28-5}.lst-kix_list_66-6>li:before{content:"-  "}.lst-kix_list_116-3>li:before{content:"\0025cf   "}ul.lst-kix_list_54-2{list-style-type:none}.lst-kix_list_92-1>li:before{content:"\0025cf   "}ul.lst-kix_list_54-1{list-style-type:none}ul.lst-kix_list_54-4{list-style-type:none}ul.lst-kix_list_54-3{list-style-type:none}ul.lst-kix_list_54-6{list-style-type:none}.lst-kix_list_65-8>li:before{content:"\0025cf   "}ul.lst-kix_list_54-5{list-style-type:none}ul.lst-kix_list_54-8{list-style-type:none}ul.lst-kix_list_54-7{list-style-type:none}ul.lst-kix_list_54-0{list-style-type:none}.lst-kix_list_136-2>li:before{content:"\0025cf   "}.lst-kix_list_38-4>li:before{content:"\0025cb   "}.lst-kix_list_108-8>li:before{content:"\0025a0   "}.lst-kix_list_16-1>li{counter-increment:lst-ctn-kix_list_16-1}.lst-kix_list_118-3>li:before{content:"\0025cf   "}.lst-kix_list_85-7>li:before{content:"\0025cf   "}.lst-kix_list_64-6>li:before{content:"\0025cf   "}.lst-kix_list_94-1>li:before{content:"\0025cf   "}.lst-kix_list_73-0>li:before{content:"\0025cf   "}.lst-kix_list_57-5>li:before{content:"\0025cf   "}.lst-kix_list_36-4>li:before{content:"\0025cf   "}.lst-kix_rxdmkrnvx4u4-6>li:before{content:"\0025cf   "}.lst-kix_36wys1csajrz-6>li:before{content:"\0025cf   "}ol.lst-kix_list_20-8{list-style-type:none}ol.lst-kix_list_20-5{list-style-type:none}.lst-kix_list_83-7>li:before{content:"\0025cf   "}ol.lst-kix_list_20-4{list-style-type:none}ol.lst-kix_list_20-7{list-style-type:none}ol.lst-kix_list_20-6{list-style-type:none}ol.lst-kix_list_20-1{list-style-type:none}ol.lst-kix_list_20-0{list-style-type:none}ol.lst-kix_list_20-3{list-style-type:none}ol.lst-kix_list_20-2{list-style-type:none}.lst-kix_list_73-8>li:before{content:"\0025a0   "}ul.lst-kix_list_67-1{list-style-type:none}ul.lst-kix_list_67-2{list-style-type:none}.lst-kix_list_16-8>li{counter-increment:lst-ctn-kix_list_16-8}ul.lst-kix_list_67-0{list-style-type:none}ul.lst-kix_list_67-5{list-style-type:none}ul.lst-kix_list_67-6{list-style-type:none}ul.lst-kix_list_67-3{list-style-type:none}.lst-kix_list_95-7>li:before{content:"" counter(lst-ctn-kix_list_95-7,decimal) ". "}ul.lst-kix_list_67-4{list-style-type:none}.lst-kix_list_74-6>li:before{content:"\0025cf   "}.lst-kix_list_20-4>li:before{content:"" counter(lst-ctn-kix_list_20-4,decimal) ". "}.lst-kix_list_99-1>li{counter-increment:lst-ctn-kix_list_99-1}.lst-kix_list_75-0>li:before{content:"\0025cf   "}ul.lst-kix_list_67-7{list-style-type:none}ul.lst-kix_list_67-8{list-style-type:none}ol.lst-kix_list_16-8.start{counter-reset:lst-ctn-kix_list_16-8 0}.lst-kix_list_55-5>li:before{content:"" counter(lst-ctn-kix_list_55-5,lower-roman) ". "}.lst-kix_list_76-6>li:before{content:"\0025cf   "}.lst-kix_list_82-1>li:before{content:"\0025cf   "}.lst-kix_list_54-7>li:before{content:"\0025cf   "}.lst-kix_list_75-8>li:before{content:"\0025cf   "}ul.lst-kix_list_85-6{list-style-type:none}ul.lst-kix_list_85-5{list-style-type:none}ul.lst-kix_list_85-4{list-style-type:none}ul.lst-kix_list_85-3{list-style-type:none}ul.lst-kix_list_85-8{list-style-type:none}ul.lst-kix_list_85-7{list-style-type:none}.lst-kix_list_35-6>li:before{content:"\0025cf   "}.lst-kix_list_77-0>li:before{content:"\0025cf   "}.lst-kix_list_77-8>li:before{content:"\0025cf   "}ul.lst-kix_list_85-2{list-style-type:none}ul.lst-kix_list_85-1{list-style-type:none}ul.lst-kix_list_85-0{list-style-type:none}.lst-kix_list_3-2>li:before{content:"\0025a0   "}.lst-kix_list_8-1>li:before{content:"\0025cb   "}ol.lst-kix_list_95-1.start{counter-reset:lst-ctn-kix_list_95-1 0}.lst-kix_list_119-1>li:before{content:"\0025cf   "}.lst-kix_list_30-7>li:before{content:"\0025cf   "}.lst-kix_list_72-6>li:before{content:"\0025cf   "}ol.lst-kix_list_95-0{list-style-type:none}ol.lst-kix_list_95-1{list-style-type:none}.lst-kix_list_21-6>li:before{content:"" counter(lst-ctn-kix_list_21-6,decimal) ". "}ol.lst-kix_list_95-2{list-style-type:none}ol.lst-kix_list_95-6.start{counter-reset:lst-ctn-kix_list_95-6 0}.lst-kix_list_138-2>li:before{content:"" counter(lst-ctn-kix_list_138-2,decimal) ". "}ol.lst-kix_list_95-3{list-style-type:none}ol.lst-kix_list_95-4{list-style-type:none}ol.lst-kix_list_95-5{list-style-type:none}.lst-kix_list_63-0>li:before{content:"\0025cf   "}.lst-kix_list_63-8>li:before{content:"\0025cf   "}.lst-kix_list_123-1>li:before{content:"\0025cf   "}ol.lst-kix_list_95-6{list-style-type:none}ol.lst-kix_list_95-7{list-style-type:none}ol.lst-kix_list_95-8{list-style-type:none}.lst-kix_list_58-7>li:before{content:"\0025cf   "}.lst-kix_list_91-7>li:before{content:"\0025cf   "}ol.lst-kix_list_100-8.start{counter-reset:lst-ctn-kix_list_100-8 0}.lst-kix_list_67-0>li:before{content:"\0025cf   "}.lst-kix_list_129-1>li:before{content:"\0025cf   "}.lst-kix_list_16-5>li:before{content:"" counter(lst-ctn-kix_list_16-5,decimal) ". "}.lst-kix_list_44-7>li:before{content:"\0025cf   "}ol.lst-kix_list_138-7{list-style-type:none}ol.lst-kix_list_138-6{list-style-type:none}.lst-kix_list_86-1>li:before{content:"\0025cf   "}.lst-kix_list_16-4>li{counter-increment:lst-ctn-kix_list_16-4}ol.lst-kix_list_138-8{list-style-type:none}ol.lst-kix_list_138-3{list-style-type:none}ol.lst-kix_list_138-2{list-style-type:none}ol.lst-kix_list_138-5{list-style-type:none}ol.lst-kix_list_138-4{list-style-type:none}ol.lst-kix_list_138-1{list-style-type:none}ol.lst-kix_list_138-0{list-style-type:none}.lst-kix_list_115-1>li:before{content:"\0025cf   "}.lst-kix_list_142-2>li:before{content:"\0025cf   "}ul.lst-kix_36wys1csajrz-5{list-style-type:none}ul.lst-kix_list_41-4{list-style-type:none}ul.lst-kix_36wys1csajrz-6{list-style-type:none}ul.lst-kix_list_41-3{list-style-type:none}ul.lst-kix_36wys1csajrz-7{list-style-type:none}ul.lst-kix_list_41-6{list-style-type:none}.lst-kix_list_69-0>li:before{content:"\0025cf   "}ul.lst-kix_36wys1csajrz-8{list-style-type:none}ul.lst-kix_list_41-5{list-style-type:none}ul.lst-kix_list_41-8{list-style-type:none}ul.lst-kix_list_41-7{list-style-type:none}.lst-kix_list_27-6>li:before{content:"\0025cf   "}ul.lst-kix_36wys1csajrz-0{list-style-type:none}ul.lst-kix_36wys1csajrz-1{list-style-type:none}ul.lst-kix_list_41-0{list-style-type:none}ul.lst-kix_36wys1csajrz-2{list-style-type:none}ul.lst-kix_36wys1csajrz-3{list-style-type:none}ul.lst-kix_list_41-2{list-style-type:none}ul.lst-kix_36wys1csajrz-4{list-style-type:none}ul.lst-kix_list_41-1{list-style-type:none}.lst-kix_list_18-5>li:before{content:"\0025cf   "}ol.lst-kix_list_100-3.start{counter-reset:lst-ctn-kix_list_100-3 0}.lst-kix_list_13-6>li:before{content:"\0025cf   "}.lst-kix_list_148-2>li:before{content:"\0025a0   "}.lst-kix_list_127-1>li:before{content:"\0025cf   "}.lst-kix_list_106-0>li:before{content:"\0025cf   "}.lst-kix_list_39-6>li:before{content:"\0025cf   "}.lst-kix_list_113-1>li:before{content:"o  "}.lst-kix_list_67-8>li:before{content:"\0025a0   "}.lst-kix_list_25-6>li:before{content:"\0025cf   "}.lst-kix_list_134-2>li:before{content:"\0025cf   "}.lst-kix_list_106-8>li:before{content:"\0025cf   "}.lst-kix_list_46-7>li:before{content:"\0025cf   "}ul.lst-kix_list_150-6{list-style-type:none}ul.lst-kix_list_150-7{list-style-type:none}.lst-kix_list_111-1>li:before{content:"\0025cf   "}ul.lst-kix_list_150-8{list-style-type:none}.lst-kix_list_132-2>li:before{content:"\0025a0   "}.lst-kix_list_29-6>li:before{content:"\0025cf   "}.lst-kix_list_11-6>li:before{content:"\0025cf   "}ul.lst-kix_list_150-2{list-style-type:none}ul.lst-kix_list_150-3{list-style-type:none}.lst-kix_list_32-7>li:before{content:"\0025cf   "}ul.lst-kix_list_150-4{list-style-type:none}ul.lst-kix_list_150-5{list-style-type:none}.lst-kix_list_1-2>li:before{content:"-  "}ul.lst-kix_list_150-0{list-style-type:none}ul.lst-kix_list_150-1{list-style-type:none}ul.lst-kix_list_98-4{list-style-type:none}ul.lst-kix_list_98-3{list-style-type:none}ul.lst-kix_list_98-2{list-style-type:none}.lst-kix_list_146-2>li:before{content:"\0025cf   "}ul.lst-kix_list_98-1{list-style-type:none}ul.lst-kix_list_98-8{list-style-type:none}ul.lst-kix_list_98-7{list-style-type:none}.lst-kix_list_104-0>li:before{content:"\0025cf   "}ul.lst-kix_list_98-6{list-style-type:none}ul.lst-kix_list_98-5{list-style-type:none}.lst-kix_list_125-1>li:before{content:"\0025cf   "}ul.lst-kix_list_98-0{list-style-type:none}.lst-kix_list_69-8>li:before{content:"\0025cf   "}.lst-kix_list_48-7>li:before{content:"\0025cf   "}.lst-kix_list_104-8>li:before{content:"\0025cf   "}ol.lst-kix_list_100-4.start{counter-reset:lst-ctn-kix_list_100-4 0}.lst-kix_list_51-4>li:before{content:"\0025cf   "}.lst-kix_list_14-1>li:before{content:"\0025cb   "}.lst-kix_list_121-5>li:before{content:"\0025cf   "}.lst-kix_list_14-4>li:before{content:"\0025cb   "}.lst-kix_list_51-1>li:before{content:"o  "}.lst-kix_list_14-5>li:before{content:"\0025a0   "}.lst-kix_list_121-8>li:before{content:"\0025cf   "}.lst-kix_list_89-8>li:before{content:"\0025a0   "}.lst-kix_list_89-7>li:before{content:"\0025cb   "}.lst-kix_list_51-8>li:before{content:"\0025cf   "}ul.lst-kix_list_70-7{list-style-type:none}.lst-kix_list_20-7>li{counter-increment:lst-ctn-kix_list_20-7}ul.lst-kix_list_70-8{list-style-type:none}ul.lst-kix_list_128-2{list-style-type:none}ul.lst-kix_list_128-1{list-style-type:none}ul.lst-kix_list_128-0{list-style-type:none}ul.lst-kix_list_70-1{list-style-type:none}ul.lst-kix_list_17-1{list-style-type:none}ul.lst-kix_list_128-6{list-style-type:none}ul.lst-kix_list_70-2{list-style-type:none}ul.lst-kix_list_17-0{list-style-type:none}ul.lst-kix_list_128-5{list-style-type:none}ul.lst-kix_list_128-4{list-style-type:none}ul.lst-kix_list_70-0{list-style-type:none}.lst-kix_list_89-4>li:before{content:"\0025cb   "}ul.lst-kix_list_128-3{list-style-type:none}ul.lst-kix_list_70-5{list-style-type:none}ul.lst-kix_list_70-6{list-style-type:none}ul.lst-kix_list_70-3{list-style-type:none}ul.lst-kix_list_128-8{list-style-type:none}ul.lst-kix_list_70-4{list-style-type:none}ul.lst-kix_list_128-7{list-style-type:none}ul.lst-kix_list_17-8{list-style-type:none}.lst-kix_list_102-4>li:before{content:"\0025cf   "}ul.lst-kix_list_17-7{list-style-type:none}.lst-kix_list_32-3>li:before{content:"\0025cf   "}ul.lst-kix_list_17-6{list-style-type:none}ul.lst-kix_list_17-5{list-style-type:none}ul.lst-kix_list_17-4{list-style-type:none}ul.lst-kix_list_17-3{list-style-type:none}.lst-kix_list_14-8>li:before{content:"\0025a0   "}ul.lst-kix_list_17-2{list-style-type:none}.lst-kix_list_32-0>li:before{content:"\0025cf   "}.lst-kix_list_70-2>li:before{content:"\0025cf   "}.lst-kix_list_140-6>li:before{content:"\0025cf   "}.lst-kix_list_5-7>li:before{content:"\0025cf   "}ul.lst-kix_list_8-4{list-style-type:none}ul.lst-kix_list_8-5{list-style-type:none}ul.lst-kix_list_8-2{list-style-type:none}ul.lst-kix_list_8-3{list-style-type:none}ul.lst-kix_list_8-8{list-style-type:none}ul.lst-kix_list_8-6{list-style-type:none}.lst-kix_list_88-2>li:before{content:"\0025cf   "}ul.lst-kix_list_8-7{list-style-type:none}ul.lst-kix_list_8-0{list-style-type:none}.lst-kix_list_120-2>li:before{content:"\0025cf   "}ul.lst-kix_list_8-1{list-style-type:none}.lst-kix_list_70-5>li:before{content:"\0025cf   "}.lst-kix_list_50-2>li:before{content:"\0025cf   "}.lst-kix_list_120-3>li:before{content:"\0025cf   "}.lst-kix_list_6-1>li:before{content:"\0025cf   "}.lst-kix_list_6-0>li:before{content:"\0025cf   "}.lst-kix_list_6-4>li:before{content:"\0025cf   "}.lst-kix_list_120-6>li:before{content:"\0025cf   "}ul.lst-kix_list_83-8{list-style-type:none}.lst-kix_list_69-1>li:before{content:"o  "}ul.lst-kix_list_83-7{list-style-type:none}.lst-kix_list_149-3>li{counter-increment:lst-ctn-kix_list_149-3}ul.lst-kix_list_115-0{list-style-type:none}ul.lst-kix_list_83-6{list-style-type:none}ul.lst-kix_list_83-5{list-style-type:none}.lst-kix_list_7-6>li:before{content:"\0025cf   "}ul.lst-kix_list_115-3{list-style-type:none}ul.lst-kix_list_115-4{list-style-type:none}ul.lst-kix_list_115-1{list-style-type:none}ul.lst-kix_list_115-2{list-style-type:none}ul.lst-kix_list_83-0{list-style-type:none}ul.lst-kix_list_115-7{list-style-type:none}ul.lst-kix_list_115-8{list-style-type:none}ul.lst-kix_list_115-5{list-style-type:none}ul.lst-kix_list_115-6{list-style-type:none}.lst-kix_list_7-2>li:before{content:"\0025a0   "}ul.lst-kix_list_83-4{list-style-type:none}ul.lst-kix_list_83-3{list-style-type:none}ul.lst-kix_list_83-2{list-style-type:none}ul.lst-kix_list_83-1{list-style-type:none}.lst-kix_list_101-1>li:before{content:"" counter(lst-ctn-kix_list_101-1,decimal) ". "}.lst-kix_list_34-8>li:before{content:"\0025cf   "}.lst-kix_list_139-8>li:before{content:"\0025cf   "}.lst-kix_list_101-5>li:before{content:"" counter(lst-ctn-kix_list_101-5,decimal) ". "}.lst-kix_list_100-4>li{counter-increment:lst-ctn-kix_list_100-4}.lst-kix_9jhhktla6bpr-3>li:before{content:"\0025cf   "}.lst-kix_list_52-7>li:before{content:"\0025cf   "}ol.lst-kix_list_28-8{list-style-type:none}ol.lst-kix_list_28-5{list-style-type:none}ol.lst-kix_list_28-4{list-style-type:none}ol.lst-kix_list_28-7{list-style-type:none}ol.lst-kix_list_28-6{list-style-type:none}ol.lst-kix_list_28-1{list-style-type:none}.lst-kix_list_52-3>li:before{content:"\0025cf   "}ol.lst-kix_list_28-0{list-style-type:none}ol.lst-kix_list_28-3{list-style-type:none}ol.lst-kix_list_28-2{list-style-type:none}.lst-kix_list_15-7>li:before{content:"\0025cf   "}.lst-kix_list_4-5>li:before{content:"\0025cf   "}.lst-kix_list_15-3>li:before{content:"\0025cf   "}ol.lst-kix_list_138-7.start{counter-reset:lst-ctn-kix_list_138-7 0}.lst-kix_list_102-7>li:before{content:"\0025cf   "}.lst-kix_list_53-5>li:before{content:"\0025cf   "}.lst-kix_list_101-8>li{counter-increment:lst-ctn-kix_list_101-8}.lst-kix_list_138-4>li{counter-increment:lst-ctn-kix_list_138-4}.lst-kix_list_55-8>li{counter-increment:lst-ctn-kix_list_55-8}.lst-kix_list_33-2>li:before{content:"\0025cf   "}.lst-kix_list_10-4>li{counter-increment:lst-ctn-kix_list_10-4}.lst-kix_list_34-4>li:before{content:"\0025cf   "}.lst-kix_list_104-3>li:before{content:"\0025cf   "}.lst-kix_list_79-3>li{counter-increment:lst-ctn-kix_list_79-3}.lst-kix_list_33-6>li:before{content:"\0025cf   "}.lst-kix_list_142-5>li:before{content:"\0025cf   "}.lst-kix_list_105-2>li:before{content:"\0025cf   "}.lst-kix_list_35-1>li:before{content:"\0025cf   "}.lst-kix_list_105-1>li:before{content:"\0025cf   "}.lst-kix_list_105-5>li:before{content:"\0025cf   "}.lst-kix_list_28-8>li{counter-increment:lst-ctn-kix_list_28-8}.lst-kix_list_142-1>li:before{content:"\0025cf   "}.lst-kix_list_3-3>li:before{content:"\0025cf   "}ol.lst-kix_list_10-7.start{counter-reset:lst-ctn-kix_list_10-7 0}.lst-kix_list_138-3>li:before{content:"" counter(lst-ctn-kix_list_138-3,decimal) ". "}.lst-kix_list_123-0>li:before{content:"\0025cf   "}.lst-kix_list_99-8>li{counter-increment:lst-ctn-kix_list_99-8}ol.lst-kix_list_138-6.start{counter-reset:lst-ctn-kix_list_138-6 0}.lst-kix_list_123-4>li:before{content:"\0025cf   "}.lst-kix_list_68-2>li:before{content:"\0025cf   "}.lst-kix_list_90-5>li:before{content:"\0025a0   "}.lst-kix_list_8-8>li:before{content:"\0025a0   "}.lst-kix_list_4-8>li:before{content:"\0025cf   "}.lst-kix_list_90-8>li:before{content:"\0025a0   "}.lst-kix_list_21-5>li{counter-increment:lst-ctn-kix_list_21-5}.lst-kix_list_87-3>li:before{content:"\0025cf   "}.lst-kix_list_137-0>li:before{content:"\0025cf   "}.lst-kix_list_16-0>li:before{content:"" counter(lst-ctn-kix_list_16-0,decimal) ". "}.lst-kix_list_137-4>li:before{content:"\0025cf   "}ul.lst-kix_list_102-1{list-style-type:none}ul.lst-kix_list_102-2{list-style-type:none}ul.lst-kix_list_35-0{list-style-type:none}ul.lst-kix_list_35-1{list-style-type:none}ul.lst-kix_list_102-0{list-style-type:none}ul.lst-kix_list_35-2{list-style-type:none}ul.lst-kix_list_102-5{list-style-type:none}ul.lst-kix_list_35-3{list-style-type:none}ul.lst-kix_list_102-6{list-style-type:none}ul.lst-kix_list_35-4{list-style-type:none}ul.lst-kix_list_102-3{list-style-type:none}.lst-kix_list_71-4>li:before{content:"\0025cf   "}ul.lst-kix_list_35-5{list-style-type:none}ul.lst-kix_list_102-4{list-style-type:none}.lst-kix_list_141-8>li:before{content:"\0025cf   "}ul.lst-kix_list_102-7{list-style-type:none}ul.lst-kix_list_102-8{list-style-type:none}.lst-kix_1yoisyi8wrza-6>li:before{content:"\0025cf   "}.lst-kix_list_87-0>li:before{content:"\0025cf   "}.lst-kix_list_143-0>li:before{content:"\0025cf   "}.lst-kix_list_71-8>li:before{content:"\0025cf   "}ul.lst-kix_list_35-6{list-style-type:none}ul.lst-kix_list_35-7{list-style-type:none}ul.lst-kix_list_35-8{list-style-type:none}.lst-kix_list_71-7>li:before{content:"\0025cf   "}.lst-kix_list_100-3>li:before{content:"" counter(lst-ctn-kix_list_100-3,decimal) ". "}.lst-kix_list_85-4>li:before{content:"\0025cf   "}.lst-kix_list_2-4>li:before{content:"\0025cf   "}ol.lst-kix_list_138-2.start{counter-reset:lst-ctn-kix_list_138-2 0}.lst-kix_list_85-8>li:before{content:"\0025cf   "}.lst-kix_9jhhktla6bpr-0>li:before{content:"\0025cf   "}.lst-kix_list_7-3>li:before{content:"\0025cf   "}.lst-kix_list_31-1>li:before{content:"\0025cf   "}.lst-kix_list_101-2>li:before{content:"" counter(lst-ctn-kix_list_101-2,decimal) ". "}.lst-kix_list_143-4>li:before{content:"\0025cf   "}.lst-kix_list_52-6>li:before{content:"\0025cf   "}.lst-kix_list_122-7>li:before{content:"\0025cf   "}.lst-kix_list_10-4>li:before{content:"" counter(lst-ctn-kix_list_10-4,lower-latin) ". "}.lst-kix_list_10-8>li:before{content:"" counter(lst-ctn-kix_list_10-8,lower-roman) ". "}.lst-kix_list_73-3>li:before{content:"\0025cf   "}ul.lst-kix_list_48-1{list-style-type:none}ul.lst-kix_list_48-0{list-style-type:none}ul.lst-kix_list_48-3{list-style-type:none}.lst-kix_list_9-7>li:before{content:"\0025cf   "}ul.lst-kix_list_48-2{list-style-type:none}.lst-kix_list_124-3>li:before{content:"\0025cf   "}.lst-kix_list_103-6>li:before{content:"\0025cf   "}ul.lst-kix_list_48-5{list-style-type:none}ul.lst-kix_list_48-4{list-style-type:none}ul.lst-kix_list_48-7{list-style-type:none}.lst-kix_list_54-2>li:before{content:"\0025cf   "}ul.lst-kix_list_48-6{list-style-type:none}ul.lst-kix_list_48-8{list-style-type:none}.lst-kix_list_12-0>li:before{content:"\0025cf   "}ul.lst-kix_list_146-2{list-style-type:none}ul.lst-kix_list_146-1{list-style-type:none}ul.lst-kix_list_146-0{list-style-type:none}ul.lst-kix_list_146-6{list-style-type:none}ul.lst-kix_list_146-5{list-style-type:none}ol.lst-kix_list_138-1.start{counter-reset:lst-ctn-kix_list_138-1 0}ul.lst-kix_list_146-4{list-style-type:none}ul.lst-kix_list_146-3{list-style-type:none}.lst-kix_list_119-2>li:before{content:"\0025cf   "}ul.lst-kix_list_146-8{list-style-type:none}ul.lst-kix_list_146-7{list-style-type:none}.lst-kix_list_33-5>li:before{content:"\0025cf   "}.lst-kix_list_2-0>li:before{content:"\0025cf   "}.lst-kix_list_49-1>li:before{content:"\0025cf   "}ul.lst-kix_list_66-2{list-style-type:none}ul.lst-kix_list_66-3{list-style-type:none}ul.lst-kix_list_66-0{list-style-type:none}ul.lst-kix_list_66-1{list-style-type:none}.lst-kix_list_93-6>li:before{content:"\0025cf   "}ul.lst-kix_list_66-6{list-style-type:none}ul.lst-kix_list_66-7{list-style-type:none}ul.lst-kix_list_66-4{list-style-type:none}ul.lst-kix_list_66-5{list-style-type:none}.lst-kix_list_56-6>li:before{content:"\0025cf   "}.lst-kix_list_93-3>li:before{content:"\0025cf   "}ul.lst-kix_list_66-8{list-style-type:none}.lst-kix_list_126-7>li:before{content:"\0025cf   "}.lst-kix_list_56-3>li:before{content:"\0025cf   "}ul.lst-kix_list_120-5{list-style-type:none}ul.lst-kix_list_120-6{list-style-type:none}ul.lst-kix_list_120-3{list-style-type:none}ul.lst-kix_list_120-4{list-style-type:none}.lst-kix_list_84-2>li:before{content:"\0025a0   "}ul.lst-kix_list_120-7{list-style-type:none}.lst-kix_list_47-2>li:before{content:"\0025cf   "}ul.lst-kix_list_120-8{list-style-type:none}.lst-kix_list_74-2>li:before{content:"\0025a0   "}.lst-kix_list_84-5>li:before{content:"\0025a0   "}ul.lst-kix_list_120-1{list-style-type:none}.lst-kix_list_19-7>li:before{content:"\0025cb   "}.lst-kix_list_144-6>li:before{content:"" counter(lst-ctn-kix_list_144-6,decimal) ". "}ul.lst-kix_list_120-2{list-style-type:none}ul.lst-kix_list_120-0{list-style-type:none}ul.lst-kix_list_133-4{list-style-type:none}ul.lst-kix_list_133-3{list-style-type:none}ul.lst-kix_list_133-2{list-style-type:none}ul.lst-kix_list_133-1{list-style-type:none}ul.lst-kix_list_133-8{list-style-type:none}ul.lst-kix_list_133-7{list-style-type:none}ul.lst-kix_list_133-6{list-style-type:none}ul.lst-kix_list_133-5{list-style-type:none}.lst-kix_2x6t2se3panh-1>li:before{content:"\0025cb   "}.lst-kix_list_107-6>li:before{content:"\0025cf   "}.lst-kix_list_66-3>li:before{content:"-  "}.lst-kix_list_46-3>li:before{content:"\0025cf   "}.lst-kix_list_116-7>li:before{content:"\0025cf   "}.lst-kix_list_37-2>li:before{content:"\0025a0   "}.lst-kix_list_37-5>li:before{content:"\0025a0   "}.lst-kix_list_46-0>li:before{content:"\0025cf   "}ul.lst-kix_list_133-0{list-style-type:none}.lst-kix_2x6t2se3panh-4>li:before{content:"\0025cb   "}.lst-kix_list_116-4>li:before{content:"\0025cf   "}.lst-kix_list_135-8>li:before{content:"\0025cf   "}.lst-kix_list_65-4>li:before{content:"\0025cf   "}.lst-kix_list_92-4>li:before{content:"\0025cf   "}.lst-kix_list_135-5>li:before{content:"\0025cf   "}.lst-kix_list_18-1>li:before{content:"\0025c6   "}.lst-kix_list_65-1>li:before{content:"\0025cf   "}.lst-kix_list_38-8>li:before{content:"\0025a0   "}ul.lst-kix_list_22-0{list-style-type:none}ul.lst-kix_list_22-1{list-style-type:none}ul.lst-kix_list_22-2{list-style-type:none}ul.lst-kix_list_22-3{list-style-type:none}.lst-kix_list_64-2>li:before{content:"\0025cf   "}ul.lst-kix_list_22-4{list-style-type:none}.lst-kix_list_134-3>li:before{content:"\0025cf   "}ul.lst-kix_list_22-5{list-style-type:none}ul.lst-kix_list_22-6{list-style-type:none}ul.lst-kix_list_22-7{list-style-type:none}.lst-kix_list_144-4>li{counter-increment:lst-ctn-kix_list_144-4}ol.lst-kix_list_10-3.start{counter-reset:lst-ctn-kix_list_10-3 0}ol.lst-kix_list_79-4.start{counter-reset:lst-ctn-kix_list_79-4 0}.lst-kix_list_127-0>li:before{content:"\0025cf   "}.lst-kix_list_10-1>li:before{content:"" counter(lst-ctn-kix_list_10-1,lower-latin) ". "}.lst-kix_list_18-4>li:before{content:"\0025c6   "}.lst-kix_rxdmkrnvx4u4-2>li:before{content:"\0025a0   "}ul.lst-kix_list_22-8{list-style-type:none}.lst-kix_list_99-2>li{counter-increment:lst-ctn-kix_list_99-2}.lst-kix_list_106-7>li:before{content:"\0025cf   "}.lst-kix_list_143-7>li:before{content:"\0025cf   "}.lst-kix_list_36-0>li:before{content:"\0025cf   "}.lst-kix_list_94-5>li:before{content:"\0025cf   "}.lst-kix_list_101-2>li{counter-increment:lst-ctn-kix_list_101-2}.lst-kix_list_20-8>li:before{content:"" counter(lst-ctn-kix_list_20-8,decimal) ". "}.lst-kix_list_29-7>li:before{content:"\0025cf   "}.lst-kix_list_75-4>li:before{content:"\0025cf   "}.lst-kix_list_55-2>li{counter-increment:lst-ctn-kix_list_55-2}.lst-kix_list_83-3>li:before{content:"\0025cf   "}.lst-kix_list_11-7>li:before{content:"\0025cf   "}.lst-kix_list_124-6>li:before{content:"\0025cf   "}.lst-kix_list_146-1>li:before{content:"\0025c6   "}.lst-kix_list_55-1>li:before{content:"" counter(lst-ctn-kix_list_55-1,lower-latin) ". "}.lst-kix_list_119-5>li:before{content:"\0025cf   "}.lst-kix_list_48-8>li:before{content:"\0025cf   "}.lst-kix_list_125-8>li:before{content:"\0025cf   "}.lst-kix_list_28-1>li:before{content:"" counter(lst-ctn-kix_list_28-1,decimal) ". "}.lst-kix_list_30-8>li:before{content:"\0025cf   "}ul.lst-kix_list_97-5{list-style-type:none}ul.lst-kix_list_97-4{list-style-type:none}ul.lst-kix_list_97-3{list-style-type:none}ul.lst-kix_list_97-2{list-style-type:none}.lst-kix_list_8-5>li:before{content:"\0025a0   "}ul.lst-kix_list_97-8{list-style-type:none}ul.lst-kix_list_97-7{list-style-type:none}ol.lst-kix_list_28-0.start{counter-reset:lst-ctn-kix_list_28-0 0}ul.lst-kix_list_97-6{list-style-type:none}.lst-kix_list_3-6>li:before{content:"\0025cf   "}ul.lst-kix_list_97-1{list-style-type:none}ul.lst-kix_list_97-0{list-style-type:none}.lst-kix_list_138-6>li:before{content:"" counter(lst-ctn-kix_list_138-6,decimal) ". "}ol.lst-kix_list_79-8.start{counter-reset:lst-ctn-kix_list_79-8 0}ul.lst-kix_list_84-7{list-style-type:none}ul.lst-kix_list_84-6{list-style-type:none}ul.lst-kix_list_84-5{list-style-type:none}ul.lst-kix_list_84-4{list-style-type:none}ul.lst-kix_list_84-8{list-style-type:none}ul.lst-kix_list_84-3{list-style-type:none}ul.lst-kix_list_84-2{list-style-type:none}ul.lst-kix_list_84-1{list-style-type:none}ul.lst-kix_list_84-0{list-style-type:none}.lst-kix_list_129-5>li:before{content:"\0025cf   "}ol.lst-kix_list_28-6.start{counter-reset:lst-ctn-kix_list_28-6 0}ol.lst-kix_list_28-5.start{counter-reset:lst-ctn-kix_list_28-5 0}.lst-kix_list_44-8>li:before{content:"\0025cf   "}.lst-kix_list_129-8>li:before{content:"\0025cf   "}ol.lst-kix_list_10-2.start{counter-reset:lst-ctn-kix_list_10-2 0}.lst-kix_list_17-2>li:before{content:"-  "}.lst-kix_list_59-4>li:before{content:"\0025cb   "}.lst-kix_list_115-5>li:before{content:"\0025cf   "}.lst-kix_list_27-2>li:before{content:"\0025cf   "}.lst-kix_list_22-3>li:before{content:"\0025cf   "}.lst-kix_list_139-5>li:before{content:"\0025cf   "}.lst-kix_list_69-4>li:before{content:"\0025cf   "}ol.lst-kix_list_28-4.start{counter-reset:lst-ctn-kix_list_28-4 0}.lst-kix_list_134-6>li:before{content:"\0025cf   "}ol.lst-kix_list_28-1.start{counter-reset:lst-ctn-kix_list_28-1 0}.lst-kix_e77bf81k0na-5>li:before{content:"\0025a0   "}.lst-kix_list_78-5>li:before{content:"\0025a0   "}ul.lst-kix_list_53-3{list-style-type:none}ul.lst-kix_list_53-2{list-style-type:none}ul.lst-kix_list_53-5{list-style-type:none}ul.lst-kix_list_53-4{list-style-type:none}.lst-kix_list_148-6>li:before{content:"\0025cf   "}ul.lst-kix_list_53-7{list-style-type:none}ul.lst-kix_list_53-6{list-style-type:none}ul.lst-kix_list_53-8{list-style-type:none}.lst-kix_36wys1csajrz-7>li:before{content:"\0025cb   "}ul.lst-kix_list_53-1{list-style-type:none}.lst-kix_list_106-4>li:before{content:"\0025cf   "}ul.lst-kix_list_53-0{list-style-type:none}.lst-kix_list_88-5>li:before{content:"\0025cf   "}.lst-kix_list_144-1>li{counter-increment:lst-ctn-kix_list_144-1}.lst-kix_list_80-2>li:before{content:"\0025aa   "}.lst-kix_list_36-3>li:before{content:"\0025cf   "}ul.lst-kix_list_40-5{list-style-type:none}ul.lst-kix_list_40-4{list-style-type:none}ul.lst-kix_list_40-7{list-style-type:none}ul.lst-kix_list_40-6{list-style-type:none}ul.lst-kix_list_40-8{list-style-type:none}ul.lst-kix_list_40-1{list-style-type:none}ul.lst-kix_list_40-0{list-style-type:none}.lst-kix_list_74-5>li:before{content:"\0025a0   "}ul.lst-kix_list_40-3{list-style-type:none}ul.lst-kix_list_40-2{list-style-type:none}.lst-kix_list_100-1>li{counter-increment:lst-ctn-kix_list_100-1}.lst-kix_list_41-4>li:before{content:"\0025cb   "}.lst-kix_list_111-5>li:before{content:"\0025cf   "}.lst-kix_list_95-7>li{counter-increment:lst-ctn-kix_list_95-7}.lst-kix_list_10-1>li{counter-increment:lst-ctn-kix_list_10-1}.lst-kix_list_60-5>li:before{content:"\0025cf   "}.lst-kix_list_97-6>li:before{content:"\0025cf   "}.lst-kix_list_61-1>li:before{content:"\0025cb   "}.lst-kix_list_55-4>li:before{content:"" counter(lst-ctn-kix_list_55-4,lower-latin) ". "}.lst-kix_list_125-5>li:before{content:"\0025cf   "}.lst-kix_list_13-2>li:before{content:"\0025a0   "}.lst-kix_list_98-2>li:before{content:"\0025cf   "}.lst-kix_list_42-8>li:before{content:"\0025a0   "}ol.lst-kix_list_20-2.start{counter-reset:lst-ctn-kix_list_20-2 0}.lst-kix_list_130-7>li:before{content:"\0025cf   "}.lst-kix_list_98-4>li:before{content:"\0025cf   "}.lst-kix_list_60-1>li:before{content:"\0025cf   "}.lst-kix_list_42-0>li:before{content:"\0025cf   "}.lst-kix_list_112-2>li:before{content:"\0025cf   "}.lst-kix_list_42-2>li:before{content:"\0025a0   "}.lst-kix_list_100-0>li{counter-increment:lst-ctn-kix_list_100-0}.lst-kix_list_112-4>li:before{content:"\0025cf   "}.lst-kix_list_130-1>li:before{content:"\0025cf   "}.lst-kix_list_149-1>li:before{content:"" counter(lst-ctn-kix_list_149-1,decimal) ". "}.lst-kix_list_112-8>li:before{content:"\0025cf   "}ol.lst-kix_list_10-4.start{counter-reset:lst-ctn-kix_list_10-4 0}.lst-kix_list_79-7>li:before{content:"" counter(lst-ctn-kix_list_79-7,decimal) ". "}.lst-kix_list_149-7>li:before{content:"" counter(lst-ctn-kix_list_149-7,decimal) ". "}.lst-kix_list_24-3>li:before{content:"-  "}.lst-kix_list_79-5>li:before{content:"" counter(lst-ctn-kix_list_79-5,decimal) ". "}.lst-kix_list_23-3>li:before{content:"\0025cf   "}.lst-kix_list_23-5>li:before{content:"\0025a0   "}ol.lst-kix_list_79-7.start{counter-reset:lst-ctn-kix_list_79-7 0}.lst-kix_list_150-4>li:before{content:"\0025cf   "}ol.lst-kix_list_144-5.start{counter-reset:lst-ctn-kix_list_144-5 0}.lst-kix_list_113-8>li:before{content:"\0025cf   "}.lst-kix_list_148-1>li:before{content:"\0025cb   "}.lst-kix_list_148-7>li:before{content:"\0025cb   "}.lst-kix_list_25-7>li:before{content:"\0025cb   "}.lst-kix_list_113-2>li:before{content:"\0025cf   "}.lst-kix_list_150-6>li:before{content:"\0025cf   "}ol.lst-kix_list_149-4.start{counter-reset:lst-ctn-kix_list_149-4 0}ul.lst-kix_list_82-8{list-style-type:none}ul.lst-kix_list_82-7{list-style-type:none}ul.lst-kix_list_82-6{list-style-type:none}.lst-kix_list_132-1>li:before{content:"\0025cb   "}.lst-kix_list_111-2>li:before{content:"\0025cf   "}ul.lst-kix_list_29-0{list-style-type:none}ul.lst-kix_list_82-1{list-style-type:none}ul.lst-kix_list_82-0{list-style-type:none}.lst-kix_list_111-4>li:before{content:"\0025cf   "}ol.lst-kix_list_95-7.start{counter-reset:lst-ctn-kix_list_95-7 0}.lst-kix_list_132-5>li:before{content:"\0025a0   "}ul.lst-kix_list_82-5{list-style-type:none}ul.lst-kix_list_82-4{list-style-type:none}ul.lst-kix_list_82-3{list-style-type:none}ul.lst-kix_list_82-2{list-style-type:none}.lst-kix_list_132-7>li:before{content:"\0025cb   "}ol.lst-kix_list_101-3.start{counter-reset:lst-ctn-kix_list_101-3 0}.lst-kix_list_131-1>li:before{content:"\0025cb   "}ul.lst-kix_list_29-1{list-style-type:none}ul.lst-kix_list_29-2{list-style-type:none}ul.lst-kix_list_29-3{list-style-type:none}ul.lst-kix_list_29-4{list-style-type:none}ul.lst-kix_list_29-5{list-style-type:none}ul.lst-kix_list_29-6{list-style-type:none}ul.lst-kix_list_29-7{list-style-type:none}ul.lst-kix_list_29-8{list-style-type:none}.lst-kix_list_110-4>li:before{content:"\0025cf   "}.lst-kix_list_147-1>li:before{content:"\0025cf   "}.lst-kix_list_131-7>li:before{content:"\0025cb   "}ul.lst-kix_list_47-0{list-style-type:none}.lst-kix_list_147-3>li:before{content:"\0025cf   "}ul.lst-kix_list_47-2{list-style-type:none}ul.lst-kix_list_47-1{list-style-type:none}ul.lst-kix_list_47-4{list-style-type:none}ul.lst-kix_list_47-3{list-style-type:none}.lst-kix_list_77-7>li:before{content:"\0025cf   "}ul.lst-kix_list_5-7{list-style-type:none}ul.lst-kix_list_5-8{list-style-type:none}ul.lst-kix_list_5-5{list-style-type:none}ul.lst-kix_list_5-6{list-style-type:none}ul.lst-kix_list_47-6{list-style-type:none}ul.lst-kix_list_47-5{list-style-type:none}ul.lst-kix_list_5-0{list-style-type:none}ul.lst-kix_list_47-8{list-style-type:none}.lst-kix_list_147-7>li:before{content:"\0025cf   "}ul.lst-kix_list_47-7{list-style-type:none}.lst-kix_list_114-2>li:before{content:"\0025cf   "}ul.lst-kix_list_5-3{list-style-type:none}ul.lst-kix_list_5-4{list-style-type:none}ul.lst-kix_list_5-1{list-style-type:none}ul.lst-kix_list_5-2{list-style-type:none}.lst-kix_list_26-1>li:before{content:"\0025cb   "}.lst-kix_list_10-2>li{counter-increment:lst-ctn-kix_list_10-2}.lst-kix_list_95-0>li:before{content:"" counter(lst-ctn-kix_list_95-0,decimal) ". "}.lst-kix_list_21-5>li:before{content:"" counter(lst-ctn-kix_list_21-5,decimal) ". "}.lst-kix_list_63-3>li:before{content:"\0025cf   "}.lst-kix_list_115-6>li:before{content:"\0025cf   "}.lst-kix_list_58-2>li:before{content:"\0025cf   "}.lst-kix_list_21-0>li{counter-increment:lst-ctn-kix_list_21-0}.lst-kix_list_25-1>li:before{content:"\0025cb   "}.lst-kix_list_129-2>li:before{content:"\0025cf   "}.lst-kix_list_45-6>li:before{content:"\0025cf   "}.lst-kix_list_44-6>li:before{content:"\0025cf   "}.lst-kix_list_144-5>li{counter-increment:lst-ctn-kix_list_144-5}ul.lst-kix_list_125-1{list-style-type:none}ul.lst-kix_list_125-0{list-style-type:none}ul.lst-kix_list_125-5{list-style-type:none}ul.lst-kix_list_125-4{list-style-type:none}ul.lst-kix_list_125-3{list-style-type:none}ul.lst-kix_list_125-2{list-style-type:none}.lst-kix_list_109-1>li:before{content:"o  "}ul.lst-kix_list_125-8{list-style-type:none}ul.lst-kix_list_125-7{list-style-type:none}ul.lst-kix_list_125-6{list-style-type:none}.lst-kix_list_16-7>li{counter-increment:lst-ctn-kix_list_16-7}.lst-kix_list_114-6>li:before{content:"\0025cf   "}.lst-kix_list_59-2>li:before{content:"\0025a0   "}.lst-kix_list_109-5>li:before{content:"\0025cf   "}ol.lst-kix_list_138-5.start{counter-reset:lst-ctn-kix_list_138-5 0}.lst-kix_list_64-3>li:before{content:"\0025cf   "}.lst-kix_list_80-4>li:before{content:"\0025cf   "}.lst-kix_list_22-5>li:before{content:"\0025cf   "}.lst-kix_list_43-2>li:before{content:"\0025cf   "}.lst-kix_list_43-6>li:before{content:"\0025cf   "}.lst-kix_list_21-2>li{counter-increment:lst-ctn-kix_list_21-2}ol.lst-kix_list_99-8.start{counter-reset:lst-ctn-kix_list_99-8 0}.lst-kix_rxdmkrnvx4u4-1>li:before{content:"\0025cb   "}.lst-kix_list_76-7>li:before{content:"\0025cf   "}.lst-kix_e77bf81k0na-7>li:before{content:"\0025cb   "}.lst-kix_list_94-0>li:before{content:"\0025cf   "}ol.lst-kix_list_55-0.start{counter-reset:lst-ctn-kix_list_55-0 0}.lst-kix_list_99-4>li:before{content:"" counter(lst-ctn-kix_list_99-4,decimal) ". "}.lst-kix_list_78-7>li:before{content:"\0025cb   "}.lst-kix_list_99-8>li:before{content:"" counter(lst-ctn-kix_list_99-8,decimal) ". "}.lst-kix_list_83-8>li:before{content:"\0025cf   "}.lst-kix_list_94-8>li:before{content:"\0025cf   "}ul.lst-kix_list_112-2{list-style-type:none}ul.lst-kix_list_112-3{list-style-type:none}ul.lst-kix_list_112-0{list-style-type:none}ul.lst-kix_list_112-1{list-style-type:none}ul.lst-kix_list_112-6{list-style-type:none}.lst-kix_list_82-8>li:before{content:"\0025cf   "}ul.lst-kix_list_112-7{list-style-type:none}ul.lst-kix_list_112-4{list-style-type:none}.lst-kix_list_96-0>li:before{content:"\0025cf   "}ul.lst-kix_list_112-5{list-style-type:none}ol.lst-kix_list_21-8.start{counter-reset:lst-ctn-kix_list_21-8 0}.lst-kix_list_61-7>li:before{content:"\0025cb   "}ul.lst-kix_list_112-8{list-style-type:none}.lst-kix_list_83-0>li:before{content:"\0025cf   "}.lst-kix_list_20-5>li:before{content:"" counter(lst-ctn-kix_list_20-5,decimal) ". "}.lst-kix_list_144-0>li{counter-increment:lst-ctn-kix_list_144-0}.lst-kix_list_75-3>li:before{content:"\0025cf   "}.lst-kix_list_96-4>li:before{content:"\0025cf   "}.lst-kix_list_41-2>li:before{content:"\0025a0   "}.lst-kix_list_20-1>li:before{content:"" counter(lst-ctn-kix_list_20-1,decimal) ". "}.lst-kix_list_81-8>li:before{content:"\0025a0   "}.lst-kix_list_21-7>li{counter-increment:lst-ctn-kix_list_21-7}.lst-kix_list_60-7>li:before{content:"\0025cf   "}.lst-kix_list_81-4>li:before{content:"\0025cb   "}.lst-kix_list_10-7>li{counter-increment:lst-ctn-kix_list_10-7}.lst-kix_list_76-3>li:before{content:"\0025cf   "}.lst-kix_list_97-4>li:before{content:"\0025cf   "}.lst-kix_list_40-2>li:before{content:"\0025cf   "}.lst-kix_list_93-2>li:before{content:"\0025cf   "}.lst-kix_list_56-4>li:before{content:"\0025cf   "}.lst-kix_list_135-3>li:before{content:"\0025cf   "}ul.lst-kix_list_78-1{list-style-type:none}ul.lst-kix_list_143-5{list-style-type:none}ul.lst-kix_list_78-2{list-style-type:none}ul.lst-kix_list_143-4{list-style-type:none}ul.lst-kix_list_143-3{list-style-type:none}ul.lst-kix_list_78-0{list-style-type:none}ul.lst-kix_list_143-2{list-style-type:none}ul.lst-kix_list_78-5{list-style-type:none}ul.lst-kix_list_78-6{list-style-type:none}ul.lst-kix_list_143-8{list-style-type:none}ul.lst-kix_list_78-3{list-style-type:none}ul.lst-kix_list_143-7{list-style-type:none}ul.lst-kix_list_78-4{list-style-type:none}ul.lst-kix_list_143-6{list-style-type:none}.lst-kix_list_117-6>li:before{content:"\0025cf   "}.lst-kix_list_84-6>li:before{content:"\0025cf   "}.lst-kix_list_74-3>li:before{content:"\0025cf   "}ul.lst-kix_list_78-7{list-style-type:none}ul.lst-kix_list_78-8{list-style-type:none}.lst-kix_list_117-4>li:before{content:"\0025cf   "}ul.lst-kix_list_143-1{list-style-type:none}.lst-kix_list_144-5>li:before{content:"" counter(lst-ctn-kix_list_144-5,lower-roman) ". "}ul.lst-kix_list_143-0{list-style-type:none}.lst-kix_list_107-1>li:before{content:"o  "}ul.lst-kix_list_65-3{list-style-type:none}ul.lst-kix_list_65-4{list-style-type:none}ul.lst-kix_list_65-1{list-style-type:none}ul.lst-kix_list_65-2{list-style-type:none}ul.lst-kix_list_65-7{list-style-type:none}ul.lst-kix_list_65-8{list-style-type:none}ul.lst-kix_list_65-5{list-style-type:none}ul.lst-kix_list_65-6{list-style-type:none}.lst-kix_list_107-7>li:before{content:"\0025cf   "}.lst-kix_tm0xq8syd7t0-2>li:before{content:"\0025a0   "}.lst-kix_2x6t2se3panh-8>li:before{content:"\0025a0   "}ul.lst-kix_list_65-0{list-style-type:none}.lst-kix_list_116-0>li:before{content:"\0025cf   "}.lst-kix_list_37-1>li:before{content:"\0025cb   "}.lst-kix_list_66-7>li:before{content:"-  "}.lst-kix_tm0xq8syd7t0-4>li:before{content:"\0025cb   "}.lst-kix_list_38-7>li:before{content:"\0025cb   "}.lst-kix_list_18-2>li:before{content:"\0025cf   "}.lst-kix_list_65-3>li:before{content:"\0025cf   "}.lst-kix_list_66-1>li:before{content:"-  "}.lst-kix_list_92-8>li:before{content:"\0025cf   "}.lst-kix_list_38-5>li:before{content:"\0025a0   "}ul.lst-kix_list_130-7{list-style-type:none}.lst-kix_list_134-5>li:before{content:"\0025cf   "}ol.lst-kix_list_95-5.start{counter-reset:lst-ctn-kix_list_95-5 0}.lst-kix_list_27-1>li:before{content:"\0025cf   "}ul.lst-kix_list_130-6{list-style-type:none}ul.lst-kix_list_130-5{list-style-type:none}.lst-kix_list_48-2>li:before{content:"\0025cf   "}ul.lst-kix_list_130-4{list-style-type:none}ul.lst-kix_list_130-8{list-style-type:none}.lst-kix_list_48-4>li:before{content:"\0025cf   "}.lst-kix_list_18-8>li:before{content:"\0025cf   "}ul.lst-kix_4sgfft60r4dw-4{list-style-type:none}ul.lst-kix_4sgfft60r4dw-5{list-style-type:none}ul.lst-kix_4sgfft60r4dw-2{list-style-type:none}ul.lst-kix_4sgfft60r4dw-3{list-style-type:none}ul.lst-kix_list_130-3{list-style-type:none}ul.lst-kix_4sgfft60r4dw-0{list-style-type:none}ul.lst-kix_list_130-2{list-style-type:none}ul.lst-kix_4sgfft60r4dw-1{list-style-type:none}ul.lst-kix_list_130-1{list-style-type:none}ul.lst-kix_list_130-0{list-style-type:none}.lst-kix_list_10-7>li:before{content:"" counter(lst-ctn-kix_list_10-7,lower-latin) ". "}ul.lst-kix_list_34-0{list-style-type:none}ul.lst-kix_list_34-1{list-style-type:none}.lst-kix_list_127-6>li:before{content:"\0025cf   "}ul.lst-kix_list_34-2{list-style-type:none}.lst-kix_list_10-5>li:before{content:"" counter(lst-ctn-kix_list_10-5,lower-roman) ". "}ul.lst-kix_list_34-3{list-style-type:none}ul.lst-kix_4sgfft60r4dw-8{list-style-type:none}ul.lst-kix_list_34-4{list-style-type:none}ul.lst-kix_list_34-5{list-style-type:none}ul.lst-kix_4sgfft60r4dw-6{list-style-type:none}ul.lst-kix_list_34-6{list-style-type:none}ul.lst-kix_4sgfft60r4dw-7{list-style-type:none}.lst-kix_list_106-3>li:before{content:"\0025cf   "}.lst-kix_list_127-4>li:before{content:"\0025cf   "}.lst-kix_list_9-2>li:before{content:"\0025cf   "}.lst-kix_list_46-4>li:before{content:"\0025cf   "}ul.lst-kix_list_34-7{list-style-type:none}ul.lst-kix_list_34-8{list-style-type:none}.lst-kix_list_124-0>li:before{content:"\0025cf   "}.lst-kix_list_145-3>li:before{content:"\0025cf   "}.lst-kix_list_124-2>li:before{content:"\0025a0   "}.lst-kix_list_124-8>li:before{content:"\0025a0   "}.lst-kix_list_49-8>li:before{content:"\0025cf   "}.lst-kix_list_9-8>li:before{content:"\0025cf   "}.lst-kix_list_28-7>li:before{content:"" counter(lst-ctn-kix_list_28-7,decimal) ". "}.lst-kix_list_1-7>li:before{content:"-  "}.lst-kix_list_1-5>li:before{content:"-  "}.lst-kix_list_28-5>li:before{content:"" counter(lst-ctn-kix_list_28-5,decimal) ". "}.lst-kix_list_126-0>li:before{content:"\0025cf   "}.lst-kix_list_125-6>li:before{content:"\0025cf   "}.lst-kix_list_35-3>li:before{content:"\0025cf   "}.lst-kix_list_142-7>li:before{content:"\0025cf   "}.lst-kix_list_30-4>li:before{content:"\0025cf   "}.lst-kix_list_20-8>li{counter-increment:lst-ctn-kix_list_20-8}.lst-kix_list_114-0>li:before{content:"\0025cf   "}.lst-kix_list_26-7>li:before{content:"\0025cb   "}.lst-kix_list_3-5>li:before{content:"\0025a0   "}.lst-kix_list_81-2>li:before{content:"\0025a0   "}.lst-kix_list_63-5>li:before{content:"\0025cf   "}.lst-kix_list_53-0>li:before{content:"\0025cf   "}.lst-kix_list_68-1>li:before{content:"o  "}.lst-kix_list_149-6>li{counter-increment:lst-ctn-kix_list_149-6}ol.lst-kix_list_149-8{list-style-type:none}.lst-kix_list_91-2>li:before{content:"\0025cf   "}.lst-kix_list_137-1>li:before{content:"\0025cf   "}ol.lst-kix_list_149-6{list-style-type:none}ol.lst-kix_list_149-7{list-style-type:none}ol.lst-kix_list_149-4{list-style-type:none}ol.lst-kix_list_149-5{list-style-type:none}.lst-kix_list_25-3>li:before{content:"\0025cf   "}ol.lst-kix_list_149-2{list-style-type:none}ol.lst-kix_list_149-3{list-style-type:none}ol.lst-kix_list_149-0{list-style-type:none}ol.lst-kix_list_149-1{list-style-type:none}.lst-kix_list_129-4>li:before{content:"\0025cf   "}.lst-kix_list_58-4>li:before{content:"\0025cf   "}.lst-kix_list_138-3>li{counter-increment:lst-ctn-kix_list_138-3}.lst-kix_list_86-6>li:before{content:"\0025cf   "}ul.lst-kix_list_96-6{list-style-type:none}.lst-kix_list_114-8>li:before{content:"\0025cf   "}ul.lst-kix_list_96-5{list-style-type:none}ul.lst-kix_list_96-4{list-style-type:none}ul.lst-kix_list_96-3{list-style-type:none}.lst-kix_list_59-8>li:before{content:"\0025a0   "}ul.lst-kix_list_96-8{list-style-type:none}ul.lst-kix_list_96-7{list-style-type:none}ul.lst-kix_list_96-2{list-style-type:none}ul.lst-kix_list_96-1{list-style-type:none}ul.lst-kix_list_96-0{list-style-type:none}ol.lst-kix_list_100-7.start{counter-reset:lst-ctn-kix_list_100-7 0}.lst-kix_list_45-0>li:before{content:"\0025cf   "}.lst-kix_list_71-5>li:before{content:"\0025cf   "}.lst-kix_9jhhktla6bpr-2>li:before{content:"\0025a0   "}.lst-kix_list_139-1>li:before{content:"\0025cf   "}.lst-kix_list_43-0>li:before{content:"\0025cf   "}.lst-kix_list_43-8>li:before{content:"\0025cf   "}.lst-kix_e77bf81k0na-1>li:before{content:"\0025cb   "}.lst-kix_list_109-7>li:before{content:"\0025cf   "}.lst-kix_list_101-4>li{counter-increment:lst-ctn-kix_list_101-4}.lst-kix_list_101-4>li:before{content:"" counter(lst-ctn-kix_list_101-4,decimal) ". "}.lst-kix_list_15-6>li:before{content:"\0025cf   "}.lst-kix_list_73-5>li:before{content:"\0025a0   "}.lst-kix_list_94-6>li:before{content:"\0025cf   "}.lst-kix_list_78-1>li:before{content:"\0025cb   "}.lst-kix_list_99-2>li:before{content:"" counter(lst-ctn-kix_list_99-2,decimal) ". "}.lst-kix_list_36-7>li:before{content:"\0025cf   "}.lst-kix_list_20-7>li:before{content:"" counter(lst-ctn-kix_list_20-7,decimal) ". "}.lst-kix_list_41-8>li:before{content:"\0025a0   "}.lst-kix_list_95-2>li{counter-increment:lst-ctn-kix_list_95-2}.lst-kix_list_102-8>li:before{content:"\0025cf   "}.lst-kix_list_33-3>li:before{content:"\0025cf   "}.lst-kix_list_83-2>li:before{content:"\0025cf   "}.lst-kix_list_53-8>li:before{content:"\0025cf   "}ol.lst-kix_list_100-5.start{counter-reset:lst-ctn-kix_list_100-5 0}ul.lst-kix_list_52-4{list-style-type:none}ul.lst-kix_list_52-3{list-style-type:none}ul.lst-kix_list_52-6{list-style-type:none}ul.lst-kix_list_52-5{list-style-type:none}ul.lst-kix_list_52-8{list-style-type:none}.lst-kix_list_119-4>li:before{content:"\0025cf   "}.lst-kix_list_76-1>li:before{content:"\0025cf   "}ul.lst-kix_list_52-7{list-style-type:none}ul.lst-kix_list_108-0{list-style-type:none}.lst-kix_list_55-0>li:before{content:"" counter(lst-ctn-kix_list_55-0,decimal) ". "}ul.lst-kix_list_108-3{list-style-type:none}ul.lst-kix_list_108-4{list-style-type:none}ol.lst-kix_list_16-1.start{counter-reset:lst-ctn-kix_list_16-1 0}ul.lst-kix_list_108-1{list-style-type:none}ul.lst-kix_list_108-2{list-style-type:none}ul.lst-kix_list_52-0{list-style-type:none}ul.lst-kix_list_108-7{list-style-type:none}ul.lst-kix_list_108-8{list-style-type:none}ul.lst-kix_list_52-2{list-style-type:none}ul.lst-kix_list_108-5{list-style-type:none}.lst-kix_list_96-6>li:before{content:"\0025cf   "}ul.lst-kix_list_52-1{list-style-type:none}ul.lst-kix_list_108-6{list-style-type:none}.lst-kix_list_61-5>li:before{content:"\0025a0   "}.lst-kix_list_79-0>li{counter-increment:lst-ctn-kix_list_79-0}.lst-kix_list_40-4>li:before{content:"\0025cf   "}.lst-kix_list_51-5>li:before{content:"\0025cf   "}ul.lst-kix_list_81-8{list-style-type:none}ul.lst-kix_list_81-7{list-style-type:none}.lst-kix_list_14-3>li:before{content:"\0025cf   "}.lst-kix_list_14-0>li:before{content:"\0025cf   "}ul.lst-kix_list_81-2{list-style-type:none}ul.lst-kix_list_81-1{list-style-type:none}ul.lst-kix_list_81-0{list-style-type:none}ul.lst-kix_list_81-6{list-style-type:none}ul.lst-kix_list_81-5{list-style-type:none}.lst-kix_list_51-2>li:before{content:"\0025aa   "}ul.lst-kix_list_81-4{list-style-type:none}.lst-kix_list_121-7>li:before{content:"\0025cf   "}ul.lst-kix_list_81-3{list-style-type:none}.lst-kix_list_14-6>li:before{content:"\0025cf   "}.lst-kix_list_32-2>li:before{content:"\0025cf   "}ol.lst-kix_list_99-6.start{counter-reset:lst-ctn-kix_list_99-6 0}ol.lst-kix_list_21-6.start{counter-reset:lst-ctn-kix_list_21-6 0}.lst-kix_list_102-2>li:before{content:"\0025cf   "}.lst-kix_list_89-2>li:before{content:"\0025a0   "}.lst-kix_list_102-5>li:before{content:"\0025cf   "}.lst-kix_list_5-0>li:before{content:"\0025cf   "}.lst-kix_list_140-4>li:before{content:"\0025cf   "}.lst-kix_list_5-3>li:before{content:"\0025cf   "}ol.lst-kix_list_144-3.start{counter-reset:lst-ctn-kix_list_144-3 0}.lst-kix_list_99-0>li{counter-increment:lst-ctn-kix_list_99-0}.lst-kix_list_70-1>li:before{content:"\0025cf   "}.lst-kix_list_5-6>li:before{content:"\0025cf   "}.lst-kix_list_140-7>li:before{content:"\0025cf   "}.lst-kix_list_70-7>li:before{content:"\0025cf   "}.lst-kix_list_70-4>li:before{content:"\0025cf   "}.lst-kix_list_88-1>li:before{content:"\0025cf   "}.lst-kix_list_120-4>li:before{content:"\0025cf   "}.lst-kix_list_50-4>li:before{content:"\0025cf   "}.lst-kix_list_120-7>li:before{content:"\0025cf   "}ol.lst-kix_list_55-2.start{counter-reset:lst-ctn-kix_list_55-2 0}.lst-kix_list_50-7>li:before{content:"\0025cf   "}.lst-kix_list_121-1>li:before{content:"\0025cf   "}.lst-kix_list_121-4>li:before{content:"\0025cf   "}.lst-kix_list_7-4>li:before{content:"\0025cb   "}.lst-kix_list_139-4>li:before{content:"\0025cf   "}.lst-kix_9jhhktla6bpr-5>li:before{content:"\0025a0   "}.lst-kix_list_69-5>li:before{content:"\0025cf   "}.lst-kix_list_122-2>li:before{content:"\0025cf   "}ol.lst-kix_list_149-6.start{counter-reset:lst-ctn-kix_list_149-6 0}ul.lst-kix_list_126-0{list-style-type:none}.lst-kix_list_31-8>li:before{content:"\0025cf   "}.lst-kix_list_52-5>li:before{content:"\0025cf   "}ul.lst-kix_list_126-4{list-style-type:none}ul.lst-kix_list_126-3{list-style-type:none}.lst-kix_list_120-1>li:before{content:"o  "}ul.lst-kix_list_126-2{list-style-type:none}ul.lst-kix_list_126-1{list-style-type:none}ul.lst-kix_list_126-8{list-style-type:none}ul.lst-kix_list_126-7{list-style-type:none}.lst-kix_list_31-2>li:before{content:"\0025cf   "}ul.lst-kix_list_126-6{list-style-type:none}ul.lst-kix_list_126-5{list-style-type:none}.lst-kix_list_101-7>li:before{content:"" counter(lst-ctn-kix_list_101-7,decimal) ". "}.lst-kix_list_88-4>li:before{content:"\0025cf   "}.lst-kix_list_4-3>li:before{content:"\0025cf   "}ol.lst-kix_list_101-1.start{counter-reset:lst-ctn-kix_list_101-1 0}.lst-kix_list_16-2>li{counter-increment:lst-ctn-kix_list_16-2}ul.lst-kix_list_139-2{list-style-type:none}ul.lst-kix_list_139-1{list-style-type:none}ul.lst-kix_list_139-0{list-style-type:none}ul.lst-kix_list_139-6{list-style-type:none}ol.lst-kix_list_16-7.start{counter-reset:lst-ctn-kix_list_16-7 0}ul.lst-kix_list_139-5{list-style-type:none}ul.lst-kix_list_139-4{list-style-type:none}ul.lst-kix_list_139-3{list-style-type:none}.lst-kix_list_123-8>li:before{content:"\0025cf   "}.lst-kix_list_55-1>li{counter-increment:lst-ctn-kix_list_55-1}ul.lst-kix_list_139-8{list-style-type:none}ul.lst-kix_list_139-7{list-style-type:none}.lst-kix_list_33-0>li:before{content:"\0025cf   "}.lst-kix_list_32-8>li:before{content:"\0025cf   "}.lst-kix_list_103-5>li:before{content:"\0025cf   "}ul.lst-kix_list_94-8{list-style-type:none}ul.lst-kix_list_94-7{list-style-type:none}.lst-kix_list_34-0>li:before{content:"\0025cf   "}ul.lst-kix_list_94-6{list-style-type:none}ul.lst-kix_list_94-5{list-style-type:none}.lst-kix_list_13-3>li:before{content:"\0025cf   "}ul.lst-kix_list_94-0{list-style-type:none}ul.lst-kix_list_94-4{list-style-type:none}ul.lst-kix_list_94-3{list-style-type:none}.lst-kix_list_34-6>li:before{content:"\0025cf   "}ul.lst-kix_list_94-2{list-style-type:none}ul.lst-kix_list_94-1{list-style-type:none}.lst-kix_list_12-5>li:before{content:"\0025a0   "}ul.lst-kix_list_113-1{list-style-type:none}ul.lst-kix_list_113-2{list-style-type:none}ul.lst-kix_list_113-0{list-style-type:none}ul.lst-kix_list_113-5{list-style-type:none}.lst-kix_list_105-6>li:before{content:"\0025cf   "}ul.lst-kix_list_113-6{list-style-type:none}ul.lst-kix_list_113-3{list-style-type:none}ul.lst-kix_list_113-4{list-style-type:none}ul.lst-kix_list_113-7{list-style-type:none}ul.lst-kix_list_113-8{list-style-type:none}.lst-kix_list_100-7>li:before{content:"" counter(lst-ctn-kix_list_100-7,decimal) ". "}.lst-kix_list_68-7>li:before{content:"\0025cf   "}.lst-kix_list_72-8>li:before{content:"\0025a0   "}.lst-kix_list_8-7>li:before{content:"\0025cb   "}.lst-kix_list_3-8>li:before{content:"\0025a0   "}ul.lst-kix_list_59-1{list-style-type:none}ul.lst-kix_list_59-0{list-style-type:none}ul.lst-kix_list_59-3{list-style-type:none}ul.lst-kix_list_59-2{list-style-type:none}.lst-kix_list_90-1>li:before{content:"\0025cb   "}.lst-kix_list_8-4>li:before{content:"\0025cb   "}.lst-kix_list_90-4>li:before{content:"\0025cb   "}.lst-kix_list_138-7>li:before{content:"" counter(lst-ctn-kix_list_138-7,decimal) ". "}ul.lst-kix_list_59-5{list-style-type:none}.lst-kix_list_138-4>li:before{content:"" counter(lst-ctn-kix_list_138-4,decimal) ". "}ul.lst-kix_list_59-4{list-style-type:none}ul.lst-kix_list_59-7{list-style-type:none}ul.lst-kix_list_59-6{list-style-type:none}ul.lst-kix_list_59-8{list-style-type:none}ol.lst-kix_list_149-2.start{counter-reset:lst-ctn-kix_list_149-2 0}.lst-kix_list_16-7>li:before{content:"" counter(lst-ctn-kix_list_16-7,decimal) ". "}ul.lst-kix_list_46-1{list-style-type:none}ul.lst-kix_list_46-0{list-style-type:none}ul.lst-kix_list_46-3{list-style-type:none}ul.lst-kix_list_46-2{list-style-type:none}ul.lst-kix_list_46-5{list-style-type:none}ul.lst-kix_list_46-4{list-style-type:none}ol.lst-kix_list_20-0.start{counter-reset:lst-ctn-kix_list_20-0 1}.lst-kix_list_87-4>li:before{content:"\0025cf   "}.lst-kix_list_16-4>li:before{content:"" counter(lst-ctn-kix_list_16-4,decimal) ". "}ul.lst-kix_list_46-7{list-style-type:none}ul.lst-kix_list_46-6{list-style-type:none}ul.lst-kix_list_46-8{list-style-type:none}.lst-kix_1yoisyi8wrza-2>li:before{content:"\0025a0   "}.lst-kix_list_87-7>li:before{content:"\0025cf   "}.lst-kix_list_55-0>li{counter-increment:lst-ctn-kix_list_55-0}.lst-kix_list_141-4>li:before{content:"\0025cf   "}.lst-kix_list_17-7>li:before{content:"-  "}.lst-kix_list_95-3>li{counter-increment:lst-ctn-kix_list_95-3}.lst-kix_list_17-4>li:before{content:"-  "}.lst-kix_list_69-2>li:before{content:"\0025cf   "}.lst-kix_list_2-8>li:before{content:"\0025cf   "}.lst-kix_list_85-0>li:before{content:"\0025cf   "}.lst-kix_9jhhktla6bpr-8>li:before{content:"\0025a0   "}.lst-kix_list_139-7>li:before{content:"\0025cf   "}.lst-kix_list_7-7>li:before{content:"\0025cb   "}.lst-kix_list_143-8>li:before{content:"\0025cf   "}.lst-kix_list_31-5>li:before{content:"\0025cf   "}.lst-kix_list_141-1>li:before{content:"\0025cf   "}.lst-kix_list_106-6>li:before{content:"\0025cf   "}ol.lst-kix_list_20-4.start{counter-reset:lst-ctn-kix_list_20-4 0}ul.lst-kix_list_15-3{list-style-type:none}.lst-kix_list_36-1>li:before{content:"\0025cf   "}.lst-kix_list_4-0>li:before{content:"\0025cf   "}ul.lst-kix_list_15-2{list-style-type:none}ul.lst-kix_list_15-1{list-style-type:none}.lst-kix_list_52-2>li:before{content:"\0025cf   "}ul.lst-kix_list_15-0{list-style-type:none}ul.lst-kix_list_15-8{list-style-type:none}ul.lst-kix_list_15-7{list-style-type:none}ul.lst-kix_list_15-6{list-style-type:none}.lst-kix_36wys1csajrz-5>li:before{content:"\0025a0   "}ul.lst-kix_list_15-5{list-style-type:none}ul.lst-kix_list_15-4{list-style-type:none}.lst-kix_list_88-7>li:before{content:"\0025cf   "}.lst-kix_list_140-1>li:before{content:"\0025cf   "}.lst-kix_list_29-8>li:before{content:"\0025cf   "}ul.lst-kix_2x6t2se3panh-5{list-style-type:none}ul.lst-kix_2x6t2se3panh-6{list-style-type:none}.lst-kix_list_144-8>li:before{content:"" counter(lst-ctn-kix_list_144-8,lower-roman) ". "}.lst-kix_list_103-2>li:before{content:"\0025cf   "}ul.lst-kix_2x6t2se3panh-3{list-style-type:none}.lst-kix_list_32-5>li:before{content:"\0025cf   "}ul.lst-kix_2x6t2se3panh-4{list-style-type:none}ul.lst-kix_2x6t2se3panh-7{list-style-type:none}ul.lst-kix_2x6t2se3panh-8{list-style-type:none}ul.lst-kix_list_6-6{list-style-type:none}ul.lst-kix_list_6-7{list-style-type:none}ul.lst-kix_list_6-4{list-style-type:none}ul.lst-kix_list_6-5{list-style-type:none}ul.lst-kix_list_6-8{list-style-type:none}.lst-kix_list_11-8>li:before{content:"\0025cf   "}ul.lst-kix_list_6-2{list-style-type:none}ul.lst-kix_list_6-3{list-style-type:none}ul.lst-kix_list_6-0{list-style-type:none}ul.lst-kix_list_6-1{list-style-type:none}.lst-kix_list_49-5>li:before{content:"\0025cf   "}.lst-kix_list_13-0>li:before{content:"\0025cf   "}.lst-kix_list_55-6>li:before{content:"" counter(lst-ctn-kix_list_55-6,decimal) ". "}.lst-kix_list_125-3>li:before{content:"\0025cf   "}.lst-kix_list_54-6>li:before{content:"\0025cf   "}.lst-kix_list_12-8>li:before{content:"\0025a0   "}.lst-kix_list_126-6>li:before{content:"\0025cf   "}.lst-kix_list_19-2>li:before{content:"\0025a0   "}ol.lst-kix_list_21-0.start{counter-reset:lst-ctn-kix_list_21-0 2}.lst-kix_list_135-0>li:before{content:"\0025cf   "}.lst-kix_list_47-7>li:before{content:"\0025cf   "}.lst-kix_list_93-8>li:before{content:"\0025cf   "}.lst-kix_list_84-0>li:before{content:"\0025cf   "}.lst-kix_list_47-4>li:before{content:"\0025cf   "}.lst-kix_list_19-8>li:before{content:"\0025a0   "}ol.lst-kix_list_20-8.start{counter-reset:lst-ctn-kix_list_20-8 0}.lst-kix_list_19-5>li:before{content:"\0025a0   "}.lst-kix_list_47-1>li:before{content:"\0025cf   "}.lst-kix_list_74-0>li:before{content:"\0025cf   "}.lst-kix_list_37-7>li:before{content:"\0025cb   "}ol.lst-kix_list_55-8.start{counter-reset:lst-ctn-kix_list_55-8 0}.lst-kix_list_136-6>li:before{content:"\0025cf   "}.lst-kix_2x6t2se3panh-2>li:before{content:"\0025a0   "}.lst-kix_list_116-6>li:before{content:"\0025cf   "}.lst-kix_list_108-4>li:before{content:"\0025cb   "}ul.lst-kix_rxdmkrnvx4u4-8{list-style-type:none}.lst-kix_list_37-4>li:before{content:"\0025cb   "}.lst-kix_list_108-1>li:before{content:"\0025cb   "}ul.lst-kix_list_77-2{list-style-type:none}.lst-kix_list_65-6>li:before{content:"\0025cf   "}.lst-kix_list_101-3>li{counter-increment:lst-ctn-kix_list_101-3}ul.lst-kix_list_77-3{list-style-type:none}.lst-kix_list_92-2>li:before{content:"\0025cf   "}.lst-kix_list_55-6>li{counter-increment:lst-ctn-kix_list_55-6}ul.lst-kix_list_77-0{list-style-type:none}ul.lst-kix_list_77-1{list-style-type:none}ul.lst-kix_list_77-6{list-style-type:none}ul.lst-kix_list_77-7{list-style-type:none}ul.lst-kix_list_77-4{list-style-type:none}ul.lst-kix_list_77-5{list-style-type:none}.lst-kix_list_108-7>li:before{content:"\0025cb   "}ol.lst-kix_list_149-0.start{counter-reset:lst-ctn-kix_list_149-0 0}.lst-kix_list_136-3>li:before{content:"\0025cf   "}.lst-kix_list_92-5>li:before{content:"\0025cf   "}ul.lst-kix_list_77-8{list-style-type:none}.lst-kix_list_85-3>li:before{content:"\0025cf   "}.lst-kix_list_2-5>li:before{content:"\0025cf   "}.lst-kix_list_100-5>li{counter-increment:lst-ctn-kix_list_100-5}ol.lst-kix_list_55-6.start{counter-reset:lst-ctn-kix_list_55-6 0}.lst-kix_list_64-0>li:before{content:"\0025cf   "}.lst-kix_list_118-1>li:before{content:"o  "}.lst-kix_list_39-5>li:before{content:"\0025a0   "}.lst-kix_list_143-5>li:before{content:"\0025cf   "}.lst-kix_list_118-7>li:before{content:"\0025cf   "}ol.lst-kix_list_99-2.start{counter-reset:lst-ctn-kix_list_99-2 0}.lst-kix_list_57-1>li:before{content:"\0025cf   "}.lst-kix_list_57-7>li:before{content:"\0025cf   "}.lst-kix_rxdmkrnvx4u4-4>li:before{content:"\0025cb   "}.lst-kix_36wys1csajrz-2>li:before{content:"\0025a0   "}.lst-kix_list_67-7>li:before{content:"\0025cb   "}ul.lst-kix_list_131-6{list-style-type:none}ul.lst-kix_list_131-5{list-style-type:none}ul.lst-kix_list_131-4{list-style-type:none}ul.lst-kix_list_131-3{list-style-type:none}.lst-kix_list_29-5>li:before{content:"\0025cf   "}ul.lst-kix_list_131-8{list-style-type:none}ul.lst-kix_list_131-7{list-style-type:none}ol.lst-kix_list_99-0.start{counter-reset:lst-ctn-kix_list_99-0 1}.lst-kix_list_11-5>li:before{content:"\0025cf   "}.lst-kix_list_54-3>li:before{content:"\0025cf   "}ol.lst-kix_list_21-2.start{counter-reset:lst-ctn-kix_list_21-2 0}ul.lst-kix_list_131-2{list-style-type:none}ul.lst-kix_list_131-1{list-style-type:none}ul.lst-kix_list_131-0{list-style-type:none}ul.lst-kix_list_33-0{list-style-type:none}ul.lst-kix_list_33-1{list-style-type:none}ul.lst-kix_list_33-2{list-style-type:none}ul.lst-kix_list_33-3{list-style-type:none}.lst-kix_list_125-0>li:before{content:"\0025cf   "}ul.lst-kix_list_33-4{list-style-type:none}ul.lst-kix_list_33-5{list-style-type:none}ul.lst-kix_list_33-6{list-style-type:none}ul.lst-kix_list_33-7{list-style-type:none}.lst-kix_list_146-3>li:before{content:"\0025cb   "}.lst-kix_list_75-6>li:before{content:"\0025cf   "}.lst-kix_list_82-5>li:before{content:"\0025cf   "}.lst-kix_list_27-7>li:before{content:"\0025cf   "}ul.lst-kix_list_33-8{list-style-type:none}.lst-kix_list_49-2>li:before{content:"\0025cf   "}.lst-kix_list_100-4>li:before{content:"" counter(lst-ctn-kix_list_100-4,decimal) ". "}.lst-kix_list_77-1>li:before{content:"o  "}.lst-kix_list_72-5>li:before{content:"\0025a0   "}.lst-kix_list_72-2>li:before{content:"\0025a0   "}.lst-kix_list_105-3>li:before{content:"\0025cf   "}.lst-kix_list_44-0>li:before{content:"\0025cf   "}.lst-kix_list_77-4>li:before{content:"\0025cf   "}ul.lst-kix_list_51-5{list-style-type:none}.lst-kix_list_133-2>li:before{content:"\0025cf   "}ul.lst-kix_list_51-4{list-style-type:none}ul.lst-kix_list_51-7{list-style-type:none}.lst-kix_list_21-2>li:before{content:"" counter(lst-ctn-kix_list_21-2,decimal) ". "}ul.lst-kix_list_51-6{list-style-type:none}.lst-kix_list_79-5>li{counter-increment:lst-ctn-kix_list_79-5}ul.lst-kix_list_51-8{list-style-type:none}ol.lst-kix_list_144-7.start{counter-reset:lst-ctn-kix_list_144-7 0}ul.lst-kix_list_109-2{list-style-type:none}ul.lst-kix_list_109-3{list-style-type:none}ul.lst-kix_list_109-0{list-style-type:none}ul.lst-kix_list_109-1{list-style-type:none}ul.lst-kix_list_51-1{list-style-type:none}ul.lst-kix_list_109-6{list-style-type:none}ul.lst-kix_list_51-0{list-style-type:none}ul.lst-kix_list_109-7{list-style-type:none}ul.lst-kix_list_51-3{list-style-type:none}.lst-kix_list_138-1>li:before{content:"" counter(lst-ctn-kix_list_138-1,decimal) ". "}.lst-kix_list_133-5>li:before{content:"\0025cf   "}ul.lst-kix_list_109-4{list-style-type:none}ul.lst-kix_list_51-2{list-style-type:none}ul.lst-kix_list_109-5{list-style-type:none}.lst-kix_list_11-2>li:before{content:"\0025cf   "}.lst-kix_list_128-1>li:before{content:"\0025cf   "}.lst-kix_list_123-2>li:before{content:"\0025cf   "}.lst-kix_list_128-4>li:before{content:"\0025cf   "}.lst-kix_list_95-3>li:before{content:"" counter(lst-ctn-kix_list_95-3,decimal) ". "}.lst-kix_list_123-5>li:before{content:"\0025cf   "}.lst-kix_list_99-6>li{counter-increment:lst-ctn-kix_list_99-6}.lst-kix_list_90-7>li:before{content:"\0025cb   "}.lst-kix_1yoisyi8wrza-5>li:before{content:"\0025a0   "}.lst-kix_list_16-1>li:before{content:"" counter(lst-ctn-kix_list_16-1,decimal) ". "}.lst-kix_list_67-1>li:before{content:"\0025cb   "}ol.lst-kix_list_149-8.start{counter-reset:lst-ctn-kix_list_149-8 0}ol.lst-kix_list_101-5.start{counter-reset:lst-ctn-kix_list_101-5 0}.lst-kix_list_44-3>li:before{content:"\0025cf   "}.lst-kix_list_141-7>li:before{content:"\0025cf   "}.lst-kix_list_115-0>li:before{content:"\0025cf   "}.lst-kix_list_39-2>li:before{content:"\0025a0   "}.lst-kix_list_87-1>li:before{content:"o  "}.lst-kix_list_100-1>li:before{content:"" counter(lst-ctn-kix_list_100-1,decimal) ". "}.lst-kix_1yoisyi8wrza-8>li:before{content:"\0025a0   "}ul.lst-kix_list_64-4{list-style-type:none}.lst-kix_list_113-5>li:before{content:"\0025cf   "}ul.lst-kix_list_64-5{list-style-type:none}ul.lst-kix_list_64-2{list-style-type:none}ul.lst-kix_list_64-3{list-style-type:none}ul.lst-kix_list_64-8{list-style-type:none}ul.lst-kix_list_64-6{list-style-type:none}.lst-kix_list_118-4>li:before{content:"\0025cf   "}ul.lst-kix_list_64-7{list-style-type:none}.lst-kix_list_7-1>li:before{content:"\0025cb   "}ul.lst-kix_list_64-0{list-style-type:none}ul.lst-kix_list_64-1{list-style-type:none}.lst-kix_list_143-2>li:before{content:"\0025cf   "}.lst-kix_list_20-3>li{counter-increment:lst-ctn-kix_list_20-3}.lst-kix_4sgfft60r4dw-5>li:before{content:"\0025a0   "}.lst-kix_list_85-6>li:before{content:"\0025cf   "}ul.lst-kix_rxdmkrnvx4u4-0{list-style-type:none}ul.lst-kix_rxdmkrnvx4u4-1{list-style-type:none}ul.lst-kix_rxdmkrnvx4u4-2{list-style-type:none}.lst-kix_list_52-8>li:before{content:"\0025cf   "}ul.lst-kix_rxdmkrnvx4u4-3{list-style-type:none}ul.lst-kix_rxdmkrnvx4u4-4{list-style-type:none}ul.lst-kix_rxdmkrnvx4u4-5{list-style-type:none}ul.lst-kix_rxdmkrnvx4u4-6{list-style-type:none}ul.lst-kix_rxdmkrnvx4u4-7{list-style-type:none}.lst-kix_list_67-4>li:before{content:"\0025cb   "}.lst-kix_list_28-4>li{counter-increment:lst-ctn-kix_list_28-4}ol.lst-kix_list_101-7.start{counter-reset:lst-ctn-kix_list_101-7 0}.lst-kix_rxdmkrnvx4u4-7>li:before{content:"\0025cb   "}.lst-kix_list_10-8>li{counter-increment:lst-ctn-kix_list_10-8}.lst-kix_list_57-4>li:before{content:"\0025cf   "}.lst-kix_list_138-8>li{counter-increment:lst-ctn-kix_list_138-8}.lst-kix_list_95-6>li:before{content:"" counter(lst-ctn-kix_list_95-6,decimal) ". "}ul.lst-kix_list_109-8{list-style-type:none}.lst-kix_list_103-8>li:before{content:"\0025cf   "}.lst-kix_list_62-5>li:before{content:"\0025cf   "}.lst-kix_list_54-0>li:before{content:"\0025cf   "}.lst-kix_list_29-2>li:before{content:"\0025cf   "}.lst-kix_list_21-1>li{counter-increment:lst-ctn-kix_list_21-1}.lst-kix_list_110-1>li:before{content:"o  "}.lst-kix_list_82-2>li:before{content:"\0025cf   "}.lst-kix_list_149-1>li{counter-increment:lst-ctn-kix_list_149-1}.lst-kix_list_105-0>li:before{content:"\0025cf   "}.lst-kix_list_34-3>li:before{content:"\0025cf   "}.lst-kix_list_2-2>li:before{content:"\0025cf   "}.lst-kix_list_146-6>li:before{content:"\0025cb   "}ol{margin:0;padding:0}table td,table th{padding:0}.c17{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:95.2pt;border-top-color:#000000;border-bottom-style:solid}.c38{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:396pt;border-top-color:#000000;border-bottom-style:solid}.c113{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:257.4pt;border-top-color:#000000;border-bottom-style:solid}.c5{margin-left:36pt;padding-top:0pt;padding-left:0pt;padding-bottom:0pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c24{margin-left:108pt;padding-top:0pt;padding-left:0pt;padding-bottom:0pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c63{margin-left:70.9pt;padding-top:0pt;text-indent:-21.3pt;padding-bottom:0pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c43{margin-left:72pt;padding-top:0pt;padding-left:0pt;padding-bottom:0pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:left}.c75{margin-left:108pt;padding-top:14pt;padding-left:0pt;padding-bottom:0pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c30{margin-left:36pt;padding-top:12pt;padding-left:0pt;padding-bottom:0pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c44{margin-left:72pt;padding-top:0pt;padding-left:0pt;padding-bottom:5pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c20{margin-left:36pt;padding-top:0pt;padding-left:0pt;padding-bottom:8pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:left}.c15{margin-left:72pt;padding-top:0pt;padding-left:0pt;padding-bottom:0pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c27{margin-left:36pt;padding-top:14pt;padding-left:0pt;padding-bottom:0pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c0{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Calibri";font-style:normal}.c16{color:#c22844;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:18pt;font-family:"Calibri";font-style:normal}.c39{padding-top:5pt;padding-bottom:5pt;line-height:1.0791666666666666;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c12{color:#e7952f;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:22pt;font-family:"Calibri";font-style:normal}.c58{padding-top:24pt;padding-bottom:6pt;line-height:1.0791666666666666;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c14{color:#c22844;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:18pt;font-family:"Calibri";font-style:normal}.c67{padding-top:18pt;padding-bottom:4pt;line-height:1.0791666666666666;page-break-after:avoid;orphans:2;widows:2;text-align:right}.c2{padding-top:5pt;padding-bottom:5pt;line-height:1.0791666666666666;page-break-after:avoid;orphans:2;widows:2;text-align:justify}.c22{color:#4a86e8;text-decoration:none;vertical-align:baseline;font-size:13.5pt;font-family:"Calibri";font-style:normal}.c69{padding-top:14pt;padding-bottom:14pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c26{padding-top:0pt;padding-bottom:8pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:left}.c46{padding-top:0pt;padding-bottom:0pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:left}.c9{padding-top:5pt;padding-bottom:5pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:center}.c35{padding-top:12pt;padding-bottom:0pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:left}.c10{padding-top:0pt;padding-bottom:0pt;line-height:1.1500000000000001;orphans:2;widows:2;text-align:justify}.c25{padding-top:0pt;padding-bottom:8pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:center}.c28{padding-top:12pt;padding-bottom:12pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:center}.c59{padding-top:5pt;padding-bottom:5pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:left}.c76{padding-top:14pt;padding-bottom:0pt;line-height:1.0;orphans:2;widows:2;text-align:justify}.c41{padding-top:0pt;padding-bottom:14pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c19{padding-top:0pt;padding-bottom:8pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c21{color:#e7952f;text-decoration:none;vertical-align:baseline;font-size:24pt;font-family:"Calibri";font-style:normal}.c72{padding-top:5pt;padding-bottom:0pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c6{padding-top:5pt;padding-bottom:5pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c50{padding-top:0pt;padding-bottom:12pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c81{margin-left:35.4pt;padding-top:0pt;text-indent:-14.2pt;padding-bottom:0pt;line-height:1.0791666666666666;text-align:justify}.c49{padding-top:0pt;padding-bottom:5pt;line-height:1.0791666666666666;orphans:2;widows:2;text-align:justify}.c91{padding-top:0pt;padding-bottom:14pt;line-height:1.0791666666666666;text-align:center}.c89{padding-top:0pt;padding-bottom:0pt;line-height:1.0791666666666666;text-align:justify}.c86{padding-top:0pt;padding-bottom:12pt;line-height:1.0791666666666666;text-align:left}.c104{padding-top:0pt;padding-bottom:14pt;line-height:1.0;text-align:justify}.c4{text-decoration-skip-ink:none;-webkit-text-decoration-skip:none;color:#1155cc;text-decoration:underline}.c98{padding-top:5pt;padding-bottom:0pt;line-height:1.1500000000000001;text-align:justify}.c106{margin-left:5pt;border-spacing:0;border-collapse:collapse;margin-right:auto}.c53{margin-left:72pt;padding-left:0pt;orphans:2;widows:2}.c36{color:#4a86e8;text-decoration:none;vertical-align:baseline;font-style:normal}.c101{padding-top:14pt;padding-bottom:0pt;line-height:1.0791666666666666;text-align:justify}.c111{padding-top:5pt;padding-bottom:5pt;line-height:1.0791666666666666;text-align:right}.c93{padding-top:12pt;padding-bottom:12pt;line-height:1.0791666666666666;text-align:left}.c37{color:#1155cc;text-decoration:none;vertical-align:baseline;font-style:normal}.c115{color:#666666;font-weight:400;font-size:24pt;font-family:"Georgia"}.c79{padding-top:12pt;padding-bottom:12pt;line-height:1.0791666666666666;text-align:justify}.c85{padding-top:14pt;padding-bottom:0pt;line-height:1.0791666666666666;text-align:center}.c34{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:left}.c119{padding-top:5pt;padding-bottom:0pt;line-height:1.0;text-align:justify}.c47{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:justify}.c107{padding-top:5pt;padding-bottom:5pt;line-height:1.0;text-align:justify}.c87{padding-top:0pt;padding-bottom:8pt;line-height:1.0791666666666666;text-align:right}.c64{background-color:#ffffff;max-width:519.5pt;padding:61.2pt 40.3pt 56.2pt 35.4pt}.c62{font-weight:400;font-size:11pt;font-family:"Arial"}.c11{font-size:10pt;font-family:"Arial";font-weight:400}.c13{font-size:11pt;font-family:"Calibri";font-weight:700}.c42{-webkit-text-decoration-skip:none;text-decoration:underline;text-decoration-skip-ink:none}.c71{color:#e6b8af;text-decoration:none;vertical-align:baseline}.c57{color:#4a86e8;vertical-align:baseline;font-style:normal}.c33{font-size:10pt;font-family:"Courier New";font-weight:400}.c68{font-size:7pt;font-family:"Times New Roman";font-weight:400}.c1{font-size:11pt;font-family:"Calibri";font-weight:400}.c31{font-size:36pt;font-family:"Calibri";font-style:normal}.c48{font-weight:400;font-size:14pt;font-family:"Calibri"}.c29{color:#000000;text-decoration:none;vertical-align:baseline}.c88{margin-left:-45pt;margin-right:-21.8pt}.c99{font-weight:400;font-family:"Courier New"}.c100{text-decoration:none;vertical-align:baseline}.c55{font-size:12pt;font-family:"Calibri"}.c118{color:#e7952f;font-family:"Calibri"}.c105{margin-left:58.5pt;margin-right:69pt}.c8{color:inherit;text-decoration:inherit}.c40{border:1px solid black;margin:5px}.c102{margin-left:-7.1pt;margin-right:-14.2pt}.c84{margin-left:-31.5pt;margin-right:-14.2pt}.c70{font-size:30pt;font-family:"Calibri"}.c92{margin-left:108pt;padding-left:0pt}.c3{padding:0;margin:0}.c77{orphans:2;widows:2}.c61{margin-left:72pt;padding-left:0pt}.c32{margin-left:36pt;padding-left:0pt}.c51{color:#1155cc}.c18{font-weight:700}.c112{margin-right:-14.2pt}.c56{margin-left:36pt}.c80{font-size:23pt}.c78{page-break-after:avoid}.c7{height:11pt}.c116{font-weight:400}.c45{background-color:#ffff00}.c66{font-size:48pt}.c52{height:18pt}.c73{font-size:11pt}.c95{margin-left:-27pt}.c94{color:#f1c232}.c117{font-size:9pt}.c97{color:#1c4587}.c120{margin-left:-63pt}.c23{font-style:italic}.c114{height:22pt}.c74{color:#000000}.c103{margin-left:72pt}.c54{vertical-align:sub}.c109{vertical-align:baseline}.c90{height:14pt}.c96{margin-left:-14.2pt}.c110{margin-left:-31.5pt}.c65{height:0pt}.c60{font-style:normal}.c82{font-size:10pt}.c108{margin-left:-40.5pt}.c83{font-size:17pt}.title{padding-top:24pt;color:#000000;font-weight:700;font-size:36pt;padding-bottom:6pt;font-family:"Calibri";line-height:1.0791666666666666;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:18pt;color:#666666;font-size:24pt;padding-bottom:4pt;font-family:"Georgia";line-height:1.0791666666666666;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Calibri"}p{margin:0;color:#000000;font-size:11pt;font-family:"Calibri"}h1{padding-top:5pt;color:#e7952f;font-weight:700;font-size:22pt;padding-bottom:5pt;font-family:"Calibri";line-height:1.0791666666666666;page-break-after:avoid;orphans:2;widows:2;text-align:justify}h2{padding-top:5pt;color:#c22844;font-size:18pt;padding-bottom:5pt;font-family:"Calibri";line-height:1.0791666666666666;page-break-after:avoid;orphans:2;widows:2;text-align:justify}h3{padding-top:5pt;color:#4a86e8;font-size:14pt;padding-bottom:5pt;font-family:"Calibri";line-height:1.0791666666666666;page-break-after:avoid;orphans:2;widows:2;text-align:justify}h4{padding-top:5pt;color:#4a86e8;font-weight:700;font-size:12pt;padding-bottom:5pt;font-family:"Calibri";line-height:1.0791666666666666;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:5pt;color:#000000;font-weight:700;font-size:10pt;padding-bottom:5pt;font-family:"Calibri";line-height:1.0791666666666666;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:10pt;color:#000000;font-weight:700;font-size:10pt;padding-bottom:2pt;font-family:"Calibri";line-height:1.0791666666666666;page-break-after:avoid;orphans:2;widows:2;text-align:left}</style></head><body class="c64 doc-content"><div><p class="c87 c77"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 107.00px; height: 35.00px;"><img alt="" src="images/image182.png" style="width: 107.00px; height: 35.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c25 c7"><span class="c0"></span></p></div><p class="c77 c87"><span class="c18 c66">Proline User Guide</span></p><p class="c67 subtitle" id="h.gjdgxs"><span class="c100 c23 c115">Release 2.3.0</span></p><hr><p class="c19 c7"><span class="c62 c29 c60"></span></p><p class="c19 c7"><span class="c29 c60 c62"></span></p><a id="id.30j0zll"></a><h1 class="c2"><span>Proline Software Suite</span></h1><p class="c6"><span class="c0">Proline is a production grade software suite, which provides an environment for large-scale MS data management, visualization, analysis and curation with the main objective of promoting the production and sharing of high quality proteomic datasets. Proline can be used (i) to produce reliable identification and quantification results through robust automated processes, (ii) for data curation, (iii) to systematically save and keep track of metadata from processing steps, parameters and generated data, and (iv) to submit highly qualified datasets to public repositories.</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c9"><span class="c1 c23">&nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 466.00px; height: 160.00px;"><img alt="" src="images/image184.png" style="width: 466.00px; height: 160.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9 c7"><span class="c0"></span></p><p class="c6"><span class="c0">A workflow in Proline is implemented as a collection of tasks (see Figure) that can be performed by the user through the graphical user interface. Users can import multiple identification results corresponding, for example, to fractions and replicates of a biological sample and combine them before or after validation. The resulting datasets can then be compared or quantified using spectral counting or DDA label-free quantification, before exporting the results in different file formats. </span></p><p class="c6"><span class="c1">The software suite is based on </span><span>two </span><span class="c1">main components: </span><span>a </span><span class="c18">server </span><span>handling processing tasks and based on </span><span class="c18">relational database management system</span><span>&nbsp;storing the data generated and two different graphical user interfaces, both allowing users to start tasks and visualize the data: </span><span class="c18">Proline Studio</span><span>&nbsp;which is a rich client interface and </span><span class="c18">Proline Web</span><span>&nbsp;the web client interface. </span><span class="c1">An additional component called </span><span class="c13">Proline</span><span class="c18">Admin </span><span>is used by system</span><span class="c1">&nbsp;administrators to </span><span>set up</span><span class="c1">&nbsp;and manage Proline. </span><span class="c18 c73">Proline Zero</span><span class="c1">&nbsp;i</span><span>s an all-in-one, &ldquo;zero installation&rdquo; solution containing the </span><span class="c18">server</span><span>&nbsp;and </span><span class="c18">Proline Studio</span><span>.</span></p><h2 class="c2"><span>Document organization </span></h2><p class="c6"><span class="c0">This document is organized in two main sections: </span></p><ul class="c3 lst-kix_list_25-0 start"><li class="c6 c32 li-bullet-0"><span>T</span><span class="c1">he </span><span class="c4 c1"><a class="c8" href="#id.1fob9te">Concepts &amp; Principles</a></span><span class="c42 c1 c74"><a class="c8" href="#id.1fob9te">&nbsp;</a></span><span>section presents </span><span class="c1">the main concepts and algorithms </span><span>implemented </span><span class="c1">in Proline as well as the different parameters </span><span>of these algorithms</span><span class="c1">. </span></li><li class="c6 c32 li-bullet-0"><span>The </span><span class="c4"><a class="c8" href="#id.39kk8xu">How to</a></span><span class="c0">&nbsp;section gives more details on how to proceed with the Proline Studio graphical interface.</span></li></ul><p class="c6"><span>You could also find a description of </span><span class="c4"><a class="c8" href="#id.o9t5nltnb7nt">available converters</a></span><span>&nbsp;to create files in </span><span class="c18">mzDB format</span><span class="c0">&nbsp;(Bouyssi&eacute; et al., 2015) from spectrometers raw files. </span></p><a id="id.1fob9te"></a><p class="c58 title" id="h.3znysh7"><span class="c29 c70 c18 c60">Proline Concepts &amp; Principles</span></p><ul class="c3 lst-kix_list_51-0 start"><li class="c27 li-bullet-0"><span class="c0">Dataset types:</span></li></ul><ul class="c3 lst-kix_list_51-1 start"><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.3dy6vkm">Result File</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.1t3h5sf">Search Results</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.3rdcrjn">Decoy Searches</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.lnxbz9">Identification Summary</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.vj9t7kx4t9zd">Modification Dataset</a></span></li></ul><ul class="c3 lst-kix_list_51-0"><li class="c5 li-bullet-0"><span class="c0">Data Processing:</span></li></ul><ul class="c3 lst-kix_list_51-1 start"><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.44sinio">Protein Inference</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.z337ya">Protein and Proteins Sets scoring</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.3as4poj">Validation Algorithm</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.1v1yuxt">FDR Estimation</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.2u6wntf">Protein Sets Filtering</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.2lwamvv">Combining datasets</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#h.111kx3o">Identify Modification Sites or Clusters</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.3l18frh">Compare with Spectral Count</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.3ygebqi">Label Free LC-MS quantitation workflow</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#h.1wmaarypjked">Isobaric labelling quantitation</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.1jlao46">Post-processing of quantitative results</a></span></li><li class="c15 li-bullet-0"><span class="c4"><a class="c8" href="#id.xvir7l">Aggregation of quantitative results</a></span></li></ul><ul class="c3 lst-kix_list_107-0 start"><li class="c47 c77 c32 li-bullet-0"><span class="c0">Data Import/Export:</span></li></ul><ul class="c3 lst-kix_list_107-1 start"><li class="c47 c53 li-bullet-0"><span class="c4"><a class="c8" href="#id.3hv69ve">Identification Summary Export</a></span></li></ul><ul class="c3 lst-kix_list_107-0"><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.vtr0o7bk97h1">Advanced Features</a></span><span class="c0">:</span></li></ul><ul class="c3 lst-kix_list_107-1 start"><li class="c41 c61 li-bullet-0"><span class="c4"><a class="c8" href="#id.799oj238ld40">Allow multiple imports in parallel</a></span><span><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></li></ul><a id="id.2et92p0"></a><h1 class="c2"><span class="c12">Identification results</span></h1><p class="c26"><span class="c0">Proline considers different types of identification data: Result Files, Search Results and Identification Summaries which will be defined in the following sections. </span></p><h2 class="c2" id="h.tyjcwt"><span>R</span><a id="id.3dy6vkm"></a><span class="c16">esult File</span></h2><p class="c6"><span class="c1">A </span><span class="c18">r</span><span class="c13">esult </span><span class="c18">f</span><span class="c13">ile</span><span class="c1">&nbsp;</span><span>produced </span><span class="c1">by a search engine can be imported into Proline in their native format. OMSSA (</span><span class="c33">.omx</span><span class="c1">&nbsp;files), Mascot (</span><span class="c33">.dat</span><span class="c1">&nbsp;files) and X!Tandem (</span><span class="c33">.xml</span><span class="c1">&nbsp;files) search engines are currently supported. </span><span class="c0">In addition, the mzIdentML format is supported to allow the output from any other search engine compatible with this standard to be imported (e.g. MS-GF+). A first version for MaxQuant support has been implemented. It is possible to import only search results or to import search results as well as quantitation (beta version) values from MaxQuant files.</span></p><p class="c6"><span class="c1">Search engines </span><span>may</span><span class="c1">&nbsp;provide different types of searches for MS and MS/MS data. It is important to highlight that Proline </span><span class="c0">only supports MS/MS ions searches at this point.</span></p><p class="c6 c7"><span class="c0"></span></p><a id="id.1t3h5sf"></a><h2 class="c2" id="h.4d34og8"><span class="c16">Search Result</span></h2><p class="c6"><span class="c1">A </span><span class="c18">s</span><span class="c13">earch </span><span class="c18">r</span><span class="c13">esult</span><span class="c1">&nbsp;is the raw interpretation of a given set of MS/MS spectra given by a search engine. It contains one or many peptides matching the submitted MS/MS spectra (</span><span class="c13">PSM</span><span class="c1">, i.e. Peptide Spectrum Match), and the protein sequences these peptides belong to. The </span><span class="c18">s</span><span class="c18 c73">earch </span><span class="c18">r</span><span class="c18 c73">esult </span><span class="c0">also contains additional information such as search parameters, used protein sequence databank, etc. </span></p><p class="c6"><span class="c1">A </span><span class="c18">s</span><span class="c13">earch </span><span class="c18">r</span><span class="c13">esult</span><span class="c1">&nbsp;is created when a </span><span class="c51">r</span><span class="c4 c1"><a class="c8" href="#id.3dy6vkm">esult </a></span><span class="c4"><a class="c8" href="#id.3dy6vkm">f</a></span><span class="c4 c1"><a class="c8" href="#id.3dy6vkm">ile</a></span><span class="c1">&nbsp;is imported in Proline. </span><span>During this step, no filtering or thresholding is applied: along with the search parameters, all submitted spectra, peptide spectrum matches (PSMs) and protein hits suggested by the search engine are retained in the Proline database to allow subsequent validation of putative identifications. </span><span class="c1">In the case of a target-decoy search, two </span><span class="c18">s</span><span class="c13">earch </span><span class="c18">r</span><span class="c13">esults</span><span class="c0">&nbsp;are created: one for the target PSMs, one for decoy PSMs. </span></p><h3 class="c2" id="h.2s8eyo1"><span>Content of a search result</span></h3><p class="c6"><span class="c1">Importing a </span><span class="c51">r</span><span class="c4"><a class="c8" href="#id.3dy6vkm">esult fil</a></span><span class="c4 c1"><a class="c8" href="#id.3dy6vkm">e</a></span><span class="c1">&nbsp;creates a new </span><span class="c18">s</span><span class="c18 c73">earch </span><span class="c18">r</span><span class="c18 c73">esult </span><span class="c0">in the database which contains the following information: </span></p><ul class="c3 lst-kix_list_110-0 start"><li class="c27 li-bullet-0"><span class="c0">Search Settings: software name and version, parameters values</span></li><li class="c5 li-bullet-0"><span class="c0">Peaklist and Spectrum information: file name, MS level, precursor m/z, &hellip;</span></li><li class="c5 li-bullet-0"><span class="c0">Search result data:</span></li></ul><ul class="c3 lst-kix_list_110-1 start"><li class="c15 li-bullet-0"><span class="c0">Protein sequences</span></li><li class="c15 li-bullet-0"><span class="c0">Peptide sequences</span></li><li class="c15 li-bullet-0"><span class="c0">Spectra</span></li><li class="c15 li-bullet-0"><span class="c0">Two kinds of Matches:</span></li></ul><ul class="c3 lst-kix_list_110-2 start"><li class="c24 li-bullet-0"><span class="c0">Peptide Spectrum Matches (PSM), i.e. the matching between a peptide and a spectrum, with some related data such as the score, fragment matches&hellip;</span></li><li class="c41 c92 li-bullet-0"><span class="c0">Protein Matches, i.e. the proteins in the databank corresponding to the PSMs identified by the search engine</span></li></ul><h4 class="c39" id="h.17dp8vu"><span class="c36 c55 c18">Search engine specificities</span></h4><ul class="c3 lst-kix_list_38-0 start"><li class="c46 c32 li-bullet-0"><span>Mascot </span></li></ul><ul class="c3 lst-kix_list_38-1 start"><li class="c43 li-bullet-0"><span class="c1">The </span><span>PSM </span><span class="c1">score corresponds to the Mascot ion score.</span></li></ul><ul class="c3 lst-kix_list_38-0"><li class="c46 c32 li-bullet-0"><span>OMSSA </span></li></ul><ul class="c3 lst-kix_list_38-1 start"><li class="c43 li-bullet-0"><span class="c1">The </span><span>PSM </span><span class="c1">score corresponds to the negative common logarithm of the E-value: </span><span>&nbsp;<br>Score = -log10(E-value). Note that Proline only supports OMSSA Result Files generated with the 2.1.9 release.</span></li></ul><ul class="c3 lst-kix_list_38-0"><li class="c46 c32 li-bullet-0"><span>X!Tandem </span></li></ul><ul class="c3 lst-kix_list_38-1 start"><li class="c26 c61 li-bullet-0"><span>The X!Tandem standard hyperscore is used as a PSM score. Note that </span><span class="c1">Proline supports X</span><span>!Tandem Result Files generated with the </span><span class="c1">Sledgehammer release (or later)</span><span>.</span></li></ul><p class="c6 c7"><span class="c0"></span></p><a id="id.3rdcrjn"></a><h2 class="c2" id="h.26in1rg"><span class="c16">Decoy Searches</span></h2><p class="c77 c107"><span class="c0">Proline handles decoy searches performed from two different strategies: </span></p><ul class="c3 lst-kix_list_109-0 start"><li class="c32 c76 li-bullet-0"><span class="c0">Concatenated searches:</span></li></ul><ul class="c3 lst-kix_list_109-1 start"><li class="c15 li-bullet-0"><span>A protein </span><span>databank</span><span class="c0">&nbsp;is created by concatenating target protein sequence to decoy protein sequence. Decoy could be created using reverse or random strategy. A unique search is done using that databank. </span></li><li class="c15 li-bullet-0"><span class="c0">When importing a Search Result from a decoy concatenated databank, decoy data are extracted from the Result File and stored in Proline databases as a decoy Search Result independent of the target Search result. Nevertheless both searches are linked to each other.</span></li></ul><ul class="c3 lst-kix_list_109-0"><li class="c5 li-bullet-0"><span class="c0">Separated searches: </span></li></ul><ul class="c3 lst-kix_list_109-1 start"><li class="c41 c61 li-bullet-0"><span class="c0">Two searches are done using the same peaklist, one on a target protein databank and one on a decoy protein databank. These searches are then combined to retrieve useful information such as FDR. Mascot allows the user to check a decoy option and automatically creates a decoy databank. </span></li><li class="c41 c61 li-bullet-0"><span>The two performed searches are stored in Proline databases and are linked together.</span><span class="c1">&nbsp;</span></li></ul><a id="id.lnxbz9"></a><h2 class="c2" id="h.35nkun2"><span class="c16">Identification Summary</span></h2><p class="c6"><span class="c1">An </span><span class="c13">Identification Summary</span><span class="c1">&nbsp;is a set of identified proteins inferred from a subset of the PSM contained in the </span><span class="c4">s</span><span class="c4 c1"><a class="c8" href="#id.1t3h5sf">earch </a></span><span class="c4"><a class="c8" href="#id.1t3h5sf">r</a></span><span class="c4 c1"><a class="c8" href="#id.1t3h5sf">esult</a></span><span>&nbsp;that have been declared valid</span><span class="c1">. The subset of PSM taken into account are the PSM that have been validated </span><span>by </span><span class="c0">a filtering process (example: PSM fulfilling some specified criteria such as score greater than a threshold value). </span></p><h3 class="c2" id="h.1ksv4uv"><span class="c36 c48">Content of an identification summary</span></h3><ul class="c3 lst-kix_list_3-0 start"><li class="c46 c32 li-bullet-0"><span>Peptide Set</span></li></ul><ul class="c3 lst-kix_list_3-1 start"><li class="c43 li-bullet-0"><span>A set of peptides identifying one or more proteins.</span></li></ul><ul class="c3 lst-kix_list_3-0"><li class="c46 c32 li-bullet-0"><span>Protein Set</span></li></ul><ul class="c3 lst-kix_list_3-1 start"><li class="c43 li-bullet-0"><span>Typical Protein or representative protein. Protein &ldquo;chosen&rdquo; to represent the ProteinSet.</span></li><li class="c43 li-bullet-0"><span>Sameset : all proteins identified by the same set of peptides than the typical one.</span></li><li class="c26 c61 li-bullet-0"><span>Subset: Proteins identified by a subset of peptides. All these peptides should also identify typical protein.</span></li></ul><a id="id.44sinio"></a><h3 class="c2" id="h.2jxsxqh"><span class="c36 c48">Protein Inference</span></h3><p class="c6"><span class="c1">All peptides identifying a protein are grouped in a </span><span class="c33">Peptides Set</span><span class="c1">. A same Peptides Set can identify many proteins, represented by one </span><span class="c33">Proteins Set</span><span class="c1">. In this case, one protein of this Protein Set is chosen to represent the set, it is the </span><span class="c33">Typical Protein</span><span class="c1">. If only a subset of peptides identify a (or some) protein(s), a new Peptide Set is created. This Peptide Set is a subset of the first one, and identified Proteins are </span><span class="c33">Subset</span><span class="c33">&nbsp;</span><span class="c33">Proteins</span><span class="c0">. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 576.65px; height: 153.65px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\pepprotset1.png" src="images/image186.png" style="width: 576.65px; height: 153.65px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><ul class="c3 lst-kix_list_43-0 start"><li class="c27 li-bullet-0"><span class="c0">In the first example, P2 and P5 are identified by the same peptide set {pe1, pe4, pe5, pe8}. P2 was chosen as typical protein. One subset composed of {pe4, pe5, pe8} identifies subset protein P4.</span></li><li class="c5 li-bullet-0"><span class="c0">In the second example, another protein set represented by P3 shares some peptides with the protein set represented by P2. Both protein sets have specific peptides.</span></li><li class="c41 c32 li-bullet-0"><span class="c0">Sharing could involve many protein sets as shown in example 3.</span></li></ul><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 454.65px; height: 130.03px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\pepprotset2.png" src="images/image185.png" style="width: 454.65px; height: 130.03px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">All </span><span>p</span><span class="c1">eptides </span><span>s</span><span class="c1">ets and associated </span><span>p</span><span class="c1">rotein</span><span>&nbsp;s</span><span class="c1">ets are represented, even if there are no specific peptides. In both cases above, no choice is done on which </span><span>p</span><span class="c1">rotein </span><span>s</span><span class="c1">et / </span><span>p</span><span class="c1">eptide</span><span>&nbsp;s</span><span class="c1">et to keep. These </span><span>p</span><span class="c1">rotein</span><span>&nbsp;s</span><span class="c1">ets could be filtered after inference (see </span><span class="c4 c1"><a class="c8" href="#id.2u6wntf">Protein sets filtering</a></span><span class="c0">). </span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.vj9t7kx4t9zd"></a><h2 class="c2" id="h.ygv3j5ll3of4"><span class="c16">Modification Dataset</span></h2><p class="c79 c77"><span class="c0">From an Identification Summary, users can ask to Identify Modification Sites and Clusters, thus creating a Modification Dataset containing the result. This dataset contains</span></p><ul class="c3 lst-kix_list_89-0 start"><li class="c30 li-bullet-0"><span>Modification Sites: a modification type at a given location on a given protein. They are only extracted for the proteins that are representative of a validated protein set. Only modifications of interest are used in this process.</span></li><li class="c32 c50 li-bullet-0"><span>Modification Clusters: co-localized modification sites are grouped into clusters. Identify Modification Sites and Clusters describe this grouping process.</span></li></ul><a id="id.z337ya"></a><h2 class="c2" id="h.3j2qqm3"><span class="c16">Proteins and Protein sets scoring</span></h2><p class="c6"><span class="c1">There are multiple algorithms tha</span><span>t</span><span class="c0">&nbsp;could be used to calculate the Proteins and Protein Sets scores. Proteins scores are computed during the importation phase while Protein Sets scores are computed during the validation phase. </span></p><h3 class="c2" id="h.1y810tw"><span class="c36 c48">Protein</span></h3><p class="c6"><span class="c1">Each individual protein match is scored according to all peptide matches associated with this protein, independently of any validation of these peptide matches. </span><span class="c0">The sum of the peptide matches scores is used as protein score (called standard scoring for Mascot result files).</span></p><h3 class="c2" id="h.4i7ojhp"><span class="c36 c48">Protein Set</span></h3><p class="c6"><span class="c1">Each individual protein set is scored according to the validated peptide matches belonging to this protein set (see </span><span class="c42 c1 c74"><a class="c8" href="#id.44sinio">inference</a></span><span class="c0">). </span></p><h3 class="c2" id="h.2xcytpi"><span class="c36 c48">Scoring schemes</span></h3><h4 class="c39" id="h.1ci93xb"><span class="c36 c55 c18">Mascot Standard Scoring</span></h4><p class="c6"><span class="c1">The score associated </span><span>with each</span><span class="c1">&nbsp;identified protein (or protein set) is the sum </span><span class="c13">of the score of all peptide matches</span><span class="c0">&nbsp;identifying this protein (or protein set). In case of duplicate peptide matches (peptide matched by multiple queries) only the match with the best score is considered. </span></p><h4 class="c39" id="h.3whwml4"><span class="c36 c55 c18">Mascot MudPIT Scoring</span></h4><p class="c6"><span class="c0">This scoring scheme is also based on the sum of all non-duplicate peptide matches score. However the score for each peptide match is not its absolute value, but the amount that it is above the threshold: the score offset. Therefore, peptide matches with a score below the threshold do not contribute to the protein score. Finally, the average of the thresholds used is added to the score. For each peptide match, the &ldquo;threshold&rdquo; is the homology threshold if it exists, otherwise it is the identity threshold. The algorithm below illustrates the MudPIT score computation procedure: </span></p><p class="c19"><span class="c29 c99 c117 c60">Protein score = 0<br>For each peptide match {<br>&nbsp;If there is a homology threshold and ions score &gt; homology threshold {<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Protein score += peptide score - homology threshold<br>&nbsp;} else if ions score &gt; identity threshold {<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Protein score += peptide score - identity threshold<br>&nbsp;}<br>}<br>Protein score += 1 * average of all the subtracted thresholds</span></p><ul class="c3 lst-kix_list_77-0 start"><li class="c27 li-bullet-0"><span class="c0">if there are no significant peptide matches, the protein score will be 0.</span></li><li class="c5 li-bullet-0"><span class="c0">homology and identity threshold values depend on a given p-value. By default Mascot and Proline compute these thresholds with a p-value of 5%.</span></li><li class="c5 li-bullet-0"><span class="c0">In the case of separated target-decoy searches we obtain two values for each threshold: one for the target search and another one for the decoy search. In order to obtain a single value we apply the following procedure:</span></li></ul><ul class="c3 lst-kix_list_77-1 start"><li class="c15 li-bullet-0"><span class="c0">the homology threshold is the decoy value if it exists else the target value</span></li><li class="c41 c61 li-bullet-0"><span class="c0">the identity threshold is the mean of target and decoy values. </span></li></ul><p class="c6"><span class="c0">The benefit of the MudPIT score over the standard score is that it removes many of the junk protein sets, which have a high standard score but no high scoring peptide matches. Indeed, protein sets with a large number of weak peptide matches do not have a good MudPIT score. </span></p><h4 class="c39" id="h.2bn6wsx"><span class="c36 c55 c18">Mascot Modified MudPIT Scoring</span></h4><p class="c6"><span class="c1">This scoring scheme, introduced by Proline, is a modified version of the Mascot MudPIT one. The difference with the latter is that it does not take into account the average of the </span><span>subtracted</span><span class="c0">&nbsp;thresholds: </span></p><p class="c19"><span class="c29 c99 c60 c117">Protein score = 0<br>For each peptide match {<br>&nbsp;If there is a homology threshold and ions score &gt; homology threshold {<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Protein score += peptide score - homology threshold<br>&nbsp;} else if ions score &gt; identity threshold {<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Protein score += peptide score - identity threshold<br>&nbsp;}<br>}</span></p><p class="c6"><span class="c1">This score has the same benefits </span><span>as</span><span class="c1">&nbsp;the MudPIT one. The main difference is that the minimum value of this modified version will be always close to zero while the genuine MudPIT score defines a minimum value which is not constant between the datasets and the proteins (i.e. the average of all the subtracted thresholds). </span></p><h4 class="c2" id="h.gyl3ow6qhbti"><span class="c36 c55 c18">Fisher</span></h4><p class="c6"><span>This scoring scheme relies on the Fisher&rsquo;s test to define protein scores and p-value from the scores of the best subset of peptides. The scoring is extensively described in (</span><span class="c4"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1021/acs.analchem.0c00328&amp;sa=D&amp;source=editors&amp;ust=1733226590794953&amp;usg=AOvVaw25EgC2RwsPLQ55SFMpwCsD">https://doi.org/10.1021/acs.analchem.0c00328</a></span><span class="c0">).</span></p><h2 class="c2" id="h.qsh70q"><span>Search result</span><a id="id.3as4poj"></a><span class="c16">&nbsp;validation</span></h2><p class="c6"><span class="c1">Once a result file ha</span><span>s</span><span class="c1">&nbsp;been imported and a search result created, the validation is performed in </span><span>four</span><span class="c0">&nbsp;main steps: </span></p><ol class="c3 lst-kix_list_79-0 start" start="1"><li class="c27 li-bullet-0"><span class="c4"><a class="c8" href="#id.1pxezwc">Peptide Matches filtering and Validation</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.44sinio">Protein Inference</a></span><span class="c37 c1">&nbsp;(peptides and proteins grouping)</span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.z337ya">Protein and Proteins Sets scoring</a></span></li><li class="c41 c32 li-bullet-0"><span class="c4"><a class="c8" href="#id.4ekz59m">Protein Sets Filtering and Validation</a></span></li></ol><p class="c6"><span class="c1">Finally, the </span><span class="c4"><a class="c8" href="#id.lnxbz9">i</a></span><span class="c4 c1"><a class="c8" href="#id.lnxbz9">dentification </a></span><span class="c4"><a class="c8" href="#id.lnxbz9">summary</a></span><span>&nbsp;</span><span class="c0">issued from these steps is stored in the identification database. Different validation of a Search Result can be performed and a new Identification Summary of this Search Result is created for each validation.</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span>When validating a merged Search Result, it is possible to propagate the same validation parameters to all childs Search Results. In this case </span><span class="c4"><a class="c8" href="#id.1pxezwc">Peptide Matches filtering and validation</a></span><span>&nbsp;will be applied on childs as well as Protein Sets filtering. </span><span class="c42">Note:</span><span class="c0">&nbsp;actually, Protein Sets validation is not propagated to childs Search Results.</span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.1pxezwc"></a><h3 class="c2" id="h.49x2ik5"><span class="c36 c48">Peptide matches filtering</span></h3><p class="c6"><span class="c1">Peptide Matches identified in search </span><span>results</span><span class="c1">&nbsp;can be filtered using one or multiple predefined filters (describe</span><span>d</span><span class="c1">&nbsp;</span><span>hereafter</span><span class="c0">). Only validated peptide matches will be considered for further steps.</span></p><h4 class="c39" id="h.2p2csry"><span class="c36 c55 c18">Score filter</span></h4><p class="c6"><span>All PSMs with a score lower than a given threshold are discarded. For some search engines, Proline computes the PSM score value itself by applying a mathematical transformation to another PSM property. For instance, the score values for PSMs from X!Tandem search results correspond to the log10 transformation of the PSMs expectation values.</span><span class="c0">&nbsp;</span></p><h4 class="c39" id="h.147n2zr"><span class="c36 c55 c18">Pretty Rank filter</span></h4><p class="c6"><span>This filter is applied after having temporarily joined target and decoy PSMs corresponding to the same query. For each query, target/decoy PSMs are then sorted by score. As in Mascot, a pretty rank is computed for each PSM depending on their ranking: PSM with almost equal score (difference &lt; 0.1) are assigned the same rank. All PSMs with a pretty rank greater than the cut-off specified are discarded.</span></p><h4 class="c39" id="h.3o7alnk"><span class="c36 c55 c18">Minimum sequence Length filter (Length)</span></h4><p class="c6"><span>PSMs corresponding to peptide sequences shorter than the cut-off stipulated will be discarded when this parameter is applied.</span><span class="c0">&nbsp;</span></p><h4 class="c39" id="h.23ckvvd"><span class="c36 c55 c18">Mascot e-Value filter (e-Value)</span></h4><p class="c6"><span>This filter is used to select PSMs based on the Mascot expectation value (e-value) which reflects the difference between the PSM&rsquo;s score and the Mascot identity threshold (p=0.05). PSMs with an e-value greater than the threshold specified are discarded.</span></p><h4 class="c39" id="h.ihv636"><span class="c36 c55 c18">Mascot adjusted e-Value filter (Adjusted e-Value)</span></h4><p class="c6"><span>Proline can compute an adjusted e-value. It first selects the lowest threshold between the identity and homology e-values (p=0.05). Then, it computes the e-value using this selected threshold. PSMs for which the adjusted e-value is greater than the specified cut-off are discarded.</span></p><h4 class="c39" id="h.32hioqz"><span class="c36 c55 c18">Mascot p-Value based on identity filter (Identity p-Value)</span></h4><p class="c6"><span>Given a specific p-value, the Mascot identity threshold is calculated for each query and all peptide matches associated with the query for which the score is lower than the identity threshold calculated are discarded.</span><span class="c0">&nbsp;</span></p><h4 class="c39" id="h.1hmsyys"><span class="c36 c55 c18">Mascot p-Value based on homology filter (Homology p-Value)</span></h4><p class="c6"><span>Given a specific p-value, the Mascot homology threshold is inferred for each query and all peptide matches associated with the query which have a score lower than the calculated homology threshold are discarded.</span></p><h4 class="c39" id="h.41mghml"><span class="c36 c55 c18">Single PSM per MS Query filter</span></h4><p class="c6"><span class="c1">This filter validates only </span><span class="c13">one PSM per Query</span><span class="c1">. To select a PSM, following rules</span><span>&nbsp;are</span><span class="c0">&nbsp;applied: </span></p><p class="c6"><span class="c0">For each query: </span></p><ul class="c3 lst-kix_list_80-0 start"><li class="c27 li-bullet-0"><span class="c0">Select PSM with higher score.</span></li><li class="c5 li-bullet-0"><span class="c0">If several PSM with same score: </span></li></ul><ul class="c3 lst-kix_list_80-1 start"><li class="c15 li-bullet-0"><span class="c0">Choose PSM which identify the protein which have the max number of valid PSM </span></li><li class="c15 li-bullet-0"><span class="c0">If still equality choose the PSM with the lower delta moz</span></li><li class="c15 li-bullet-0"><span class="c0">If still equality, select the first PSM using alphabetic order</span></li><li class="c15 li-bullet-0"><span class="c0">If still equality, select the first one. Warning : may be different in different versions of Prolline...</span></li></ul><p class="c6 c7"><span class="c0"></span></p><h4 class="c39" id="h.2grqrue"><span class="c36 c55 c18">Single PSM per Rank filter</span></h4><p class="c6"><span>This filter selects only </span><span class="c18">one PSM per pretty rank</span><span class="c0">, which is already the case when a given pretty rank is associated with a single PSM. When multiple PSMs have the same pretty rank, the same selection than &ldquo;Single PSM per Query&rdquo; is used</span></p><p class="c81 c77"><span class="c11">&#9679;</span><span class="c68">&nbsp; &nbsp; </span><span class="c0">Select PSM that identifies the protein which has the max number of valid PSM.</span></p><p class="c77 c81"><span class="c11">&#9679;</span><span class="c68">&nbsp; &nbsp; </span><span class="c0">If several PSM are equals</span></p><p class="c63"><span class="c11">o</span><span class="c68">&nbsp; &nbsp; &nbsp; </span><span class="c0">Choose the PSM with the lower delta moz.</span></p><p class="c63"><span class="c11">o</span><span class="c68">&nbsp; &nbsp; &nbsp; </span><span class="c0">If still equality, select the first PSM using alphabetic order.</span></p><p class="c63"><span class="c11">o</span><span class="c68">&nbsp; &nbsp; &nbsp; </span><span class="c0">If still equality, select the first one. Warning : may be different in different versions of Proline...</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">Thus, if this filter is combined with the &ldquo;Pretty rank&rdquo; filter, the result obtained should be identical to the result of the &ldquo;Single PSM per MS query&rdquo; filter.</span></p><p class="c6 c7"><span class="c0"></span></p><h4 class="c39" id="h.16wurjp0l80b"><span class="c36 c55 c18">Single Sequence per Rank filter</span></h4><p class="c6"><span>This filter selects the best </span><span class="c18">PSM per pretty rank</span><span>&nbsp;and all PSMs that have the </span><span class="c18">same sequence</span><span class="c0">, without considering Post translational modification. The best PSM is selected using the same rules as the Single PSM per Rank filter described above. </span></p><p class="c6 c7"><span class="c0"></span></p><h4 class="c2" id="h.ahszc2l0m93x"><span class="c36 c55 c18">Isotope Offset filter</span></h4><p class="c26"><span class="c0">This filter validates PSMs which have been validated using an isotope offset less or equal than specified one. This corresponds to identification search engine parameters such as Mascot isotope error and could be used to invalidate PSM identified using one or more 13C .</span></p><p class="c26 c7"><span class="c0"></span></p><a id="id.vx1227"></a><h3 class="c2" id="h.3fwokq0"><span class="c36 c48">Peptide matches validation</span></h3><p class="c6"><span class="c0">In addition to these filters a PSM level validation step may be specified using:</span></p><ul class="c3 lst-kix_list_74-0 start"><li class="c72 c32 li-bullet-0"><span>A target-decoy approach (Elias and Gygi, 2007), which &nbsp;can be performed by adjusting a user-specified validation criterion until it reaches a user-specified false discovery rate (FDR). The search engine score can be used as a generic validation criterion for any of the search engines supported by Proline. For results obtained with the Mascot search engine, other criteria can be used to control the FDR: Mascot e-Value, Mascot adjusted e-Value, Mascot p-Value based on identity threshold, or Mascot p-Value based on homology threshold. </span></li><li class="c49 c32 li-bullet-0"><span>A method based on the Benjamini-Hochberg procedure to control the FDR proposed in (</span><span class="c4"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1021/acs.analchem.0c00328&amp;sa=D&amp;source=editors&amp;ust=1733226590799213&amp;usg=AOvVaw1pyFk6kMLP8RCwSmZO9Sbc">https://doi.org/10.1021/acs.analchem.0c00328</a></span><span>) </span></li></ul><p class="c6 c7"><span class="c0"></span></p><a id="id.1v1yuxt"></a><h3 class="c2" id="h.4f1mdlm"><span class="c36 c48">FDR estimation for Target Decoy validation</span></h3><p class="c6"><span class="c0">There are several ways to calculate FDR depending on the database search type. In Proline the FDR is calculated at PSM and protein levels using the following rules: </span></p><ul class="c3 lst-kix_list_47-0 start"><li class="c27 li-bullet-0"><span>if the Search has been done on a concatenated Target/Decoy bank or if rank filter has been used during validation :</span></li></ul><p class="c77 c85"><img src="images/image1.png"></p><p class="c77 c101"><span class="c42">Note</span><span class="c0">: when computing PSM FDR, peptide sequences matching a Target Protein and a Decoy Protein are taken into account in both cases.</span></p><p class="c85 c77 c7"><span class="c0"></span></p><ul class="c3 lst-kix_list_47-0"><li class="c41 c32 li-bullet-0"><span>if the Search has been done on a separated Target/Decoy bank :</span></li></ul><p class="c77 c91"><img src="images/image2.png"></p><p class="c6"><span class="c1">&nbsp;</span></p><a id="id.2u6wntf"></a><h3 class="c2" id="h.19c6y18"><span class="c36 c48">Protein sets filtering</span></h3><p class="c6"><span>Any Identification Summary, generated by a </span><span class="c4"><a class="c8" href="#id.3as4poj">validation</a></span><span class="c51">&nbsp;</span><span>process or by </span><span class="c4"><a class="c8" href="#id.18vjpp8">merging</a></span><span class="c51">&nbsp;</span><span class="c0">datasets could be filtered. </span></p><p class="c6"><span class="c0">Filtering consists in invalidating Protein Sets which don&#39;t follow specified criteria. Invalidated Protein Sets are not taken into account for further algorithms or display. </span></p><p class="c6"><span class="c0">Available filtering criteria are defined below. </span></p><h4 class="c39" id="h.3tbugp1"><span class="c36 c55 c18">Specific peptides filter</span></h4><p class="c6"><span class="c0">This filter invalidates protein sets that don&#39;t have at least x peptides identifying only that protein set. The specificity is considered at the DataSet level. </span></p><p class="c6"><span class="c0">This filtering goes through all Protein Sets from worse score to best score. For each, if the protein set is invalidated, associated peptides properties are updated before going to the next protein set. Peptide property is the number of identified protein sets. </span></p><h4 class="c39" id="h.28h4qwu"><span class="c36 c55 c18">Peptides count filter</span></h4><p class="c6"><span class="c0">This filter invalidates protein sets that don&#39;t have at least x peptides identifying that protein set, independently of the number of protein sets identified by the same peptide. </span></p><p class="c6"><span class="c0">This filtering goes through all Protein Sets. For each, if the protein set is invalidated, associated peptides properties are updated before going to the next protein set. Peptide property is the number of identified protein sets. </span></p><h4 class="c39" id="h.nmf14n"><span class="c36 c55 c18">Peptide sequence count filter</span></h4><p class="c6"><span class="c0">This filter invalidates protein sets that don&#39;t have at least x different peptide sequences (independently of PTMs) identifying that protein set. </span></p><p class="c6"><span class="c0">This filtering goes through all Protein Sets from worse score to best score. For each, if the protein set is invalidated, associated peptides properties are updated before going to the next protein set. Peptide property is the number of identified protein sets.</span></p><h4 class="c39" id="h.37m2jsg"><span class="c36 c55 c18">Protein set score filter</span></h4><p class="c6"><span>This filter invalidates protein sets which score is below a given value.</span><span class="c0">&nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><a id="id.1mrcu09"></a><h3 class="c2" id="h.46r0co2"><span class="c36 c48">Protein sets validation</span></h3><p class="c6"><span class="c1">Once prefilters (see above) have been applied, a validation algorithm can be run to control the FDR. </span><span class="c0">In the same way as for PSM, FDR may be controlled using :</span></p><ul class="c3 lst-kix_list_132-0 start"><li class="c72 c32 li-bullet-0"><span>the Benjamini-Hochberg procedure, see (</span><span class="c4"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1021/acs.analchem.0c00328&amp;sa=D&amp;source=editors&amp;ust=1733226590807176&amp;usg=AOvVaw3GjNIcrtDADJA0cgIGenBA">https://doi.org/10.1021/acs.analchem.0c00328</a></span><span class="c0">) </span></li><li class="c49 c32 li-bullet-0"><span>a target decoy strategy.</span></li></ul><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c42 c1 c74 c109 c60">Target-Decoy Strategy:</span></p><p class="c6"><span class="c1">See how</span><span class="c42 c1 c74"><a class="c8" href="#id.1v1yuxt">&nbsp;</a></span><span class="c4 c1"><a class="c8" href="#id.1v1yuxt">FDR is calculated</a></span><span>.</span></p><p class="c6"><span class="c1">At the moment, it is only possible to control the FDR by changing the Protein Set Score threshold. Three different </span><span class="c4 c1"><a class="c8" href="#id.z337ya">protein set scoring functions</a></span><span class="c0">&nbsp;are available. </span></p><p class="c6"><span class="c1">Given an expected FDR, the system tr</span><span>ies</span><span class="c0">&nbsp;to estimate the best score threshold to reach this FDR. Two validation rules (R1 and R2) corresponding to two different groups of protein sets (see below the detailed procedure) are optimized by the algorithm. Each rule defines the optimum score threshold allowing to obtain the closest FDR to the expected one for the corresponding group of protein sets. </span></p><p class="c6"><span class="c0">Here is the procedure used for FDR optimization: </span></p><ul class="c3 lst-kix_list_65-0 start"><li class="c27 li-bullet-0"><span class="c0">protein sets are segregated in two groups, the ones identified by a single validated peptide (G1) and the ones identified by multiple validated peptides (G2), with potentially multiple identified PSMs per peptide.</span></li></ul><ul class="c3 lst-kix_list_68-0 start"><li class="c5 li-bullet-0"><span class="c0">for each of the validation rules, the FDR computation is performed by merging target and decoy protein sets and by sorting them by descending score. The score threshold is then modulated by using successively the score of each protein set of this sorted list. For each new threshold, a new FDR is computed by counting the number of target/decoy protein sets having a score above or equivalent to this value. The procedure stops when there are no more protein sets in the list or when a maximum FDR of 50% is reached. It has to be noted that the two validation rules are optimized separately:</span></li></ul><ul class="c3 lst-kix_list_68-1 start"><li class="c15 li-bullet-0"><span class="c0">G2 FDR is first optimized leading to the R2 score threshold. The validation status of G2 protein sets is then fixed.</span></li><li class="c41 c61 li-bullet-0"><span class="c0">final FDR (G1+G2) is then optimized leading to the R1 score threshold. Only the G1 protein sets are here used for the score threshold modulation procedure. However the FDR is computed by taking into account the G2 validated target/decoy protein sets.</span></li></ul><p class="c6"><span class="c1">The separation of </span><span>protein</span><span class="c1">&nbsp;sets in two groups allows to increase the power of discrimination between target and decoy hits. Indeed, the score threshold of the G1 group is often much higher than the G2 one. If we were using a single average threshold, this w</span><span>ould</span><span class="c1">&nbsp;reduce the number of G2 validated proteins, leading to a decrease in sensitivity for a same value of FDR.</span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.2lwamvv"></a><h1 class="c2"><span>Combining datasets</span></h1><p class="c6"><span class="c0">Identification results can be combined to construct a parent dataset, and create a non-redundant list of identified peptides and proteins. This combination can be performed either before validation (on search results) or after validation (on identification summaries). Since this operation could be recursively performed, it leads to hierarchical structuring of search results and/or identification summaries. On the one hand, combination before validation (taking into account all PSMs identified by the search engine) may, for example, be relevant when analyzing results obtained after peptide fractionation: in that case, several peptides belonging to the same protein may be spread across different result sets; these sets should be merged before protein validation. On the other hand, merging identification summaries is appropriate when seeking to group the validated results from a series of individual samples to be compared or when combining data from different search engines.</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">When datasets are combined, their PSMs are collected to generate a non-redundant set of peptides before recomputing protein inference. Additionally, the mappings between peptides and FASTA entries observed across the different datasets are also collected and merged into a single final mapping list. This list reflects thus the whole set of peptide and protein matches that were observed in the individual datasets.</span></p><p class="c6"><span class="c0">Users can combine search results or identification summaries. The main difference is the set of spectra and peptides (and thus PSMs) considered. When combining search results, all spectra, peptides and PSMs in the dataset are considered, whereas when combining identification summaries, only validated PSMs are taken into account. In addition, Proline can be used to control how PSMs are collected in the parent dataset: in union mode, PSMs originating from combined datasets are added, while in aggregation mode, all PSMs identifying the same peptide are aggregated into a single representative PSM.</span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 491.00px; height: 388.00px;"><img alt="" src="images/image188.png" style="width: 491.00px; height: 388.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c19 c105"><span class="c1 c29 c23">Combining datasets in Proline. Datasets are represented as a tripartite graph composed of spectra, peptides and proteins; edges between spectrum and peptides represent PSM. When the blue and orange datasets are combined, PSMs from both datasets are collected together, generating a non-redundant set of peptides. The combination can be performed in 2 modes: union or aggregation mode.</span></p><p class="c19 c7"><span class="c0"></span></p><h1 class="c2" id="h.111kx3o"><span class="c12">Identifying Modification sites and clusters</span></h1><p class="c19"><span>A list of Post translational modification (PTM) sites identified among the peptides of an identification summary can be extracted by Proline. A modification site is characterized by a modification type, at a given location on a given protein. The list of modification sites extracted by the software is restricted to the proteins that are representative of a validated protein set and to the modifications of interest specified by the user. This means that a peptide identified with two different modifications of equal interest to the user will appear twice in the list, one for each modification location.</span></p><p class="c19"><span class="c0">In a second phase, co-localized modification sites are grouped into clusters as soon as evidence of their co-existence exists. The required evidence is a peptide sequence identified in the dataset with all the clusterized modification sites.</span></p><p class="c19"><span class="c0">The figure below represents different peptides (blue rectangles), co-localized on a protein sequence (in red). In this example, modifications of interest (Phosphorylation (Ph)) are shown in orange. Peptide 1 proves that Phosphorylation at positions m, x and y occurs simultaneously. Peptides 2 and 3 are considered as isomorphic since the oxidation of peptide 2 is ignored (only Phosphorylations have been declared of interest by the user in this example). Peptides 4 and 5 are partially isomorphic: they confirm Phosphorylation respectively at position (m, x) and (x, y). Conversely peptides 6 and 7 are not in accordance, suggesting that there are two other proteoforms, one with Phosphorylations at position (m,y) but no modification at x and one with a Phosphorylation at position m but not at x. These two peptides could be grouped into another cluster.</span></p><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 615.85px; height: 339.85px;"><img alt="" src="images/image187.png" style="width: 615.85px; height: 339.85px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c19"><span class="c0">The user can choose between two different modification clustering methods:</span></p><ul class="c3 lst-kix_list_19-0 start"><li class="c5 li-bullet-0"><span>Partially Isomorphic Matching (aka Exact Position matching): &nbsp;Two sequence matches are clusterized if they are Isomorphic or Partially Isomorphic. Partially Isomorphic matches could belong to multiple clusters.</span></li><li class="c19 c32 li-bullet-0"><span>Only Isomorphic Matching: two sequence matches are clusterized if they are Isomorphic.</span></li></ul><p class="c79 c77"><span class="c0">In Proline, clusters co-localisation concept is used. Co-localized clusters are clusters that have overlapping on protein sequence. These clusters may share one or more sites, but they could also cover the same protein sequence without any common site. See co-localized clusters example below.</span></p><p class="c28"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 479.26px; height: 367.53px;"><img alt="" src="images/image190.png" style="width: 479.26px; height: 367.53px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.3l18frh"></a><h1 class="c2"><span class="c12">Spectral Counting</span></h1><h2 class="c2" id="h.206ipza"><span class="c16">Principles</span></h2><p class="c19"><span class="c0">Proline can be used to compare protein sets based on spectral counts through a previously presented algorithm (Hesse et al., 2016). This algorithm notably computes a weighted spectral count metric (called adjusted spectral count in the original publication). Basically, the algorithm takes both unique and shared peptides into account, and for each shared peptide, the proportion of MS/MS spectra that should be attributed to the different protein sets is determined. This proportion (also called weight) is based on the spectral counting of proteotypic (or specific) peptides identifying the different protein sets sharing the peptide to be attributed. </span></p><p class="c6"><span>Spectral counting is calculated from a hierarchy of </span><span class="c33">identification summaries</span><span>. The parent </span><span class="c33">identification summary</span><span class="c0">&nbsp;(at the top of the hierarchy) is where the list of protein sets to compare and the list of specific peptides are created. The list of specific peptides is then used to compute the protein sets&#39; respective weights but users can choose any&ldquo;child&rdquo; dataset where the weights must be calculated. </span></p><p class="c6"><span>First, Proline compute the </span><span class="c33">peptide spectral count</span><span class="c0">&nbsp;at each level of the dataset hierarchy using the following rules: </span></p><ul class="c3 lst-kix_list_45-0 start"><li class="c27 li-bullet-0"><span>If the dataset is a &ldquo;leaf&rdquo; </span><span class="c33">identification summary</span><span>&nbsp;of the hierarchy (not issued from a merge, no child dataset), the </span><span class="c99">p</span><span class="c33">eptide spectral count</span><span class="c0">&nbsp;is the number of MS/MS spectra matching a peptide (equal to the number of peptide spectrum matches).</span></li><li class="c5 li-bullet-0"><span>If the dataset is a merged </span><span class="c33">identification summary</span><span>, the</span><span class="c82">&nbsp;</span><span class="c33">peptide spectral count</span><span class="c82">&nbsp; </span><span>is the sum of its child </span><span class="c33">peptide spectral count.</span></li><li class="c41 c32 li-bullet-0"><span>If the dataset is a merged of </span><span class="c99">search results</span><span>, the </span><span class="c99">p</span><span class="c33">eptide spectral count</span><span>&nbsp;is the sum of </span><span class="c18">validated children</span><span>&nbsp;</span><span class="c33">peptide spectral count</span><span>. Validated children are PSMs that meet validation criteria applied to parent </span><span class="c33">Identification Summaries</span><span class="c0">.</span></li></ul><p class="c6"><span>Once, </span><span class="c33">peptide spectral count</span><span>&nbsp;is calculated for each peptide, the </span><span class="c99">protein spectral count </span><span class="c0">is computed using the following rules:</span></p><ul class="c3 lst-kix_list_58-0 start"><li class="c5 li-bullet-0"><span class="c33">Protein basic spectral count</span><span>&nbsp;(BSC) is the sum of the </span><span class="c33">peptide spectral count</span><span class="c0">&nbsp;of all peptides matching a protein.</span></li><li class="c5 li-bullet-0"><span class="c33">Protein specific spectral count</span><span>&nbsp;(SSC) is the sum of the </span><span class="c33">peptide spectral count</span><span class="c0">&nbsp;of specific peptides. A specific peptide is a peptide which does not identify any other protein sets in the context of the identification summaries.</span></li><li class="c41 c32 li-bullet-0"><span class="c33">Protein weighted spectral count</span><span>&nbsp;(WSC) is the </span><span class="c33">Protein specific spectral count</span><span class="c82">&nbsp;(SSC)</span><span>&nbsp;+ weighted </span><span class="c33">peptide spectral count</span><span class="c0">&nbsp;of shared peptides. </span></li></ul><h3 class="c2" id="h.4k668n3"><span class="c36 c48">Simple example</span></h3><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 372.15px; height: 159.23px;"><img alt="" src="images/image189.png" style="width: 372.15px; height: 159.23px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9 c7"><span class="c13 c29 c60"></span></p><p class="c9 c7"><span class="c13 c29 c60"></span></p><h2 class="c2"><span>Protein sets weights </span></h2><p class="c6"><span class="c1">The </span><span>protein set respective </span><span class="c1">weights computation is based on the proteotypic peptides</span><span>. The level in the dataset hierarchy where these weights are calculated can be chosen by the user, it could be the top level of the dataset hierarchy or at a lower level. In the following example, the weights (W</span><span class="c54">P1</span><span>&nbsp;and W</span><span class="c54">P2</span><span class="c0">) are calculated at the &ldquo;parent dataset&rdquo; level. At this level, P1 and P2 are respectively identified by the red and dark blue peptides/psm, each protein set weight is calculated using their specific spectral counting. These weights are thus used to calculate the weighted spectral count of P1 and P2 in the two &ldquo;child&rdquo; dataset &ldquo;Condition 1&rdquo; and &ldquo;Condition 2&rdquo;. </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 390.74px; height: 269.42px;"><img alt="" src="images/image192.png" style="width: 390.74px; height: 269.42px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2" id="h.2zbgiuw"><span class="c16">FAQ</span></h2><h3 class="c2" id="h.1egqt2p"><span class="c36 c48">Why is the BSC less than Peptide Count ?</span></h3><p class="c19"><span>When running SC even on a simple hierarchy (1 parent, 2 childs) in some cases we obtain a BSC value smaller than the peptide count of the protein set. This occurs only for </span><span class="c23">invalid protein sets</span><span>. </span><span class="c23">Invalid protein sets</span><span>&nbsp;are the ones that are present at the parent level but are filtered at child level (if a </span><span class="c23">specific peptide</span><span class="c0">&nbsp;filter has been applied for example).</span></p><p class="c19"><span class="c0">Indeed, the peptide count value is read in the child protein sets. On the other hand, the BSC is calculated by getting the spectral count information at child level for each peptide identified at parent level. If a protein set is invalid, its peptides are not taken into account during the merging so some of them could be missing at the parent level if they were not identified in the other child. </span></p><p class="c26"><span class="c0">This case is illustrated in the following figure: </span></p><p class="c25 c108"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 455.65px; height: 371.55px;"><img alt="" src="images/image191.png" style="width: 455.65px; height: 371.55px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.3ygebqi"></a><h1 class="c2"><span>Label-free LC-MS quantitation workflow</span></h1><p class="c10"><span>Proline detects chromatographic peaks from raw data </span><span class="c18">converted to the mzDB format</span><span>&nbsp;(Bouyssi&eacute; et al., 2015). Depending on the Proline version and the instrument used to acquire data, two </span><span class="c4"><a class="c8" href="#id.o9t5nltnb7nt">converters are available</a></span><span>&nbsp;to generate mzdbFile. <br></span></p><p class="c10"><span>After a first </span><span class="c18">signal extraction </span><span>step, the algorithm associates the chromatographic peaks detected with validated PSMs, first by retrieving the corresponding MS/MS spectra acquired during the peptide elution, and then by </span><span class="c18">matching</span><span>&nbsp;</span><span class="c18">the precursor</span><span>&nbsp;m/z value of these spectra to the chromatographic peak m/z value. After the </span><span class="c18">deisotoping </span><span>step, the abundance of each ion is estimated from the apex of the chromatographic peak, which corresponds to the theoretically most abundant isotopologue (inferred from the peptide&rsquo;s atomic composition). The software then </span><span class="c18">aligns </span><span>the retention time of these annotated ions for all the LC-MS runs to be compared, and uses this information to </span><span class="c18">cross-assign</span><span class="c0">&nbsp;MS/MS data to ions (i.e. chromatographic peaks) that were detected but not identified in other runs . The resulting ion abundances are finally stored in the Proline database, making them available for rapid data visualization and further post-processing.</span></p><p class="c10"><span>Finally, peptide ion measurements can be </span><span class="c18">summarized as protein abundances</span><span class="c0">&nbsp;using different computational methods. The user can opt to perform additional operations such as excluding peptides or ions based on their characteristics (missed cleavages, variable modifications, sequence specificity, etc.) or normalizing peptide and protein abundances between runs. These post-processing steps can be executed on-demand using different parameters or methods; there is no need to repeat the whole quantification process when changes are made.</span></p><p class="c7 c10"><span class="c0"></span></p><h2 class="c2" id="h.2dlolyb"><span class="c16">Signal extraction</span></h2><p class="c10"><span class="c0">During an LC-MS experiment, the m/z and intensity values for each peptide ion detected are recorded in MS1 scans acquired during the elution of this peptide from the chromatographic column. Most existing peak picking algorithms analyze these MS scans individually or sequentially. The Proline algorithm performs the signal detection in a different way. It first takes advantage of the mzDB format to detect chromatographic peaks in spectrum slices (5 m/z wide by default) across the whole chromatographic time. Then, in a given slice, m/z peaks are sorted in decreasing order of intensity. Thus, starting from the most intense m/z peak (apex), the algorithm searches for a peak with the same m/z value in the previous and subsequent MS1 scans, while applying a user-defined m/z tolerance. This lookup procedure stops when the ion signal is absent from more than a predefined number of consecutive scans. The [RT, m/z, intensity] peak list obtained, which is comparable to an extracted ion chromatogram (XIC), is then smoothed using a Savitzky-Golay filter (Savitzky and Golay, 1964). The resulting smoothed chromatogram is then split into the time dimension to form chromatographic peaks, by applying a peak picking procedure that will search for significant minima and maxima of signal intensity. When the signals of two ions overlap in the time dimension, a minimum is generally surrounded by two maxima. If the corresponding valley is deep enough, i.e. at least 66% of the lower surrounding maximum, this minimum will be considered significant (and thus will trigger the generation of two peaks). Once the smoothed chromatogram has been fully analyzed, the algorithm removes the corresponding detected peaks from the current spectrum slice, and performs another lookup using the next available apex. The result of this whole procedure is a list of chromatographic peaks defined by an m/z value, an apex elution time and an elution time range.</span></p><h3 class="c2" id="h.sqyw64"><span class="c36 c48">Parameters</span></h3><p class="c6"><span class="c0">These parameters are used by signal extraction algorithms.</span></p><ul class="c3 lst-kix_list_148-0 start"><li class="c72 c32 li-bullet-0"><span class="c0">Extraction m/z tolerance: this corresponds to the error tolerance used when searching for a m/z peak in the previous and subsequent MS1 scans (see above). </span></li><li class="c49 c32 li-bullet-0"><span>Use previous/last peakel detection: If a quantitation has already been done on a run, use the previously detected peakels file for this run. </span><span class="c18">Warning</span><span class="c0">: In this case, the extraction m/z tolerance parameter is ignored since the signal extraction is already done ! </span></li></ul><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span>PSM assignment and deisotoping</span></h2><p class="c10"><span class="c0">In a single run, validated PSMs are MS/MS spectra assigned to a peptide sequence, and each spectrum is characterized by a precursor mass, a charge state and a retention time (RT). The algorithm assigns PSMs to detected chromatographic peaks by matching the spectrum precursor m/z ratio to the chromatographic peak m/z and verifies that the spectrum retention time falls within the peak time range. The PSM charge state assigned is then used to search for chromatographic peaks corresponding to the ion&rsquo;s isotopologues, considering the precursor mass-to-charge ratio of the spectrum as the monoisotope. The peptide ion intensity is summarized by retaining only the apex of the peak corresponding to the theoretically most abundant isotopologue (inferred from the peptide&rsquo;s atomic composition). All peptide ion signals (a.k.a. LC-MS features) extracted from an mzDB file and assigned to a PSM are used to construct an LC-MS map. For the sake of simplicity no distinction is made between LC-MS runs and LC-MS maps in this manuscript.</span></p><p class="c10 c7"><span class="c0"></span></p><h3 class="c77 c78 c98" id="h.3cqmetx"><span class="c36 c48">Parameters</span></h3><p class="c10 c7"><span class="c0"></span></p><ul class="c3 lst-kix_list_81-0 start"><li class="c10 c32 li-bullet-0"><span>PSM/Peakel matching m/z tolerance is the m/z error allowed to match a peakel to a PSM</span></li></ul><p class="c10 c7"><span class="c0"></span></p><p class="c6"><span class="c0">Clustering is applied to group peakels that are matching the same identified ion. </span></p><p class="c89 c77 c56 c7"><span class="c0"></span></p><ul class="c3 lst-kix_list_56-0 start"><li class="c5 li-bullet-0"><span class="c0">Cluster time computation: you have the choice between 2 computation methods: most intense or median. For &quot;most intense&rdquo; method, the cluster time corresponds to the time of the most intense feature composing the cluster. For the &ldquo;median&rdquo; method, cluster time is the median of the feature times forming the cluster.</span></li></ul><ul class="c3 lst-kix_list_33-0 start"><li class="c41 c32 li-bullet-0"><span class="c0">Cluster intensity computation: you have the choice between 2 computation methods: most intense or sum. For &ldquo;most intense&rdquo; method, the cluster intensity corresponds to the intensity of the most intense feature of features forming the cluster. For the &ldquo;sum&ldquo; method, cluster intensity is the sum of the intensities of features composing the cluster.</span></li></ul><h2 class="c2"><span>Retention time alignment</span></h2><p class="c19"><span>As soon as the PSM are matched to peakels, the software then aligns the retention time of the annotated ions for all the LC-MS runs to be compared, and uses this information to cross-assign MS/MS data to ions (i.e. chromatographic peaks) that were detected but not identified in other runs. </span><span class="c1">Because chromatographic separation is not completely reproducible, LC-MS </span><span>runs </span><span class="c1">must be aligned. The retention time alignment procedure is a critical step in MS1 label-free quantification. Proline&rsquo;s alignment algorithm selects a</span><span class="c18 c73">&nbsp;reference run</span><span class="c1">&nbsp;and generates a set of functions that will be used to predict the RT (retention time) for missing features from another run. These functions are obtained by performing pairwise alignments between the different runs to be compared (Bylund et al., 2002; Jaitly et al., 2006; Sadygov et al., 2006).</span><span>&nbsp;</span><span class="c1">The first step consists in computing a scatter plot (</span><span>see Figure below</span><span class="c1">) of the observed time difference between the two runs as a function of the reference run&rsquo;s time-scale. This mapping can be based on the </span><span class="c18 c73">peptide identity</span><span class="c0">&nbsp;(same sequence and same post-translational modifications) of identified features, or by mapping the detected features of the pair of runs, taking user-defined time and mass error ranges into account (the default feature mapping time and m/z tolerance values are set to 600 seconds and 5 ppm, respectively).</span></p><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 602.00px; height: 365.00px;"><img alt="" src="images/image193.png" style="width: 602.00px; height: 365.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c79 c77"><span class="c18 c23">RT prediction using computed alignments</span><span class="c23">. The two scatter plots on the left correspond to computed run alignments between the reference run and two other runs (A and B). The red curves on these plots correspond to the median RT prediction for each alignment, obtained by applying a moving median calculation. The graphs on the right illustrate the case of a peptide ion that is present in runs A and B, but has only been fragmented by MS/MS in run A. Knowing the retention time in run A (T</span><span class="c23 c54">A</span><span class="c23">), we can predict T</span><span class="c23 c54">B</span><span class="c23">&nbsp;by two consecutive time conversions. T</span><span class="c23 c54">A</span><span class="c23">&nbsp;is first converted to the reference run scale (T</span><span class="c23 c54">REF</span><span class="c23">&nbsp;= T</span><span class="c23 c54">A</span><span class="c23">&nbsp;+ D</span><span class="c23 c54">A</span><span class="c23">) using the first run alignment, then to T</span><span class="c23 c54">B</span><span class="c23">&nbsp;= T</span><span class="c23 c54">REF</span><span class="c23">&nbsp;+ D</span><span class="c23 c54">B</span><span class="c1 c29 c23">&nbsp;using the alignment for the second run.</span></p><p class="c25 c7"><span class="c0"></span></p><p class="c19"><span class="c1">RT prediction functions are then obtained by </span><span class="c18 c73">smoothing </span><span class="c1">these scatter plots using a moving median calculation or a local regression. To decrease the number of alignment combinations (i.e., pairs of maps), the reference run is determined by an iterative method. The algorithms begin by selecting a random run as a reference and compute all alignments against this map. The algorithm then determines a new </span><span class="c73">reference run</span><span class="c1">&nbsp;by selecting the run with the smallest sum of RT differences in the resulting run alignments. The iteration stops after a user-specified maximum number of iterations or when the reference run remains unchanged between two iterations. The software can also be configured to compute all possible RT alignment combinations (all possible pairs of maps, &ldquo;exhaustive&rdquo; option), but this can be very computationally expensive when there is a high number of maps to be compared. The alignments computed can then be used to predict the retention times for peptide ions in a specific sample where they were not identified.</span></p><h3 class="c2" id="h.1rvwp1q"><span class="c36 c48">Parameters</span></h3><h4 class="c39" id="h.4bvk7pj"><span class="c36 c18 c55">Alignment</span></h4><ul class="c3 lst-kix_list_72-0 start"><li class="c76 c32 li-bullet-0"><span class="c0">Map Alignment: alignment can be disabled. In this case the cross assignment step could be executed but without any retention time prediction </span></li><li class="c76 c32 li-bullet-0"><span class="c0">Method : is the name of the method used to determine the reference run</span></li></ul><ul class="c3 lst-kix_list_72-1 start"><li class="c15 li-bullet-0"><span class="c0">Exhaustive: the exhaustive algorithm computes the distance between maps for each possible couple of maps and selects the map with the lowest sum of distances to be the reference map. </span></li><li class="c41 c61 li-bullet-0"><span class="c0">Iterative: &nbsp;the algorithm iteratively selects the best reference run as described above.</span></li></ul><ul class="c3 lst-kix_list_72-2 start"><li class="c41 c92 li-bullet-0"><span class="c0">Maximum number of iterations: this option is available only for the iterative method. This is a stop condition of the iterative algorithm, when the algorithm has reached its maximum number of iterations, it stops.</span></li></ul><ul class="c3 lst-kix_list_14-0 start"><li class="c27 li-bullet-0"><span class="c0">Ignore alignment error : allow or not to continue quantitation even if there are some errors in alignment.</span></li></ul><h4 class="c2" id="h.2r0uhxc"><span class="c36 c55 c18">Feature mapping</span></h4><ul class="c3 lst-kix_list_53-0 start"><li class="c5 li-bullet-0"><span class="c0">Feature mapping is needed to build the alignment scatter plot &nbsp;of the observed time difference between two runs. The method used could be:</span></li></ul><ul class="c3 lst-kix_list_53-1 start"><li class="c15 li-bullet-0"><span class="c0">Feature coordinates : the mapping is based on features RT and m/z coordinates taking into account a tolerance for each of these values.</span></li></ul><ul class="c3 lst-kix_list_53-2 start"><li class="c24 li-bullet-0"><span class="c0">m/z tolerance : m/z window used to match features between two compared maps.</span></li></ul><ul class="c3 lst-kix_list_53-1"><li class="c15 li-bullet-0"><span class="c0">Peptide Identity : &nbsp;the mapping is based on peptides identification (same sequence and same post-translational modifications)</span></li></ul><ul class="c3 lst-kix_list_27-0 start"><li class="c41 c32 li-bullet-0"><span class="c0">Time tolerance in seconds: time window used to match features/peptides between two compared maps.</span></li></ul><h4 class="c2" id="h.1664s55"><span>Alignment smoothing</span></h4><p class="c6"><span class="c0">When features of two runs are matched, a trend can be extracted from the scatter plot by using a smoothing method. </span></p><ul class="c3 lst-kix_list_42-0 start"><li class="c27 li-bullet-0"><span class="c0">Smoothing method: you have the choice between three smoothing methods: LOESS, time window or landmark range:</span></li></ul><ul class="c3 lst-kix_list_42-1 start"><li class="c53 c101 li-bullet-0"><span class="c0">LOESS : no specific parameter</span></li><li class="c53 c101 li-bullet-0"><span class="c0">Landmark range</span></li></ul><ul class="c3 lst-kix_list_42-2 start"><li class="c41 c92 li-bullet-0"><span class="c0">Number of landmarks/window size: time of aligned map is corrected using median computed on windows containing a specified number of landmarks. The run is divided into windows of size the specified number of landmarks. You have to provide the number of landmarks by window. The smoothing method is applied considering the number of landmarks present in the window, and computes the median point for this window.</span></li><li class="c41 c92 li-bullet-0"><span class="c0">Sliding window overlap: overlap is used to compute the step to move the smoothing window forward to calculate a smoothing point for this new smoothing window. Overlap gives the percentage of overlapping between two consecutive windows. For example, if window size is 200 (seconds or landmarks depending on which smoothing method is selected) and overlap is 20%, the step forward = 200*((100-20)/100) = 160 seconds or landmarks, i.e. the smoothing window is moved forward by a step of 160, so two successive windows overlaps each other by a step of 40 seconds or landmarks corresponding to 20% of 200</span></li></ul><ul class="c3 lst-kix_list_42-1"><li class="c15 li-bullet-0"><span class="c0">Time window : time of aligned map is corrected using median in a time window. </span></li></ul><ul class="c3 lst-kix_list_42-2 start"><li class="c24 li-bullet-0"><span class="c0">Window size/time interval: You have to provide the time interval. This time interval corresponds to the window size in which time median will be computed. </span></li><li class="c24 li-bullet-0"><span class="c0">Minimum number of landmarks: This allows you to specify the minimum number of landmarks a window must contain to compute a median on it, it is not significant to compute a median on less landmarks.</span></li><li class="c41 c92 li-bullet-0"><span class="c0">Sliding window overlap: overlap is used to compute the step to move the smoothing window forward to calculate a smoothing point for this new smoothing window. Overlap gives the percentage of overlapping between two consecutive windows. For example, if window size is 200 (seconds or landmarks depending on which smoothing method is selected) and overlap is 20%, the step forward = 200*((100-20)/100) = 160 seconds or landmarks, i.e. the smoothing window is moved forward by a step of 160, so two successive windows overlap each other by a step of 40 seconds or landmarks corresponding to 20% of 200.</span></li></ul><h2 class="c2" id="h.z3aaghl3hev6"><span class="c16">MoZ calibration</span></h2><p class="c26"><span class="c0">Delta moz (between experimental and theoretical moz) may change during a run. By specifying a moz calibration method, you can specify to use the recalibrated theoretical moz to cross assign instead of using the best feature moz. In both cases, a moz tolerance should be specified. see below </span></p><p class="c7 c26"><span class="c0"></span></p><h2 class="c2"><span class="c16">Cross Assignment</span></h2><p class="c79 c77"><span class="c0">Proline uses a hybrid approach to retrieve intensity values for ions that were not identified. As indicated above, identified and quantified features are obtained by detecting chromatographic peaks in raw files without a-priori, using an identification-based deisotoping method. During the PSM assignment step, the identification data provides the monoisotopic mass and the charge state for the ion, guiding the deisotoping procedure to group together the detected chromatographic peaks. These grouped peaks are then removed from the list of peaks to be assigned, thereby reducing the data density when annotating subsequent chromatographic peaks during the cross-assignment step. Ions that were not identified in a run can then be sought out in this restricted list of detected peaks using their m/z and RT coordinates. The m/z value is the theoretical m/z value obtained following identification of this ion in another run. The RT value is predicted from the apex RT of the feature detected in the run providing the highest identification score. This RT prediction computation may involve the use of one or two alignment functions. Using these two coordinates, associated with user-defined m/z and RT tolerances, the algorithm seeks a corresponding signal among the chromatographic peaks that have not already been assigned to an identified ion. To avoid the propagation of erroneous cross-assignments between runs, an additional but optional control (named &ldquo;use only confident features&rdquo;) is applied to ensure that this peak is the monoisotope of a peptide ion with a charge state identical to the master feature one. This is done by fitting its observed isotope pattern to a theoretical one.</span></p><h3 class="c2" id="h.3q5sasy"><span class="c36 c48">Parameters </span></h3><ul class="c3 lst-kix_list_64-0 start"><li class="c32 c72 li-bullet-0"><span class="c0">allow cross assignment between all runs or only between runs of the same groups: the user can completely disable the cross assignment or can control runs between which cross assignment is applied: </span></li></ul><ul class="c3 lst-kix_list_64-1 start"><li class="c15 li-bullet-0"><span class="c0">Between all runs</span></li><li class="c44 li-bullet-0"><span>Within groups only: cross assignment is applied only between runs belonging to the same group.</span></li></ul><h4 class="c2" id="h.25b2l0r"><span class="c36 c55 c18">Feature mapping</span></h4><ul class="c3 lst-kix_list_64-0"><li class="c27 li-bullet-0"><span class="c0">m/z tolerance (ppm or dalton): when mapping features from two different maps of the map set, the delta m/z between features must be lower than the m/z tolerance to be considered as the same feature seen on two different maps.</span></li></ul><ul class="c3 lst-kix_list_117-0 start"><li class="c5 li-bullet-0"><span class="c0">Time/RT tolerance (seconds): when mapping features from two different maps of the map set, delta time between features must be lower than the time tolerance to be considered as the same feature seen on two different maps.</span></li></ul><h4 class="c2" id="h.kgcv8k"><span class="c36 c55 c18">Filtering/Correction</span></h4><ul class="c3 lst-kix_list_105-0 start"><li class="c47 c77 c32 li-bullet-0"><span class="c0">Use only confident features: is applied to ensure that the cross-assigned peak is the monoisotope of a peptide ion with a charge state identical to the master feature one. This is done by fitting its observed isotope pattern to a theoretical one. </span></li></ul><ul class="c3 lst-kix_list_84-0 start"><li class="c104 c77 c32 li-bullet-0"><span class="c0">Master feature intensity filter (optional): a filter can be applied to the map features to keep the best features (above threshold) to build the master map. </span></li></ul><ul class="c3 lst-kix_list_84-1 start"><li class="c104 c53 li-bullet-0"><span>Two methods are available to filter features: the filter can be applied directly on intensity values (</span><span class="c23">Intensity method</span><span>) or it can be a proportion of the map median intensity (</span><span class="c23">Relative intensity method</span><span class="c0">). </span></li><li class="c104 c53 li-bullet-0"><span>intensity threshold/value: this provides the threshold for the filtering method. Only features above or below (depending on the </span><span class="c23">operator</span><span class="c0">) this threshold are considered for the master map building process. </span></li></ul><p class="c41"><span class="c0">If you choose Relative intensity for master feature filter type, the only possibility you have is percent, so features which intensities are beyond the relative intensity threshold in percentage of the median intensity are removed. If you choose Intensity for master feature filter type, you also have only one possibility at the moment of the intensity method: basic. Features which intensities are beyond the intensity threshold are removed and not considered for the master map building process. </span></p><h2 class="c2" id="h.34g0dwd"><span>Normalizing LC-MS maps</span></h2><p class="c6"><span class="c1">The comparison of LC-MS maps is confronted to another problem which is the variability of the MS signals measured by the instrument. This variability can be technical or biological. Technical variations between MS signals in two analyses can depend on the injected quantity of material, the reproducibility of the instrument configuration and also the software used for the signal processing. The observed systematic biases on the intensity measurements between two successive and similar analysis are mainly due to errors in the total amount of injected material in each case, or the LC-MS system instabilities that can cause variable performances during a series of analysis and thus a different response in MS signal for peptides having the same abundance. Data may not be used if the difference is too important. It is always recommended to do a quality control of the acquisition before considering any computational analysis. However, there are always biases in any analytic measurement but they can usually be fixed by normalizing the signals. Numerous normalization methods have been developed, each of them using a different mathematical approach (Christin, Bischoff et al. 2011). Methods are usually split in two categories, linear and non-linear calculation methods, and it has been demonstrated that linear methods can fix most of the biases (Callister, Barry et al. 2006). Three different linear methods have been implemented in Proline by calculating normalization factors as the ratio of the sum of the intensities, as the ratio of the median of the intensities, or as the ratio of the median of the intensities. </span></p><h3 class="c2"><span>Sum of the intensities</span></h3><p class="c6"><span class="c1">How to c</span><span>alculate</span><span class="c0">&nbsp;this factor: </span></p><ol class="c3 lst-kix_list_149-0 start" start="1"><li class="c27 li-bullet-0"><span class="c0">For each map, sum the intensities of the features</span></li></ol><ol class="c3 lst-kix_list_99-0 start" start="2"><li class="c5 li-bullet-0"><span class="c0">The reference map is the median map</span></li></ol><ol class="c3 lst-kix_list_100-0 start" start="3"><li class="c41 c32 li-bullet-0"><span class="c0">The normalization factor of a map = sum of the intensities of the reference map / sum of the intensities of the map</span></li></ol><h3 class="c2"><span>Median of the intensities</span></h3><p class="c6"><span>How to calculate this factor: </span></p><ol class="c3 lst-kix_list_101-0 start" start="1"><li class="c27 li-bullet-0"><span class="c0">For each map, calculate the median of the intensities in the map</span></li></ol><ol class="c3 lst-kix_list_20-0 start" start="2"><li class="c5 li-bullet-0"><span class="c0">The reference map is the median map</span></li></ol><ol class="c3 lst-kix_list_21-0 start" start="3"><li class="c41 c32 li-bullet-0"><span class="c0">The normalization factor of a map = median of the intensities of the reference map / median of the intensities of the map</span></li></ol><h3 class="c2"><span>Median of ratios</span></h3><p class="c6"><span class="c1">This last strategy </span><span>was</span><span class="c1">&nbsp;published in 2006 (Dieterle, Ross et al. 2006) and gives the best results. It consists </span><span>in</span><span class="c1">&nbsp;calculating the intensity ratios between two maps to be compared then set the normalization factor as the inverse value of the median of these ratios (cf. figure </span><span>below</span><span class="c0">). The procedure is the following: </span></p><ol class="c3 lst-kix_list_28-0 start" start="1"><li class="c27 li-bullet-0"><span class="c0">For each map in a &ldquo;map set&rdquo;, sum the intensities of the features</span></li></ol><ol class="c3 lst-kix_list_16-0 start" start="2"><li class="c5 li-bullet-0"><span class="c0">The reference map is the median map</span></li></ol><ol class="c3 lst-kix_list_138-0 start" start="3"><li class="c5 li-bullet-0"><span class="c0">For each feature of the master map, ratio = intensity of the feature in the reference map / intensity of the feature for this map</span></li></ol><ol class="c3 lst-kix_list_95-0 start" start="4"><li class="c41 c32 li-bullet-0"><span class="c0">Normalization factor = median of these ratios</span></li></ol><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 255.74px; height: 194.97px;"><img alt="" src="images/image15.jpg" style="width: 255.74px; height: 194.97px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1 c29 c23">Distribution of the ratios transformed in log2 and calculated with the intensities of features observed in two LC-MS maps. The red line representing the median is slightly off-centered. The normalization factor is equal to the inverse of this median value. The normalization process will refocus the ratio distribution on 0 which is represented by the black arrow</span></p><p class="c6"><span class="c0">Proline makes this normalization process for each match with the reference map and has a normalization factor for each map, independently of the choice of the algorithm. The normalization factor for the reference map is equal to 1. <br></span></p><h1 class="c2" id="h.1wmaarypjked"><span>Isobaric labeling quantitation workflow</span></h1><p class="c10"><span class="c0">Common isobaric labeling quantitation methods, such as TMT 6 to 18plex or iTRAQ 4 or 8plex, are supported in Proline. Adding a missing xPlex method can be easily done. This technique may be used to compare multiple samples in a single run. This is done by reading all reporters ion peak intensity each corresponding to a different sample. </span></p><p class="c10"><span>Data is red from the MS2 spectrum stored in mzDB file (see </span><span class="c4"><a class="c8" href="#id.o9t5nltnb7nt">available converters </a></span><span class="c0">paragraph). These quantitation results are stored in the Proline database and aggregated to define peptide ion abundances which are also stored in the Proline database, making them available for rapid data visualization and further post-processing.</span></p><p class="c10"><span>Finally, in the same way as for label free quantitation, peptide ion measurements can be </span><span class="c18">summarized as peptides and then protein abundances</span><span class="c0">&nbsp;using different computational methods. </span></p><p class="c10 c7"><span class="c0"></span></p><p class="c19"><span>The user can opt to perform additional operations such as excluding peptides or ions based on their characteristics (precursor intensity fraction, missed cleavages, variable modifications, sequence specificity, etc.). A purity correction matrix can also be provided to correct the reporter ion abundances. Indeed some manufacturers provide such &nbsp;matrices indicating the percentages of each reporter ion that have masses differing by +/- n Da from the nominal reporter ion mass due to isotopic variants. </span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.1jlao46"></a><h1 class="c2"><span>Post-processing of quantitative results</span></h1><p class="c26"><span class="c0">This procedure is used to compute peptide and protein abundances. It is available for LC-MS and isobaric labeling quantitative results. Several filters can also be set to increase the quality of quantitative results. </span></p><p class="c6"><span class="c1">Here is the description of the parameters that </span><span>can </span><span class="c0">be modified by the user. </span></p><h4 class="c2" id="h.yu4b1sf8p7dz"><span class="c36 c55 c18">PSM filters </span></h4><p class="c26"><span class="c18 c94">For isobaric labeling only</span></p><ul class="c3 lst-kix_9jhhktla6bpr-0 start"><li class="c20 li-bullet-0"><span>Discard PSM on Precursor Ion Fraction value. &ldquo;The PIF is defined as the fraction of ion current in the isolation window that is due to the targeted precursor ion and therefore ranges between 0 and 1. The PIF is determined for each targeted precursor ion and each tandem mass spectrum based on the closest full scan, which could be the previous or the consecutive one&rdquo; (</span><span class="c4 c82"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1021/pr101060v&amp;sa=D&amp;source=editors&amp;ust=1733226590823628&amp;usg=AOvVaw0quWQdkWqSGP-Mptu1s4MJ">J. Proteome Res. 2011, 10, 4, 1785&ndash;1793</a></span><span class="c0">). <br></span></li></ul><h4 class="c2" id="h.j4ai9q3ir8mz"><span class="c36 c55 c18">Purity correction matrix </span></h4><p class="c26"><span class="c18 c94">For isobaric labelling only</span></p><p class="c19"><span>In isobaric labelling quantitation, isobaric tags are used in samples to compare their abundances. Due to isotopic contamination in tags, reporter ion peaks will contribute to those of neighboring reporter ions. A purity correction matrix is provided with the labelling kit by the manufacturer. &nbsp;This matrix can be used to correct the abundance of the reporter ions .</span></p><h4 class="c2"><span>Peptide filters</span></h4><ul class="c3 lst-kix_list_86-0 start"><li class="c27 li-bullet-0"><span class="c18">Peptides selection method:</span><span class="c0">&nbsp;define which peptides are to be considered for quantitation.</span></li></ul><ul class="c3 lst-kix_list_86-1 start"><li class="c53 c47 li-bullet-0"><span class="c18">All peptides: </span><span>all peptides are taken into account at this step. Further filters may invalidate some peptides.</span></li><li class="c53 c47 li-bullet-0"><span class="c18">Specific peptides</span><span>: peptides shared between different protein sets are discarded for protein set abundance calculation and &nbsp;statistical analysis.</span></li><li class="c53 c47 li-bullet-0"><span class="c18">Razor and specific:</span><span>&nbsp;In addition to specific peptides, quantitation of shared peptides are considered for the &ldquo;best&rdquo; ProteinsSet. The best is actually based on the score.</span></li></ul><ul class="c3 lst-kix_list_11-0 start"><li class="c5 li-bullet-0"><span class="c18">Discard missed cleaved peptides</span><span class="c0">: if checked, peptides containing missed cleavages are discarded from the statistical analysis. It has to be noted that perfect tryptic peptides whose sequence is included in an observed missed cleaved peptide are also discarded if this option is enabled.</span></li></ul><ul class="c3 lst-kix_list_130-0 start"><li class="c5 li-bullet-0"><span class="c18">Discard modified peptides</span><span class="c0">: if checked, peptides containing specific modifications are discarded for protein set abundance calculation and &nbsp;the statistical analysis. </span></li></ul><ul class="c3 lst-kix_list_130-1 start"><li class="c15 li-bullet-0"><span class="c0">only those modified peptides or also non-modified peptides whose sequence is the same as these peptides may be &nbsp;discarded depending on the chosen filtering method.</span></li></ul><h4 class="c2" id="h.80smys4yzk8a"><span class="c36 c55 c18">Peptide and protein common parameters</span></h4><ul class="c3 lst-kix_list_127-0 start"><li class="c69 c32 li-bullet-0"><span>Normalization: the normalization factors are computed as the median of the ratio distributions between each run and a run of reference. A similar procedure is used for the </span><span class="c42"><a class="c8" href="#h.34g0dwd">normalization of LC-MS features.</a></span><span><br></span></li></ul><h4 class="c2" id="h.ofg49vwsicv2"><span class="c36 c55 c18">Summarize PSM (reporter ions) into peptide ions abundance</span></h4><p class="c26"><span class="c18 c94">For isobaric labelling only</span><span class="c13 c29 c60">.</span></p><p class="c26"><span>To calculate peptide ions abundance, associated reporter ions abundances can be </span><span class="c18">summed </span><span>or the </span><span class="c18">median </span><span class="c0">can be computed. <br></span></p><h4 class="c2" id="h.43ky6rz"><span class="c36 c55 c18">Summarize peptides ions into peptide abundance</span></h4><p class="c26"><span class="c0">To calculate peptide abundance, associated peptide ion abundances can be summed or the best ion is used. &nbsp;To choose the best peptide ion following rules are defined : </span></p><ul class="c3 lst-kix_list_78-0 start"><li class="c46 c32 li-bullet-0"><span>Select the peptide ions identified and quantified in the maximum of quant channels(runs)</span></li><li class="c46 c32 li-bullet-0"><span>If more than one, select peptide ions with the maximum of PSMs &nbsp;count</span></li><li class="c20 li-bullet-0"><span>If more than one, choose the peptide ion with the higher abundance.<br></span></li></ul><h4 class="c2"><span>Summarize peptides abundances into protein abundance</span></h4><p class="c6"><span class="c0">Peptide abundances can be summarized into protein abundances using several mathematical methods: </span></p><ul class="c3 lst-kix_list_34-0 start"><li class="c27 li-bullet-0"><span class="c18">sum</span><span class="c0">: for each quantitative channel (raw file) the sum of observed peptides abundances is computed</span></li></ul><ul class="c3 lst-kix_list_129-0 start"><li class="c5 li-bullet-0"><span class="c18">mean</span><span class="c0">: for each quantitative channel (raw file) the mean of observed peptides abundances is computed</span></li></ul><ul class="c3 lst-kix_list_36-0 start"><li class="c5 li-bullet-0"><span class="c18">mean of TOP3</span><span class="c0">&nbsp;peptides: same procedure but applied on the 3 most abundant peptides. Peptides are sorted by descending median abundances (computed across all compared samples for peptide). Then the 3 first peptides are kept.</span></li></ul><ul class="c3 lst-kix_list_35-0 start"><li class="c5 li-bullet-0"><span class="c18">median</span><span class="c0">: for each quantitative channel, the median of observed peptides abundances is computed</span></li></ul><ul class="c3 lst-kix_list_143-0 start"><li class="c5 li-bullet-0"><span class="c18">median profile</span><span>: a matrix of peptide abundance ratios is first computed (rows correspond to peptides and columns to quantitative channels). The median of these ratios is then computed for each column. The relative values are then converted back into absolute values using a scaling factor. This factor is computed as the maximum value from the </span><span class="c18">means of TOP3 peptides</span><span class="c0">&nbsp;abundances.</span></li></ul><ul class="c3 lst-kix_list_141-0 start"><li class="c5 li-bullet-0"><span class="c18">normalized median profile</span><span>: matrix of peptide abundance ratios is first computed (rows correspond to peptides and columns to quantitative channels). This matrix is then normalized and then summarized using the median method described above. The obtained median abundances are then adjusted by using a scaling factor. This factor is computed as the maximum value from the </span><span class="c18">means of TOP3 peptides</span><span class="c0">&nbsp;abundances.</span></li><li class="c5 li-bullet-0"><span class="c18">Median Ratio Fitting</span><span>: MRF computes a matrix of abundance ratios calculated between any two runs from ion abundances for each protein. For each pair-wise ratio, the median of the ion ratios is then calculated and used to represent the protein ratio between these two runs. A least-squares regression is then performed to approximate the relative abundance of the protein in each run in the dataset. This abundance is finally rescaled to the sum of the ion abundances across runs.</span></li></ul><p class="c89 c77 c7"><span class="c0 c45"></span></p><h1 class="c72 c78" id="h.2iq8gzs"><span>A</span><a id="id.xvir7l"></a><span class="c12">ggregation of quantitative results</span></h1><p class="c89 c77 c7"><span class="c0 c45"></span></p><p class="c19"><span class="c0">Two or more label free quantitations can be combined such that an ion quantified in multiple aggregated quantitations is represented only once in the aggregation result. The abundance of this ion is a combination of its abundance measured in the different aggregated quantitations. This could be useful to combine for example quantitation of fractions into a single quantitation result. <br>For Isobaric labelling quantitations, the same process is used but combining reporter ions abundances. These combined PSMs will then be summarized to get peptide ions abundances.</span></p><p class="c19"><span class="c0">The experimental design of the aggregation is based on the experimental design of the aggregated quantitation: the number of group/condition and the number of replicates per condition remains the same. However, the user can modify the correspondence between the groups and replicates if needed. In the following example the abundance of ion1 in the aggregated quantitation (in green) is based on the quantitation of this same ion in &quot;Fraction 1&quot; and &quot;Fraction 2&quot;. Since ion2 is quantified only in &quot;Fraction 1&quot;, its abundance values in the aggregated quantitation are the same as the abundances measured in &quot;Fraction 1&quot;.</span></p><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 314.66px; height: 274.90px;"><img alt="" src="images/image16.png" style="width: 314.66px; height: 274.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c19"><span class="c0">In this simple example, the correspondence between experimental designs is such that the abundance of ion1 in the replicate Repl1 is based on the measured abundance of ion1 in Repl1&rsquo; in &ldquo;Fraction 1&rdquo; and Repl1&rsquo;&rsquo; in &ldquo;Fraction 2&rdquo;. This could be modified by the user to take into account differences in the replicates order in aggregated quantitations or to account for the absence of a replicate (see for example replicate 3 in &ldquo;Fraction 1&rdquo;).</span></p><p class="c19"><span class="c0">In the current version, the abundance at the aggregation level is the sum of the abundances in aggregated quantitations. </span></p><p class="c26"><span class="c0">&nbsp; </span></p><p class="c89 c77 c7"><span class="c0 c45"></span></p><a id="id.3hv69ve"></a><h1 class="c2"><span>Identification Summary Export</span></h1><p class="c6"><span class="c0">When exporting a whole Identification Summary in an excel file, the following sheets may be generated: </span></p><ul class="c3 lst-kix_list_147-0 start"><li class="c27 li-bullet-0"><span class="c23">Search settings and info</span><span class="c0">&nbsp;: Contains information on project and search settings parameters</span></li></ul><ul class="c3 lst-kix_list_126-0 start"><li class="c5 li-bullet-0"><span class="c23">Import and filters</span><span class="c0">&nbsp;: Summary of used parameters during import, filtering and validation process</span></li></ul><ul class="c3 lst-kix_list_136-0 start"><li class="c5 li-bullet-0"><span class="c23">Protein sets</span><span>&nbsp;: List of all Protein Sets, valid or invalidated (configurable in custom option) during </span><span class="c42 c74"><a class="c8" href="#id.4ekz59m">Protein Sets Filtering</a></span><span class="c0">. Some columns description :</span></li></ul><ul class="c3 lst-kix_list_120-1 start"><li class="c15 li-bullet-0"><span class="c23">#sequences</span><span>&nbsp;(</span><span class="c23">#specific sequences</span><span class="c0">&nbsp;) : number of different peptide sequences identifying the Protein Set (specific : which does not identify any other valid Protein Set )</span></li></ul><ul class="c3 lst-kix_list_118-1 start"><li class="c15 li-bullet-0"><span class="c23">#peptides</span><span>&nbsp;(</span><span class="c23">#peptides</span><span class="c0">) : number of different peptide (sequence + PTM) identifying the Protein Set (specific : which does not identify any other valid Protein Set )</span></li></ul><ul class="c3 lst-kix_list_113-1 start"><li class="c15 li-bullet-0"><span class="c23">#spectral_count</span><span>&nbsp;(</span><span class="c23">specific_spectral_count</span><span class="c0">) : number of different peptide spectrum matches identifying the Protein Set (specific : which does not identify any other valid Protein Set )</span></li></ul><ul class="c3 lst-kix_list_122-0 start"><li class="c5 li-bullet-0"><span class="c23">Protein matches in protein set</span><span class="c0">&nbsp;: list of Protein Matches in each Protein Set. A same Protein Match could thus appear a few times if it belongs to different Protein Sets. (same column as protein set)</span></li><li class="c5 li-bullet-0"><span class="c23">Best PSM from protein sets</span><span class="c0">&nbsp;: List of best peptide spectrum matches (a single PSM per peptide is listed) for each Protein Set. Some columns description :</span></li></ul><ul class="c3 lst-kix_list_98-1 start"><li class="c15 li-bullet-0"><span class="c23">#psm_prot_sets</span><span class="c0">&nbsp;: number of Valid Protein Sets identified by this PSM.</span></li></ul><ul class="c3 lst-kix_list_97-1 start"><li class="c15 li-bullet-0"><span class="c23">#psm_prot_matches</span><span class="c0">&nbsp;: number of Protein Match, which belong to at least 1 valid Protein Set, identified by this PSM.</span></li></ul><ul class="c3 lst-kix_list_69-1 start"><li class="c15 li-bullet-0"><span class="c0">#psm_db_prot_matches : number of Protein Match, validated or not, identified by this PSM. This is equivalent to the number of proteins in fasta files containing the PSM.</span></li></ul><ul class="c3 lst-kix_list_2-0 start"><li class="c5 li-bullet-0"><span class="c23">All PSMs from protein sets</span><span class="c0">&nbsp;: List of all peptide spectrum matches for each Protein Set. (same column as best PSM from protein sets)</span></li><li class="c5 li-bullet-0"><span>PTM Cluster (filled only if &ldquo;identify Modifications sites&rdquo; has been run): All clusters (protein-set of localised ptms) identified in this identification summary. </span></li></ul><ul class="c3 lst-kix_list_142-0 start"><li class="c32 c41 li-bullet-0"><span class="c23">Dataset statistics</span><span class="c0">&nbsp;: Some statistical values for the exported Identification Summary : number of Protein Set, modified peptides &hellip; </span></li></ul><a id="id.vtr0o7bk97h1"></a><h1 class="c2" id="h.ytz0x72ixnrv"><span class="c12">Advanced features</span></h1><a id="id.799oj238ld40"></a><h2 class="c2" id="h.g7dmgsd1ute6"><span class="c16">Allowing &nbsp;multiple imports</span></h2><p class="c19"><span class="c0">By default, Proline does not allow importing multiple results files at the same time. This was due to the obsolete Peptide database which has been removed in Proline version 2.0. Nevertheless, if multiple results files could now be imported at the same time there is still a restriction at project level. &nbsp;Indeed, result files should be inserted sequentially in the same project. In addition, depending on Proline Server configuration, there could not be as many import threads as wanted or as the number of &nbsp;projects. </span></p><p class="c19"><span class="c0">This is why a configuration allows the user (administrator) to specify groups of projects which will belong to the same &ldquo;import thread&rdquo;. For instance, if Proline Server enables 3 threads for import services, you can specify 2 groups of projects:</span></p><ul class="c3 lst-kix_list_39-0 start"><li class="c5 li-bullet-0"><span>Import Thread 1 : project 1, project 3, project 5</span></li><li class="c5 li-bullet-0"><span class="c0">Import Thread 2 : project 6, project 7</span></li><li class="c19 c32 li-bullet-0"><span class="c0">Import Thread 3 : all other project (and new projects)</span></li></ul><p class="c19"><span class="c0">This means that for projects 1, 3 and 5 imports will be sequential but in parallel of import to project 6 or 7&hellip;</span></p><p class="c19"><span class="c0">If you specify more groups than allowed thread, some project groups will be grouped into a single one.</span></p><p class="c19"><span>To define a project group, you should add the following &nbsp;string to </span><span class="c23">project.serialized_properties</span><span class="c0">&nbsp;in uds_db database. Currently this could be done only with PGAdmin (or any database server administration tool) :</span></p><p class="c19"><span class="c0">{&ldquo;import_group&rdquo;:&rdquo;1&rdquo;} </span></p><p class="c19"><span>In the following example, the user has defined 3 specific groups, and the default group is always defined for all other projects</span><span class="c29 c55 c60 c116">. </span></p><p class="c19"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 693.00px; height: 129.88px;"><img alt="" src="images/image17.png" style="width: 693.00px; height: 171.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><hr><p class="c19 c7"><span class="c0"></span></p><hr style="page-break-before:always;display:none;"><p class="c19 c7"><span class="c0"></span></p><a id="id.o9t5nltnb7nt"></a><p class="c58 title" id="h.x5dq3q88n2vl"><span class="c29 c18 c31">Available converters to mzDB</span></p><a id="id.1x0gk37"></a><h1 class="c2" id="h.4h042r0"><span class="c12">Raw2mzdb</span></h1><h2 class="c2" id="h.2w5ecyt"><span>Raw file conversion to mzDB</span></h2><p class="c10"><span>This </span><span class="c4"><a class="c8" href="https://www.google.com/url?q=https://github.com/mzdb/pwiz-mzdb&amp;sa=D&amp;source=editors&amp;ust=1733226590830829&amp;usg=AOvVaw1yfEfMhLmmJ9XJFso3_T_4">converter</a></span><span>, named </span><span class="c4"><a class="c8" href="#id.1x0gk37">raw2mzdb</a></span><span class="c0">&nbsp;is based on ProteoWizard, ensuring compatibility with a wide range of instrument vendors. </span></p><h3 class="c2" id="h.1baon6m"><span class="c36 c48">Installation</span></h3><ol class="c3 lst-kix_list_10-3 start" start="1"><li class="c46 c32 li-bullet-0"><span class="c0">get the zip archive on Proline download section</span></li></ol><ol class="c3 lst-kix_list_10-0 start" start="1"><li class="c46 c32 li-bullet-0"><span>install of MSFileReader from Thermo ( </span><span class="c4"><a class="c8" href="https://www.google.com/url?q=http://sjsupport.thermofinnigan.com/public/detail.asp?id%3D703&amp;sa=D&amp;source=editors&amp;ust=1733226590831546&amp;usg=AOvVaw05QUuR1V76lIWd4NNvOiuk">here</a></span><span class="c0">, will install all necessary c++ redistribuables)</span></li><li class="c20 li-bullet-0"><span class="c0">ensure your regional settings parameters are &#39;.&#39; for the decimal symbol and &#39;,&#39; for the list separator</span></li></ol><h3 class="c2" id="h.3vac5uf"><span class="c36 c48">Use case procedure</span></h3><p class="c26"><span class="c0">Open a command line window in the directory containing raw2mzdb.exe</span></p><p class="c26"><span class="c0">Enter: </span></p><p class="c26"><span class="c13 c29 c60">raw2mzdb.exe -i &lt;rawfilename&gt; -o &lt;outputfilename&gt;</span></p><p class="c19"><span>By default, the raw file will be converted in the &ldquo;fitted&rdquo; mode for the MS1 (MS2 is often in centroid mode and can not be converted in fitted mode). If the MS2 (or superior) are acquired in high resolution (i.e in profile mode), you could specify that you want to convert several MSs in the required mode: </span><span class="c18">raw2mzdb.exe -i &lt;rawfilename&gt; -o &lt;outputfilename&gt; -f 1-2</span><span class="c0">&nbsp;will try to convert MS1 to MS2 in fitted mode.</span></p><p class="c26"><span class="c0">There are two other available conversion modes:</span></p><ol class="c3 lst-kix_list_144-0 start" start="1"><li class="c46 c32 li-bullet-0"><span>&ldquo;</span><span class="c23">profile</span><span>&rdquo;, the command line is then: </span><span class="c18">raw2mzdb.exe -i &lt;rawfilename&gt; -o &lt;outputfilename&gt; -p 1</span><span class="c0">&nbsp;(means you want profile mode for MS1, others MS will be stored as they were stored in the raw file)</span></li><li class="c20 li-bullet-0"><span>&ldquo;</span><span class="c23">centroid</span><span>&rdquo; : </span><span class="c18">raw2mzdb.exe -i &lt;rawfilename&gt; -o &lt;outputfilename&gt; -c 1</span><span class="c0">&nbsp;(means you want centroid mode for MS1, others MS will be stored as they were stored in the raw file)</span></li></ol><p class="c26 c7"><span class="c0"></span></p><h1 class="c2" id="h.s3r6nftf1kgr"><span>mzDBConverter</span></h1><p class="c26"><span class="c0">New converter based on manufacturer libraries. It currently supports Thermo raw files and Bruker TimsTof files. </span></p><p class="c26"><span>TO COMPLETE</span><sup><a href="#cmnt1" id="cmnt_ref1">[a]</a></sup><span><br></span><hr style="page-break-before:always;display:none;"></p><a id="id.2afmg28"></a><p class="c58 title" id="h.pkwqa1"><span class="c29 c70 c18 c60">Proline Studio</span></p><h1 class="c2"><span>How to</span></h1><a id="id.39kk8xu"></a><p class="c6"><span class="c1">Note: Read the </span><span class="c4 c1"><a class="c8" href="#id.1fob9te">Concepts &amp; Principles documentation</a></span><span class="c0">&nbsp;to understand main concepts and algorithms used in Proline. </span></p><p class="c6 c7"><span class="c0"></span></p><ul class="c3 lst-kix_1yoisyi8wrza-0 start"><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.1opuj5n">List of Abbreviations</a></span></li></ul><p class="c77 c56 c7 c89"><span class="c0"></span></p><h3 class="c2"><span class="c36 c48">Creation/Deletion</span></h3><ul class="c3 lst-kix_36wys1csajrz-0 start"><li class="c27 li-bullet-0"><span class="c4"><a class="c8" href="#id.2nusc19">Open a session and access to my projects</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.1302m92">Create a new project</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.3mzq4wv">Create a Dataset</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.319y80a">Import a Search Result</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.4du1wux">Delete Data</a></span><span class="c0">&nbsp;</span></li><li class="c41 c32 li-bullet-0"><span class="c42 c97"><a class="c8" href="#id.2szc72q">Connection Management</a></span></li></ul><h3 class="c2"><span>Display</span></h3><ul class="c3 lst-kix_e77bf81k0na-0 start"><li class="c27 li-bullet-0"><span class="c4"><a class="c8" href="#id.184mhaj">Display MSQueries, Peptides/PSM or Proteins</a></span><span class="c51">&nbsp;</span><span>of a Search Result</span><span class="c37 c1">&nbsp;</span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.279ka65">Display MSQueries, PSM, Peptides, Protein Sets, PTM Protein Sites or Adjacency Matrices</a></span><span class="c51">&nbsp;</span><span class="c0">of an Identification Summary</span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.ak81bdcki9j0">Display Modification Sites or Clusters of a Modification Dataset</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.meukdy">Display Search Result &amp; Identification Summary Properties</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.36ei31r">Display Spectral Counts</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.1ljsd9k">Display Quantitation &nbsp;results</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.zu0gcz">Create and Save a User Window</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.45jfvxd">Frame Toolbars Functionalities</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.3jtnz0s">Filter tables</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.1yyy98l">Search tables</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.4iylrwe">Graphics : Scatter Plot / Histogram</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.1qoc8b1">Statistical Reports (MSDiag)</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.3oy7u29">MS Files (MzScope)</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#h.1smtxgf">General Settings</a></span></li></ul><h3 class="c2"><span>Save, import and export</span></h3><ul class="c3 lst-kix_rxdmkrnvx4u4-0 start"><li class="c27 li-bullet-0"><span class="c4"><a class="c8" href="#id.319y80a">Import Search Result file</a></span><span>&nbsp;( Mascot/OMSSA/X!Tandem/MzIdent/MaxQuant)</span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.338fx5o">Export data</a></span></li><li class="c41 c32 li-bullet-0"><span class="c4"><a class="c8" href="#id.1idq7dh">Generate Spectrum Matches</a></span></li></ul><h3 class="c2"><span>Algorithm and other operation</span></h3><ul class="c3 lst-kix_tm0xq8syd7t0-0 start"><li class="c27 li-bullet-0"><span class="c4"><a class="c8" href="#id.42ddq1a">Validate a Search Result</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.2hio093">Filter Protein Sets</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.wnyagw">Change Typical Protein of a Protein Set</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.3gnlt4p">Combine datasets</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.1vsw3ci">Data Analyzer</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.1nia2ey">Calculator</a></span></li><li class="c47 c77 c32 li-bullet-0"><span class="c4"><a class="c8" href="#id.47hxl2r">Update Spectrum using Peaklist software</a></span></li><li class="c47 c77 c32 li-bullet-0"><span class="c4"><a class="c8" href="#h.2wwbldi">Identify Modification Sites/Cluster</a></span></li><li class="c47 c32 c77 li-bullet-0"><span class="c4"><a class="c8" href="#id.6z6uo9u38ru">Annotate and Edit Modification Clusters</a></span></li><li class="c77 c32 c104 li-bullet-0"><span class="c4"><a class="c8" href="#id.1c1lvlb">Administration</a></span></li></ul><h3 class="c2"><span>Quantitation</span></h3><ul class="c3 lst-kix_list_46-0 start"><li class="c27 li-bullet-0"><span class="c4"><a class="c8" href="#id.2mn7vak">Spectral Count</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.3ls5o66">Label Free Quantitation</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#h.17lhjkd63lf3">Isobaric Labeling Quantitation</a></span></li><li class="c5 li-bullet-0"><span class="c4"><a class="c8" href="#id.2eclud0">Quantitations Aggregation</a></span></li><li class="c41 c32 li-bullet-0"><span class="c4"><a class="c8" href="#id.thw4kt">Compute Post Processing on abundances</a></span></li></ul><p class="c19 c7"><span class="c0"></span></p><p class="c19 c7"><span class="c0"></span></p><hr style="page-break-before:always;display:none;"><p class="c19 c7"><span class="c0"></span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.1opuj5n"></a><h1 class="c2" id="h.48pi1tg"><span class="c12">List of Abbreviations</span></h1><p class="c6"><span class="c18">Calc. Mass</span><span class="c0">: Calculated Mass </span></p><p class="c6"><span class="c18">Delta MoZ</span><span class="c0">: Delta Mass to Charge Ratio </span></p><p class="c6"><span class="c18">Exp. MoZ</span><span class="c0">: Experimental Mass to Charge Ratio </span></p><p class="c6"><span class="c18">Ion Parent Int.</span><span class="c0">: Ion Parent Intensity </span></p><p class="c6"><span class="c18">Missed Cl.</span><span class="c0">: Missed Cleavage </span></p><p class="c6"><span class="c18">Modification D. Mass:</span><span class="c0">&nbsp;Modification Delta Mass</span></p><p class="c6"><span class="c18">Modification Loc.: </span><span class="c0">Modification Location</span></p><p class="c6"><span class="c18">Next AA</span><span class="c0">: Next Amino-Acid </span></p><p class="c6"><span class="c18">Prev. AA</span><span class="c0">: Previous Amino-Acid </span></p><p class="c6"><span class="c18">Protein Loc.:</span><span class="c0">&nbsp;Protein Location of the Modification</span></p><p class="c6"><span class="c18">Protein S. Matches</span><span class="c0">: Protein Set Matches </span></p><p class="c6"><span class="c18">PSM</span><span class="c0">: Peptide Spectrum Match </span></p><p class="c6"><span class="c18">PTM</span><span class="c0">: Post Translational Modification </span></p><p class="c6"><span class="c18">PTM D. Mass:</span><span class="c0">&nbsp;PTM Delta Mass</span></p><p class="c6"><span class="c18">RT</span><span class="c0">: Retention Time </span></p><p class="c6"><span class="c18">SC</span><span class="c0">: Spectral Counting</span></p><p class="c6 c7"><span class="c0"></span></p><a id="id.2nusc19"></a><h1 class="c2"><span>Server Connection</span></h1><p class="c6"><span class="c0">When you start Proline Studio for the first time, the Server Connection Dialog is automatically displayed. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 276.00px; height: 218.03px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\login_dialog.png" src="images/image18.png" style="width: 670.51px; height: 744.02px; margin-left: -229.75px; margin-top: -238.55px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">You must fill the following fields: </span></p><p class="c6"><span class="c1">- </span><span class="c13">Server Host</span><span class="c1">: this information must be asked to your IT Administrator. It correspon</span><span>ds to the Proline server </span><span class="c23">name</span></p><p class="c6"><span class="c1">- </span><span class="c13">User</span><span class="c0">: your username (an account must have been previously created by the IT Administrator). </span></p><p class="c6"><span class="c1">- </span><span class="c13">Password</span><span class="c0">: password corresponding to your account (username). </span></p><p class="c6"><span class="c1">If the field &ldquo;Remember Password&rdquo; is checked, the password </span><span>is</span><span class="c1">&nbsp;saved for future use. Server connection dialog continues to open with Proline Studio, the user though does not need to fill in his password, unless the last one is changed after his last login. </span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.1302m92"></a><h1 class="c2"><span>Create a New Project</span></h1><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 222.68px; height: 234.90px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\addprojectpopupprolinev3.png" src="images/image19.png" style="width: 222.68px; height: 234.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c1">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 221.48px; height: 273.52px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\addprojectdialogprolinev3.png" src="images/image20.png" style="width: 221.48px; height: 273.52px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">To create a Projec</span><span>t, c</span><span class="c1">lick on </span><span class="c13">&ldquo;+&ldquo;</span><span class="c1">&nbsp;button at the right of the Project Combobox. The Add Project Dialog ope</span><span>ns.</span></p><p class="c6"><span class="c0">Fill the following fields: </span></p><p class="c6"><span class="c1">- </span><span class="c13">Name</span><span class="c0">: name of your project </span></p><p class="c6"><span class="c1">- </span><span class="c13">Description</span><span class="c0">: description of your project </span></p><p class="c6"><span class="c0">You can specify other people to share this new project with them. Then click on OK Button </span></p><p class="c6"><span class="c1">Creation of a Project can </span><span>take</span><span class="c0">&nbsp;a few seconds. During its creation, the Project is displayed grayed with a small hourglass over it. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 211.15px; height: 131.86px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\newprojectcreatedprolinev3.png" src="images/image21.png" style="width: 211.15px; height: 131.86px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.3mzq4wv"></a><h1 class="c2"><span>Create a Dataset</span></h1><p class="c6"><span>In the Identification tree, y</span><span class="c0">ou can create a Dataset to group your data</span></p><p class="c6"><span class="c0">To create a Dataset: </span></p><p class="c6"><span class="c0">- right click on Identifications or on a Dataset to display the popup. </span></p><p class="c6"><span class="c0">- click on the menu &ldquo;Add Dataset&hellip;&rdquo; </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 294.00px; height: 315.41px;"><img alt="" src="images/image22.png" style="width: 294.00px; height: 532.99px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">On the dialog opened: </span></p><p class="c6"><span class="c0">- fill the name of the Dataset </span></p><p class="c6"><span class="c0">- choose the type of the Dataset </span></p><p class="c6"><span class="c0">- optional: click on &ldquo;Create Multiple Datasets&rdquo; and select the number of datasets you want to create </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 311.03px; height: 192.22px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\addaggregatewindowprolinev3.png" src="images/image23.png" style="width: 376.06px; height: 266.41px; margin-left: -39.71px; margin-top: -60.95px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">Let&#39;s see the result of the creation of 3 datasets named &ldquo;Replicate&rdquo;: </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 199.03px; height: 185.65px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\addaggregateresultprolinev4.png" src="images/image24.png" style="width: 199.03px; height: 185.65px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c19 c7"><span class="c0"></span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.2250f4o"></a><h1 class="c2" id="h.haapch"><span class="c12">Create a Folder</span></h1><p class="c6"><span class="c0">In both Identification and Quantitation tree, you can create Folders to organize your data</span></p><p class="c6"><span class="c0">To create a Folder : </span></p><p class="c6"><span class="c0">- right click on Identifications, Quantitations or on a Folder to display the popup. </span></p><p class="c6"><span class="c0">- click on the menu &ldquo;Add Identification Folder&hellip;&rdquo; or &ldquo;Add Quantitation Folder&hellip;&rdquo;</span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 348.00px; height: 502.00px;"><img alt="" src="images/image5.png" style="width: 348.00px; height: 502.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><a id="id.319y80a"></a><h1 class="c2"><span class="c12">Import a Search Result</span></h1><p class="c26"><span>See </span><span class="c4"><a class="c8" href="#id.3dy6vkm">Concept &amp; Principle</a></span><span class="c0">&nbsp;section </span></p><h2 class="c2" id="h.1gf8i83"><span class="c16">Import Mascot/X!Tandem/OMSSA/MzIdentML</span></h2><p class="c6"><span class="c0">There are two possibilities to import Search Results: </span></p><p class="c6"><span class="c0">- import multiple Search Results in &ldquo;All Imported&rdquo; and put them later in different datasets. </span></p><p class="c6"><span class="c0">- import directly a Search Result in a dataset. </span></p><h3 class="c2" id="h.40ew0vw"><span class="c36 c48">Import in &quot;All Imported&quot;</span></h3><p class="c6"><span class="c0">To import in &ldquo;All Imported&rdquo;: </span></p><p class="c6"><span class="c0">- right click on &ldquo;All Imported&rdquo; to show the popup </span></p><p class="c6"><span class="c1">- click on the menu &ldquo;</span><span>Import</span><span class="c0">&nbsp;Search Result&hellip;&rdquo; </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 239.03px; height: 166.83px;"><img alt="" src="images/image6.png" style="width: 239.03px; height: 166.83px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><h3 class="c2" id="h.2fk6b3p"><span class="c36 c48">Import directly in a Dataset</span></h3><p class="c6"><span class="c0">It is possible to import Search Results directly in a Dataset. Even in this case, Search Results are &nbsp;available in &ldquo;All Imported&rdquo;. <br></span></p><table class="c106"><tr class="c65"><td class="c113" colspan="1" rowspan="1"><p class="c6"><span class="c0">To import a Search Result in a Dataset, right click on a dataset and then click on &ldquo;Import Search Result&hellip;&rdquo; menu. Same dialog and parameters as in &ldquo;Import in &ldquo;All Imported&rdquo;&rdquo; above will be displayed.</span></p></td><td class="c113" colspan="1" rowspan="1"><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 235.18px; height: 488.91px;"><img alt="" src="images/image7.png" style="width: 235.18px; height: 488.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr></table><p class="c6"><span class="c0">&nbsp;</span></p><h3 class="c2" id="h.upglbi"><span class="c36 c48">Import Search Result Parameters</span></h3><p class="c6"><span class="c0">In the Import Search Results Dialog: </span></p><p class="c6"><span class="c0">- select the file(s) you want to import thanks to the file button (the Parser will be automatically selected according to the type of file selected) </span></p><p class="c6"><span class="c0">- select the different parameters (see description below) </span></p><p class="c6"><span class="c0">- click on OK button </span></p><p class="c6"><span class="c13">Note 1</span><span class="c1">: You can only browse the files accessible from the server according to the configuration done by your IT Administrator. Ask him if your files are not reachable. (Look for </span><span>&ldquo;</span><span class="c1 c23">Setting up Mount-points</span><span class="c23">&rdquo;</span><span class="c1">&nbsp;paragraph in </span><span class="c4 c1"><a class="c8" href="#id.1fob9te">Installation &amp; Setup</a></span><span class="c0">&nbsp;page).</span></p><p class="c6"><span class="c13">Note </span><span class="c18">2</span><span class="c0">: Proline is able to import OMSSA files compressed with BZip2. </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c13 c42">Parameters description</span><span class="c13">:</span><span class="c0">&nbsp;</span></p><ul class="c3 lst-kix_list_32-0 start"><li class="c27 li-bullet-0"><span class="c0">Software Engine: the software which generated your Result File (this parameter will be automatically set when files are selected or you can select it)</span></li></ul><ul class="c3 lst-kix_list_31-0 start"><li class="c5 li-bullet-0"><span class="c0">Instrument: mass-spectrometer (with specific configuration) used for sample analysis</span></li><li class="c5 li-bullet-0"><span>Fragmentation Rule Set: The fragmentation rules specified in software. by clicking on </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 16.00px; height: 16.00px;"><img alt="" src="images/image8.png" style="width: 16.00px; height: 16.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>you could visualize all rules for a specific rule set. This is necessary to generate spectrum matches.</span></li></ul><ul class="c3 lst-kix_list_71-0 start"><li class="c5 li-bullet-0"><span class="c0">Peaklist Software: the software used for the peaklist creation (mandatory for LCMS XIC quantitation)</span></li></ul><ul class="c3 lst-kix_list_54-0 start"><li class="c5 li-bullet-0"><span class="c0">Decoy Strategy: The type of decoy search which was performed. </span></li></ul><ul class="c3 lst-kix_list_75-1 start"><li class="c15 li-bullet-0"><span class="c0">&ldquo;No Decoy&rdquo;: if the search was performed against a target database only.</span></li></ul><ul class="c3 lst-kix_list_50-1 start"><li class="c15 li-bullet-0"><span class="c0">&ldquo;Concatenated Decoy&rdquo;: if target and decoy sequences were merged into a single database.</span></li></ul><ul class="c3 lst-kix_list_63-1 start"><li class="c15 li-bullet-0"><span class="c0">&ldquo;Software Engine Decoy&rdquo;: if the decoy sequences were generated on-the-fly by your search engine.</span></li></ul><ul class="c3 lst-kix_list_40-1 start"><li class="c15 li-bullet-0"><span class="c0">Decoy Accession Regex: for concatenated searches only. Select the rule to apply for the discrimination of target and decoy protein matches.</span></li></ul><ul class="c3 lst-kix_list_52-0 start"><li class="c5 li-bullet-0"><span class="c0">Parser Parameters: according to your Software Engine, some extra-parameters are displayed:</span></li></ul><ul class="c3 lst-kix_list_30-1 start"><li class="c15 li-bullet-0"><span class="c0">Mascot:</span></li></ul><ul class="c3 lst-kix_list_41-2 start"><li class="c41 c92 li-bullet-0"><span>Subset Threshold: the percentage of score between a given protein match and the master protein match (superset). Protein matches with a relative score lower than <br> &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c33">Master_protein_score * (1-subset threshold)</span><span class="c33"><br></span><span class="c1">won&#39;t be imported. </span></li></ul><ul class="c3 lst-kix_list_61-1 start"><li class="c53 c101 li-bullet-0"><span class="c0">Omssa:</span></li></ul><ul class="c3 lst-kix_list_61-2 start"><li class="c24 li-bullet-0"><span class="c0">Usermods file path: an XML file containing the definitions for each user defined PTM used in the OMSSA search.</span></li><li class="c41 c92 li-bullet-0"><span>PTM Composition file path: a text file containing the chemical composition for each user defined PTM. This is required for PTMs not already imported in another Search Result. The format is the following: <br></span><span class="c33">PTM name=&lt;PTM composition&gt; (one per line). <br><br></span><span>Example:<br></span><span class="c33">Acetyl peptide N-term=H(-6) C(-7) O(-1)</span></li></ul><ul class="c3 lst-kix_list_61-1"><li class="c53 c101 li-bullet-0"><span class="c0">X!Tandem:</span></li></ul><ul class="c3 lst-kix_list_61-2 start"><li class="c75 li-bullet-0"><span>Protein parsing rule: Specify the rule (regular expression) to be used to retrieve &nbsp;protein accession from protein info. As an example, to use the Uniprot </span><span class="c23">Entry Name</span><span>&nbsp;as protein accession, use the following rule: &nbsp;</span><span class="c99">..\|[^|]*\|([^ ]*)</span></li></ul><p class="c101 c77 c7 c103"><span class="c0"></span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 378.29px; height: 535.76px;"><img alt="" src="images/image9.png" style="width: 378.29px; height: 535.76px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9 c7"><span class="c0"></span></p><p class="c6"><span class="c1">Importing a Search Result can take some time. While the import is not finished, the &ldquo;All Imported&rdquo; or </span><span>&ldquo;selected dataset&rdquo; </span><span class="c1">is shown grayed with an ho</span><span>ur</span><span class="c0">glass and you can follow the imports in the Tasks Log Window (Menu Window &gt; Tasks Log to show it). &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 613.03px; height: 137.06px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\importidentificationlogv3.png" src="images/image10.png" style="width: 613.03px; height: 137.06px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">To show all the Search Results imported, double click on &ldquo;All Imported&rdquo;, or right click to popup the contextual menu and select &ldquo;Display List&rdquo; </span></p><p class="c6"><span>If needed, f</span><span class="c0">rom the All Imported window, you can drag and drop one or multiple Search Result to an existing dataset. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 559.10px; height: 172.39px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\draganddropsearchresultprolinev3.png" src="images/image11.png" style="width: 559.10px; height: 172.39px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9 c7"><span class="c0"></span></p><a id="id.3ep43zb"></a><h2 class="c2" id="h.1tuee74"><span class="c16">Import MaxQuant result</span></h2><p class="c26"><span class="c0">To import a MaxQuant Search Result, right click on a dataset and then select &ldquo;Import MaxQuant&rdquo;</span></p><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 236.53px; height: 271.91px;"><img alt="" src="images/image12.png" style="width: 309.12px; height: 271.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c25 c7"><span class="c0"></span></p><p class="c6"><span class="c18">Note 1:</span><span class="c0">&nbsp;MaxQuant import will generate a dataset hierarchy with the result from the different acquisition.</span></p><p class="c19"><span class="c0">The following dialog will be displayed </span></p><p class="c19"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 466.23px; height: 397.88px;"><img alt="" src="images/image13.png" style="width: 466.23px; height: 397.88px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">- select the directory containing the files generated by MaxQuant. &nbsp;This folder should look like:<br> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;root_folder&gt;\mqpar.xml<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;root_folder&gt;\combined\txt\summary.txt<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;root_folder&gt;\combined\txt\proteinGroups.txt<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;root_folder&gt;\combined\txt\parameters.txt<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;root_folder&gt;\combined\txt\msmsScans.txt<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;root_folder&gt;\combined\txt\msms.txt<br></span></p><p class="c6"><span class="c0">- select the Instrument: mass-spectrometer used for sample analysis different parameters</span></p><p class="c6"><span class="c0">- specify, if needed, the regular expression to extract protein accessions from &nbsp;MaxQuant protein ids.</span></p><p class="c6"><span class="c0">- you can choose to import also quantitative data</span></p><p class="c6"><span class="c0">- click on OK button </span></p><p class="c6 c7"><span class="c0"></span></p><a id="id.4du1wux"></a><h1 class="c2"><span>Delete Data</span></h1><p class="c6"><span class="c1">You can delete Search Results, Identification Summaries and Datasets in the data tree. You can also delete XIC or Spectral Counts in the quantitation tree. </span></p><p class="c6"><span class="c18">Delete</span><span>&nbsp;the Datasets (identification or quantitation&hellip;) </span><span class="c18">from the tree view</span><span class="c0">&nbsp;(Search Result always accessible from &ldquo;All Imported&rdquo; view&hellip;).</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">There are two ways to delete data: use the contextual popup or drag and drop data to the Trash. </span></p><h2 class="c2"><span>Delete Data from the contextual popup</span></h2><p class="c6"><span class="c1">Select the data you want to delete, r</span><span>i</span><span class="c0">ght-click to open the contextual menu and click on delete menu. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 243.15px; height: 392.78px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\deletedatasetpopup.png" src="images/image14.png" style="width: 243.15px; height: 392.78px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">The selected data is put in the Trash. So it is possible to restore it while the Trash has not been emptied. </span></p><h2 class="c2"><span>Delete Data by Drag and Drop</span></h2><p class="c6"><span class="c0">Select the data you want to delete and drag it to the Trash. It is possible to restore data while the Trash has not been emptied </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 223.00px; height: 258.00px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\deletedatasetprolinedndprolinev3.png" src="images/image45.png" style="width: 223.00px; height: 258.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c2"><span>Empty the Trash</span></h2><p class="c6"><span class="c0">To empty the Trash, you have to Right click on it and select the &ldquo;Empty Trash&rdquo; menu. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 216.00px; height: 247.00px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\emptytrashprolinev3.png" src="images/image46.png" style="width: 216.00px; height: 247.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">A confirmation dialog is displayed and if accepted Dataset will be removed from the Trash. </span></p><p class="c6"><span class="c0">Search Results are not completely removed, you can retrieve them from the &ldquo;All Imported&rdquo; window.</span></p><h2 class="c2"><span>Delete a Project</span></h2><p class="c6"><span class="c1">It is not possible to delete a Project by yourself. If you need to do it, </span><span>ask your</span><span class="c0">&nbsp;IT Administrator.</span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.2szc72q"></a><h1 class="c2"><span>Connection Management</span></h1><p class="c6"><span class="c1">Once user is connected (see </span><span class="c42 c1 c74"><a class="c8" href="#id.2nusc19">Server Connection</a></span><span class="c0">), it is possible to: </span></p><ul class="c3 lst-kix_list_6-0 start"><li class="c69 c32 li-bullet-0"><span class="c0">Reconnect with a different login</span></li></ul><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 242.87px; height: 189.69px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\reconnect_dialog.png" src="images/image48.png" style="width: 242.87px; height: 189.69px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><ul class="c3 lst-kix_list_119-0 start"><li class="c32 c69 li-bullet-0"><span class="c0">Change password</span></li></ul><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 248.15px; height: 149.94px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\changepassword.png" src="images/image50.png" style="width: 248.15px; height: 149.94px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><a id="id.184mhaj"></a><h1 class="c2"><span>Display MS Queries, Peptides/PSM or Proteins of a Search Result</span></h1><p class="c26"><span class="c0">All information, validated or not, can be accessible from this menu. Indeed, Search Result contains all data imported from a result file without any validation consideration.</span></p><h2 class="c2"><span>Functionality Access</span></h2><p class="c6"><span class="c0">To display data of a Search Result: </span></p><p class="c6"><span class="c0">- right click on a Search Result </span></p><p class="c6"><span class="c0">- click on the menu &ldquo;Display Search Result &gt;&rdquo; and on the sub-menu &ldquo;MSQueries&rdquo; or &ldquo;PSM&rdquo; or &ldquo;Proteins&rdquo; </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 413.65px; height: 275.42px;"><img alt="" src="images/image52.png" style="width: 413.65px; height: 275.42px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c2"><span>MSQueries Window</span></h2><p class="c6"><span class="c1">If you click on MSQueries sub-menu, you obtain this window: </span></p><p class="c9 c95"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 707.59px; height: 326.90px;"><img alt="" src="images/image54.png" style="width: 707.59px; height: 326.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c13">Upper View</span><span class="c1">: list of MSQueries. &nbsp;Some </span><span>columns</span><span class="c1">&nbsp;may not be (correctly</span><span>) filled if the Peaklist software were not correctly specified during import. It is possible to change this information using &lsquo;</span><span class="c4"><a class="c8" href="#id.47hxl2r">Update Spectrum ..</a></span><span>&rsquo;</span></p><p class="c6"><span class="c13">Bottom Window</span><span class="c0">: list of all Peptides linked to the current selected MSQuery. </span></p><p class="c6"><span class="c1">Note: Abbreviations used are listed </span><span class="c4 c1"><a class="c8" href="#id.3hej1je">here</a></span><span class="c37 c1">&nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><a id="id.3s49zyc"></a><h2 class="c2"><span>PSMs (Peptides) Window</span></h2><p class="c6"><span class="c1">If you click on PSMs sub-menu, you obtain this window: </span></p><p class="c9 c120"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 669.41px; height: 432.48px;"><img alt="" src="images/image56.png" style="width: 669.41px; height: 432.48px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c13">Upper View</span><span class="c1">: list of all Pe</span><span>ptide </span><span class="c1">Spectrum M</span><span>atches </span></p><p class="c6"><span class="c13">Middle View</span><span class="c1">: Spectrum, Spectrum Error, </span><span>Spectrum Values </span><span class="c0">and Fragmentation Table of the selected PSM. If no annotation is displayed, you can generate Spectrum Matches by clicking on the according button </span></p><p class="c6"><span class="c13">Bottom Window</span><span class="c1">: list of all Proteins </span><span>identified by </span><span class="c0">the currently selected Peptide. </span></p><p class="c6"><span class="c1">Note: Abbreviations used are listed </span><span class="c4 c1"><a class="c8" href="#id.1opuj5n">here</a></span><span class="c37 c1">&nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span>Proteins Window</span></h2><p class="c6"><span class="c0">If you click on Proteins sub-menu, you obtain this window: </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 592.23px; height: 481.77px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\displaysearchresultproteindialogv3.png" src="images/image58.png" style="width: 592.23px; height: 481.77px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c13">Upper View</span><span class="c0">: list of all Proteins </span></p><p class="c6"><span class="c13">Bottom View</span><span class="c1">: list of all Peptides </span><span>identifying</span><span class="c0">&nbsp;the selected Protein. </span></p><p class="c6"><span class="c1">Note: Abbreviations used are listed </span><span class="c4 c1"><a class="c8" href="#id.1opuj5n">here</a></span><span class="c37 c1">&nbsp;</span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.279ka65"></a><h1 class="c2"><span>Display MS Queries, PSM, Peptides, Protein Sets or Adjacency Matrices of an Identification Summary</span></h1><p class="c26"><span class="c0">From this menu, all displayed information is Identification Summary data, which has been validated according to user specified rules. To view the raw information as defined at import, use the Search Result sub menu.</span></p><h2 class="c2"><span>Functionality Access</span></h2><p class="c6"><span class="c0">To display data of an Identification Summary: </span></p><p class="c6"><span class="c0">- right click on an Identification Summary </span></p><p class="c6"><span class="c1">- click on the menu &ldquo;Display Identification Summary &gt;&rdquo; and on the sub-menu &ldquo;MSQueries&rdquo;, &ldquo;PSM&rdquo;, &ldquo;Peptides&rdquo;, &ldquo;Protein Sets&rdquo;, &ldquo;PTM Protein Sites&rdquo; or &ldquo;Adjacency Matri</span><span>x</span><span class="c0">&rdquo; </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 337.09px; height: 260.07px;"><img alt="" src="images/image60.png" style="width: 337.09px; height: 260.07px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9 c7"><span class="c0"></span></p><h2 class="c2"><span>MSQueries Window</span></h2><p class="c6"><span class="c0">If you click on MSQueries sub-menu, you obtain this window: </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 596.26px; height: 316.09px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\msqueryrsm.png" src="images/image62.png" style="width: 596.26px; height: 316.09px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c18">Upper View</span><span class="c0">: list of MSQueries. </span></p><p class="c6"><span class="c18">Bottom Window</span><span>: list of all Peptides linked to the current selected MSQuery. </span></p><p class="c6"><span class="c1">Note: Abbreviations used are listed </span><span class="c4 c1"><a class="c8" href="#id.1opuj5n">here</a></span><span class="c37 c1">&nbsp;</span></p><p class="c6"><span class="c0">This view contains all MSQueries even if it doesn&rsquo;t bring an identification.</span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span>PSMs Window</span></h2><p class="c6"><span class="c0">If you click on PSM sub-menu, you obtain this window: </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 637.26px; height: 491.15px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\displayidspsmdialogv3.png" src="images/image27.png" style="width: 637.26px; height: 491.15px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">Note: Abbreviations used are listed </span><span class="c4 c1"><a class="c8" href="#id.1opuj5n">here</a></span><span class="c37 c1">&nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span>Peptides Window</span></h2><p class="c6"><span class="c0">If you click on Peptides sub-menu, you obtain this window: </span></p><p class="c9 c110"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 656.45px; height: 309.90px;"><img alt="" src="images/image28.png" style="width: 656.45px; height: 309.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c13">Upper View</span><span class="c1">: list of all Peptides </span><span>with</span><span class="c1">&nbsp;</span><span>best PSM information (charge, score ...)</span></p><p class="c6"><span class="c13">Middle View</span><span class="c1">: list of all Protein Sets </span><span>identified by </span><span class="c0">the selected peptide. </span></p><p class="c6"><span class="c13">Bottom Left View</span><span class="c0">: list of all Proteins of the selected Protein Set </span></p><p class="c6"><span class="c13">Bottom Right View</span><span class="c0">: list of all Peptides of the selected Protein </span></p><p class="c6"><span class="c1">Note: Abbreviations used are listed </span><span class="c4 c1"><a class="c8" href="#id.1opuj5n">here</a></span><span class="c37 c1">&nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span>Protein Sets Window</span></h2><p class="c6"><span class="c0">If you click on Protein Sets sub-menu, you obtain this window: </span></p><p class="c9 c108"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 684.58px; height: 408.69px;"><img alt="" src="images/image30.png" style="width: 684.58px; height: 408.69px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c13">View 1 (upp</span><span class="c18">er)</span><span class="c1">: list of all Protein Sets of th</span><span>e identification Summary</span></p><p class="c6"><span class="c1">Note: In the column </span><span class="c1 c23">Proteins</span><span class="c1">, 8 (2, 6) means that there are 8 proteins in the </span><span>p</span><span class="c1">rotein</span><span>&nbsp;s</span><span class="c1">et</span><span>&nbsp;:</span><span class="c0">&nbsp;2 in the sameset, 6 in the subset. </span></p><p class="c6"><span class="c13">View 2</span><span class="c1">: list of all Proteins of the selected Protein Set</span><span>, sameset or subset</span><span class="c0">. </span></p><p class="c6"><span class="c13">View 3</span><span class="c1">: list of all Peptides of the selected Protein</span><span>. If a subset is selected only peptides matching that protein will be listed.</span></p><p class="c6"><span class="c13">View 4a</span><span class="c1">: Protein Sequence of the previously selected Protein and Spectrum of the selected Peptide. Other tabs di</span><span class="c0">splay Spectrum, Spectrum Error and Fragmentation Table.</span></p><p class="c6"><span class="c18">View 4b: </span><span class="c0">Graphic representation of the Protein with matching peptide and associated modifications.</span></p><p class="c6"><span class="c1">Note: Abbreviations used are listed </span><span class="c4 c1"><a class="c8" href="#id.1opuj5n">here</a></span><span class="c1 c51">&nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span>Adjacency Matrix Window</span></h2><p class="c6"><span class="c1">If you click on Adjacency Matrix sub-menu, you obtain this window:</span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 370.96px; height: 548.90px;"><img alt="" src="images/image32.png" style="width: 370.96px; height: 548.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c13">View 1</span><span class="c1">: All the matrices. Each matrix </span><span>corresponds to</span><span class="c0">&nbsp;a cluster composed of linked Proteins/Peptides. </span></p><p class="c6"><span class="c0">Note: use the Search tool to display an Adjacency Matrix for a particular Protein or Peptide</span></p><p class="c6"><span class="c13">View 2</span><span class="c0">: The currently selected matrix. </span></p><p class="c6"><span class="c1">In the example, you can see two different protein sets</span><span>&nbsp;which</span><span class="c1">&nbsp;share only </span><span>two </span><span class="c0">peptides.</span></p><p class="c6"><span>Thanks to the settings you can hide proteins with exactly the same peptides.</span><span class="c0">&nbsp;</span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.ak81bdcki9j0"></a><h1 class="c2" id="h.jf37vrpng5s8"><span class="c12">Display Modification Site or Cluster of a Modification Dataset</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 268.74px; height: 158.67px;"><img alt="" src="images/image159.png" style="width: 268.74px; height: 158.67px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></h1><h2 class="c2" id="h.qfzhv2qj6act"><span class="c16">Functionality Access</span></h2><p class="c79 c77"><span class="c0">To display modification data associated to an Identification Summary:</span></p><ul class="c3 lst-kix_list_8-0 start"><li class="c30 li-bullet-0"><span>right click on the Dataset </span></li><li class="c50 c32 li-bullet-0"><span>click on the Display Modification &gt; &ldquo;Sites&rdquo;; &ldquo;Clusters&rdquo;; &ldquo;Annotated Clusters&rdquo;</span></li></ul><p class="c79 c77"><span>If you click on one of &ldquo;</span><span class="c23">Display Modifications&rdquo; </span><span class="c0">sub-menu, you can obtain the following warning dialog.</span></p><p class="c6"><span class="c0">This is due to the fact that you must run beforehand the &ldquo;Identify Modification Sites&rdquo; process. To do that, mouse right click on your Identification Summary and select the &ldquo;Identify Modification Sites&rdquo; menu.</span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 645.00px; height: 122.00px;"><img alt="" src="images/image35.png" style="width: 645.00px; height: 122.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">The Annotated Clusters menu may also cause the display of this warning. Indeed, Annotated Clusters display is only available after Clusters have been modified and saved.</span></p><p class="c6"><span>All displays, Sites and Clusters, are structured in the same way. In Sites windows, the upper view will list all individual Sites while in Clusters windows, Sites will be clustered using rules specified by user (see </span><span class="c4"><a class="c8" href="#h.111kx3o">Identify Modification Sites</a></span><span class="c0">)</span></p><p class="c77 c79"><span>&nbsp;</span><span class="c18 c74 c83">Clusters/Sites Windows</span></p><p class="c19"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 710.91px; height: 378.25px;"><img alt="" src="images/image37.png" style="width: 710.91px; height: 378.25px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c18">Upper View</span><span class="c0">: This view lists all Modification Sites or Clusters. In this example, a Cluster view is shown. For each of them, many information is given : the number of Peptides belonging to the Cluster, how many sites have been clustered, The Confidence for the site combination, ...</span></p><p class="c6"><span class="c18">Bottom Left view</span><span class="c0">: Graphic illustration of the cluster/site peptides mapped to the protein sequence. All modification locations are represented above the protein sequence. An overview graphic is also given below.</span></p><p class="c6"><span class="c18">Bottom Right view</span><span class="c0">: Cluster/Site peptides list. The PSMs list is accessible in a hidden tab.</span></p><p class="c9 c7 c88"><span class="c0"></span></p><a id="id.meukdy"></a><h1 class="c2"><span>Display Additional Information on Search Result/Identification Summary</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 169.23px; height: 331.51px;"><img alt="" src="images/image169.png" style="width: 169.23px; height: 331.51px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></h1><h2 class="c2"><span>Functionality Access</span></h2><p class="c6"><span class="c1">To display properties of a Search Result</span><span>&nbsp;or </span><span class="c0">Identification Summary: </span></p><ul class="c3 lst-kix_list_90-0 start"><li class="c72 c32 li-bullet-0"><span class="c1">right click on a Search Result/Identification Summary </span></li><li class="c49 c32 li-bullet-0"><span class="c1">click on the menu &ldquo;Properties&rdquo; </span></li></ul><p class="c6"><span class="c0">Note: it is possible to select multiple Search Results/Identification Summaries to compare the values. </span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span>Properties Window</span></h2><p class="c6"><span class="c0">Property window opened: </span></p><p class="c9 c7"><span class="c0"></span></p><p class="c6"><span class="c18">General Information: </span><span class="c0">Various information on the analysis (instrument name, peaklist software&hellip;)</span></p><p class="c6"><span class="c18">Search Properties: </span><span class="c0">Information extracted from the Result File (date, software version, search settings...)</span></p><p class="c6"><span class="c18">Search Result Information: </span><span class="c0">Amount of Queries, PSM and Proteins in the Search Result.</span></p><p class="c6"><span class="c18">Identification Summary Information:</span><span class="c0">&nbsp;Information obtained after validation process</span></p><p class="c6"><span class="c18">Validation xxx:</span><span class="c0">&nbsp;Information on validation process : parameters used to validate and result </span></p><p class="c6"><span class="c18">Sql Ids:</span><span class="c0">&nbsp;Database ids related to this item</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c19"><span class="c42">Note:</span><span class="c0">&nbsp;Identification Summary Number may differ from Validation Results. Indeed, on one hand, peptide matches count in Validation Results takes into account all PSMs that have been validated . On the other hand, the PSM Number in the &ldquo;Identification Summary Information&rdquo; section considers only PSMs that identify a valid Protein Sets.</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c19 c7"><span class="c0"></span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 605.77px; height: 614.93px;"><img alt="" src="images/image39.png" style="width: 605.77px; height: 614.93px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">Property window opened with multiple Identification summaries selected:</span></p><p class="c19"><span class="c0">The color of the type column indicates if the values are the same (white) or different (yellow)</span></p><p class="c19"><span>.</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 543.71px; height: 226.91px;"><img alt="" src="images/image41.png" style="width: 543.71px; height: 226.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><a id="id.36ei31r"></a><h1 class="c2"><span>Display a Spectral Count</span></h1><p class="c6"><span class="c0">You can display a generated Spectral Count by using the right mouse popup. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 384.03px; height: 222.83px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\displayscpopup.png" src="images/image89.png" style="width: 384.03px; height: 222.83px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">To have more details about the results, see </span><span class="c4 c1"><a class="c8" href="#h.11si5id">spectral_count_result</a></span><span class="c37 c1">&nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c9 c108"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 689.51px; height: 432.12px;"><img alt="" src="images/image90.png" style="width: 689.51px; height: 432.12px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">The overview is based by default on the weighted spectral count values. (Note: if you sort on the overview column, the sort is based on max (value-mean (values))/mean (values). So, you will obtain the most homogenous and confident rows first) </span></p><p class="c6"><span class="c1">For each compa</span><span>red</span><span class="c0">&nbsp;dataset, are displayed: </span></p><p class="c6"><span class="c0">- status ( typical, sameset, / ) </span></p><p class="c6"><span class="c0">- peptide numbers </span></p><p class="c6"><span class="c0">- the basic spectral count </span></p><p class="c6"><span class="c0">- the specific spectral count </span></p><p class="c6"><span class="c0">- the weighted spectral count </span></p><p class="c6"><span class="c0">- the selection level </span></p><p class="c6"><span>User can change the information displayed by the overview using the table settings icon ( </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 16.00px; height: 16.00px;"><img alt="" src="images/image92.png" style="width: 16.00px; height: 16.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>)</span><span class="c0">&nbsp;.</span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.1ljsd9k"></a><h1 class="c2"><span>Display a quantitation result</span></h1><p class="c6"><span class="c1">To display a </span><span>quantitation, other than spectral count</span><span class="c0">, right click on the selected node in the Quantitation tree, and select &ldquo;Display Abundances&rdquo;, and then the level you want to display: </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 344.00px; height: 143.20px;"><img alt="" src="images/image94.png" style="width: 344.00px; height: 143.20px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">Note: You can also display the identification summary used as reference for the quantitation from the popup menu in the quantitation tree:</span></p><p class="c59 c7"><span class="c0"></span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 482.14px; height: 229.53px;"><img alt="" src="images/image96.png" style="width: 482.14px; height: 229.53px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9 c7"><span class="c0"></span></p><h3 class="c2"><span>Display Protein Sets</span></h3><h4 class="c2"><span>Protein Sets</span></h4><p class="c6"><span class="c0">By clicking on &ldquo;Display Abundances&rdquo; / &ldquo;Protein Sets&rdquo;, you can see all quantified protein sets. For each quantified protein set, you can see below all peptides linked to the selected protein set and peptides Ions linked to the selected peptide. For each peptide Ion, you can see the different features and the graph of the peakels in each quantitation channel. </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c59"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 653.01px; height: 469.33px;"><img alt="" src="images/image98.png" style="width: 653.01px; height: 469.33px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">The overview is based by default on the </span><span>abundance</span><span class="c1">&nbsp;values. </span></p><p class="c6"><span class="c1">Note: if you sort on the overview column, the sort is based on max (value-mean (values))/mean (values). So, you obtain the most homogenous and confident rows first</span><span>.</span><span class="c0">&nbsp;</span></p><p class="c6"><span class="c0">For each quantitation channel, are displayed: </span></p><p class="c6"><span class="c0">- the raw abundance </span></p><p class="c6"><span class="c0">- the peptide match count </span></p><p class="c6"><span class="c0">- the abundance </span></p><p class="c6"><span class="c0">- the selection level </span></p><p class="c6"><span class="c1">By clicking on the </span><span>using the &ldquo;table setting&rdquo; icon </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 16.00px; height: 16.00px;"><img alt="" src="images/image92.png" style="width: 16.00px; height: 16.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>, </span><span class="c0">you can choose the information you want to display or change the overview. </span></p><p class="c6"><span class="c0">&nbsp;</span></p><h4 class="c2"><span>Peptides</span></h4><p class="c6"><span>The middle part of the window lists all peptides of the selected Protein set with the same kind of quantitative data. The status column indicates whether the peptide was used or not for protein set abundances. On the right, a</span><span class="c0">&nbsp;graph allows you to see the variations of the abundance (or raw abundance) of a peptide in the different quantitation channels.</span></p><p class="c6 c7"><span class="c0"></span></p><h4 class="c2"><span>Features and Peakels</span></h4><p class="c6"><span class="c1">You can see the different features in the different quantitation channels and the graph of the peakels: </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 600.13px; height: 127.08px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\xic_features.png" src="images/image101.png" style="width: 600.13px; height: 127.08px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">By clicking on </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 16.00px; height: 16.00px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\chart-arrow.png" src="images/image102.png" style="width: 16.00px; height: 16.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c0">you can display either: </span></p><p class="c6"><span class="c0">- the peaks of isotope 0 in all quantitation channels </span></p><p class="c6"><span class="c0">- all isotopes for the selected quantitation channel: </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 687.60px; height: 196.91px;"><img alt="" src="images/image103.png" style="width: 687.60px; height: 196.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">By clicking on </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 16.00px; height: 16.00px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\wave.png" src="images/image87.png" style="width: 16.00px; height: 16.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c1">&nbsp;you can see the chromatograms of the features and their first time scan and last time scan in mzScope. For more details see the </span><span class="c4 c1"><a class="c8" href="#id.3oy7u29">mzScope section</a></span><span class="c0">. </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">It is also possible to extract the corresponding chromatogram for one or all of the features.</span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 302.50px; height: 255.62px;"><img alt="" src="images/image67.png" style="width: 302.50px; height: 255.62px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c0">&nbsp;</span></p><p class="c6"><span class="c0">The resulting chromatograms will be displayed in the same windows as peakel.</span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 653.01px; height: 213.33px;"><img alt="" src="images/image70.png" style="width: 653.01px; height: 213.33px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><h3 class="c2"><span>Display Peptides</span></h3><p class="c6"><span class="c0">By clicking on &ldquo;Display Abundances&ldquo; / &ldquo;Peptides&rdquo;, you can see: </span></p><p class="c6"><span class="c0">- identified and quantified Peptides </span></p><p class="c6"><span class="c0">- non identified but quantified peptides </span></p><p class="c6"><span class="c0">- identified but not quantified peptides (linked to a quantified protein) </span></p><p class="c6 c112"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 699.59px; height: 253.91px;"><img alt="" src="images/image72.png" style="width: 699.59px; height: 253.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7 c84"><span class="c0"></span></p><p class="c6 c102"><span class="c0">The lower view lists all peptide ions (specific charge) of selected peptide. The status column indicates if the ion is valid or not and if it was used for peptide quantitation.</span></p><p class="c6 c7 c102"><span class="c0"></span></p><h3 class="c2"><span>Display Peptides Ions</span></h3><p class="c6"><span class="c0">By clicking on &ldquo;Display Abundances&rdquo; / &ldquo;Peptides Ions&rdquo;, you can see: </span></p><p class="c6"><span class="c0">- all identified and quantified Peptides Ions </span></p><p class="c6"><span class="c0">- non identified but quantified peptides Ions </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 653.01px; height: 272.00px;"><img alt="" src="images/image74.png" style="width: 653.01px; height: 272.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h3 class="c2 c90" id="h.9q5vc8a2r40"><span class="c36 c48"></span></h3><h3 class="c2" id="h.u15y8c1gscrd"><span>Display Modification Sites/Clusters</span></h3><p class="c6"><span>By clicking on &ldquo;Display Abundances&rdquo; / &ldquo;Modification XXX&rdquo; (Site, Clusters or Annotated Clusters), you can see the same Clusters/Sites information as in &ldquo;Identification Summary/Display / Modification XX&rdquo; , See </span><span class="c4"><a class="c8" href="#h.jf37vrpng5s8">Display Modification</a></span><span class="c0">&nbsp;description</span></p><p class="c6"><span class="c0">In addition, quantitative information is computed and displayed.</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 692.70px; height: 369.33px;"><img alt="" src="images/image76.png" style="width: 692.70px; height: 369.33px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c18">Upper View</span><span class="c0">: This view lists all Modification Sites or Clusters. For each of them, in addition to cluster information, quantitative data is given. &nbsp;Abundance is calculated using peptide&rsquo;s abundance sum.</span></p><p class="c6"><span class="c18">2nd view</span><span class="c0">: Cluster/Site peptides list. The PSMs list is accessible in a hidden tab. Peptide quantitative information are displayed the same way it is done in other quantitative views.</span></p><p class="c6"><span class="c18">3rd view</span><span class="c0">&nbsp;: Graphic illustration of the cluster/site peptides mapped to the protein sequence is shown on the left. &nbsp;On the right, a graph allows you to see the variations of the abundance (or raw abundance) of these peptides in the different quantitation channels.</span></p><p class="c6"><span class="c18">Bottom Right view</span><span>: This view is the same as the &ldquo;Peptide&rdquo; view in &ldquo;</span><span class="c4"><a class="c8" href="#id.1ljsd9k">Display Xic Protein Sets</a></span><span class="c0">&rdquo; above.</span></p><p class="c6 c7"><span class="c0"></span></p><h3 class="c2"><span>Display Experimental Design and Parameters</span></h3><p class="c6"><span class="c0">By clicking on &ldquo;Exp. Design &gt; Parameters&rdquo;, you can see the experimental design and the parameters of the selected XIC. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 322.80px; height: 331.36px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\displayxicdesign.png" src="images/image78.png" style="width: 322.80px; height: 331.36px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 324.65px; height: 245.00px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\displayxicparam.png" src="images/image80.png" style="width: 762.41px; height: 245.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">If you have launched the </span><span>&ldquo;compute post processing ...&rdquo; </span><span class="c1">on the XIC, you can also display the </span><span>corresponding </span><span class="c0">parameters. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 448.55px; height: 262.91px;"><img alt="" src="images/image82.png" style="width: 448.55px; height: 262.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9 c7"><span class="c0"></span></p><h3 class="c2"><span>Display Map Alignment</span></h3><p class="c6"><span class="c0">By clicking on &ldquo;Exp. Design &gt; Map Alignment&rdquo;, you can see the map of the variation of the alignment of the maps compared to the map alignment of the selected XIC. You can also calculate the predicted time in a map from an elution time in another map. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 604.80px; height: 464.07px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\displaymapalignment.png" src="images/image83.png" style="width: 604.80px; height: 464.07px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><a id="id.45jfvxd"></a><h1 class="c2" id="h.2koq656"><span class="c12">Frame Toolbars Functionalities</span></h1><p class="c26 c7"><span class="c0"></span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 545.74px; height: 392.90px;"><img alt="" src="images/image64.png" style="width: 545.74px; height: 392.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c18">A</span><span class="c0">: Display Decoy Data. </span></p><p class="c6"><span class="c18">B</span><span>: </span><span class="c4"><a class="c8" href="#id.1yyy98l">Search</a></span><span class="c51">&nbsp;</span><span class="c0">in the Table. (Using * and ? wild cards) </span></p><p class="c6"><span class="c18">C</span><span>: </span><span class="c4"><a class="c8" href="#id.3jtnz0s">Filter</a></span><span class="c51">&nbsp;</span><span class="c0">data displayed in the Table </span></p><p class="c6"><span class="c18">D</span><span class="c0">: Display settings dialog (you can modify displayed columns and perform double sorting)</span></p><p class="c6"><span class="c18">E</span><span>: </span><span class="c4"><a class="c8" href="#id.338fx5o">Export data</a></span><span class="c51">&nbsp;</span><span class="c0">displayed in the Table </span></p><p class="c6"><span class="c18">F</span><span>: Send to </span><span class="c4"><a class="c8" href="#id.1vsw3ci">Data Analyzer</a></span><span class="c0">&nbsp;to compare data from different views </span></p><p class="c6"><span class="c18">G</span><span>: Create a </span><span class="c4"><a class="c8" href="#id.4iylrwe">Graphic</a></span><span class="c51">&nbsp;</span><span class="c0">: histogram or scatter plot . Only on PSMs table</span></p><p class="c6"><span class="c18">H</span><span class="c0">: Display number of entities in the table (number of PSMs / Peptides / Proteins&hellip;)</span></p><p class="c6"><span class="c18">I</span><span class="c0">: Right click on the marker bar to display Line Numbers or add Annotations/Bookmarks </span></p><p class="c6"><span class="c18">J</span><span class="c0">: Export view as an image </span></p><p class="c6"><span class="c18">K</span><span class="c0">: Generate Spectrum Matches (specific to spectrum grahic)</span></p><p class="c6"><span class="c18">L</span><span class="c0">: Expands the frame to its maximum (other frames are hidden). Click again to undo.</span></p><p class="c6"><span class="c18">M</span><span class="c0">: Gather the frame with the previous one as a tab. </span></p><p class="c6"><span class="c18">N</span><span class="c0">: Split the last tab as a frame underneath </span></p><p class="c6"><span class="c18">O</span><span class="c0">: Remove the last Tab or Frame </span></p><p class="c6"><span class="c18">P</span><span class="c0">: Open a dialog to let the user add a View (as a Frame, a Tab or a splitted Frame) </span></p><p class="c6"><span class="c18">Q</span><span>: </span><span class="c4"><a class="c8" href="#id.zu0gcz">Save the window as a user window</a></span><span class="c0">, to display the same window with different data later </span></p><p class="c6 c7"><span class="c0"></span></p><a id="id.zu0gcz"></a><h1 class="c2"><span>Create a custom User Window</span></h1><p class="c6"><span class="c0">You can lay out your own user window with the desired views. </span></p><p class="c6"><span class="c0">You can do it from an already displayed window, or by using the right click mouse popup on a dataset like in the following example (Use menu &ldquo;Search Result&gt;New User Window&hellip;&rdquo; or &ldquo;Identification Summary&gt;New User Window&hellip;&rdquo;) </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 382.03px; height: 366.65px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\newuserwindowpopup.png" src="images/image65.png" style="width: 382.03px; height: 366.65px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">In the example, the user has clicked on &ldquo;Identification Summary&gt;New User Window&hellip;&rdquo; and selects the Peptides View as the first view of his window. </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 430.03px; height: 231.01px;"><img alt="" src="images/image132.png" style="width: 430.03px; height: 231.01px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">You can add other views by using the &#39;+&#39; button. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 614.27px; height: 345.08px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\newuserwindowpeptides.png" src="images/image134.png" style="width: 614.27px; height: 345.08px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">In this example, the user has added a Spectrum View and he saves his window by clicking on the &ldquo;Disk&rdquo; Button. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 598.40px; height: 336.54px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\newuserwindowpeptidesandspectrum.png" src="images/image136.png" style="width: 598.40px; height: 336.54px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">The user selects &#39;Peptides Spectrum&#39; as his user window name </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 612.42px; height: 344.08px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\newuserwindowsave.png" src="images/image138.png" style="width: 612.42px; height: 344.08px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">Now, the user can use his new &#39;Peptides Spectrum&#39; on a different Identification Summary. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 346.61px; height: 349.31px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\newuserwindowpopupwithsavedwindow.png" src="images/image140.png" style="width: 346.61px; height: 349.31px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.3jtnz0s"></a><h1 class="c2"><span>Filter Tables</span></h1><p class="c6"><span class="c0">You can filter data displayed in the different tables thanks to the filter button at the top right corner of a table. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 407.00px; height: 226.00px;"><img alt="" src="images/image142.png" style="width: 407.00px; height: 226.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">When you have clicked on the filter button, a dialog is opened. In this dialog you can select the columns of the table you want to filter thanks to the &ldquo;+&rdquo; button. </span></p><p class="c6"><span class="c0">In the following example, we have added two filters: </span></p><p class="c6"><span class="c1">- </span><span class="c13">one on the Protein Name column</span><span class="c0">&nbsp;(available wildcards are * to replace multiple characters and ? to replace one character) </span></p><p class="c6"><span class="c1">- </span><span class="c13">one on the Score Column</span><span class="c0">&nbsp;(Score must be at least 100 and there is no maximum specified). </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 295.50px; height: 197.90px;"><img alt="" src="images/image144.png" style="width: 295.50px; height: 197.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">The result is all the proteins starting with GLPK (correspond to GLPK*) and with a score greater or equal than 100. </span></p><p class="c6"><span class="c0">Note: for String filters, you can use the following wildcards: * matches zero or more characters, ? matches one character. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 600.03px; height: 136.52px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\filterresultprolinev2.png" src="images/image146.png" style="width: 600.03px; height: 136.52px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.1yyy98l"></a><h1 class="c2"><span>Search Tables</span></h1><p class="c6"><span class="c0">In some tables, a Search Functionality is available thanks to the search button at the top right corner. </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 422.00px; height: 209.00px;"><img alt="" src="images/image128.png" style="width: 422.00px; height: 209.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">When you have clicked on the search button, a floating panel is opened. In this panel you can select the column searched and fill in the searched expression, or the value range. </span></p><p class="c6"><span class="c0">For searched expressions, two wild cards are available: </span></p><ul class="c3 lst-kix_list_106-0 start"><li class="c27 li-bullet-0"><span class="c0">&#39;*&#39; : can replace all characters</span></li></ul><ul class="c3 lst-kix_list_116-0 start"><li class="c41 c32 li-bullet-0"><span class="c0">&#39;?&#39; : can replace one character</span></li></ul><p class="c6"><span class="c1">In the following example, the user </span><span>searches</span><span class="c1">&nbsp;for a </span><span>p</span><span class="c1">rotein </span><span>s</span><span class="c1">et whose name </span><span>contains</span><span class="c0">&nbsp;&ldquo;PGK&rdquo;. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 425.00px; height: 218.00px;"><img alt="" src="images/image129.png" style="width: 425.00px; height: 218.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">You can do an incremental search by clicking again on the search button of the floating panel, or by pressing the Enter key. </span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.4iylrwe"></a><h1 class="c2"><span>Graphics</span></h1><h2 class="c2"><span>Create a Graphic</span></h2><p class="c6"><span class="c0">There are two ways to obtain a graphic from data: </span></p><ol class="c3 lst-kix_list_55-0 start" start="1"><li class="c27 li-bullet-0"><span>In the windows with PSM of a Search Result or of an Identification Summary, you can ask for the display of a histogram in a new window to check the quality of your identification.</span></li><li class="c41 c32 li-bullet-0"><span>In any window, you can click on the &#39;+&#39; button to add a graphic (Scatter Plot or Histogram) as a view in the same window </span></li></ol><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 596.25px; height: 354.41px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\histogrambutton.png" src="images/image130.png" style="width: 596.25px; height: 354.41px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">If you have clicked on the &#39;+&#39; button, the Add View Dialog is opened and you must select the Graphic View </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 596.18px; height: 354.74px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\addaview.png" src="images/image113.png" style="width: 596.18px; height: 354.74px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c2"><span class="c16">Graphic options</span></h2><h3 class="c2" id="h.2y3w247"><span class="c36 c48">Graphic Toolbar</span></h3><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 686.18px; height: 601.33px;"><img alt="" src="images/image115.png" style="width: 686.18px; height: 601.33px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c18">A</span><span>: When this button is selected, you have the &ldquo;</span><span class="c18">Pointer Mode</span><span class="c0">&rdquo; activated.</span></p><p class="c6"><span class="c0">In this mode :</span></p><p class="c6"><span class="c0">- If you move with the left mouse button pressed on the middle of the graphic, you can scroll along the X and Y Axis.</span></p><p class="c6"><span class="c0">- If you move with the right mouse button pressed from the top/left corner to the bottom/right corner, a zooming rectangle is displayed. When you release the mouse button, a zoom in according to the zooming rectangle is performed.</span></p><p class="c6"><span class="c0">- If you move with the right mouse button pressed from the bottom/right corner to the left/top corner, a view all is done.</span></p><p class="c6"><span class="c18">B</span><span>: When this button is selected, you have the &ldquo;</span><span class="c18">Selection Mode</span><span class="c0">&rdquo; activated.</span></p><p class="c6"><span class="c0">By clicking on the black right arrow, you can switch between the square selection mode and the lasso selection mode.</span></p><p class="c6"><span class="c0">In this mode:</span></p><ul class="c3 lst-kix_list_66-0 start"><li class="c72 c32 li-bullet-0"><span>Use the left mouse button to do a square or lasso selection.</span></li><li class="c49 c32 li-bullet-0"><span>The right mouse button works as with the &ldquo;Pointer Mode&rdquo;</span></li></ul><p class="c6"><span class="c18">C</span><span>: Zoom out / Zoom in / View all. Click on the black right arrow to select the zooming mode.</span></p><p class="c6"><span class="c18">D</span><span class="c0">: Display/Remove Grid toggle button </span></p><p class="c6"><span class="c18">E</span><span class="c0">: Display/Hide View All Map. The goal of this map is to display the whole graphic in a small zone even when you have zoomed</span></p><p class="c6"><span class="c18">F</span><span class="c1">: </span><span>Open a settings dialog for the graphic. You can m</span><span class="c1">odify </span><span>for example colors or bins of an histogram.</span><span class="c0">&nbsp;</span></p><p class="c6"><span class="c18">G</span><span class="c1">: Lock/Unlock incoming data. If it is unlocked, the graphic is updated when the user </span><span>applies</span><span class="c0">&nbsp;a new filter to the previous view (for instance Peptide Score &gt;= 50) If it is locked, changing filtering on the previous view does not modify the graphic. </span></p><p class="c6"><span class="c18">H</span><span class="c1">: Select Data in the graphic according to data selected </span><span>in the table</span><span class="c0">&nbsp;in the previous view. </span></p><p class="c6"><span class="c18">I</span><span class="c0">: Select data in the table of the previous view according to data selected in the graphic. </span></p><p class="c6"><span class="c18">J</span><span class="c0">: Export graphic to image </span></p><p class="c6"><span class="c18">K</span><span class="c0">: Select the graphic type: Scatter Plot / Histogram </span></p><p class="c6"><span class="c18">L</span><span class="c13">/</span><span class="c18">M</span><span class="c0">: Select data used for X / Y axis. </span></p><p class="c6 c7"><span class="c0"></span></p><h3 class="c2" id="h.1d96cc0"><span class="c36 c48">Popup Menu on Graphic Area</span></h3><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 678.00px; height: 315.00px;"><img alt="" src="images/image116.png" style="width: 678.00px; height: 315.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c6"><span class="c0">By right mouse click on the graphic area, you get a popup with several menus:</span></p><ul class="c3 lst-kix_list_1-0 start"><li class="c72 c32 li-bullet-0"><span class="c0">Add Group: after selecting some data, you can create a group defined by a name and a color (there are two groups in the example : PSM xyz and PSM abc)</span></li><li class="c5 li-bullet-0"><span>Select Group: Right click on a data group to select them.</span></li><li class="c5 li-bullet-0"><span>Delete Group: Right click on a data group and delete it</span></li><li class="c5 li-bullet-0"><span class="c0">Add Vertical Cursor: Add a vertical cursor, its X position is displayed on the X Axis.</span></li><li class="c5 li-bullet-0"><span class="c0">Add Horizontal Cursor: Add an horizontal cursor, its Yposition is displayed on the Y Axis.</span></li><li class="c49 c32 li-bullet-0"><span class="c0">Delete Cursor: Right click on a cursor and delete it.</span></li></ul><p class="c6 c7 c56"><span class="c0"></span></p><h3 class="c2" id="h.3x8tuzt"><span class="c36 c48">Popup Menu on Axis</span></h3><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 670.00px; height: 277.00px;"><img alt="" src="images/image118.png" style="width: 670.00px; height: 277.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">By right mouse click on an axis, you get a popup with several menus:</span></p><ul class="c3 lst-kix_list_1-0"><li class="c72 c32 li-bullet-0"><span>Log 10 / Linear Axis : toggle the display of the axis between log and linear.</span></li><li class="c5 li-bullet-0"><span>Hide / Display Title : hide the axis title if you want more space for the graphic</span></li><li class="c5 li-bullet-0"><span>Remove / Add Vertical Grid</span></li><li class="c5 li-bullet-0"><span>Set Visible Range : to set the minimum and maximum values of the axis</span></li><li class="c49 c32 li-bullet-0"><span>Width measurement : display 2 cursors and the measurement of the space between the cursors.</span></li></ul><p class="c6 c7"><span class="c0"></span></p><h3 class="c2"><span>Zooming</span></h3><p class="c6 c7"><span class="c13 c29 c60"></span></p><p class="c6"><span class="c0">There are several ways to perform zoom actions.</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c18">Zoom in</span><span class="c0">:</span></p><ul class="c3 lst-kix_list_17-0 start"><li class="c6 c32 li-bullet-0"><span>Click on the Zoom in button in the toolbar.</span></li><li class="c6 c32 li-bullet-0"><span class="c0">Press the right mouse button and drag to the right bottom direction. A red box is displayed. Release the mouse button when you have selected the area to zoom in. </span></li><li class="c6 c32 li-bullet-0"><span>Put the mouse over an axis and roll upward the mouse wheel. The zoom in is centered around the middle of the axis. If you press on Ctrl Key at the same time, the zoom in is performed around the mouse position. </span></li></ul><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c18">Zoom out</span><span class="c0">:</span></p><ul class="c3 lst-kix_list_17-0"><li class="c72 c32 li-bullet-0"><span class="c0">Click on the Zoom out button in the toolbar.</span></li><li class="c49 c32 li-bullet-0"><span class="c0">Put the mouse over an axis and roll downward the mouse wheel. The zoom out is centered around the middle of the axis. If you press on Ctrl Key at the same time, the zoom out is performed around the mouse position. </span></li></ul><p class="c6 c56 c7"><span class="c0"></span></p><p class="c6"><span class="c13 c29 c60">View All:</span></p><ul class="c3 lst-kix_list_17-0"><li class="c6 c32 li-bullet-0"><span class="c0">Click on the View All button in the toolbar.</span></li><li class="c6 c32 li-bullet-0"><span class="c0">Press the right mouse button and drag to the left top direction. When you release the mouse button, the zooming is reset to view all. </span></li></ul><p class="c6 c7"><span class="c0"></span></p><h3 class="c2" id="h.2ce457m"><span class="c36 c48">Selecting</span></h3><p class="c26"><span class="c0">To be able to select data, you must be in &ldquo;Selection Mode&rdquo;</span></p><p class="c6"><span class="c13">Select</span><span class="c1">: </span><span>You can select with a rectangle area or with a lasso according to the selected button. </span><span class="c1">Press the left mouse button and drag the mouse to surround the data you want to select. When you release the button, the selection is done. Or left click on the data you want to select. It is possible to </span><span>use the Ctrl</span><span class="c0">&nbsp;key to add to the previous selection. </span></p><p class="c6"><span class="c13">Unselect</span><span class="c0">: Left click on an empty area to clear the selection. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 560.00px; height: 261.00px;"><img alt="" src="images/image120.png" style="width: 560.00px; height: 261.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h3 class="c2 c90" id="h.rjefff"><span class="c36 c48"></span></h3><h3 class="c2" id="h.3bj1y38"><span class="c36 c48">View All Map</span></h3><h1 class="c2"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 686.18px; height: 276.00px;"><img alt="" src="images/image121.png" style="width: 686.18px; height: 276.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></h1><p class="c26"><span class="c0">Click on View All Map button to display the map. This map always displays the whole graphic and the zoomed area. You can directly zoom on the view all map. You can resize it and move it.</span></p><a id="id.1qoc8b1"></a><h1 class="c2"><span class="c80">Quality Control</span></h1><h2 class="c2" id="h.4anzqyu"><span class="c16">Search Result QC</span></h2><p class="c6"><span class="c0">You can run a Quality Control on any leaf Search Result, that is to say an imported Result File not a merged search result. It consists in a transversal view of the imported data: rather than visualising the results per PSM or Proteins, results are sorted according to the score, charge state...</span></p><p class="c6"><span class="c0">&nbsp;</span></p><p class="c6"><span class="c0">Choose the menu option:</span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 406.67px; height: 241.03px;"><img alt="" src="images/image123.png" style="width: 406.67px; height: 241.03px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h3 class="c2" id="h.2pta16n"><span class="c36 c48">Settings</span></h3><p class="c6"><span class="c0">Configure some settings before launching the process</span></p><ul class="c3 lst-kix_list_124-0 start"><li class="c72 c32 li-bullet-0"><span>Score windows: you can split your data in different groups based on the score. The default groups are : less than 20, between 20 and 40, between 40 and 60, over 60</span></li><li class="c49 c32 li-bullet-0"><span>Max rank: data can be filtered to get a view focused on the best ranks. Default is to consider only the first rank.</span></li></ul><p class="c9"><span>&nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 352.00px; height: 205.00px;"><img alt="" src="images/image107.png" style="width: 352.00px; height: 205.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><h3 class="c2" id="h.14ykbeg"><span class="c36 c48">QC results</span></h3><p class="c6"><span class="c0">The report will appear in a matter of seconds (depending on the amount of data to be processed). You will get the following tabs:</span></p><p class="c6 c7"><span class="c0"></span></p><table class="c106"><tr class="c65"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c18">Assigned and unassigned spectra:</span><span class="c0">&nbsp;Pie chart presenting the ratio of assigned spectra</span></p></td><td class="c38" colspan="1" rowspan="1"><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 491.06px; height: 309.08px;"><img alt="" src="images/image108.png" style="width: 491.06px; height: 309.08px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr><tr class="c65"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c18">Score repartition for Target/Decoy data:</span><span class="c0">&nbsp;Histogram presenting the amount of PSM per group of score, separating target and decoy data</span></p></td><td class="c38" colspan="1" rowspan="1"><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 479.14px; height: 296.08px;"><img alt="" src="images/image109.png" style="width: 479.14px; height: 296.08px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr><tr class="c65"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c18">PSM per charge and score: </span><span class="c0">Histogram presenting the amount of PSM per group of score and charge state</span></p></td><td class="c38" colspan="1" rowspan="1"><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 495.43px; height: 308.08px;"><img alt="" src="images/image111.png" style="width: 495.43px; height: 308.08px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr><tr class="c65"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c18">Experimental M/z per charge and score:</span><span class="c0">&nbsp;Box plot presenting M/z information for each category of score and charge state</span></p></td><td class="c38" colspan="1" rowspan="1"><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 487.77px; height: 301.41px;"><img alt="" src="images/image158.png" style="width: 487.77px; height: 301.41px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr><tr class="c65"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c18">Number of matches per minute of RT and score:</span><span class="c0">&nbsp;histogram presenting the amount of PSM per score and retention time. This view is only calculated when retention time is available.</span></p></td><td class="c38" colspan="1" rowspan="1"><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 501.26px; height: 309.74px;"><img alt="" src="images/image161.png" style="width: 501.26px; height: 309.74px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr><tr class="c65"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c0">Each graph is also available in a table view</span></p></td><td class="c38" colspan="1" rowspan="1"><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 499.07px; height: 113.82px;"><img alt="" src="images/image163.png" style="width: 499.07px; height: 113.82px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr></table><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">&nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c19 c7"><span class="c0"></span></p><h2 class="c2 c52"><span class="c16"></span></h2><hr style="page-break-before:always;display:none;"><h2 class="c2 c52"><span class="c16"></span></h2><a id="id.3oy7u29"></a><h1 class="c2" id="h.243i4a2"><span class="c12">Ms Files Tab</span></h1><p class="c26"><span class="c0">In order to facilitate different actions on Ms Files, Proline Studio contains an homonym tab providing the end user with a view over his local and server remote file system, called Local File System and Proline Server File System respectively. </span></p><p class="c26"><span class="c0">Furthermore, on local file system a series of actions can take place, &nbsp;through an appropriate popup menu, on the encountered .mzdb and .raw files, including among others the:</span></p><ul class="c3 lst-kix_list_73-0 start"><li class="c46 c32 li-bullet-0"><span>Conversion of a .raw file to an .mzdb file</span></li><li class="c46 c32 li-bullet-0"><span>The upload of an .mzdb file</span></li><li class="c46 c32 li-bullet-0"><span>View .mzdb files</span></li><li class="c20 li-bullet-0"><span>Detect its peakels</span></li></ul><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 335.32px; height: 599.91px;"><img alt="" src="images/image73.png" style="width: 335.32px; height: 599.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c0">Apart from the popup menu supported functionality, since Proline Studio 1.5, uploads can be triggered via drag and drop mechanism. </span></p><p class="c26 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c14"></span></p><p class="c6 c7"><span class="c14"></span></p><p class="c6 c7"><span class="c14"></span></p><p class="c6 c7"><span class="c14"></span></p><p class="c6 c7"><span class="c14"></span></p><p class="c6 c7"><span class="c14"></span></p><p class="c6 c7"><span class="c14"></span></p><p class="c6 c7"><span class="c14"></span></p><p class="c6 c7"><span class="c14"></span></p><p class="c6 c7"><span class="c14"></span></p><p class="c6 c7"><span class="c14"></span></p><h3 class="c2 c90"><span class="c36 c48"></span></h3><h3 class="c2 c90"><span class="c36 c48"></span></h3><h3 class="c2 c90"><span class="c36 c48"></span></h3><h3 class="c2"><span class="c36 c48">mzDB File Upload</span></h3><p class="c26"><span class="c0">As mentioned earlier, after selecting a number of files, the user can either drag and drop them inside the remote site, or use the popup menu as shown in the following screenshot. It is important to precise that both approaches are not compatible with a selected group consisting of different file types. </span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 376.23px; height: 360.93px;"><img alt="" src="images/image165.png" style="width: 376.23px; height: 443.50px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c0">As we can see, clicking on upload opens a dedicated dialog packing a series of uploading options:</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 287.63px; height: 290.91px;"><img alt="upload_dialog.png" src="images/image68.png" style="width: 287.63px; height: 290.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><ul class="c3 lst-kix_list_67-0 start"><li class="c46 c32 li-bullet-0"><span>The deletion of the file(s) after the successful upload</span></li><li class="c46 c32 li-bullet-0"><span>The creation of the file(s) parent directory in destination: only the direct parent folder will be created. Otherwise, the file will be uploaded at the mount point root.</span></li><li class="c20 li-bullet-0"><span>The mounting point at the server: depending on server mounting points configuration.</span></li></ul><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span class="c0">Furthermore, the dialog permits us to add or remove .mzdb files to upload. The uploading tasks status could be viewed in &nbsp;the Logs tab.</span></p><h3 class="c2" id="h.j8sehv"><span class="c36 c48">Raw File Conversion</span></h3><p class="c26"><span class="c0">In the same way, when the user desires to convert and upload a raw file, he or she can use the respective dialog through the popup menu.</span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 507.00px; height: 488.00px;"><img alt="convert_dialog.png" src="images/image177.png" style="width: 507.00px; height: 488.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span class="c0">For the upload step, the same options as described above are shown.</span></p><p class="c26"><span>For the conversion, the path to the converter exe file should be specified. This value will be saved upon different executions. A default path may be specified in the &nbsp;</span><span class="c4"><a class="c8" href="#h.1smtxgf">general settings</a></span><span class="c0">&nbsp;dialog. The same way mzdb file may be deleted after a successful upload, raw files could be deleted after a conversion .</span></p><p class="c26 c7"><span class="c0"></span></p><h3 class="c2"><span>TIC or BPI chromatogram</span></h3><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">When the user chooses to &ldquo;View&rdquo; an mzDB file, the MzScope window is opened. </span></p><p class="c6"><span class="c0">By default, the TIC chromatogram is displayed. You can click on &ldquo;BPI&rdquo; to see the best peak intensity graph. </span></p><p class="c6"><span class="c1">By clicking in the graph, you can see below the scan at the selected time. </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 620.17px; height: 251.74px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\mzscope.png" src="images/image176.png" style="width: 620.17px; height: 251.74px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">You can choose to display 2 or more chromatograms on the same graph, by selecting 2 files and clicking on &ldquo;View</span><span class="c0">&rdquo;</span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 579.07px; height: 267.91px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\mzscopemultifile.png" src="images/image179.png" style="width: 579.07px; height: 267.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c0">You can extract a chromatogram at a given mass by entering the specified value in the panel above. </span></p><h3 class="c2"><span>Scan</span></h3><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 588.73px; height: 154.74px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\mzscopescan.png" src="images/image178.png" style="width: 588.73px; height: 154.74px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">You can navigate through the scans </span></p><p class="c6"><span class="c0">- by increasing or decreasing the scan Ids </span></p><p class="c6"><span class="c0">- by entering a retention time </span></p><p class="c6"><span class="c0">- by clicking the keys arrows on the keyboard (Ctrl+Arrows to keep the same ms level) </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c1">By double clicking on the scan, the corresponding chromatogram </span><span>is</span><span class="c0">&nbsp;displayed above (The Alt key or the check box &ldquo;XIC overlay&rdquo; allows you to overlay the chromatograms in the same graph). </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 571.77px; height: 376.41px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\mzscopechromato.png" src="images/image173.png" style="width: 571.77px; height: 376.41px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h3 class="c2 c90"><span class="c36 c48"></span></h3><h3 class="c2 c90"><span class="c36 c48"></span></h3><hr style="page-break-before:always;display:none;"><h3 class="c2 c90"><span class="c36 c48"></span></h3><h3 class="c2"><span>Peakels</span></h3><p class="c6"><span class="c0">By selecting a file, you can click on &ldquo;Detect Peakels&rdquo; in the popup menu. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 233.00px; height: 268.00px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\mzscope_detectpeakels_menu.png" src="images/image172.png" style="width: 233.00px; height: 268.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">A dialog allows you to choose the parameters of the peakels detection: the tolerance and eventually a range of m/z, or a m/z value: </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 260.00px; height: 568.00px;"><img alt="" src="images/image175.png" style="width: 260.00px; height: 568.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">The results are displayed in a table:</span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 599.93px; height: 165.20px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\mzscopepeakels.png" src="images/image174.png" style="width: 599.93px; height: 165.20px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">You can double-click (or through the popup menu) on a row to display the peakel in the corresponding raw file: </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 593.27px; height: 196.83px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\mzscopepeakelview.png" src="images/image171.png" style="width: 593.27px; height: 196.83px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h1 class="c2 c114"><span class="c12"></span></h1><a id="id.338fx5o"></a><h1 class="c2"><span>Export Data / Image</span></h1><p class="c6"><span class="c0">There are many ways to do an export: </span></p><p class="c6"><span class="c0">- Export a Table using the export button (supported formats: {xlsx, xls, csv}) </span></p><p class="c6"><span class="c0">- Export data using Copy/Paste from the selected rows of a Table to an application like Excel. </span></p><p class="c6"><span class="c1">- Export all data corresponding to an Identification Summary, X</span><span>IC</span><span class="c0">&nbsp;or Spectral Count </span></p><p class="c6"><span class="c0">- Export an image of a view </span></p><p class="c6"><span class="c1">- Export Identification Summary data into M</span><span>zIdentML </span><span class="c1">format </span><span>( for ProteomeXchange) </span><span class="c0">. </span></p><p class="c6"><span class="c0">- Export Identification Summary spectra list. </span></p><h2 class="c2"><span>1. Export a Table</span></h2><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 580.35px; height: 260.18px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\before_export.png" src="images/image170.png" style="width: 580.35px; height: 260.18px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">To export a table, click on the Export Button at the left top of a table. </span></p><p class="c6"><span class="c1">An export </span><span>d</span><span class="c0">ialog is opened, you can select the file path for the export and the format of the export (supported formats: {xlsx, xls, csv}). </span></p><p class="c6"><span class="c1">In case that the selected format is either .xls or .xlsx, the user has now the ability to maintain in his exported excel document any rich text format elements (color, font weight etc.) apparent on the original table in Proline Studio. Choice is done using the </span><span>checkbox </span><span class="c0">shown on the following screenshot. </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 235.50px; height: 190.56px;"><img alt="" src="images/image154.png" style="width: 235.50px; height: 190.56px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>&nbsp; &nbsp; &nbsp; &nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 233.54px; height: 188.98px;"><img alt="" src="images/image151.png" style="width: 233.54px; height: 188.98px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span>To perform the export, click on the Export Button. The task can take a few seconds if the table has a lot of rows and so a progress bar is displayed. </span></p><p class="c19 c7"><span class="c13 c29 c60"></span></p><h2 class="c2"><span>2. Copy/Paste a Table</span></h2><p class="c6"><span class="c0">To copy/Paste a Table: </span></p><p class="c6"><span class="c0">- Select rows you want to copy </span></p><p class="c6"><span class="c1">- Press Ctrl and C keys at the same time. </span><span>The column titles are also copied</span></p><p class="c6"><span class="c1">- Open your spreadsheet editor and press Ctrl and V keys at the same time to paste the copied rows. If </span><span class="c0">paste is done in a text editor, the column separator used is the tabulations. </span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span class="c16">3. Export an Identification Summary, a XIC or a Spectral Count</span></h2><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 284.89px; height: 411.90px;"><img alt="" src="images/image125.png" style="width: 284.89px; height: 411.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c1">To Export all data of a dataset </span><span>(</span><span class="c1">Identification Summary, XIC or Spectral Count), right</span><span>-</span><span class="c1">click on the dataset to open the contextual </span><span>menu</span><span class="c1">&nbsp;and select the &ldquo;Export&rdquo; </span><span>m</span><span class="c1">enu and then </span><span>&ldquo;Excel...&rdquo; sub-menu</span><span class="c0">. </span></p><p class="c6"><span class="c0">You can also export multiple dataset simultaneously, if they have the same type (Identification Summary or XIC or Spectral Count). </span></p><p class="c6"><span class="c1">An Export Dialog is opened, you can select the file path and the type of the export </span><span>: </span><span class="c1">Excel (.xlsx) or Tabulation </span><span>s</span><span class="c1">eparated </span><span>v</span><span class="c0">alues (.tsv). </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c1">You can export with the default parameters or perform a custom export. To enable custom export, click on the </span><span>tick box</span><span class="c0">&nbsp;located on the right of the dialog: </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 460.50px; height: 150.72px;"><img alt="" src="images/image153.png" style="width: 460.50px; height: 150.72px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">Custom export allows a number of parameters in addition to the file format to be chosen</span><span>. </span></p><ul class="c3 lst-kix_list_22-0 start"><li class="c27 li-bullet-0"><span>Sheets to be created. You can </span><span class="c18">enable/disable</span><span>&nbsp;sheets to be exported, </span><span class="c18">rename </span><span>them (by double clicking on the title) or </span><span class="c18">change their order</span><span class="c0">&nbsp;by drag and drop. &nbsp;Available sheets are dependent on the identification summary type, but you can export which parameters were used (search settings or import and filters), the list of protein sets, all proteins, PSMs&hellip; , modification Clusters or statistics data. </span></li></ul><ul class="c3 lst-kix_list_96-0 start"><li class="c5 li-bullet-0"><span class="c0">For each sheet, you can define the fields to be exported: change the title used in the exported file if needed, change the order of the fields using drag and drop of the corresponding row at the desired location.</span></li></ul><ul class="c3 lst-kix_list_94-0 start"><li class="c5 li-bullet-0"><span class="c0">Symbol to use for decimal output (comma &#39;,&#39; or point &#39;.&#39;)</span></li></ul><ul class="c3 lst-kix_list_83-0 start"><li class="c5 li-bullet-0"><span class="c0">Timestamp format to use</span></li></ul><ul class="c3 lst-kix_list_82-0 start"><li class="c5 li-bullet-0"><span class="c0">Protein sets to export (&#39;all&#39; or &#39;validated only&#39;)</span></li></ul><ul class="c3 lst-kix_list_134-0 start"><li class="c41 c32 li-bullet-0"><span class="c0">Export profile (&#39;best&#39; or &#39;all)</span></li></ul><p class="c6"><span class="c18">Description</span><span>&nbsp;of the exported file is available </span><span class="c42"><a class="c8" href="#id.3hv69ve">here</a></span><span class="c0">. </span></p><p class="c6"><span class="c42">Note</span><span class="c0">: The parameters could be saved and loaded in further export using the Save / Load buttons of the export dialog. </span></p><p class="c41 c7"><span class="c0"></span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 528.50px; height: 549.94px;"><img alt="" src="images/image152.png" style="width: 528.50px; height: 549.94px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span>4. Export an Image</span></h2><p class="c6"><span>Any </span><span class="c1">graphics in </span><span>proline</span><span class="c1">&nbsp;can be exported.</span><span>&nbsp;C</span><span class="c1">lick on the Export Image Button </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 16.00px; height: 16.00px;"><img alt="" src="images/image149.png" style="width: 16.00px; height: 16.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c0">at the left top of the image. </span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 437.35px; height: 257.84px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\export_image_button.png" src="images/image148.png" style="width: 437.35px; height: 486.92px; margin-left: -0.00px; margin-top: -229.08px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">An Export Dialog is opened</span><span>&nbsp;where you</span><span class="c1">&nbsp;can select the file path and the export type</span><span>. Available formats are &nbsp;PNG or SVG formats. SVG format produces a vector image that can be edited and resized afterwards.</span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 459.03px; height: 289.23px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\image_export_dialog.png" src="images/image150.png" style="width: 459.03px; height: 289.23px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c2"><span class="c16">5. Export Identification Summary for ProteomeXchange</span></h2><p class="c26"><span class="c0">Actually it is possible to export Identification Summary into MzIdentML, Pride isn&rsquo;t supported any more.</span></p><p class="c6"><span class="c1 c23 c42">Note</span><span class="c0">: Before exporting data all spectrum matches should have been generated. To do so, right click on the dataset and select &ldquo;Generate Spectrum Matches&rdquo;. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 227.27px; height: 318.90px;"><img alt="" src="images/image124.png" style="width: 227.27px; height: 318.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">Right click on the dataset you want to export and select the &ldquo;Export&rdquo; menu and then &ldquo;MzIdentML...&rdquo; sub-menu</span></p><p class="c59"><span class="c0">A dialog is opened where user information may be specified (name, organization &hellip;) </span></p><p class="c77 c111"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 289.54px; height: 213.10px;"><img alt="" src="images/image147.png" style="width: 289.54px; height: 213.10px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9 c7"><span class="c0"></span></p><p class="c59"><span class="c0">The file name and path should be specified in the next step. A progress bar is shown until the file is generated. The generated file contains identification and validation data issues from the dataset. All meta information including instrument configuration as well as search engine parameters are also extracted from dataset associated data.</span></p><p class="c9 c7"><span class="c0"></span></p><h2 class="c2"><span>6. Export Identification Summary spectra list</span></h2><p class="c6"><span class="c1">To export valid PSM Spectra from an Identification Summary or from a XIC Dataset. The exported tsv file is compatible with Peakview</span><span class="c0">.</span></p><p class="c6"><span class="c0">Note: all Spectrum Matches must be generated first.</span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.1idq7dh"></a><h1 class="c2"><span>Generate Spectrum Matches</span></h1><p class="c6"><span class="c1">When importing a Search Result in Proline, </span><span>users</span><span class="c1">&nbsp;can view PSM with their associated Spectrum but by default no annotation is defined. </span><span>Users</span><span class="c1">&nbsp;</span><span>need to </span><span class="c1">generate (and save) this information</span><span>&nbsp;explicitly.</span></p><ul class="c3 lst-kix_list_85-0 start"><li class="c27 li-bullet-0"><span>For a single PSM, select the icon near the Spectrum (see </span><span class="c4"><a class="c8" href="#id.3s49zyc">Display Peptide and PSM</a></span><span class="c0">&nbsp;)</span></li></ul><ul class="c3 lst-kix_list_48-0 start"><li class="c5 li-bullet-0"><span class="c0">For a whole Search Result, Identification Summary or Quantitation Result :</span></li></ul><ul class="c3 lst-kix_list_91-1 start"><li class="c15 li-bullet-0"><span class="c0">right click on a Dataset </span></li></ul><ul class="c3 lst-kix_list_87-1 start"><li class="c41 c61 li-bullet-0"><span class="c0">select &ldquo;Generate Spectrum Matches&rdquo; </span></li></ul><p class="c41"><span class="c0">In both cases, the following dialog will be opened. User can </span></p><ul class="c3 lst-kix_list_7-0 start"><li class="c5 li-bullet-0"><span>Choose to force spectrum matches to be calculated even if it is defined. This could be useful if previous spectrum matches generation was done with the wrong rules set.</span></li><li class="c5 li-bullet-0"><span>Use the fragmentation rules set defined at import. In some cases, this &nbsp;information is not accessible and the user should then specify it.</span></li><li class="c41 c32 li-bullet-0"><span>Specify the &nbsp;fragmentation rules set to use without taking into account the rules specified at import.</span></li></ul><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 524.00px; height: 356.00px;"><img alt="" src="images/image167.png" style="width: 524.00px; height: 356.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">Once executed, the dataset views need to be loaded again &nbsp;to effectively view the spectrum matches.</span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.42ddq1a"></a><h1 class="c2"><span>How to validate a Search Result</span></h1><p class="c6"><span class="c1">See description of </span><span class="c42 c1 c74"><a class="c8" href="#id.3as4poj">Validation Algorithm</a></span><span class="c0">. </span></p><p class="c6"><span class="c0">It is possible to validate identification Search Result or merged ones. In the latest case, the filters and validation threshold can be propagated to child Search Results.</span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span class="c16">Starting Validation</span></h2><p class="c26 c7"><span class="c0"></span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 288.69px; height: 482.74px;"><img alt="" src="images/image166.png" style="width: 288.69px; height: 482.74px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>&nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 291.62px; height: 482.91px;"><img alt="" src="images/image160.png" style="width: 291.62px; height: 482.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">To validate a Search Result: </span></p><p class="c59"><span class="c0">- Select one or multiple Search Results to validate </span></p><p class="c6"><span class="c0">- Right Click to display the popup </span></p><p class="c6"><span class="c1">- Click on &ldquo;Validate&hellip;&rdquo; menu </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span>Validation Dialog</span></h2><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 456.67px; height: 727.62px;"><img alt="" src="images/image157.png" style="width: 456.67px; height: 727.62px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">In the Validation Dialog, fill the different Parameters (see </span><span class="c4 c1"><a class="c8" href="#id.3as4poj">Validation description</a></span><span class="c0">): </span></p><p class="c6"><span class="c1">- you can add multiple PSM Prefilter Parameters ( Rank, Length, Score, e-Value, Identity p-Value, Homology p-Value) by selecting them in the combobox</span><span>.</span></p><p class="c6"><span class="c1">- you can ensure a FDR on PSMs using BH or Taget/Deco</span><span>y strategy. If using the latest, FDR </span><span class="c0">will be reached according to the variable selected ( Score, e-Value, Identity p-Value, Homology p-Value,&hellip; ) </span></p><p class="c6"><span class="c1">- you can add a Protein Set Prefilter on Specific Peptides </span><span>count</span><span class="c1">, peptides or </span><span>peptides sequence </span><span class="c1">count &nbsp;</span><span>or on Protein Sets score. </span><span class="c0">&nbsp;</span></p><p class="c6"><span class="c1">- you can also ensure a FDR on protein Sets o</span><span>r peptides using BH or Target/Decoy strategy. The strategy must be consistent between all levels where an FDR control is done (PSM, Peptide and ProteinSet) </span></p><p class="c6"><span class="c0">Note: FDR can be used only for Search Results with Decoy Data. </span></p><p class="c6"><span class="c0">If you run validation on a merged Search Result, you can choose to propagate it to child Search Result. Specified prefilters will be used as defined. For the FDR Filter, it is the threshold found by the validation algorithm which will be used for childs, as a prefilter.</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span>In the second tab, you can define rules for choosing the Typical Protein of a Protein Set by using a match string with wildcards ( * or ? ) on Protein Accession or Protein Description. (see </span><span class="c42"><a class="c8" href="#id.wnyagw">Change Typical Protein of Protein Sets</a></span><span class="c0">).</span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 385.33px; height: 425.03px;"><img alt="" src="images/image164.png" style="width: 385.33px; height: 425.03px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9 c7"><span class="c0"></span></p><p class="c59"><span class="c42">Note</span><span class="c0">: All validation parameters can be saved and loaded using appropriated buttons.</span></p><p class="c59 c7"><span class="c0"></span></p><h2 class="c2"><span>Validation Processing</span></h2><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 503.03px; height: 223.95px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\validationprocessingprolinev2.png" src="images/image162.png" style="width: 503.03px; height: 223.95px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">Validating a Search Result can take some time. While it is not finished, the Search Results are shown greyed with an hourglass over them. The tasks are displayed as running in the &ldquo;Tasks Log Dialog&rdquo;. </span></p><h2 class="c2"><span>Validation Done</span></h2><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 216.24px; height: 234.47px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\validationdoneprolinev2.png" src="images/image155.png" style="width: 216.24px; height: 234.47px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">When the validation is finished, the icon becomes orange and blue. Orange part </span><span>corresponds</span><span class="c0">&nbsp;to the Identification Summary. Blue is for the Search Result part. </span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.2hio093"></a><h1 class="c2"><span>How to filter Protein Sets</span></h1><p class="c6"><span class="c1">See description of </span><span class="c4 c1"><a class="c8" href="#id.2u6wntf">Protein Sets Filtering</a></span><span class="c0">. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 15.00px; height: 15.00px;"><img alt=":!:" src="images/image122.gif" style="width: 15.00px; height: 15.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c1">The protein sets windows are not update</span><span>d</span><span class="c1">&nbsp;after filtering Protein Set</span><span>.</span><span class="c1">&nbsp;You should close and reopen the window </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 16.00px; height: 16.00px;"><img alt=":!:" src="images/image122.gif" style="width: 16.00px; height: 16.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c2"><span>Starting filtering</span></h2><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 348.03px; height: 287.71px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\filterprotset.png" src="images/image156.png" style="width: 348.03px; height: 287.71px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">To filter Protein sets of Identification Summaries: </span></p><p class="c6"><span class="c0">- Select one or multiple Identification Summaries to filter </span></p><p class="c6"><span class="c0">- Right Click to display the popup </span></p><p class="c6"><span class="c0">- Click on &ldquo;Filter ProteinSets&hellip;&rdquo; menu </span></p><h2 class="c2"><span>Filtering Dialog</span></h2><p class="c6 c7"><span class="c0"></span></p><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 302.50px; height: 145.48px;"><img alt="" src="images/image119.png" style="width: 302.50px; height: 145.48px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">you can add multiple filters (Specific </span><span>P</span><span class="c1">eptide</span><span>s, Peptide count, Peptide sequence count, Protein Set Score</span><span class="c1">) by selecting them in the combobox</span><span>.</span></p><p class="c6"><span class="c1">Once the filtering is done, you will have to </span><span>open a new</span><span class="c0">&nbsp;protein sets window in order to see modification. </span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.wnyagw"></a><h1 class="c2"><span>Change Typical Protein of Protein Sets</span></h1><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 15.00px; height: 15.00px;"><img alt="" src="images/image122.gif" style="width: 15.00px; height: 15.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c1">&nbsp;The protein sets windows are not update</span><span>d</span><span class="c1">&nbsp;after </span><span>c</span><span class="c1">hang</span><span>ing</span><span class="c1">&nbsp;Typical Protein</span><span>.</span><span class="c1">&nbsp;You should close and reopen the window </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 15.00px; height: 15.00px;"><img alt=":!:" src="images/image122.gif" style="width: 15.00px; height: 15.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c2"><span>Open the Dialog</span></h2><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 244.03px; height: 312.33px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\changetypicalproteinpopupv2.png" src="images/image114.png" style="width: 244.03px; height: 312.33px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">To change the Typical Protein of the Protein Sets of an Identification Summary: </span></p><p class="c6"><span class="c0">- Select one or multiple Identification Summaries </span></p><p class="c6"><span class="c0">- Right Click to display the popup </span></p><p class="c6"><span class="c0">- Click on &ldquo;Change Typical Protein&hellip;&rdquo; menu </span></p><h2 class="c2"><span>Dialog Parameters</span></h2><p class="c9"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 351.55px; height: 357.09px;"><img alt="" src="images/image112.png" style="width: 351.55px; height: 357.09px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">You can set the choice for the Typical Protein of Protein Sets by using a match string with wildcards (* or ?) on Protein Accession or Protein Description. </span></p><p class="c6"><span class="c1">For Advanced </span><span>users</span><span class="c0">, a fully regular expression could be specified. In this case, check the corresponding option. </span></p><p class="c6"><span class="c1">Three rules could be specified</span><span>. T</span><span class="c1">hey </span><span>are</span><span class="c1">&nbsp;applied in priority order</span><span>, i.e.</span><span class="c1">&nbsp;if no protein of a protein set </span><span>satisfies</span><span class="c1">&nbsp;the first rule, the second one </span><span>is</span><span class="c0">&nbsp;tested and so on. </span></p><h2 class="c2"><span>Processing</span></h2><p class="c6"><span class="c0">The modification of Typical Proteins can take some time. During the processing, Identification Summaries are displayed grayed with an hourglass and the tasks are displayed in the Tasks Log Window </span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.3gnlt4p"></a><h1 class="c2"><span>Combine datasets</span></h1><p class="c6"><span class="c1">Merge can be done on Search Results or on Identification Summaries. You </span><span>have also to specify which merge mode is to be used (aggregation or union). </span><span class="c1">See description for </span><span class="c4"><a class="c8" href="#id.2lwamvv">combining</a></span><span class="c4 c1"><a class="c8" href="#id.2lwamvv">&nbsp;Search Results </a></span><span class="c4"><a class="c8" href="#id.2lwamvv">or</a></span><span class="c4 c1"><a class="c8" href="#id.2lwamvv">&nbsp;Identification Summarie</a></span><span class="c4"><a class="c8" href="#id.2lwamvv">s</a></span><span>.</span></p><h2 class="c2"><span>Merge on Search Results</span></h2><p class="c59"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 366.50px; height: 348.00px;"><img alt="" src="images/image117.png" style="width: 366.50px; height: 348.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">To merge a dataset with multiple Search Results: </span></p><p class="c6"><span class="c0">- Select the parent dataset </span></p><p class="c6"><span class="c0">- Right Click to display the popup </span></p><p class="c6"><span class="c0">- Click on &ldquo;Merge&rdquo; menu </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 200.74px; height: 274.91px;"><img alt="" src="images/image86.png" style="width: 200.74px; height: 274.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c1">When the merge is finished, the dataset is displayed with an </span><span>U or A </span><span class="c1">in the blue part of the icon, indicating that the </span><span>m</span><span class="c1">erge has been done u</span><span>si</span><span class="c1">ng U</span><span>nion or Aggregation</span><span class="c0">&nbsp;at a Search Result level. </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span>Merge on Identification Summaries</span></h2><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 214.22px; height: 290.91px;"><img alt="" src="images/image85.png" style="width: 214.22px; height: 290.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">If you merge a dataset containing Identification Summaries. The merge is done on an Identification Summary level. Therefore the dataset is displayed with an </span><span>U or A </span><span class="c0">in the orange part of the icon. </span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.1vsw3ci"></a><h1 class="c2"><span>Data Analyzer</span></h1><p class="c6"><span class="c0">The purpose of the Data Analyzer is to easily do calculations/comparisons on data. </span></p><p class="c6"><span class="c0">To open the data analyzer, you have two possibilities: </span></p><p class="c6"><span class="c0">- you can use the dedicated button that you can find in the toolbar of all views. If you use this button, the corresponding data is directly sent to the data analyzer. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 322.03px; height: 249.45px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\startdatamixer.png" src="images/image106.png" style="width: 322.03px; height: 249.45px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">- you can use the menu &ldquo;Window &gt; Data Analyzer&rdquo; </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 231.03px; height: 308.04px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\menudataanalyzer.png" src="images/image110.png" style="width: 231.03px; height: 308.04px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c1">In the Data Analyzer view, you can </span><span>access all</span><span class="c1">&nbsp;data views, to some functions and graphics. In the following example, we create a graph by adding by Drag &amp; Drop the Spectral Count Data and the </span><span>corresponding differential analysis function (</span><span class="c0">beta-binomial BBinomial). Then we link them together. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 437.00px; height: 325.00px;"><img alt="" src="images/image141.png" style="width: 437.00px; height: 325.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">You have to specify the parameters of the Function: right click on the function and select the &ldquo;settings&rdquo; menu </span></p><p class="c6"><span class="c0">In the settings menu, select the two groups of columns on which you want to perform the BBinomial function. When the parameters are set, the calculation is started immediately and an hourglass icon is shown. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 255.00px; height: 463.00px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\datamixersettingsdialog.png" src="images/image139.png" style="width: 509.97px; height: 612.54px; margin-left: -183.99px; margin-top: -149.54px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">When the calculation is finished: the hourglass icon becomes a green tick, and the user can right click and select the &ldquo;Display&rdquo; menu to see the result (or click on </span><span>the</span><span class="c1">&nbsp;</span><span>&ldquo;table&quot; icon)</span><span class="c0">. &nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 587.19px; height: 221.77px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\datamixerresult.png" src="images/image145.png" style="width: 587.19px; height: 221.77px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c2"><span class="c16">Available Functions</span></h2><h3 class="c2" id="h.4fsjm0b"><span class="c42 c57 c48">STATISTICS FUNCTIONS</span></h3><h3 class="c2"><span>FDR Computation Function</span></h3><p class="c6"><span class="c0">This function is used by ProStar Macro to compute the FDR. </span></p><p class="c59"><span class="c1">More information: </span><span class="c42 c1 c74"><a class="c8" href="https://www.google.com/url?q=http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf&amp;sa=D&amp;source=editors&amp;ust=1733226590919147&amp;usg=AOvVaw1oKLf1BxvvlND8xVcRu0So">http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf</a></span><span class="c0">&nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><h3 class="c2"><span>PValue Adjustment Function / calibration Plot</span></h3><p class="c6"><span>Calibration Plot for Proteomics is described here: </span><span class="c42"><a class="c8" href="https://www.google.com/url?q=https://cran.r-project.org/web/packages/cp4p/index.html&amp;sa=D&amp;source=editors&amp;ust=1733226590919893&amp;usg=AOvVaw3VEQDwbyqASZN5wwMebwSe">https://cran.r-project.org/web/packages/cp4p/index.html</a></span><span class="c0">&nbsp;</span></p><h3 class="c2"><span>SC Differential Analysis Function</span></h3><p class="c6"><span class="c0">beta binomial function, useful for Spectral Count Quantitations </span></p><h3 class="c2" id="h.2uxtw84"><span>Xic Differential Analysis Function</span></h3><p class="c6"><span class="c0">This function is used by ProStar Macro. Two tests are available: Welch t-test and Limma t-test. </span></p><p class="c59"><span>More information: </span><span class="c42"><a class="c8" href="https://www.google.com/url?q=http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf&amp;sa=D&amp;source=editors&amp;ust=1733226590920760&amp;usg=AOvVaw1ToVwDQgcVhtuT23X6usZK">http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf</a></span><span class="c0">&nbsp;</span></p><h3 class="c2"><span>Missing values filter Function</span></h3><p class="c6"><span class="c0">This function is used by ProStar Macro to remove rows with too many missing quantitative values. </span></p><p class="c6"><span class="c0">The available missing values algorithm are: </span></p><ul class="c3 lst-kix_list_121-0 start"><li class="c27 li-bullet-0"><span class="c0">Whole Groups: The lines (across all groups) in the quantitative dataset which contain less non-missing value than a user-defined threshold are deleted.</span></li></ul><ul class="c3 lst-kix_list_123-0 start"><li class="c5 li-bullet-0"><span class="c0">For every group: The lines for which each condition contains less non-missing value than a user-defined threshold are deleted.</span></li></ul><ul class="c3 lst-kix_list_128-0 start"><li class="c41 c32 li-bullet-0"><span class="c0">At least one group: The lines for which at least one condition contains less non-missing value than a user-defined threshold are deleted.</span></li></ul><h3 class="c2"><span>Missing values imputation Function</span></h3><p class="c6"><span class="c0">This function is used by ProStar Macro to impute missing values. </span></p><p class="c59"><span>More information: </span><span class="c42"><a class="c8" href="https://www.google.com/url?q=http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf&amp;sa=D&amp;source=editors&amp;ust=1733226590921981&amp;usg=AOvVaw13mNcOa0PG6sHkUw_WtY2-">http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf</a></span><span class="c0">&nbsp;</span></p><h3 class="c2"><span>Normalization Function</span></h3><p class="c6"><span class="c0">This function is used by ProStar Macro to normalize quantitative values. </span></p><p class="c59"><span>More information on algorithms: </span><span class="c42"><a class="c8" href="https://www.google.com/url?q=http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf&amp;sa=D&amp;source=editors&amp;ust=1733226590922586&amp;usg=AOvVaw0rqx7WXnnzCcyQEQLD5sHh">http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf</a></span><span class="c0">&nbsp;</span></p><h3 class="c2 c90" id="h.1a346fx"><span class="c36 c48"></span></h3><h3 class="c2" id="h.3u2rp3q"><span class="c42 c48 c57">TABLE FUNCTIONS</span></h3><h3 class="c2"><span>Join Function</span></h3><p class="c6"><span class="c1">Join data </span><span>from two</span><span class="c1">&nbsp;tables according </span><span>to the selected</span><span class="c0">&nbsp;key. </span></p><h3 class="c2"><span>Difference Function</span></h3><p class="c6"><span class="c1">Perform a difference between two joined table data according to a selected key. When a key value is not found in one of the data source </span><span>tables</span><span class="c0">, the line is displayed as empty. For numerical values a difference is done and for string values, the &#39;&lt;&gt;&#39; symbol is displayed when values are different. </span></p><h3 class="c2" id="h.2981zbj"><span class="c36 c48">Columns Filter Function</span></h3><p class="c26"><span class="c0">Columns filter, let the user remove unnecessary columns in a matrix. A combobox, with prefix and suffix of the columns allows to select multiple similar columns to filter them rapidly.</span></p><h3 class="c2"><span>Rows Filter Function</span></h3><p class="c6"><span>Rows </span><span class="c1">filter function </span><span>lets</span><span class="c1">&nbsp;the user filter some rows of a matrix according to settings on columns</span><span class="c0">.</span></p><h3 class="c2" id="h.odc9jc"><span class="c36 c48">Log Function</span></h3><p class="c6"><span class="c0">Create a column by calculating the Log (2 or 10) of an existing column. </span></p><h3 class="c2"><span>CSV/TSV Import </span></h3><p class="c6"><span class="c0">This module lets you import data from a CSV or TSV file. Then you can do calculations and display these data directly in Proline Studio.</span></p><p class="c6"><span class="c1">&nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 379.51px; height: 567.91px;"><img alt="" src="images/image143.png" style="width: 379.51px; height: 567.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">The separator is automatically selected according to the csv file. But you can modify it.</span></p><p class="c6"><span class="c0">The preview zone displays the first lines of the file as it will be loaded.</span></p><h3 class="c2" id="h.38czs75"><span class="c36 c48">Expression Builder</span></h3><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 568.50px; height: 490.32px;"><img alt="" src="images/image133.png" style="width: 568.50px; height: 490.32px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c0">The expression builder lets you create an expression with built-in functions or comparators and variables (columns from the linked matrix). In the example, we calculate the mean of a column in the matrix.</span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span class="c16">Prostar Macro</span></h2><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 542.99px; height: 353.74px;"><img alt="" src="images/image131.png" style="width: 542.99px; height: 353.74px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">1 </span><span>or </span><span class="c1">2: Add XIC Data to Data Analyzer from the Protein Set View</span><span>&nbsp;or by importing data from a csv file.</span></p><p class="c6"><span>3</span><span class="c1">: Add Prostar Macro by a drag and drop and link XIC Data to the Macro</span><span>. And do the calculation by</span><span class="c0">&nbsp;clicking on the button Process Graph.</span></p><p class="c6"><span class="c0">During the process, the Data Analyzer will ask you settings for each function. </span></p><p class="c6"><span class="c0">4: Filter unnecessary columns from your data if. Settings can be validated with no parameters if you don&#39;t need it. </span></p><p class="c6"><span>5</span><span class="c0">: Filter is needed only if you want to remove contaminants. Settings can be validated with no parameters if you don&#39;t need it. </span></p><p class="c6"><span>6</span><span class="c0">: Log is needed to log abundances (Data from Proline). For Data coming from MaxQuant, data is already logged. </span></p><p class="c6"><span>7</span><span class="c1">&nbsp;to 1</span><span>3</span><span class="c0">: follow the settings asked ( you can find some help in Prostar documentation, or information in corresponding functions.) </span></p><p class="c6"><span class="c1">During the process, results will be automatically displayed</span><span class="c0">:</span></p><p class="c6"><span class="c0">14: FDR Result</span></p><p class="c6"><span class="c0">15: Calibration Plots</span></p><p class="c6"><span class="c0">16: Result Table with differential Proteins Table and the corresponding scatter plot. You can select differential proteins in the table, to import them in the scatter plot and create a colored group with them.</span></p><p class="c6"><span class="c0">If you want to look at other results, right click on a function and select &ldquo;Display in New Window&rdquo; </span></p><p class="c6"><span class="c1">Prostar User Manual:</span></p><p class="c6"><span class="c42 c1 c74"><a class="c8" href="https://www.google.com/url?q=http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf&amp;sa=D&amp;source=editors&amp;ust=1733226590927485&amp;usg=AOvVaw3ERQTdn9KzrE4mStPQ3yFw">http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf</a></span><span class="c0">&nbsp;</span></p><p class="c6"><span class="c1">Prostar Tutorial :</span></p><p class="c6"><span class="c42 c1 c74"><a class="c8" href="https://www.google.com/url?q=http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_Tutorial.pdf&amp;sa=D&amp;source=editors&amp;ust=1733226590928130&amp;usg=AOvVaw2Ud1F7djr_R1Qvn_lzuCIz">http://bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_Tutorial.pdf</a></span><span class="c0">&nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.1nia2ey"></a><h1 class="c2"><span>Calculator</span></h1><p class="c6"><span class="c0">Calculator lets you write python scripts to manipulate freely viewed data. </span></p><p class="c6"><span class="c0">1) To open the calculator, click on the calculator icon (not available on all views for the moment) </span></p><p class="c6"><span class="c1">On the left part of the calculator, you can </span><span>access all</span><span class="c0">&nbsp;viewed data, double click to add a table or a column to the script. </span></p><p class="c6"><span class="c0">2) Write your python script on the text area </span></p><p class="c6"><span class="c0">3) Execute it by clicking on the green Arrow. </span></p><p class="c6"><span class="c0">4) When the script has been executed, the results of the calculations (variables, new columns) are available in the &ldquo;Results&rdquo; tab. Double click on a new column to add it to the table. Or like in the example, directly add the column to the table programmatically. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 565.03px; height: 511.14px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\calculatorv2.png" src="images/image137.png" style="width: 565.03px; height: 511.14px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h3 class="c2"><span>Examples</span></h3><h4 class="c2"><span>Script to calculate a log column</span></h4><p class="c19"><span class="c33 c29 c60">#### Algorithm to calculate the logarithm of a column ####<br><br># get the Table 3 which corresponds to table newSC Quanti Protein Set<br>t = Table.get(3)<br><br># get the constant column 10 of the table t ( Specific SC column)<br># mutable() is called to be able to modify data<br>specificSCCol = t[10].mutable()<br><br># number of rows of the column<br>&nbsp;nb = len(specificSCCol)<br><br># loop on the data of the column<br>for i in range (0,nb): <br>&nbsp;# calculate the log (NaN values for errors)<br>&nbsp;v = specificSCCol[i]<br>&nbsp;if v &lt;= 0:<br>&nbsp; specificSCCol[i] = float(&#39;NaN&#39;) <br>&nbsp;else:<br>&nbsp; specificSCCol[i] = math.log(v)<br><br># set the column name which will be used to the user<br>specificSCCol.setColumnName(&quot;log(specificSC)&quot;)<br><br># add the created column to the table t<br>t.addColumn(specificSCCol)</span></p><h4 class="c2"><span>Script to perform a difference and a mean between two columns</span></h4><p class="c19"><span class="c33 c29 c60">#### Algorithm to perform a difference and a mean between two columns ####<br></span></p><p class="c19"><span class="c33 c29 c60">t = Table.get(9)<br>colAbundance1 = t[3]<br>colAbundance2 = t[5]<br><br># difference between two columns<br>colDiff = colAbundance1-colAbundance2<br><br># set the name of the column<br>colDiff.setColumnName(&quot;diff&quot;)<br><br># mean between two columns<br>colMean = (colAbundance1+colAbundance2)/2<br><br># set the name of the column<br>colMean.setColumnName(&quot;mean&quot;)<br><br># add columns to the table<br>t.addColumn(colDiff)<br>t.addColumn(colMean)</span></p><h4 class="c2"><span>Script to perform a perform a pvalue and a ttd on a XIC quantitation table</span></h4><p class="c19"><span class="c33 c29 c60">#### Algorithm to perform a pvalue and a ttd on abundances column of a XIC quantitation ####<br>t = Table.get(1)<br><br>pvalueCol = Stats.pvalue( (t[2], t[3]), (t[4],t[5]) )<br>ttdCol = Stats.ttd( (t[2], t[3]), (t[4],t[5]) )<br><br>pvalueCol.setColumnName(&quot;pvalue&quot;)<br>ttdCol.setColumnName(&quot;ttd&quot;)<br><br>t.addColumn(pvalueCol)<br>t.addColumn(ttdCol)</span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.47hxl2r"></a><h1 class="c2"><span>Update Spectrum using Peaklist software</span></h1><p class="c6"><span class="c1">When </span><span class="c4"><a class="c8" href="#id.319y80a">importing a search</a></span><span class="c4 c1"><a class="c8" href="#id.319y80a">&nbsp;result</a></span><span class="c1">, the software used for the peaklist creation has to be specified. This parameter is mandatory for the </span><span class="c4 c1"><a class="c8" href="#id.3ls5o66">XIC quantitation</a></span><span class="c1">&nbsp;as it is used to find scan number or RT in the spectrum title. Indeed, this information is then used to extract abundances in the </span><span class="c1 c23">raw files</span><span class="c0">. </span></p><p class="c6"><span class="c0">If an invalid software has been specified when importing, it is possible to change the peaklist software afterwards. This option is only valid for Identification DataSets. </span></p><p class="c6"><span class="c0">Right click on the identification DataSet, and select &ldquo;Update Spectrum using Peaklist software&rdquo; </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 373.03px; height: 365.35px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\howtoupdatepeaklistsoftware.png" src="images/image135.png" style="width: 373.03px; height: 365.35px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">The following dialog will be displayed </span><span>allowing the user</span><span class="c1">&nbsp;to select the peaklist software to </span><span>use</span><span class="c0">. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 288.03px; height: 170.66px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\updatespectrummatches.png" src="images/image127.png" style="width: 288.03px; height: 170.66px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c19 c7"><span class="c0"></span></p><a id="id.2mn7vak"></a><h1 class="c2" id="h.11si5id"><span>Spectral Count</span></h1><p class="c6"><span class="c1">See description of </span><span class="c4 c1"><a class="c8" href="#id.3l18frh">Compare Identification Summaries with Spectral Count</a></span><span class="c0">. </span></p><h2 class="c2"><span>Generate a Spectral Count</span></h2><p class="c6"><span class="c1">To obtain a spectral count, right click on a Dataset with merged Identification Summaries and select the &ldquo;Quantify </span><span>&rarr;</span><span class="c1">&nbsp;Spectral C</span><span>ounting</span><span class="c1">&rdquo; menu in the popup. </span><span>This Dataset is used as the reference Dataset and Protein Set list as well as specifics peptides are defined there.</span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 471.11px; height: 548.91px;"><img alt="" src="images/image126.png" style="width: 471.11px; height: 548.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">In the Spectral Count window, fill the name and description of your Spectral Count and press Next. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 248.03px; height: 248.03px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\sc_step1.png" src="images/image84.png" style="width: 248.03px; height: 248.03px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">Then select the Identification Summaries on which you want to perform the Spectral Count and press Next. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 310.03px; height: 296.05px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\sc_step2.png" src="images/image77.png" style="width: 310.03px; height: 296.05px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">Finally select the DataSet where shared peptides spectral count weight </span><span>should </span><span class="c0">be calculated and press OK. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 299.03px; height: 284.23px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\sc_step3.png" src="images/image75.png" style="width: 299.03px; height: 284.23px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">A Spectral Count is created and added to the Quantitations Panel. </span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 221.59px; height: 421.34px;"><img alt="D:\Proline_Data\Documentation\1.3\ProlineSuite_V1.3_MERGED_ODF\sc_dataset.png" src="images/image81.png" style="width: 221.59px; height: 421.34px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c2"><span>Display a Spectral Count</span></h2><p class="c6"><span class="c1">You can then display a Spectral Count, see </span><span class="c4 c1"><a class="c8" href="#id.36ei31r">Display a Spectral Count</a></span><span class="c0">&nbsp;</span></p><p class="c19 c7"><span class="c0"></span></p><h1 class="c2"><span>Label Free Quantitation</span></h1><p class="c6"><span class="c1">For description on LC-MS Quantitation you can first read the principles in this page: </span><span class="c4 c1"><a class="c8" href="#id.3ygebqi">La</a></span><span class="c4"><a class="c8" href="#id.3ygebqi">bel-Free q</a></span><span class="c4 c1"><a class="c8" href="#id.3ygebqi">uantitation principles</a></span><span class="c0">&nbsp;</span></p><h2 class="c2" id="h.20xfydz"><span class="c16">Create a New Label Free Design</span></h2><p class="c59"><span class="c0">You can create a new label-free design by clicking on the &ldquo;Quantify &rarr; Label Free&rdquo; menu from Quantitation or from a dataset in the Identification tree. In this case, the selected dataset will be used as reference for the quantitation (from where the list of peptides and protein is extracted).</span></p><p class="c59 c56"><span>&nbsp; </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 404.00px; height: 111.00px;"><img alt="" src="images/image79.png" style="width: 404.00px; height: 111.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><h3 class="c2" id="h.4kx3h1s"><span class="c36 c48">Settle Design</span></h3><p class="c6"><span class="c0">When you start the creation of a new XIC Design, the following dialog is opened:</span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 610.82px; height: 390.74px;"><img alt="xic_step1_create_group_sample.png" src="images/image69.png" style="width: 610.82px; height: 390.74px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c0">In the left Area, you can create your design :</span></p><ul class="c3 lst-kix_list_37-0 start"><li class="c46 c32 li-bullet-0"><span>by adding Group and Samples (thanks to mouse right click popup)</span></li><li class="c46 c32 li-bullet-0"><span>by &nbsp;renaming every items ( thanks to F2 or mouse right click popup)</span></li><li class="c20 li-bullet-0"><span>by rearranging order of groups and samples (thanks to drag &amp; drop)</span></li></ul><p class="c26"><span class="c0">From the right Area, you can drag and drop desired identifications to the left Area. If you drop in the XIC or in a Group node, &nbsp;Group and Sample nodes will be automatically added. But you can drop directly in a Sample node.</span></p><p class="c26"><span class="c0">Note : &nbsp;it is recommended that the XIC node is renamed.</span></p><p class="c26"><span class="c0">When your design is ready, click on Next Button.</span></p><h3 class="c2" id="h.302dr9l"><span class="c36 c48">Link to Raw Files</span></h3><p class="c26"><span class="c0">The following panel is displayed:</span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 616.74px; height: 405.47px;"><img alt="" src="images/image66.png" style="width: 616.74px; height: 405.47px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c0">In order to be able to perform any XiC design, all participating Sample Analysis must be associated with a corresponding raw file. The association is done at the second step of the XIC design by : </span></p><ul class="c3 lst-kix_list_13-0 start"><li class="c72 c32 li-bullet-0"><span>drag &amp; drop the &ldquo;mzDB File&rdquo; into the raw corresponding to the correct sample analysis. It must be noted that although it is possible to overwrite an existing association (done in a previous quantitation using the same sample analysis dataset), there is no verification done to ensure compatibility between the mzDb file and the sample analysis dataset. </span></li><li class="c32 c49 li-bullet-0"><span>using the Drop Zone. This feature can be quite helpful in cases where multiple associations are missing or when the plethora of uploaded .mzdb files intimidates the user from manually searching the files one by one. The feature itself is extremely easy to use as it just requires dragging a set of files or folders containing .mzdb files from the user part. As soon as a drop takes place, all missing connections will be automatically created as long as a matching .mzdb file has been dropped in the Drop zone. It must also be noted that since version 1.5, users have now at their disposal indices about the association source. Furthermore, in order to protect &nbsp;from a possible corruption of an existing association in the database, the latter ones cannot be overwritten.</span></li></ul><p class="c6 c56 c7"><span class="c0"></span></p><p class="c26"><span class="c0">When all participating Sample Analysis are associated with a raw file, click on Next Button.</span></p><h3 class="c2" id="h.1f7o1he"><span class="c36 c48">Quantitation Parameters</span></h3><p class="c6"><span>Now you must </span><span class="c1">select the parameters. See </span><span class="c4 c1"><a class="c8" href="#id.3ygebqi">Label-free LC-MS quantitation configuration</a></span><span class="c1">&nbsp;to have more details about the different parameters. </span></p><p class="c6"><span class="c0">These parameters are not all displayed. You can display a complete set of parameters by clicking on the &ldquo;Advances Parameters&rdquo;.</span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 566.00px; height: 574.00px;"><img alt="" src="images/image168.png" style="width: 566.00px; height: 574.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c1">Note: all the parameters are already set with default values, or with previous </span><span>values if a quantitation has already been run</span><span class="c0">. </span></p><h2 class="c2 c52" id="h.lf8wwjwabfxq"><span class="c16"></span></h2><h2 class="c2" id="h.3z7bk57"><span class="c16">Clone a Label Free Design</span></h2><p class="c6"><span class="c0">You can copy a XIC design by selecting the &ldquo;Clone &amp; Extract Abundances&rdquo; option from an existing quantitation result. The new XIC is generated using an existing Experimental Design and parameters setting.</span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 333.50px; height: 339.14px;"><img alt="clone_extract_abundances.png" src="images/image71.png" style="width: 455.65px; height: 412.89px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c2 c52"><span class="c16"></span></h2><h2 class="c2"><span>Displays</span></h2><p class="c6"><span class="c0">Newly generated quantitation designs are immediately added to the Quantitation Tree. Through the latter one, and via a popup menu, the end user has the capacity either to view a design&rsquo;s properties as seen to the following screenshot, or to apply a series of actions on it, including among others:</span></p><p class="c7 c119"><span class="c0"></span></p><ul class="c3 lst-kix_list_108-0 start"><li class="c47 c32 li-bullet-0"><span>Display experimental Design parameters, results and abundances &nbsp;: see how to </span><span class="c4"><a class="c8" href="#id.1ljsd9k">Display a quantitation result</a></span></li><li class="c32 c47 li-bullet-0"><span>Delete a quantitation Design, see how to </span><span class="c4"><a class="c8" href="#id.4du1wux">Delete Data</a></span><span>&nbsp;</span></li><li class="c47 c32 li-bullet-0"><span>Rename a quantitation Design, by clicking on &ldquo;Rename&hellip;&rdquo; in the popup menu. </span></li><li class="c47 c32 li-bullet-0"><span>Export the quantitation results, see how to </span><span class="c4"><a class="c8" href="#id.338fx5o">Export Data</a></span><span class="c0">&nbsp;</span></li></ul><p class="c9 c96"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 668.67px; height: 940.08px;"><img alt="xic_properties.png" src="images/image63.png" style="width: 668.67px; height: 940.08px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h1 class="c39" id="h.17lhjkd63lf3"><span class="c12">Isobaric labeling Quantitation</span></h1><p class="c6"><span>For description on isobaric labeling quantitation you can first read the principles in this page: </span><span class="c4"><a class="c8" href="#h.1wmaarypjked">Isobaric labeling quantitation workflow</a></span><span>&nbsp;</span><span class="c0">&nbsp;</span></p><h2 class="c2" id="h.l5okxbnxuj6"><span class="c16">Create a New Design</span></h2><p class="c59"><span>You can create a new label-free design by clicking on the &ldquo;Quantify &rarr; Label Free&rdquo; menu from Quantitation or from a dataset in the Identification tree. In this case which is recommended, the selected dataset will be used as </span><span class="c18">reference </span><span class="c0">for the quantitation (from where the list of peptides and protein is extracted).</span></p><p class="c59 c56"><span>&nbsp; </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 370.63px; height: 101.83px;"><img alt="" src="images/image79.png" style="width: 370.63px; height: 101.83px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><h3 class="c2" id="h.1r7xxq4bwt2a"><span class="c36 c48">Settle Design</span></h3><p class="c6"><span class="c0">When you start the creation of a new Design, the following dialog is opened:</span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 584.36px; height: 444.39px;"><img alt="" src="images/image180.png" style="width: 584.36px; height: 444.39px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">First select the method name in the drop-down list. </span></p><p class="c6"><span class="c0">In case of &ldquo;simple&rdquo; isobaric quantitation, each run is quantified separately, the different conditions are indeed pulled in the same sample with a specific tag. However, Proline allows you to configure the quantisation of several runs at once, in the case of fractions for example. Each run will produce a separate quantitation result as if you had run them individually.</span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 427.26px; height: 483.82px;"><img alt="" src="images/image183.png" style="width: 427.26px; height: 483.82px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span>But it is also possible to execute a multi-batch quantitation. In this case, a single quantitation will be performed and the result of all the runs will be concatenated as different channels. <br></span><span class="c18">Warning</span><span class="c0">: If multi-batch quantitation is selected, at the next step it is mandatory to use &ldquo;Rescale reporter abundances&hellip;&rdquo;</span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 491.26px; height: 242.58px;"><img alt="" src="images/image3.png" style="width: 491.26px; height: 242.58px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><h3 class="c2" id="h.4bcu34y0b2j2"><span class="c36 c48">Isobaric labeling quantitation parameters</span></h3><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span class="c0">When you click next, a view with isobaric labeling specific parameters is displayed. </span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 359.26px; height: 272.03px;"><img alt="" src="images/image181.png" style="width: 359.26px; height: 272.03px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><ul class="c3 lst-kix_2x6t2se3panh-0 start"><li class="c20 li-bullet-0"><span class="c18">Reporter ions m/z tolerance</span><span class="c0">&nbsp;(ppm): m/z window used to match a peak in MSMS spectra with an isobaric tag mz</span></li><li class="c20 li-bullet-0"><span class="c18">Reporter ions will be extracted from</span><span class="c0">: &nbsp;specify where intensity values are read </span></li></ul><ul class="c3 lst-kix_2x6t2se3panh-1 start"><li class="c26 c61 li-bullet-0"><span class="c18">Proline spectrum </span><span>: from spectrum stored in Proline database and read in </span><span class="c4"><a class="c8" href="#id.3dy6vkm">result file</a></span><span>&nbsp;when </span><span class="c4"><a class="c8" href="#id.319y80a">importing search result</a></span><span class="c0">. </span></li><li class="c26 c61 li-bullet-0"><span class="c18">mzDB MS2 spectrum</span><span class="c0">&nbsp;: from MS2 spectrum read in mzDB associated with the run. </span></li><li class="c26 c61 li-bullet-0"><span class="c18">mzDB MS3 spectrum</span><span class="c0">&nbsp;(beta version): from MS3 spectrum read in mzDB associated with the run. </span></li></ul><ul class="c3 lst-kix_2x6t2se3panh-0"><li class="c20 li-bullet-0"><span class="c18">Rescale reporter abundance to MS1 signal </span><span class="c0">(mandatory for multi-batch) : all reporter ions abundance (read in MS2 or MS3 spectra) are rescaled on the MS1 APEX. If this option is selected, you will be prompted to give &ldquo;label-free parameters&rdquo; in order to compute MS1 quantitation, and thus MS1 APEX.</span></li></ul><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span class="c0">The next steps will be:</span></p><ul class="c3 lst-kix_4sgfft60r4dw-0 start"><li class="c20 li-bullet-0"><span>Associate mzDB files to sample analysis: &nbsp;see </span><span class="c4"><a class="c8" href="#h.302dr9l">Link raw files</a></span></li><li class="c20 li-bullet-0"><span>Label-free parameter, if rescale to MS1 was selected : see </span><span class="c4"><a class="c8" href="#h.1f7o1he">Quantitation parameters</a></span><span>.</span></li></ul><p class="c26 c7"><span class="c0"></span></p><a id="id.2eclud0"></a><h1 class="c2"><span class="c12">Quantitations Aggregation</span></h1><p class="c26"><span class="c0">It is possible to aggregate data from different label-free or isobaric labeling designs.</span></p><p class="c26"><span class="c0">First you must select two or more XIC Designs in the Quantitation tree and do a mouse right click to obtain a popup and then select the menu &ldquo;Aggregate Quantitations&rdquo;.</span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 402.00px; height: 220.00px;"><img alt="" src="images/image105.png" style="width: 402.00px; height: 220.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c0">The following dialog is opened:</span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 686.18px; height: 537.33px;"><img alt="" src="images/image104.png" style="width: 686.18px; height: 537.33px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c0">This dialog is divided in three areas:</span></p><ul class="c3 lst-kix_list_24-0 start"><li class="c46 c32 li-bullet-0"><span class="c18">left area</span><span>: aggregated experimental design. This design is inferred from the quantitations that will be aggregated.</span></li><li class="c46 c32 li-bullet-0"><span class="c18">center area</span><span>&nbsp;: quantitation channels mapping. Each quantitation channel of the aggregation will correspond to sample analysis of aggregated quantitations.</span></li><li class="c20 li-bullet-0"><span class="c18">right area</span><span>&nbsp;: XIC Sources</span></li></ul><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span class="c0">You can completely modify the inferred aggregated experimental design. You can use drag &amp; drop to modify the order of the items and the right mouse popup to create, delete or rename items.</span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 426.00px; height: 210.00px;"><img alt="" src="images/image97.png" style="width: 426.00px; height: 210.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c0">You can modify the quantitation channels mapping by drag and drop from the right area to the center area. You can also remove or reorder the mapping thanks to the right mouse popup or thanks to the floating panel.</span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 686.18px; height: 225.33px;"><img alt="" src="images/image95.png" style="width: 686.18px; height: 225.33px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c0">When you select a Channel, you can click on the help button to check what the aggregation will exactly do.</span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 667.00px; height: 330.00px;"><img alt="" src="images/image100.png" style="width: 667.00px; height: 330.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><a id="id.thw4kt"></a><h1 class="c2"><span>Compute Post Processing on abundances</span></h1><h2 class="c2"><span>Advanced Protein Sets abundances</span></h2><p class="c6"><span class="c1">Right click on the selected XIC node in the Quantitation tree, and select &ldquo;Compute Post Processing on Abundances</span><span>&hellip;&rdquo; </span><span class="c0">&nbsp;</span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 343.00px; height: 397.00px;"><img alt="" src="images/image99.png" style="width: 343.00px; height: 397.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c2 c52"><span class="c16"></span></h2><h2 class="c2"><span class="c16">Configuration</span></h2><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 565.91px; height: 505.07px;"><img alt="" src="images/image4.png" style="width: 565.91px; height: 505.07px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6 c7"><span class="c0"></span></p><p class="c6"><span class="c0">In the dialog, you can: </span></p><p class="c6"><span class="c0">- specify peptides to consider for quantitation </span></p><p class="c6"><span class="c1">- configure </span><span>abundances computing </span><span class="c0">parameters </span></p><p class="c6"><span class="c1">- </span><span>select </span><span class="c1">a</span><span>ggregation mode </span></p><p class="c6"><span class="c1">For more details, see </span><span class="c42 c1 c74"><a class="c8" href="#id.1jlao46">Post-processing of LC-MS quantitative results</a></span><span class="c0">&nbsp;</span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2"><span>Advanced XIC results</span></h2><p class="c6"><span class="c1">You can see the results by displaying the XIC (</span><span class="c42 c1 c74"><a class="c8" href="#id.1ljsd9k">Display a XIC</a></span><span class="c1">) or export them </span><span class="c42 c1 c74"><a class="c8" href="#id.338fx5o">Export Data</a></span><span class="c0">&nbsp;</span></p><hr style="page-break-before:always;display:none;"><p class="c6 c7"><span class="c0"></span></p><a id="id.3dhjn8m"></a><h1 class="c2" id="h.1smtxgf"><span class="c12">General Settings</span></h1><p class="c26"><span class="c0">Since version 1.4 Proline Studio includes a general settings dialog which can be accessed from the top menu bar clicking on &ldquo;General Settings&rdquo; as shown in the screenshot below.</span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 601.24px; height: 360.74px;"><img alt="access_general_settings.png" src="images/image88.png" style="width: 601.24px; height: 360.74px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c19"><span class="c0">The latter one consists of a constantly but slowly growing number of user preferences regarding various aspects of the utilization of Proline Studio. Based on their context, for the time being, preferences are organized into the following four tabs:</span></p><ul class="c3 lst-kix_list_59-0 start"><li class="c46 c32 li-bullet-0"><span>JMS Settings</span></li><li class="c46 c32 li-bullet-0"><span>Conversion/Upload Settings</span></li><li class="c46 c32 li-bullet-0"><span class="c0">Table Parameters</span></li><li class="c20 li-bullet-0"><span>General Application Settings</span></li></ul><p class="c26 c7"><span class="c0"></span></p><h2 class="c2" id="h.4cmhg48"><span class="c16">JMS Settings</span></h2><p class="c19"><span class="c0">JMS Settings tab contains parameters that concern the exchange of messages between your local machine and the JMS Server. It should be made clear that compared to other preferences, preferences that are included in this tab should be treated with caution. Mistreating a communication preference can lead either to communication/connection problems or to users&rsquo; confusion to whether they are connected to the correct server version. </span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 607.68px; height: 181.41px;"><img alt="jms_settings.png" src="images/image93.png" style="width: 607.68px; height: 181.41px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><h3 class="c2" id="h.2rrrqc1"><span class="c36 c48">Service Request Queue Name</span></h3><p class="c26"><span class="c0">Parameter can be seen as a name which represents a server address. The parameter&rsquo;s existence is justified by the fact that multiple server versions might run on the same server machine imposing the need to be able to canalize Studio&rsquo;s messages appropriately. </span></p><p class="c26 c7"><span class="c0"></span></p><a id="id.16x20ju"></a><h2 class="c2" id="h.3qwpj7n"><span class="c16">Table Parameters</span></h2><p class="c26"><span class="c0">Table parameters&rsquo; tab encapsulates a short list of preferences regarding all tables generated throughout Proline Studio. More specifically those preferences control the arrangement of the participating columns as well as their respective width.</span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 582.73px; height: 173.96px;"><img alt="table_parameters.png" src="images/image91.png" style="width: 582.73px; height: 173.96px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h3 class="c2 c90" id="h.261ztfg"><span class="c36 c48"></span></h3><h3 class="c2" id="h.l7a3n9"><span class="c36 c48">Columns Arrangement</span></h3><p class="c26"><span class="c0">This field dictates the spatial arrangement of table columns. Three arrangements are possible:</span></p><ul class="c3 lst-kix_list_26-0 start"><li class="c46 c32 li-bullet-0"><span>Fixed Column Size</span></li><li class="c46 c32 li-bullet-0"><span>Automatic Columns Size</span></li><li class="c19 c32 li-bullet-0"><span>Smart Columns Size</span></li></ul><p class="c19"><span class="c0">When &ldquo;Automatic Columns Size&rdquo; is used, all columns are width-wise readjusted in a way that they all fit to their container. Given that it is a &ldquo;fit-to-screen&rdquo; approach, it lacks scrollbars and does not guarantee the readability of the presented date, especially when the number of columns is high. </span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 502.03px; height: 550.99px;"><img alt="automatic.png" src="images/image38.png" style="width: 502.03px; height: 550.99px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span class="c0">On the other hand a simpler approach that guarantees readability is to select &ldquo;Fixed Column Size&rdquo;. In this case all table columns have a fixed width, explicitly dictated by the user using the parameter &ldquo;Column Width&rdquo;. </span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 604.77px; height: 665.33px;"><img alt="fixed.png" src="images/image42.png" style="width: 604.77px; height: 665.33px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c19"><span class="c0">The less clear option is &ldquo;Smart Column Size&rdquo; which serves as a trade off between the aforementioned ones. It tackles with the cases that either we have too many columns to visualize using the &ldquo;Automatic&rdquo; approach or too few and the selected default and globally applied width imposes unneeded scrollbars ending in hiding some columns at the same time. In this context, &ldquo;Smart Column Size&rdquo; can be seen as a simple rule based on the ratio between the mean column width needed in case of &ldquo;Automatic Column Size&rdquo; for a specific table and the globally selected width. For the sake of simplicity we have set a threshold of 0.7 or 70% which on its turn determines which one of the two first modes will be used given a table. If the ratio is smaller than 0.7 then the table in question will be presented in &ldquo;Fixed Column Size&rdquo; mode. On contraire if ratio is equal or greater than 0.7, then we consider that using &ldquo;Automatic Column Size&rdquo; mode is more appropriate as it balances between a possible slightly smaller than desired width and the possibility of hiding a column using scrollbars.</span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><h3 class="c2" id="h.356xmb2"><span class="c36 c48">Column Width</span></h3><p class="c26"><span class="c0">The second preference is more or less self explanatory and corresponds to the globally desired columns&rsquo; width when fixed rune is applied, either directly or as a result of the smart mode.</span></p><p class="c26 c7"><span class="c0"></span></p><h2 class="c2" id="h.1kc7wiv"><span class="c16">General Application Settings</span></h2><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 576.07px; height: 185.53px;"><img alt="general_application_settings.png" src="images/image40.png" style="width: 576.07px; height: 185.53px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span class="c0">In this tab we can find a diverse set of preferences regarding various tasks encountered in Proline Studio. For the time being those preferences are:</span></p><ul class="c3 lst-kix_list_146-0 start"><li class="c46 c32 li-bullet-0"><span>Hide gettings Started Dialog</span></li><li class="c46 c32 li-bullet-0"><span>Default Search Name Source</span></li><li class="c46 c32 li-bullet-0"><span>Export Decorated</span></li><li class="c20 li-bullet-0"><span>Use dataset type to create a XiC design by DnD</span></li></ul><p class="c26 c7"><span class="c0"></span></p><h3 class="c2" id="h.44bvf6o"><span class="c36 c48">Default Search Name Source</span></h3><p class="c19"><span class="c0">Unlike the last three, Hide Getting Started Dialog is pretty much self explanatory. The second preference on the other hand, &ldquo;Default Search Name Source&rdquo; affects the way identification datasets are named on importation. For this preference we have three possible options:</span></p><ul class="c3 lst-kix_list_18-0 start"><li class="c46 c32 li-bullet-0"><span>Search Name (E.g Gamme Levure UPS1)</span></li><li class="c46 c32 li-bullet-0"><span>Peaklist (E.g OEMMA121101_36.raw)</span></li><li class="c32 c46 li-bullet-0"><span>MSI Search Filename (E.g F054967)</span></li><li class="c20 li-bullet-0"><span>Mascot Rule (...)</span></li></ul><h3 class="c2 c90" id="h.2jh5peh"><span class="c36 c48"></span></h3><h3 class="c2" id="h.ymfzma"><span class="c36 c48">Export Decorated</span></h3><p class="c26"><span class="c0">This parameter affects the .xls and .xlsx files that are produced in the process of export (client side). It could be easily described as a preservation of any existing Rich Text Feature in a table. (Colors, Font Weight etc.)</span></p><p class="c6 c7"><span class="c0"></span></p><h3 class="c2" id="h.3im3ia3"><span class="c36 c48">Use dataset type to create a XiC design by DnD</span></h3><p class="c26 c7"><span class="c0"></span></p><h2 class="c2" id="h.1xrdshw"><span class="c16">MsFiles</span></h2><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 595.28px; height: 389.33px;"><img alt="" src="images/image33.png" style="width: 595.28px; height: 389.33px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><h3 class="c2" id="h.4hr1b5p"><span class="c36 c48">Converter (.exe)</span></h3><p class="c19"><span class="c0">Corresponds to the default raw2mzDB converter. While it is left at the discretion of the user, which version to choose, it must be noted that different versions do tend to work better with specific type raw files. Said that, it is also important to understand that in order for a conversion to be successful within Proline Studio, all system requirements set by the specific raw2mzDB version must be met.</span></p><p class="c19 c7"><span class="c0"></span></p><h1 class="c2" id="h.2wwbldi"><span class="c12">Identify Modification Sites or Clusters</span></h1><p class="c26"><span>For description on sites and clusters you can first read the principles in this page: </span><span class="c4"><a class="c8" href="#h.111kx3o">Identifying Modification sites and clusters</a></span></p><p class="c26"><span>To identify Modification sites and create Modification clusters, right click on the Identification Summary and select </span><span class="c23">Identify Modification sites</span><span class="c0">. </span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 289.61px; height: 400.10px;"><img alt="" src="images/image31.png" style="width: 289.61px; height: 400.10px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26"><span>A dialog is displayed where you can choose Modification of interest and configure the clustering method (see &nbsp;</span><span class="c4"><a class="c8" href="#h.111kx3o">Identifying Modification sites and clusters</a></span><span class="c0">&nbsp;for more details).</span></p><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 369.71px; height: 411.90px;"><img alt="" src="images/image36.png" style="width: 369.71px; height: 411.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><a id="id.6z6uo9u38ru"></a><h1 class="c2" id="h.kadd5msugege"><span class="c12">Annotate and Edit Modification Clusters</span></h1><p class="c93 c77"><span>After running Identify Modification for a specific Identification Summary or Quantitation dataset (see above and &nbsp;</span><span class="c51">Identifying Modification sites and clusters</span><span class="c0">&nbsp;for more details), users can annotate and edit these clusters.</span></p><p class="c77 c93"><span class="c0">In the associated display, few operations are allowed.</span></p><ul class="c3 lst-kix_list_12-0 start"><li class="c35 c32 li-bullet-0"><span>Change the status of the cluster by clicking on the </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 16.00px; height: 16.00px;"><img alt="" src="images/image34.png" style="width: 16.00px; height: 16.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>&nbsp;icon. The following dialog appears, allowing user to</span></li></ul><ul class="c3 lst-kix_list_12-1 start"><li class="c43 li-bullet-0"><span class="c68">&nbsp;</span><span>Validate/Invalidate the cluster</span></li><li class="c43 li-bullet-0"><span class="c68">&nbsp;</span><span>Add a confidence level : any number which signification is user specific</span></li><li class="c53 c86 li-bullet-0"><span>Add a free description </span></li></ul><p class="c28"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 345.85px; height: 258.75px;"><img alt="" src="images/image26.png" style="width: 345.85px; height: 258.75px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><ul class="c3 lst-kix_list_131-0 start"><li class="c79 c77 c32 li-bullet-0"><span>Edit the selected cluster by clicking on the </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 16.00px; height: 16.00px;"><img alt="" src="images/image25.png" style="width: 16.00px; height: 16.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>&nbsp;icon. In this case, the status dialog is displayed in addition with the list of peptides of the cluster. Same status annotation as described above can be done. Users can also remove some peptides from the cluster. The removed peptides will be removed from the whole &ldquo;PTM Dataset&rdquo;. They will not be added to an existing or new cluster. Note: Peptide used as reference could not be deleted, a single peptide could be deleted at a time.</span></li></ul><p class="c28"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 545.41px; height: 343.91px;"><img alt="" src="images/image29.png" style="width: 545.41px; height: 343.91px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><ul class="c3 lst-kix_list_23-0 start"><li class="c32 c35 li-bullet-0"><span class="c68">&nbsp;</span><span>Merge co-localized clusters by clicking on </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 16.00px; height: 17.00px;"><img alt="" src="images/image57.png" style="width: 16.00px; height: 17.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>&nbsp;icon. Users must select clusters that should be merged. These clusters must be co-localized, see </span><span class="c4"><a class="c8" href="#h.2wwbldi">Identify Modification Sites or Clusters</a></span><span>&nbsp; .</span></li><li class="c50 c32 li-bullet-0"><span>Save edition by clicking in </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 16.00px; height: 16.00px;"><img alt="" src="images/image55.png" style="width: 16.00px; height: 16.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>&nbsp;icon. All modifications done on clusters will not be saved in datastore until user explicitly ask for it. When saving annotation, an Annotated Modification Dataset is created and will be accessible through the </span><span class="c23">Annotated Clusters</span><span>&nbsp;menu. This permits users to still access the initial Modification Dataset. Warning : there is only one Annotated Modification Dataset per Modification Dataset. If new annotation is done and saved from initial Modification Dataset, previous Annotated Modification Dataset will be erased</span></li></ul><a id="id.1c1lvlb"></a><h1 class="c2" id="h.3w19e94"><span class="c12">Administration</span></h1><p class="c26"><span>Some administration views and operations are accessible through the Proline Studio </span><span class="c23">File &gt; Admin</span><span class="c0">&nbsp;menu. Edit functions are only permitted to advanced users, who at the same time hold the status of Admin..</span></p><h2 class="c2" id="h.2b6jogx"><span class="c16">User Accounts</span></h2><p class="c26"><span>The </span><span class="c23">&ldquo;User Accounts&rdquo;</span><span>&nbsp;tab, list all registered users with the group they belong to. </span><span class="c23">Admin </span><span class="c0">users can add new users &nbsp;or modify existing one. Modifications consist in changing user groups or passwords.</span></p><h2 class="c2" id="h.9xhmol3rzx4x"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 498.91px; height: 323.16px;"><img alt="user_accounts.png" src="images/image61.png" style="width: 498.91px; height: 323.16px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></h2><p class="c26 c7"><span class="c0"></span></p><p class="c25"><span>&nbsp; &nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 276.48px; height: 134.50px;"><img alt="modify_existing_user.png" src="images/image59.png" style="width: 276.48px; height: 134.50px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c25"><span class="c13 c29 c60">User account modification dialog</span></p><p class="c6 c7"><span class="c0"></span></p><h2 class="c2" id="h.qbtyoq"><span class="c16">Peaklist Softwares</span></h2><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 440.73px; height: 332.90px;"><img alt="" src="images/image49.png" style="width: 440.73px; height: 332.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c0">Peaklist Softwares tab permits users &nbsp;to view the list of all available Peaklist Software. A detailed view could be displayed when selecting the &ldquo;View Peaklist Software&rdquo; button. </span></p><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 440.86px; height: 305.90px;"><img alt="" src="images/image47.png" style="width: 440.86px; height: 305.90px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c0">In both windows, It is possible to test a spectrum title to verify which data are extracted for a given Peaklist Software. </span></p><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 444.26px; height: 305.02px;"><img alt="" src="images/image53.png" style="width: 444.26px; height: 305.02px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c0">Admin users also have the possibility to add new Peaklist Software definition (name, version and extraction rules using regular expression) or to modify existing definitions by changing the name or version. Rules are not modifiable since some already parsed data may not be consistent with the new definition. New rules definition may be done by creation of a new peaklist software.</span></p><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 455.85px; height: 300.51px;"><img alt="new_peaklist_software.png" src="images/image51.png" style="width: 455.85px; height: 300.51px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26 c7"><span class="c0"></span></p><h2 class="c2" id="h.3abhhcj"><span class="c16">Projects and Databases </span></h2><p class="c26"><span class="c0">This tab lists all projects defined on the Proline Suite. </span></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 692.70px; height: 284.00px;"><img alt="" src="images/image44.png" style="width: 692.70px; height: 284.00px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c18">Upper view:</span><span class="c0">&nbsp;List of projects with associated generic information (owner, description) and more &nbsp;administration information such as the name of the databases containing data, size of these databases, last dataset creation&hellip; &nbsp;The satus column, first one, indicates if the project is active (green), inactive (blue) or archived (grey)</span></p><p class="c26"><span class="c18">Lower view:</span><span class="c0">&nbsp;for selected project, list of the mzdb files used for quantitation. This is useful to do some cleanup on disk when archiving projects. The status indicated if this mzdb file is used in active projects only (green), in active and archived projects (yellow) or in archived projects only (grey).</span></p><p class="c26 c7"><span class="c0"></span></p><h2 class="c2" id="h.1pgrrkc"><span class="c16">Fragmentation Rule Sets</span></h2><p class="c26"><span>This tab show all fragmentation rule sets defined in Proline Suite and the rules they use. The display is similar to the one defined by Mascot &reg; </span><hr style="page-break-before:always;display:none;"></p><p class="c26"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 692.70px; height: 497.33px;"><img alt="" src="images/image43.png" style="width: 692.70px; height: 497.33px; margin-left: -0.00px; margin-top: -0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span>This information is used for </span><span class="c4"><a class="c8" href="#id.1idq7dh">generating spectrum matches.</a></span><span class="c0">&nbsp;Custom fragmentation rule set can be added if needed by &ldquo;admin&rdquo; user.</span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><p class="c26 c7"><span class="c0"></span></p><div><p class="c87 c77 c7"><span class="c0"></span></p></div><div class="c40"><p class="c34"><a href="#cmnt_ref1" id="cmnt1">[a]</a><span class="c62 c29 c60">A faire</span></p></div></body></html>